{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "Bioinformatika projekat 2.ipynb",
      "provenance": [],
      "collapsed_sections": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "sz9kYbhXsgvA",
        "outputId": "7649cac8-cba4-4a62-8eb7-b1ba77617891"
      },
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Mounted at /content/drive\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "rrLOqCd5Al2m"
      },
      "source": [
        "# Projekat 2 - 20 bodova\n",
        "\n",
        "Dati su FASTQ fajlovi koji su rezultat sekvenciranja dela exoma koji se nalazi na hromozomu 11 (podaci su uzeti iz uzoraka 1000 Genomes projekta).\n",
        "(tabela sa imenom uzoraka koji je dodeljen svakom studentu je data u fajlu Lista zadataka 2 2021).\n",
        "\n",
        "### Rok za predaju je 3. Jun, do kraja dana.\n",
        "Do datog roka je potrebno poslati rezultate emailom, na adresu grakocevic@raf.rs.\n",
        "\n",
        "Kao rezultat projekta se potrebno je predati:  \n",
        "1. Kod kojim su generisani rezultati (ipynb ili python skripta)\n",
        "2. Kraći izveštaj sa tekstualnim odgovorima na pitanja i grafikonima **u PDF formatu**\n",
        "\n",
        "Projekat je moguće raditi pojedinačno ili u paru. **Ukoliko se projekat radi u paru, svaki student predaje i brani projekat posebno, uz napomenu da je projekat urađen u paru; pri tome svaki student treba da obradi podatke iz eksperimenta koji mu dodeljen, a rad u paru se odnosi na programski kod i šablon za tekstualni izveštaj.**\n",
        "\n",
        "\n",
        "\n",
        "### Pitanja:\n",
        "\n",
        "1. [2 boda] Izvršiti kontrolu kvaliteta FASTQ fajlova alatom FastQC. Priložiti izvrštaj i diskutovati rezultate (da li je neki od kriterijuma koje analizira FastQC označen kao problematičan, i šta taj kriterijum znači?)\n",
        " \n",
        "2. [3 boda] Mapirati sekvencirane readove na referentni genom hg38 upotrebom alata BWA Mem. Upotrebom python biblioteke pysam odrediti sledeće:  \n",
        " 2a. Koliko je readova uspešno mapirano?  \n",
        " 2b. Koliko je parova readova mapirano tako da su oba para mapirana?  \n",
        " 2c. Nacrtati histogram dužina sekvenciranih fragmenata (*template_length*).   \n",
        "   \n",
        "3. [4 boda] Izvršiti obradu dobijenog BAM fajla prema GATK protokolu (Markiranje Duplikata, rekalibracija kvaliteta baza)\n",
        " 3a. Koliki su procenati PCR i optičkih duplikata?\n",
        "\n",
        "4. [4 bodova] Identifikovati mutacije upotrebom alata Haplotype Caller i filtirtati mutacije predefinisanim filterima (hard filtering) prema Broad preporukama (kao što je rađeno na vežbama).\n",
        " 4a. Koliko je ukupno mutacija identifikovano, koliko od njih su SNP-ovi, a koliko INDEL-i?\n",
        " 4b. Koliko mutacija prolazi, a koliko ne prolazi kriterijume filtriranja.\n",
        " 4c. Izračunati Ti/Tv odnos pre i posle filtriranja.\n",
        "\n",
        "5. [2 bodova] Anotirati mutacije alatom Funcotator\n",
        " 5a. Izbrojati različite vrednosti ClinVar značajnosti (anotacija *ClinVar_VCF_CLNSIG*, koliko mutacija je označeno kao *Benign*, *Likely_benign*, itd.)-.\n",
        "\n",
        "5. [5 bodova] Svi uzorici sadrže određenu količinu kontaminacije DNK materijalom bakterijskog ili virusnog porekla. Većina ovakvih readova se neće mapirati na ljudski genom. Izvući readove koji nisu mapirani u procesu mapiranja, asemblovati ih alatom abyss, i identifikovati organizam od kojeg potiče najduži skafold upotrebom alata Blast."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "NhANv2_NXKPz",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "aeda021e-f312-4226-c770-5ad9c2f79c92"
      },
      "source": [
        "!apt-get install samtools\n",
        "\n",
        "!pip install intervaltree\n",
        "\n",
        "!pip install pysam\n",
        "\n",
        "!wget https://github.com/broadinstitute/picard/releases/download/2.21.6/picard.jar\n",
        "\n",
        "!wget -O gatk-4.1.4.1.zip \"https://github.com/broadinstitute/gatk/releases/download/4.1.4.1/gatk-4.1.4.1.zip\"\n",
        "!unzip gatk-4.1.4.1.zip\n",
        "!R -e 'install.packages(c(\"gplots\", \"gsalib\"))'\n",
        "\n",
        "!gdown --id 13ESrnOy32eWbPO2jEEMTtLj2vn7NBseS\n",
        "!gdown --id 1_rvfN87uprNVbIX09ZcdkhLPNjoohgrQ\n",
        "\n",
        "!git clone https://github.com/lh3/bwa.git\n",
        "!cd bwa && make"
      ],
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  cwltool libc-ares2 libhts2 libjs-bootstrap nodejs nodejs-doc\n",
            "  python-asn1crypto python-avro python-cachecontrol python-certifi\n",
            "  python-cffi-backend python-chardet python-cryptography python-enum34\n",
            "  python-html5lib python-idna python-ipaddress python-isodate python-lockfile\n",
            "  python-mistune python-openssl python-pkg-resources python-pyparsing\n",
            "  python-rdflib python-rdflib-jsonld python-requests python-ruamel.yaml\n",
            "  python-schema-salad python-shellescape python-six python-sparqlwrapper\n",
            "  python-typing python-urllib3 python-webencodings\n",
            "Suggested packages:\n",
            "  python-cryptography-doc python-cryptography-vectors python-enum34-doc\n",
            "  python-genshi python-lxml python-lockfile-doc python-openssl-doc\n",
            "  python-openssl-dbg python-setuptools python-pyparsing-doc python-rdflib-doc\n",
            "  python-rdflib-tools python-socks python-ntlm\n",
            "The following NEW packages will be installed:\n",
            "  cwltool libc-ares2 libhts2 libjs-bootstrap nodejs nodejs-doc\n",
            "  python-asn1crypto python-avro python-cachecontrol python-certifi\n",
            "  python-cffi-backend python-chardet python-cryptography python-enum34\n",
            "  python-html5lib python-idna python-ipaddress python-isodate python-lockfile\n",
            "  python-mistune python-openssl python-pkg-resources python-pyparsing\n",
            "  python-rdflib python-rdflib-jsonld python-requests python-ruamel.yaml\n",
            "  python-schema-salad python-shellescape python-six python-sparqlwrapper\n",
            "  python-typing python-urllib3 python-webencodings samtools\n",
            "0 upgraded, 35 newly installed, 0 to remove and 39 not upgraded.\n",
            "Need to get 8,568 kB of archives.\n",
            "After this operation, 41.0 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-pkg-resources all 39.0.1-2 [128 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-six all 1.11.0-2 [11.3 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-isodate all 0.6.0-1 [23.8 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-pyparsing all 2.2.0+dfsg1-2 [52.1 kB]\n",
            "Get:5 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-rdflib all 4.2.1-2 [237 kB]\n",
            "Get:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-certifi all 2018.1.18-2 [144 kB]\n",
            "Get:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-chardet all 3.0.4-1 [80.3 kB]\n",
            "Get:8 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-idna all 2.6-1 [32.4 kB]\n",
            "Get:9 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-urllib3 all 1.22-1ubuntu0.18.04.2 [86.0 kB]\n",
            "Get:10 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-requests all 2.18.4-2ubuntu0.1 [58.5 kB]\n",
            "Get:11 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-ruamel.yaml amd64 0.15.34-1 [183 kB]\n",
            "Get:12 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-cachecontrol all 0.11.7-1 [15.2 kB]\n",
            "Get:13 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-lockfile all 1:0.12.2-2 [14.6 kB]\n",
            "Get:14 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-mistune all 0.8.3-2 [13.8 kB]\n",
            "Get:15 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-rdflib-jsonld all 0.4.0-3 [12.2 kB]\n",
            "Get:16 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-typing all 3.6.2-1 [22.2 kB]\n",
            "Get:17 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-avro all 1.8.2+dfsg-1 [23.2 kB]\n",
            "Get:18 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libjs-bootstrap all 3.3.7+dfsg-2 [228 kB]\n",
            "Get:19 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-schema-salad all 2.6.20171201034858-3 [260 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-shellescape all 3.4.1-2 [4,432 B]\n",
            "Get:21 http://archive.ubuntu.com/ubuntu bionic/universe amd64 cwltool all 1.0.20180302231433-1 [143 kB]\n",
            "Get:22 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libhts2 amd64 1.7-2 [300 kB]\n",
            "Get:23 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 nodejs-doc all 8.10.0~dfsg-2ubuntu0.4 [752 kB]\n",
            "Get:24 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-asn1crypto all 0.24.0-1 [72.7 kB]\n",
            "Get:25 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-cffi-backend amd64 1.11.5-1 [63.4 kB]\n",
            "Get:26 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-enum34 all 1.1.6-2 [34.8 kB]\n",
            "Get:27 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-ipaddress all 1.0.17-1 [18.2 kB]\n",
            "Get:28 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-cryptography amd64 2.1.4-1ubuntu1.4 [276 kB]\n",
            "Get:29 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-webencodings all 0.5-2 [10.3 kB]\n",
            "Get:30 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-html5lib all 0.999999999-1 [83.6 kB]\n",
            "Get:31 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-openssl all 17.5.0-1ubuntu1 [41.3 kB]\n",
            "Get:32 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-sparqlwrapper all 1.7.6-2 [21.8 kB]\n",
            "Get:33 http://archive.ubuntu.com/ubuntu bionic/universe amd64 samtools amd64 1.7-1 [288 kB]\n",
            "Get:34 http://archive.ubuntu.com/ubuntu bionic/main amd64 libc-ares2 amd64 1.14.0-1 [37.1 kB]\n",
            "Get:35 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 nodejs amd64 8.10.0~dfsg-2ubuntu0.4 [4,796 kB]\n",
            "Fetched 8,568 kB in 1s (7,339 kB/s)\n",
            "Extracting templates from packages: 100%\n",
            "Selecting previously unselected package python-pkg-resources.\n",
            "(Reading database ... 160772 files and directories currently installed.)\n",
            "Preparing to unpack .../00-python-pkg-resources_39.0.1-2_all.deb ...\n",
            "Unpacking python-pkg-resources (39.0.1-2) ...\n",
            "Selecting previously unselected package python-six.\n",
            "Preparing to unpack .../01-python-six_1.11.0-2_all.deb ...\n",
            "Unpacking python-six (1.11.0-2) ...\n",
            "Selecting previously unselected package python-isodate.\n",
            "Preparing to unpack .../02-python-isodate_0.6.0-1_all.deb ...\n",
            "Unpacking python-isodate (0.6.0-1) ...\n",
            "Selecting previously unselected package python-pyparsing.\n",
            "Preparing to unpack .../03-python-pyparsing_2.2.0+dfsg1-2_all.deb ...\n",
            "Unpacking python-pyparsing (2.2.0+dfsg1-2) ...\n",
            "Selecting previously unselected package python-rdflib.\n",
            "Preparing to unpack .../04-python-rdflib_4.2.1-2_all.deb ...\n",
            "Unpacking python-rdflib (4.2.1-2) ...\n",
            "Selecting previously unselected package python-certifi.\n",
            "Preparing to unpack .../05-python-certifi_2018.1.18-2_all.deb ...\n",
            "Unpacking python-certifi (2018.1.18-2) ...\n",
            "Selecting previously unselected package python-chardet.\n",
            "Preparing to unpack .../06-python-chardet_3.0.4-1_all.deb ...\n",
            "Unpacking python-chardet (3.0.4-1) ...\n",
            "Selecting previously unselected package python-idna.\n",
            "Preparing to unpack .../07-python-idna_2.6-1_all.deb ...\n",
            "Unpacking python-idna (2.6-1) ...\n",
            "Selecting previously unselected package python-urllib3.\n",
            "Preparing to unpack .../08-python-urllib3_1.22-1ubuntu0.18.04.2_all.deb ...\n",
            "Unpacking python-urllib3 (1.22-1ubuntu0.18.04.2) ...\n",
            "Selecting previously unselected package python-requests.\n",
            "Preparing to unpack .../09-python-requests_2.18.4-2ubuntu0.1_all.deb ...\n",
            "Unpacking python-requests (2.18.4-2ubuntu0.1) ...\n",
            "Selecting previously unselected package python-ruamel.yaml.\n",
            "Preparing to unpack .../10-python-ruamel.yaml_0.15.34-1_amd64.deb ...\n",
            "Unpacking python-ruamel.yaml (0.15.34-1) ...\n",
            "Selecting previously unselected package python-cachecontrol.\n",
            "Preparing to unpack .../11-python-cachecontrol_0.11.7-1_all.deb ...\n",
            "Unpacking python-cachecontrol (0.11.7-1) ...\n",
            "Selecting previously unselected package python-lockfile.\n",
            "Preparing to unpack .../12-python-lockfile_1%3a0.12.2-2_all.deb ...\n",
            "Unpacking python-lockfile (1:0.12.2-2) ...\n",
            "Selecting previously unselected package python-mistune.\n",
            "Preparing to unpack .../13-python-mistune_0.8.3-2_all.deb ...\n",
            "Unpacking python-mistune (0.8.3-2) ...\n",
            "Selecting previously unselected package python-rdflib-jsonld.\n",
            "Preparing to unpack .../14-python-rdflib-jsonld_0.4.0-3_all.deb ...\n",
            "Unpacking python-rdflib-jsonld (0.4.0-3) ...\n",
            "Selecting previously unselected package python-typing.\n",
            "Preparing to unpack .../15-python-typing_3.6.2-1_all.deb ...\n",
            "Unpacking python-typing (3.6.2-1) ...\n",
            "Selecting previously unselected package python-avro.\n",
            "Preparing to unpack .../16-python-avro_1.8.2+dfsg-1_all.deb ...\n",
            "Unpacking python-avro (1.8.2+dfsg-1) ...\n",
            "Selecting previously unselected package libjs-bootstrap.\n",
            "Preparing to unpack .../17-libjs-bootstrap_3.3.7+dfsg-2_all.deb ...\n",
            "Unpacking libjs-bootstrap (3.3.7+dfsg-2) ...\n",
            "Selecting previously unselected package python-schema-salad.\n",
            "Preparing to unpack .../18-python-schema-salad_2.6.20171201034858-3_all.deb ...\n",
            "Unpacking python-schema-salad (2.6.20171201034858-3) ...\n",
            "Selecting previously unselected package python-shellescape.\n",
            "Preparing to unpack .../19-python-shellescape_3.4.1-2_all.deb ...\n",
            "Unpacking python-shellescape (3.4.1-2) ...\n",
            "Selecting previously unselected package cwltool.\n",
            "Preparing to unpack .../20-cwltool_1.0.20180302231433-1_all.deb ...\n",
            "Unpacking cwltool (1.0.20180302231433-1) ...\n",
            "Selecting previously unselected package libhts2:amd64.\n",
            "Preparing to unpack .../21-libhts2_1.7-2_amd64.deb ...\n",
            "Unpacking libhts2:amd64 (1.7-2) ...\n",
            "Selecting previously unselected package nodejs-doc.\n",
            "Preparing to unpack .../22-nodejs-doc_8.10.0~dfsg-2ubuntu0.4_all.deb ...\n",
            "Unpacking nodejs-doc (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "Selecting previously unselected package python-asn1crypto.\n",
            "Preparing to unpack .../23-python-asn1crypto_0.24.0-1_all.deb ...\n",
            "Unpacking python-asn1crypto (0.24.0-1) ...\n",
            "Selecting previously unselected package python-cffi-backend.\n",
            "Preparing to unpack .../24-python-cffi-backend_1.11.5-1_amd64.deb ...\n",
            "Unpacking python-cffi-backend (1.11.5-1) ...\n",
            "Selecting previously unselected package python-enum34.\n",
            "Preparing to unpack .../25-python-enum34_1.1.6-2_all.deb ...\n",
            "Unpacking python-enum34 (1.1.6-2) ...\n",
            "Selecting previously unselected package python-ipaddress.\n",
            "Preparing to unpack .../26-python-ipaddress_1.0.17-1_all.deb ...\n",
            "Unpacking python-ipaddress (1.0.17-1) ...\n",
            "Selecting previously unselected package python-cryptography.\n",
            "Preparing to unpack .../27-python-cryptography_2.1.4-1ubuntu1.4_amd64.deb ...\n",
            "Unpacking python-cryptography (2.1.4-1ubuntu1.4) ...\n",
            "Selecting previously unselected package python-webencodings.\n",
            "Preparing to unpack .../28-python-webencodings_0.5-2_all.deb ...\n",
            "Unpacking python-webencodings (0.5-2) ...\n",
            "Selecting previously unselected package python-html5lib.\n",
            "Preparing to unpack .../29-python-html5lib_0.999999999-1_all.deb ...\n",
            "Unpacking python-html5lib (0.999999999-1) ...\n",
            "Selecting previously unselected package python-openssl.\n",
            "Preparing to unpack .../30-python-openssl_17.5.0-1ubuntu1_all.deb ...\n",
            "Unpacking python-openssl (17.5.0-1ubuntu1) ...\n",
            "Selecting previously unselected package python-sparqlwrapper.\n",
            "Preparing to unpack .../31-python-sparqlwrapper_1.7.6-2_all.deb ...\n",
            "Unpacking python-sparqlwrapper (1.7.6-2) ...\n",
            "Selecting previously unselected package samtools.\n",
            "Preparing to unpack .../32-samtools_1.7-1_amd64.deb ...\n",
            "Unpacking samtools (1.7-1) ...\n",
            "Selecting previously unselected package libc-ares2:amd64.\n",
            "Preparing to unpack .../33-libc-ares2_1.14.0-1_amd64.deb ...\n",
            "Unpacking libc-ares2:amd64 (1.14.0-1) ...\n",
            "Selecting previously unselected package nodejs.\n",
            "Preparing to unpack .../34-nodejs_8.10.0~dfsg-2ubuntu0.4_amd64.deb ...\n",
            "Unpacking nodejs (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "Setting up python-idna (2.6-1) ...\n",
            "Setting up nodejs-doc (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "Setting up python-asn1crypto (0.24.0-1) ...\n",
            "Setting up python-mistune (0.8.3-2) ...\n",
            "Setting up python-certifi (2018.1.18-2) ...\n",
            "Setting up python-avro (1.8.2+dfsg-1) ...\n",
            "Setting up python-pkg-resources (39.0.1-2) ...\n",
            "Setting up python-ruamel.yaml (0.15.34-1) ...\n",
            "Setting up python-cffi-backend (1.11.5-1) ...\n",
            "Setting up python-six (1.11.0-2) ...\n",
            "Setting up python-isodate (0.6.0-1) ...\n",
            "Setting up python-shellescape (3.4.1-2) ...\n",
            "Setting up python-enum34 (1.1.6-2) ...\n",
            "Setting up libjs-bootstrap (3.3.7+dfsg-2) ...\n",
            "Setting up python-lockfile (1:0.12.2-2) ...\n",
            "Setting up python-ipaddress (1.0.17-1) ...\n",
            "Setting up libc-ares2:amd64 (1.14.0-1) ...\n",
            "Setting up libhts2:amd64 (1.7-2) ...\n",
            "Setting up python-webencodings (0.5-2) ...\n",
            "Setting up python-typing (3.6.2-1) ...\n",
            "Setting up python-pyparsing (2.2.0+dfsg1-2) ...\n",
            "Setting up python-urllib3 (1.22-1ubuntu0.18.04.2) ...\n",
            "Setting up python-chardet (3.0.4-1) ...\n",
            "Setting up nodejs (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "update-alternatives: using /usr/bin/nodejs to provide /usr/bin/js (js) in auto mode\n",
            "Setting up python-rdflib (4.2.1-2) ...\n",
            "Setting up python-sparqlwrapper (1.7.6-2) ...\n",
            "Setting up python-rdflib-jsonld (0.4.0-3) ...\n",
            "Setting up python-cryptography (2.1.4-1ubuntu1.4) ...\n",
            "Setting up python-requests (2.18.4-2ubuntu0.1) ...\n",
            "Setting up python-cachecontrol (0.11.7-1) ...\n",
            "Setting up python-schema-salad (2.6.20171201034858-3) ...\n",
            "Setting up python-html5lib (0.999999999-1) ...\n",
            "Setting up samtools (1.7-1) ...\n",
            "Setting up python-openssl (17.5.0-1ubuntu1) ...\n",
            "Setting up cwltool (1.0.20180302231433-1) ...\n",
            "Processing triggers for libc-bin (2.27-3ubuntu1.2) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/python3.7/dist-packages/ideep4py/lib/libmkldnn.so.0 is not a symbolic link\n",
            "\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n",
            "Requirement already satisfied: intervaltree in /usr/local/lib/python3.7/dist-packages (2.1.0)\n",
            "Requirement already satisfied: sortedcontainers in /usr/local/lib/python3.7/dist-packages (from intervaltree) (2.4.0)\n",
            "Collecting pysam\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/20/85/335857b9888f6d9a13b03a8f21b0a6228b180c361631d9d70e7be3e22163/pysam-0.16.0.1-cp37-cp37m-manylinux1_x86_64.whl (9.9MB)\n",
            "\u001b[K     |████████████████████████████████| 9.9MB 28.4MB/s \n",
            "\u001b[?25hInstalling collected packages: pysam\n",
            "Successfully installed pysam-0.16.0.1\n",
            "--2021-06-03 11:33:23--  https://github.com/broadinstitute/picard/releases/download/2.21.6/picard.jar\n",
            "Resolving github.com (github.com)... 140.82.112.3\n",
            "Connecting to github.com (github.com)|140.82.112.3|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://github-releases.githubusercontent.com/18225913/da29a580-2272-11ea-80e8-04574eb02117?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210603%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210603T113323Z&X-Amz-Expires=300&X-Amz-Signature=67883d422bab0d7f66b2d9413c08d3c68f6da7b5d49345f55e5f272c7933d0e2&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=18225913&response-content-disposition=attachment%3B%20filename%3Dpicard.jar&response-content-type=application%2Foctet-stream [following]\n",
            "--2021-06-03 11:33:23--  https://github-releases.githubusercontent.com/18225913/da29a580-2272-11ea-80e8-04574eb02117?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210603%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210603T113323Z&X-Amz-Expires=300&X-Amz-Signature=67883d422bab0d7f66b2d9413c08d3c68f6da7b5d49345f55e5f272c7933d0e2&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=18225913&response-content-disposition=attachment%3B%20filename%3Dpicard.jar&response-content-type=application%2Foctet-stream\n",
            "Resolving github-releases.githubusercontent.com (github-releases.githubusercontent.com)... 185.199.108.154, 185.199.109.154, 185.199.110.154, ...\n",
            "Connecting to github-releases.githubusercontent.com (github-releases.githubusercontent.com)|185.199.108.154|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 16582508 (16M) [application/octet-stream]\n",
            "Saving to: ‘picard.jar’\n",
            "\n",
            "picard.jar          100%[===================>]  15.81M  --.-KB/s    in 0.1s    \n",
            "\n",
            "2021-06-03 11:33:23 (130 MB/s) - ‘picard.jar’ saved [16582508/16582508]\n",
            "\n",
            "--2021-06-03 11:33:23--  https://github.com/broadinstitute/gatk/releases/download/4.1.4.1/gatk-4.1.4.1.zip\n",
            "Resolving github.com (github.com)... 140.82.113.3\n",
            "Connecting to github.com (github.com)|140.82.113.3|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://github-releases.githubusercontent.com/27452807/b23b9c00-1123-11ea-8ec4-28af5face303?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210603%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210603T113323Z&X-Amz-Expires=300&X-Amz-Signature=8527e0b78d3e32991de6610ad7ad87d061507d7dbb0559fdb914fe885e659e33&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=27452807&response-content-disposition=attachment%3B%20filename%3Dgatk-4.1.4.1.zip&response-content-type=application%2Foctet-stream [following]\n",
            "--2021-06-03 11:33:23--  https://github-releases.githubusercontent.com/27452807/b23b9c00-1123-11ea-8ec4-28af5face303?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210603%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210603T113323Z&X-Amz-Expires=300&X-Amz-Signature=8527e0b78d3e32991de6610ad7ad87d061507d7dbb0559fdb914fe885e659e33&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=27452807&response-content-disposition=attachment%3B%20filename%3Dgatk-4.1.4.1.zip&response-content-type=application%2Foctet-stream\n",
            "Resolving github-releases.githubusercontent.com (github-releases.githubusercontent.com)... 185.199.108.154, 185.199.109.154, 185.199.110.154, ...\n",
            "Connecting to github-releases.githubusercontent.com (github-releases.githubusercontent.com)|185.199.108.154|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 404902600 (386M) [application/octet-stream]\n",
            "Saving to: ‘gatk-4.1.4.1.zip’\n",
            "\n",
            "gatk-4.1.4.1.zip    100%[===================>] 386.14M  93.3MB/s    in 4.0s    \n",
            "\n",
            "2021-06-03 11:33:27 (95.8 MB/s) - ‘gatk-4.1.4.1.zip’ saved [404902600/404902600]\n",
            "\n",
            "Archive:  gatk-4.1.4.1.zip\n",
            "   creating: gatk-4.1.4.1/\n",
            "   creating: gatk-4.1.4.1/gatkdoc/\n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_ASEReadCounter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_replicates_CollectIndependentReplicateMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ChromosomeCounts.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_LibraryReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_FilterVariantTranches.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonChimericOriginalAlignmentReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_MeanQualityByCycle.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintReadsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_GenotypeConcordance.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CheckTerminatorBlock.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadLengthReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_Pileup.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_OverclippedReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CleanSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_GtcToVcf.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_CheckPileup.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_BwaMemIndexImageCreator.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OrientationBiasReadCounts.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_RenameSampleInVcf.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityAvailableReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_CountNs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectOxoGMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaReferenceMaker.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AlleleFraction.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_QualityScoreDistributionSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerSampleHC.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_SampleReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_AnalyzeCovariates.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RMSMappingQuality.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$FirstOfPairReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_artifacts_ConvertSequencingArtifactToOxoG.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountBasesSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountReadsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_Funcotator.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectQualityYieldMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CreateReadCountPanelOfNormals.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqPipelineSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_CalcMetadataSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SecondOfPairReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_AlignmentAgreesWithHeaderReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_StrandOddsRatio.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_filter_FilterVcf.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_ParallelCopyGCSDirectoryIntoHDFSSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_AnnotatePairOrientation.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_VariantEval.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectRrbsMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_MendelianViolations_FindMendelianViolations.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSecondaryAlignmentReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_CountReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_FixVcfHeader.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_directed_CollectHsMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_gnarlyGenotyper_GnarlyGenotyper.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentEndReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_EstimateLibraryComplexity.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReferenceBases.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_PrintReadsHeader.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ViewSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithExpectedAlleleFraction.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CountVariants.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_IndexFeatureFile.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$AllowAllReadsReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_LiftOverIntervalList.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_MetricsReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotDuplicateReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBadGenomicKmersSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fastq_BamToBfq.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_AddCommentsToBam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_QualByDepth.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_AnalyzeSaturationMutagenesis.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_GenotypeGVCFs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_TandemRepeat.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_BuildBamIndex.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_VariantAnnotator.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_FilterSamReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_BaseQualityRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_MergePedIntoVcf.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_Concordance.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_FragmentLengthReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_SplitNCigarReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PrimaryLineReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroFragmentLengthReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectMultipleMetricsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_StructuralVariationDiscoveryPipelineSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_EvaluateInfoFieldConcordance.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_CountBases.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandOddsRatio.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_SplitVcfs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/index.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SplitSamByLibrary.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FilterFuncotations.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_CheckFingerprint.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantTrain.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$HasReadGroupReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_CrosscheckFingerprints.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_PileupSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DetermineGermlineContigPloidy.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBreakpointEvidenceSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_UniqueAltReadCount.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ProperlyPairedReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQuality.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$CigarContainsNoNOperator.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotDenoisedCopyRatios.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_QualByDepth.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_VariantsToTable.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqFilterSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_VariantRecalibrator.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectBaseDistributionByCycleSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_utils_ExtractOriginalAlignmentRecordsByNameSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_reference_NormalizeFasta.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_UnmarkDuplicates.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_RMSMappingQuality.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantWriteTensors.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_SplitIntervals.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaAlternateReferenceMaker.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountVariantsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_UpdateVCFSequenceDictionary.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_IlluminaBasecallsToFastq.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SamFormatConverter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_FixCallSetSampleOrdering.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_ClusterCrosscheckMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_ClipReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SamErrorMetric_CollectSamErrorMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_SoftClippedReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_GetSampleName.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_MappingQualityRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_IntervalOverlapReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerAlleleBySample.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroReferenceLengthAlignmentReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CalculateReadGroupChecksum.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQuality.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SeqIsStoredReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SetNmMdAndUqTags.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CompareMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotateSegments.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_GatherBamFiles.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_CalculateFingerprintMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotatorDataSourceDownloader.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_ConvertHeaderlessHadoopBamShardToBam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildReferenceTaxonomy.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_FisherStrand.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_validation_CompareBaseQualities.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_bwa_BwaSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupBlackListReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_FastqToSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectAlignmentSummaryMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_CollectIlluminaBasecallingMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SortSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithBamDepth.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SamToFastq.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_artifacts_CollectSequencingArtifactMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqScoreSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_RevertSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_GatherVcfs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappedReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_SplitReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PassesVendorQualityCheckReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_reference_ExtractSequences.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_MarkDuplicatesWithMateCigar.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_stratifications_AlleleFrequency.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectRnaSeqMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_GatherBQSRReports.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_CalculateContamination.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_MergeSamFiles.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_ApplyBQSR.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_GatherVcfsCloud.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_inference_CpxVariantReInterpreterSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_AnnotateIntervals.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildKmers.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_MakeSitesOnlyVcf.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_IntervalListTools.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_InbreedingCoeff.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectInsertSizeMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ReadLengthEqualsCigarLengthReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandBiasBySample.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_BaitDesigner.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ValidateSamFile.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectReadCounts.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_SortSamSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_LeftAlignIndels.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_FilterIntervals.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityNotZeroReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_SortVcf.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_FixMateInformation.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintVariantsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_HaplotypeCallerSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectJumpingLibraryMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SplitSamByNumberOfReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_MethylationTypeCaller.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateDifferentStrandReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_reference_NonNFastaSize.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_filtering_FilterMutectCalls.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_BaseRecalibratorSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CalculateMixingFractions.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_SelectVariants.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_AccumulateVariantCallingMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_IlluminaBasecallsToSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_FlagStat.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PreprocessIntervals.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_QualityScoreDistribution.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_MappingQualityReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/gatkDoc.css  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectMultipleMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_CalculateGenotypePosteriors.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_genomicsdb_GenomicsDBImport.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_MergeVcfs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectQualityYieldMetricsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformUnitReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNScoreVariants.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_RevertOriginalBaseQualitiesAndAddMateCigar.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_ExtractSVEvidenceSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ValidateVariants.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectAllelicCounts.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_CreateHadoopBamSplittingIndex.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_PrintReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_LiftoverVcf.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BwaAndMarkDuplicatesPipelineSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_AddOATag.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_HaplotypeCaller.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_ApplyBQSRSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_CreateVerifyIDIntensityContaminationMetricsFile.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosition.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_Coverage.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentStartReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateOnSameContigOrNoMappedMateReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_ReadsPipelineSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadStrandFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ReplaceSamHeader.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CountFalsePositives.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateUnmappedAndUnmappedReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_AddOrReplaceReadGroups.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_VcfFormatConverter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_BamIndexStats.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_CollectIlluminaLaneMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_CountBasesInReference.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CombineGVCFs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectGcBiasMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_VcfToIntervalList.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SetNmAndUqTags.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBwaSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotModeledSegments.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_MarkIlluminaAdapters.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_CollectVariantCallingMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_SampleList.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_UpdateVcfSequenceDictionary.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MatchingBasesAndQualsReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_DiscoverVariantsFromContigAlignmentsSAMSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_GermlineCNVCaller.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_MeanQualityByCycleSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentLength.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_CheckIlluminaDirectory.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadNameReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_directed_CollectTargetedPcrMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectBaseDistributionByCycle.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_ApplyVQSR.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ReblockGVCF.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CallCopyRatioSegments.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_InbreedingCoeff.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_ScatterIntervalsByNs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_MergeBamAlignment.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_FixMisencodedBaseQualityReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityZero.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_LeftAlignAndTrimVariants.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_ModelSegments.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_GatherTranches.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_RevertSamSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_ExtractIlluminaBarcodes.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_RevertBaseQualityScores.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_DownsampleSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ClippingRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_AmbiguousBaseReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_BedToIntervalList.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ExcessHet.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_RemoveNearbyIndels.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_filters_VariantFiltration.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_LikelihoodRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_validation_CompareDuplicatesSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectRawWgsMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_exome_FilterByOrientationBias.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_PossibleDeNovo.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectInsertSizeMetricsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQualityRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_SvDiscoverFromLocalAssemblyContigAlignmentsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DenoiseReadCounts.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_IntervalListToBed.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CompareSAMs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_basicshortmutpileup_ValidateBasicSomaticShortMutations.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OriginalAlignment.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BQSRPipelineSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_MarkDuplicates.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_UmiAwareMarkDuplicatesWithMateCigar.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CreateSequenceDictionary.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_VcfToAdpc.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$GoodCigarReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_realignmentfilter_FilterAlignmentArtifacts.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_SimpleMarkDuplicatesWithMateCigar.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_GenotypeSummaries.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_FlagStatSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectWgsMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ReorderSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PostprocessGermlineCNVCalls.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSupplementaryAlignmentReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_ReadPosRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_FifoBuffer.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_PositionBasedDownsampleSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_GetPileupSummaries.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PairedReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ReorderSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PostprocessGermlineCNVCalls.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSupplementaryAlignmentReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_GenotypeSummaries.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_FlagStatSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_SimpleMarkDuplicatesWithMateCigar.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_realignmentfilter_FilterAlignmentArtifacts.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectWgsMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PairedReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_ReadPosRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_GetPileupSummaries.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_FifoBuffer.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_PositionBasedDownsampleSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_AmbiguousBaseReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_BedToIntervalList.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ExcessHet.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_RemoveNearbyIndels.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_filters_VariantFiltration.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectRawWgsMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_validation_CompareDuplicatesSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_LikelihoodRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_ExtractIlluminaBarcodes.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_RevertBaseQualityScores.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_RevertSamSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_GatherTranches.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_DownsampleSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ClippingRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_basicshortmutpileup_ValidateBasicSomaticShortMutations.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CompareSAMs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BQSRPipelineSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OriginalAlignment.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_UmiAwareMarkDuplicatesWithMateCigar.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_MarkDuplicates.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_VcfToAdpc.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$GoodCigarReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CreateSequenceDictionary.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_PossibleDeNovo.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_exome_FilterByOrientationBias.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectInsertSizeMetricsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DenoiseReadCounts.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQualityRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_SvDiscoverFromLocalAssemblyContigAlignmentsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_IntervalListToBed.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_ApplyVQSR.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ReblockGVCF.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_MeanQualityByCycleSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_DiscoverVariantsFromContigAlignmentsSAMSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_GermlineCNVCaller.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_UpdateVcfSequenceDictionary.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MatchingBasesAndQualsReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_CheckIlluminaDirectory.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentLength.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_directed_CollectTargetedPcrMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadNameReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectBaseDistributionByCycle.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_FixMisencodedBaseQualityReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_MergeBamAlignment.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityZero.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_ModelSegments.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_LeftAlignAndTrimVariants.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CallCopyRatioSegments.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_ScatterIntervalsByNs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_InbreedingCoeff.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_BamIndexStats.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CountFalsePositives.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateUnmappedAndUnmappedReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_VcfFormatConverter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_AddOrReplaceReadGroups.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBwaSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SetNmAndUqTags.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotModeledSegments.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_CollectVariantCallingMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_MarkIlluminaAdapters.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_SampleList.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CombineGVCFs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_CountBasesInReference.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_CollectIlluminaLaneMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectGcBiasMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_VcfToIntervalList.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_LiftoverVcf.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BwaAndMarkDuplicatesPipelineSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_AddOATag.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_CreateHadoopBamSplittingIndex.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_PrintReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosition.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_ReadsPipelineSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentStartReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateOnSameContigOrNoMappedMateReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_Coverage.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ReplaceSamHeader.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadStrandFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_HaplotypeCaller.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_ApplyBQSRSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_CreateVerifyIDIntensityContaminationMetricsFile.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_CalculateGenotypePosteriors.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_genomicsdb_GenomicsDBImport.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_MergeVcfs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectMultipleMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ValidateVariants.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_ExtractSVEvidenceSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectAllelicCounts.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformUnitReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectQualityYieldMetricsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNScoreVariants.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_RevertOriginalBaseQualitiesAndAddMateCigar.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_SelectVariants.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_IlluminaBasecallsToSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_AccumulateVariantCallingMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectJumpingLibraryMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SplitSamByNumberOfReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateDifferentStrandReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_reference_NonNFastaSize.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_MethylationTypeCaller.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_BaseRecalibratorSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_filtering_FilterMutectCalls.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CalculateMixingFractions.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PreprocessIntervals.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_MappingQualityReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_QualityScoreDistribution.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_FlagStat.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_BaitDesigner.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandBiasBySample.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_MakeSitesOnlyVcf.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildKmers.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_IntervalListTools.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_InbreedingCoeff.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ReadLengthEqualsCigarLengthReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectInsertSizeMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityRankSumTest.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_FilterIntervals.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_LeftAlignIndels.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityNotZeroReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_FixMateInformation.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_SortVcf.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_HaplotypeCallerSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintVariantsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectReadCounts.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ValidateSamFile.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_SortSamSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_FastqToSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_bwa_BwaSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupBlackListReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectAlignmentSummaryMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SamToFastq.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SortSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithBamDepth.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_CollectIlluminaBasecallingMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_GatherBamFiles.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotatorDataSourceDownloader.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_CalculateFingerprintMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_FisherStrand.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildReferenceTaxonomy.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_ConvertHeaderlessHadoopBamShardToBam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_validation_CompareBaseQualities.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_MarkDuplicatesWithMateCigar.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_MergeSamFiles.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_ApplyBQSR.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_CalculateContamination.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectRnaSeqMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_stratifications_AlleleFrequency.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_GatherBQSRReports.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_GatherVcfsCloud.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_AnnotateIntervals.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_inference_CpxVariantReInterpreterSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_artifacts_CollectSequencingArtifactMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqScoreSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_RevertSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_GatherVcfs.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_reference_ExtractSequences.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PassesVendorQualityCheckReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_SplitReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappedReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_SoftClippedReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SamErrorMetric_CollectSamErrorMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_MappingQualityRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_GetSampleName.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SamFormatConverter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_FixCallSetSampleOrdering.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_ClusterCrosscheckMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_ClipReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQuality.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SeqIsStoredReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SetNmMdAndUqTags.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CompareMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotateSegments.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerAlleleBySample.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_IntervalOverlapReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CalculateReadGroupChecksum.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroReferenceLengthAlignmentReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectBaseDistributionByCycleSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_VariantRecalibrator.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqFilterSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_utils_ExtractOriginalAlignmentRecordsByNameSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_reference_NormalizeFasta.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotDenoisedCopyRatios.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_QualByDepth.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_VariantsToTable.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_SplitIntervals.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantWriteTensors.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaAlternateReferenceMaker.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_UpdateVCFSequenceDictionary.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_IlluminaBasecallsToFastq.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountVariantsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_UnmarkDuplicates.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_RMSMappingQuality.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectMultipleMetricsSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_StructuralVariationDiscoveryPipelineSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandOddsRatio.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_CountBases.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_EvaluateInfoFieldConcordance.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_SplitNCigarReads.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PrimaryLineReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroFragmentLengthReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DetermineGermlineContigPloidy.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBreakpointEvidenceSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_UniqueAltReadCount.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_PileupSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ProperlyPairedReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$CigarContainsNoNOperator.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQuality.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_SplitVcfs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FilterFuncotations.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_SplitSamByLibrary.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_CrosscheckFingerprints.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$HasReadGroupReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fingerprint_CheckFingerprint.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantTrain.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_GenotypeGVCFs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_TandemRepeat.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBadGenomicKmersSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_fastq_BamToBfq.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_AddCommentsToBam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_AnalyzeSaturationMutagenesis.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_QualByDepth.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_VariantAnnotator.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_FilterSamReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_BaseQualityRankSumTest.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_MergePedIntoVcf.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_Concordance.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_FragmentLengthReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_BuildBamIndex.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectRrbsMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSecondaryAlignmentReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_MendelianViolations_FindMendelianViolations.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_directed_CollectHsMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_FixVcfHeader.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_CountReads.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_filter_FilterVcf.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_ParallelCopyGCSDirectoryIntoHDFSSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_AnnotatePairOrientation.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_VariantEval.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReferenceBases.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_PrintReadsHeader.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_ViewSam.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithExpectedAlleleFraction.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_IndexFeatureFile.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$AllowAllReadsReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CountVariants.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_util_LiftOverIntervalList.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_MetricsReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotDuplicateReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_gnarlyGenotyper_GnarlyGenotyper.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_markduplicates_EstimateLibraryComplexity.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentEndReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaReferenceMaker.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_SampleReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_QualityScoreDistributionSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerSampleHC.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AlleleFraction.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_artifacts_ConvertSequencingArtifactToOxoG.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$FirstOfPairReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RMSMappingQuality.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_AnalyzeCovariates.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OrientationBiasReadCounts.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_CountNs.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_RenameSampleInVcf.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityAvailableReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectOxoGMetrics.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CreateReadCountPanelOfNormals.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_CalcMetadataSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqPipelineSpark.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_AlignmentAgreesWithHeaderReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SecondOfPairReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_StrandOddsRatio.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_Funcotator.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountReadsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountBasesSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectQualityYieldMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_MeanQualityByCycle.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_vcf_GenotypeConcordance.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintReadsSpark.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_replicates_CollectIndependentReplicateMetrics.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_ASEReadCounter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ChromosomeCounts.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_FilterVariantTranches.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_LibraryReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonChimericOriginalAlignmentReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_CheckPileup.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_arrays_GtcToVcf.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_BwaMemIndexImageCreator.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CheckTerminatorBlock.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadLengthReadFilter.html  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_engine_filters_OverclippedReadFilter.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_Pileup.json  \n",
            "  inflating: gatk-4.1.4.1/gatkdoc/picard_sam_CleanSam.html  \n",
            "  inflating: gatk-4.1.4.1/gatkPythonPackageArchive.zip  \n",
            "  inflating: gatk-4.1.4.1/gatk-completion.sh  \n",
            "  inflating: gatk-4.1.4.1/gatk-package-4.1.4.1-spark.jar  \n",
            "  inflating: gatk-4.1.4.1/gatkcondaenv.yml  \n",
            "  inflating: gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar  \n",
            "  inflating: gatk-4.1.4.1/README.md  \n",
            "   creating: gatk-4.1.4.1/scripts/\n",
            "   creating: gatk-4.1.4.1/scripts/sv/\n",
            "  inflating: gatk-4.1.4.1/scripts/sv/create_cluster.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/delete_cluster.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/sanity_checks.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/manage_sv_pipeline.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/run_whole_pipeline.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/copy_sv_results.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/default_init.sh  \n",
            "   creating: gatk-4.1.4.1/scripts/sv/stepByStep/\n",
            "  inflating: gatk-4.1.4.1/scripts/sv/stepByStep/scanBam.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/stepByStep/discoverVariants.sh  \n",
            "  inflating: gatk-4.1.4.1/scripts/sv/stepByStep/svDiscover.sh  \n",
            "   creating: gatk-4.1.4.1/scripts/cnv_wdl/\n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/cnv_common_tasks.wdl  \n",
            "   creating: gatk-4.1.4.1/scripts/cnv_wdl/germline/\n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/germline/cnv_germline_cohort_workflow.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/germline/README.md  \n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/germline/cnv_germline_case_workflow.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/germline/cnv_germline_case_scattered_workflow.wdl  \n",
            "   creating: gatk-4.1.4.1/scripts/cnv_wdl/somatic/\n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/somatic/cnv_somatic_oncotator_workflow.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/somatic/README.md  \n",
            "  inflating: gatk-4.1.4.1/scripts/cnv_wdl/somatic/cnv_somatic_funcotate_seg_workflow.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/dataproc-cluster-ui  \n",
            "   creating: gatk-4.1.4.1/scripts/mutect2_wdl/\n",
            "  inflating: gatk-4.1.4.1/scripts/mutect2_wdl/mutect2.wdl  \n",
            "   creating: gatk-4.1.4.1/scripts/mutect2_wdl/mutect_resources_json/\n",
            "  inflating: gatk-4.1.4.1/scripts/mutect2_wdl/mutect_resources_json/mutect_resources_process_gnomAD_2.1.json  \n",
            "  inflating: gatk-4.1.4.1/scripts/mutect2_wdl/README.md  \n",
            "  inflating: gatk-4.1.4.1/scripts/mutect2_wdl/mutect2_pon.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/mutect2_wdl/mutect2_multi_sample.wdl  \n",
            "  inflating: gatk-4.1.4.1/scripts/mutect2_wdl/mutect_resources.wdl  \n",
            "  inflating: gatk-4.1.4.1/gatk       \n",
            "  inflating: gatk-4.1.4.1/GATKConfig.EXAMPLE.properties  \n",
            "\n",
            "R version 4.1.0 (2021-05-18) -- \"Camp Pontanezen\"\n",
            "Copyright (C) 2021 The R Foundation for Statistical Computing\n",
            "Platform: x86_64-pc-linux-gnu (64-bit)\n",
            "\n",
            "R is free software and comes with ABSOLUTELY NO WARRANTY.\n",
            "You are welcome to redistribute it under certain conditions.\n",
            "Type 'license()' or 'licence()' for distribution details.\n",
            "\n",
            "  Natural language support but running in an English locale\n",
            "\n",
            "R is a collaborative project with many contributors.\n",
            "Type 'contributors()' for more information and\n",
            "'citation()' on how to cite R or R packages in publications.\n",
            "\n",
            "Type 'demo()' for some demos, 'help()' for on-line help, or\n",
            "'help.start()' for an HTML browser interface to help.\n",
            "Type 'q()' to quit R.\n",
            "\n",
            "> install.packages(c(\"gplots\", \"gsalib\"))\n",
            "Installing packages into ‘/usr/local/lib/R/site-library’\n",
            "(as ‘lib’ is unspecified)\n",
            "also installing the dependencies ‘bitops’, ‘gtools’, ‘caTools’\n",
            "\n",
            "trying URL 'https://cran.rstudio.com/src/contrib/bitops_1.0-7.tar.gz'\n",
            "Content type 'application/x-gzip' length 10809 bytes (10 KB)\n",
            "==================================================\n",
            "downloaded 10 KB\n",
            "\n",
            "trying URL 'https://cran.rstudio.com/src/contrib/gtools_3.8.2.tar.gz'\n",
            "Content type 'application/x-gzip' length 197529 bytes (192 KB)\n",
            "==================================================\n",
            "downloaded 192 KB\n",
            "\n",
            "trying URL 'https://cran.rstudio.com/src/contrib/caTools_1.18.2.tar.gz'\n",
            "Content type 'application/x-gzip' length 64960 bytes (63 KB)\n",
            "==================================================\n",
            "downloaded 63 KB\n",
            "\n",
            "trying URL 'https://cran.rstudio.com/src/contrib/gplots_3.1.1.tar.gz'\n",
            "Content type 'application/x-gzip' length 520248 bytes (508 KB)\n",
            "==================================================\n",
            "downloaded 508 KB\n",
            "\n",
            "trying URL 'https://cran.rstudio.com/src/contrib/gsalib_2.1.tar.gz'\n",
            "Content type 'application/x-gzip' length 4523 bytes\n",
            "==================================================\n",
            "downloaded 4523 bytes\n",
            "\n",
            "* installing *source* package ‘bitops’ ...\n",
            "** package ‘bitops’ successfully unpacked and MD5 sums checked\n",
            "** using staged installation\n",
            "** libs\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c bit-ops.c -o bit-ops.o\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c cksum.c -o cksum.o\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c init.c -o init.o\n",
            "gcc -std=gnu99 -shared -L/usr/lib/R/lib -Wl,-Bsymbolic-functions -Wl,-z,relro -o bitops.so bit-ops.o cksum.o init.o -L/usr/lib/R/lib -lR\n",
            "installing to /usr/local/lib/R/site-library/00LOCK-bitops/00new/bitops/libs\n",
            "** R\n",
            "** byte-compile and prepare package for lazy loading\n",
            "** help\n",
            "*** installing help indices\n",
            "** building package indices\n",
            "** testing if installed package can be loaded from temporary location\n",
            "** checking absolute paths in shared objects and dynamic libraries\n",
            "** testing if installed package can be loaded from final location\n",
            "** testing if installed package keeps a record of temporary installation path\n",
            "* DONE (bitops)\n",
            "* installing *source* package ‘gtools’ ...\n",
            "** package ‘gtools’ successfully unpacked and MD5 sums checked\n",
            "** using staged installation\n",
            "** libs\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c gtools_load.c -o gtools_load.o\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c roman2int.c -o roman2int.o\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c setTCPNoDelay.c -o setTCPNoDelay.o\n",
            "gcc -std=gnu99 -shared -L/usr/lib/R/lib -Wl,-Bsymbolic-functions -Wl,-z,relro -o gtools.so gtools_load.o roman2int.o setTCPNoDelay.o -L/usr/lib/R/lib -lR\n",
            "installing to /usr/local/lib/R/site-library/00LOCK-gtools/00new/gtools/libs\n",
            "** R\n",
            "** data\n",
            "** inst\n",
            "** byte-compile and prepare package for lazy loading\n",
            "** help\n",
            "*** installing help indices\n",
            "** building package indices\n",
            "** testing if installed package can be loaded from temporary location\n",
            "** checking absolute paths in shared objects and dynamic libraries\n",
            "** testing if installed package can be loaded from final location\n",
            "** testing if installed package keeps a record of temporary installation path\n",
            "* DONE (gtools)\n",
            "* installing *source* package ‘gsalib’ ...\n",
            "** package ‘gsalib’ successfully unpacked and MD5 sums checked\n",
            "** using staged installation\n",
            "** R\n",
            "** inst\n",
            "** byte-compile and prepare package for lazy loading\n",
            "** help\n",
            "*** installing help indices\n",
            "** building package indices\n",
            "** testing if installed package can be loaded from temporary location\n",
            "** testing if installed package can be loaded from final location\n",
            "** testing if installed package keeps a record of temporary installation path\n",
            "* DONE (gsalib)\n",
            "* installing *source* package ‘caTools’ ...\n",
            "** package ‘caTools’ successfully unpacked and MD5 sums checked\n",
            "** using staged installation\n",
            "** libs\n",
            "g++ -std=gnu++14 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c Gif2R.cpp -o Gif2R.o\n",
            "g++ -std=gnu++14 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c GifTools.cpp -o GifTools.o\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c caTools_init.c -o caTools_init.o\n",
            "gcc -std=gnu99 -I\"/usr/share/R/include\" -DNDEBUG      -fpic  -g -O2 -fdebug-prefix-map=/build/r-base-vB4ZXq/r-base-4.1.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c runfunc.c -o runfunc.o\n",
            "g++ -std=gnu++14 -shared -L/usr/lib/R/lib -Wl,-Bsymbolic-functions -Wl,-z,relro -o caTools.so Gif2R.o GifTools.o caTools_init.o runfunc.o -L/usr/lib/R/lib -lR\n",
            "installing to /usr/local/lib/R/site-library/00LOCK-caTools/00new/caTools/libs\n",
            "** R\n",
            "** byte-compile and prepare package for lazy loading\n",
            "** help\n",
            "*** installing help indices\n",
            "** building package indices\n",
            "** testing if installed package can be loaded from temporary location\n",
            "** checking absolute paths in shared objects and dynamic libraries\n",
            "** testing if installed package can be loaded from final location\n",
            "** testing if installed package keeps a record of temporary installation path\n",
            "* DONE (caTools)\n",
            "* installing *source* package ‘gplots’ ...\n",
            "** package ‘gplots’ successfully unpacked and MD5 sums checked\n",
            "** using staged installation\n",
            "** R\n",
            "** data\n",
            "** inst\n",
            "** byte-compile and prepare package for lazy loading\n",
            "** help\n",
            "*** installing help indices\n",
            "** building package indices\n",
            "** installing vignettes\n",
            "** testing if installed package can be loaded from temporary location\n",
            "** testing if installed package can be loaded from final location\n",
            "** testing if installed package keeps a record of temporary installation path\n",
            "* DONE (gplots)\n",
            "\n",
            "The downloaded source packages are in\n",
            "\t‘/tmp/Rtmpxpo8Lp/downloaded_packages’\n",
            "> \n",
            "> \n",
            "Downloading...\n",
            "From: https://drive.google.com/uc?id=13ESrnOy32eWbPO2jEEMTtLj2vn7NBseS\n",
            "To: /content/SRR10708177_1.fastq.gz\n",
            "13.3MB [00:00, 80.9MB/s]\n",
            "Downloading...\n",
            "From: https://drive.google.com/uc?id=1_rvfN87uprNVbIX09ZcdkhLPNjoohgrQ\n",
            "To: /content/SRR10708177_2.fastq.gz\n",
            "15.0MB [00:00, 91.1MB/s]\n",
            "Cloning into 'bwa'...\n",
            "remote: Enumerating objects: 4326, done.\u001b[K\n",
            "remote: Counting objects: 100% (31/31), done.\u001b[K\n",
            "remote: Compressing objects: 100% (23/23), done.\u001b[K\n",
            "remote: Total 4326 (delta 9), reused 20 (delta 6), pack-reused 4295\u001b[K\n",
            "Receiving objects: 100% (4326/4326), 1.68 MiB | 21.56 MiB/s, done.\n",
            "Resolving deltas: 100% (3084/3084), done.\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  utils.c -o utils.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  kthread.c -o kthread.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  kstring.c -o kstring.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  ksw.c -o ksw.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwt.c -o bwt.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bntseq.c -o bntseq.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwa.c -o bwa.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwamem.c -o bwamem.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwamem_pair.c -o bwamem_pair.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwamem_extra.c -o bwamem_extra.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  malloc_wrap.c -o malloc_wrap.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  QSufSort.c -o QSufSort.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwt_gen.c -o bwt_gen.o\n",
            "\u001b[01m\u001b[Kbwt_gen.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KBWTIncBuildRelativeRank\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kbwt_gen.c:879:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KoldInverseSa0RelativeRank\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K-Wunused-but-set-variable\u001b[m\u001b[K]\n",
            "  bgint_t \u001b[01;35m\u001b[KoldInverseSa0RelativeRank\u001b[m\u001b[K = 0;\n",
            "          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kbwt_gen.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KBWTIncMergeBwt\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kbwt_gen.c:953:15:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KbitsInWordMinusBitPerChar\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K-Wunused-but-set-variable\u001b[m\u001b[K]\n",
            "  unsigned int \u001b[01;35m\u001b[KbitsInWordMinusBitPerChar\u001b[m\u001b[K;\n",
            "               \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  rope.c -o rope.o\n",
            "\u001b[01m\u001b[Krope.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Krope_restore_node\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Krope.c:282:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "  \u001b[01;35m\u001b[Kfread(&is_bottom, 1, 1, fp)\u001b[m\u001b[K;\n",
            "  \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krope.c:283:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "  \u001b[01;35m\u001b[Kfread(&n, 2, 1, fp)\u001b[m\u001b[K;\n",
            "  \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krope.c:291:4:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "    \u001b[01;35m\u001b[Kfread(p[i].c, 8, 6, fp)\u001b[m\u001b[K;\n",
            "    \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krope.c:292:4:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "    \u001b[01;35m\u001b[Kfread(q, 2, 1, fp)\u001b[m\u001b[K;\n",
            "    \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krope.c:293:4:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "    \u001b[01;35m\u001b[Kfread(q + 1, 1, *q, fp)\u001b[m\u001b[K;\n",
            "    \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krope.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Krope_restore\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Krope.c:312:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "  \u001b[01;35m\u001b[Kfread(&r->max_nodes, 4, 1, fp)\u001b[m\u001b[K;\n",
            "  \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krope.c:313:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "  \u001b[01;35m\u001b[Kfread(&r->block_len, 4, 1, fp)\u001b[m\u001b[K;\n",
            "  \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  rle.c -o rle.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  is.c -o is.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtindex.c -o bwtindex.o\n",
            "ar -csru libbwa.a utils.o kthread.o kstring.o ksw.o bwt.o bntseq.o bwa.o bwamem.o bwamem_pair.o bwamem_extra.o malloc_wrap.o QSufSort.o bwt_gen.o rope.o rle.o is.o bwtindex.o\n",
            "ar: `u' modifier ignored since `D' is the default (see `U')\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwashm.c -o bwashm.o\n",
            "\u001b[01m\u001b[Kbwashm.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbwa_shm_stage\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kbwashm.c:29:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kftruncate\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "  \u001b[01;35m\u001b[Kftruncate(shmid, BWA_CTL_SIZE)\u001b[m\u001b[K;\n",
            "  \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kbwashm.c:66:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kftruncate\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "  \u001b[01;35m\u001b[Kftruncate(shmid, idx->l_mem)\u001b[m\u001b[K;\n",
            "  \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwase.c -o bwase.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwaseqio.c -o bwaseqio.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtgap.c -o bwtgap.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtaln.c -o bwtaln.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bamlite.c -o bamlite.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwape.c -o bwape.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  kopen.c -o kopen.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  pemerge.c -o pemerge.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  maxk.c -o maxk.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtsw2_core.c -o bwtsw2_core.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtsw2_main.c -o bwtsw2_main.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtsw2_aux.c -o bwtsw2_aux.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwt_lite.c -o bwt_lite.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtsw2_chain.c -o bwtsw2_chain.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  fastmap.c -o fastmap.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  bwtsw2_pair.c -o bwtsw2_pair.o\n",
            "gcc -c -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS  main.c -o main.o\n",
            "gcc -g -Wall -Wno-unused-function -O2 -DHAVE_PTHREAD -DUSE_MALLOC_WRAPPERS bwashm.o bwase.o bwaseqio.o bwtgap.o bwtaln.o bamlite.o bwape.o kopen.o pemerge.o maxk.o bwtsw2_core.o bwtsw2_main.o bwtsw2_aux.o bwt_lite.o bwtsw2_chain.o fastmap.o bwtsw2_pair.o main.o -o bwa -L. -lbwa -lm -lz -lpthread -lrt\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "lxts6VVv2uBw",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "d3694ca6-e901-41ee-b9c8-a06c441d33a8"
      },
      "source": [
        "!gzip -d '/content/drive/MyDrive/sample_25.chrom11.exome.pe1.fq.gz'\n",
        "!gzip -d '/content/drive/MyDrive/sample_25.chrom11.exome.pe2.fq.gz'"
      ],
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "gzip: /content/drive/MyDrive/sample_25.chrom11.exome.pe1.fq.gz: No such file or directory\n",
            "gzip: /content/drive/MyDrive/sample_25.chrom11.exome.pe2.fq.gz: No such file or directory\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "IkS4hCv6ycJi"
      },
      "source": [
        "### 1. Izvršiti kontrolu kvaliteta FASTQ fajlova alatom FastQC. Priložiti izvrštaj i diskutovati rezultate (da li je neki od kriterijuma koje analizira FastQC označen kao problematičan, i šta taj kriterijum znači?)\n",
        "\n",
        "....(detaljno u izveštaju)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "n76HdgkM1FI-"
      },
      "source": [
        "### 2. Mapirati sekvencirane readove na referentni genom hg38 upotrebom alata BWA Mem. Upotrebom python biblioteke pysam odrediti sledeće:\n",
        "2a. Koliko je readova uspešno mapirano?\n",
        "\n",
        "2b. Koliko je parova readova mapirano tako da su oba para mapirana?\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "w0Yv83MO3OEQ",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "5dc0527a-0bda-492f-e176-b1310bbc97ea"
      },
      "source": [
        "!pip install bwa"
      ],
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting bwa\n",
            "  Downloading https://files.pythonhosted.org/packages/c8/f0/dabce80929f9a646ae330837bf84c2e9746fca3691fb552827f7243f28d0/bwa-1.1.1-py3-none-any.whl\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.7/dist-packages (from bwa) (2.23.0)\n",
            "Collecting yagmail>=0.11.214\n",
            "  Downloading https://files.pythonhosted.org/packages/07/c9/f8252868c64eb53256d6f3847f99c902e151325260c4246d09ddd967fb83/yagmail-0.14.256-py2.py3-none-any.whl\n",
            "Collecting python-telegram-bot\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/e4/44/1726f407387a7332a08b4e95211380b3bda185cedbe73d1058010136e695/python_telegram_bot-13.5-py3-none-any.whl (455kB)\n",
            "\u001b[K     |████████████████████████████████| 460kB 22.9MB/s \n",
            "\u001b[?25hRequirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests->bwa) (2.10)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests->bwa) (1.24.3)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests->bwa) (2020.12.5)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests->bwa) (3.0.4)\n",
            "Collecting premailer\n",
            "  Downloading https://files.pythonhosted.org/packages/d9/02/ba01e611eb82df8be03849de6564851387336c9af24b33b588b809b3d09f/premailer-3.9.0-py2.py3-none-any.whl\n",
            "Requirement already satisfied: tornado>=5.1 in /usr/local/lib/python3.7/dist-packages (from python-telegram-bot->bwa) (5.1.1)\n",
            "Collecting APScheduler==3.6.3\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/f3/34/9ef20ed473c4fd2c3df54ef77a27ae3fc7500b16b192add4720cab8b2c09/APScheduler-3.6.3-py2.py3-none-any.whl (58kB)\n",
            "\u001b[K     |████████████████████████████████| 61kB 6.1MB/s \n",
            "\u001b[?25hRequirement already satisfied: pytz>=2018.6 in /usr/local/lib/python3.7/dist-packages (from python-telegram-bot->bwa) (2018.9)\n",
            "Collecting cssutils\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/24/c4/9db28fe567612896d360ab28ad02ee8ae107d0e92a22db39affd3fba6212/cssutils-2.3.0-py3-none-any.whl (404kB)\n",
            "\u001b[K     |████████████████████████████████| 409kB 43.6MB/s \n",
            "\u001b[?25hCollecting cssselect\n",
            "  Downloading https://files.pythonhosted.org/packages/3b/d4/3b5c17f00cce85b9a1e6f91096e1cc8e8ede2e1be8e96b87ce1ed09e92c5/cssselect-1.1.0-py2.py3-none-any.whl\n",
            "Requirement already satisfied: cachetools in /usr/local/lib/python3.7/dist-packages (from premailer->yagmail>=0.11.214->bwa) (4.2.2)\n",
            "Requirement already satisfied: lxml in /usr/local/lib/python3.7/dist-packages (from premailer->yagmail>=0.11.214->bwa) (4.2.6)\n",
            "Requirement already satisfied: setuptools>=0.7 in /usr/local/lib/python3.7/dist-packages (from APScheduler==3.6.3->python-telegram-bot->bwa) (57.0.0)\n",
            "Requirement already satisfied: tzlocal>=1.2 in /usr/local/lib/python3.7/dist-packages (from APScheduler==3.6.3->python-telegram-bot->bwa) (1.5.1)\n",
            "Requirement already satisfied: six>=1.4.0 in /usr/local/lib/python3.7/dist-packages (from APScheduler==3.6.3->python-telegram-bot->bwa) (1.15.0)\n",
            "Requirement already satisfied: importlib-metadata; python_version < \"3.8\" in /usr/local/lib/python3.7/dist-packages (from cssutils->premailer->yagmail>=0.11.214->bwa) (4.0.1)\n",
            "Requirement already satisfied: typing-extensions>=3.6.4; python_version < \"3.8\" in /usr/local/lib/python3.7/dist-packages (from importlib-metadata; python_version < \"3.8\"->cssutils->premailer->yagmail>=0.11.214->bwa) (3.7.4.3)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata; python_version < \"3.8\"->cssutils->premailer->yagmail>=0.11.214->bwa) (3.4.1)\n",
            "Installing collected packages: cssutils, cssselect, premailer, yagmail, APScheduler, python-telegram-bot, bwa\n",
            "Successfully installed APScheduler-3.6.3 bwa-1.1.1 cssselect-1.1.0 cssutils-2.3.0 premailer-3.9.0 python-telegram-bot-13.5 yagmail-0.14.256\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "AUsO8JzpULBe"
      },
      "source": [
        "!gsutil ls gs://genomics-public-data"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Jp397GNcUO35"
      },
      "source": [
        "!gsutil ls gs://genomics-public-data/resources/broad/hg38/v0"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "GCTIJmc7UyZG",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f64e8be6-b1d2-454f-e662-d650540adaaa"
      },
      "source": [
        "!gsutil cp gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta ."
      ],
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Copying gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta...\n",
            "\\ [1 files][  3.0 GiB/  3.0 GiB]   55.4 MiB/s                                   \n",
            "Operation completed over 1 objects/3.0 GiB.                                      \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "lvni0BFjxw8j",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9e1ad0c7-b5ca-40b7-e856-6c0b3d28fa80"
      },
      "source": [
        "!gsutil cp gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta.fai ."
      ],
      "execution_count": 37,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Copying gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta.fai...\n",
            "/ [0 files][    0.0 B/157.2 KiB]                                                \r/ [1 files][157.2 KiB/157.2 KiB]                                                \r\n",
            "Operation completed over 1 objects/157.2 KiB.                                    \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "k2QFUCFz3f9Z",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "d11f591d-db69-4d81-9f98-49327c0fe2ca"
      },
      "source": [
        "!bwa/bwa index Homo_sapiens_assembly38.fasta"
      ],
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "[bwa_index] Pack FASTA... tcmalloc: large alloc 1073741824 bytes == 0x55694eff8000 @  0x7fb02c3602a4 0x5568fd38bf7b 0x5568fd374759 0x5568fd38fe30 0x5568fd39051f 0x5568fd350043 0x7fb02b57abf7 0x5568fd35017a\n",
            "tcmalloc: large alloc 1608679424 bytes == 0x55698f060000 @  0x7fb02c3602a4 0x5568fd38bf7b 0x5568fd374a54 0x5568fd38fe30 0x5568fd39051f 0x5568fd350043 0x7fb02b57abf7 0x5568fd35017a\n",
            "28.74 sec\n",
            "[bwa_index] Construct BWT for the packed sequence...\n",
            "[BWTIncCreate] textLength=6434693834, availableWord=464768632\n",
            "tcmalloc: large alloc 1859076096 bytes == 0x55698f05c000 @  0x7fb02c361001 0x5568fd38be9c 0x5568fd38cc7b 0x5568fd38e4c1 0x5568fd38e874 0x5568fd390334 0x5568fd39051f 0x5568fd350043 0x7fb02b57abf7 0x5568fd35017a\n",
            "[BWTIncConstructFromPacked] 10 iterations done. 99999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 20 iterations done. 199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 30 iterations done. 299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 40 iterations done. 399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 50 iterations done. 499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 60 iterations done. 599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 70 iterations done. 699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 80 iterations done. 799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 90 iterations done. 899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 100 iterations done. 999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 110 iterations done. 1099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 120 iterations done. 1199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 130 iterations done. 1299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 140 iterations done. 1399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 150 iterations done. 1499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 160 iterations done. 1599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 170 iterations done. 1699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 180 iterations done. 1799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 190 iterations done. 1899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 200 iterations done. 1999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 210 iterations done. 2099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 220 iterations done. 2199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 230 iterations done. 2299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 240 iterations done. 2399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 250 iterations done. 2499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 260 iterations done. 2599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 270 iterations done. 2699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 280 iterations done. 2799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 290 iterations done. 2899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 300 iterations done. 2999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 310 iterations done. 3099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 320 iterations done. 3199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 330 iterations done. 3299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 340 iterations done. 3399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 350 iterations done. 3499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 360 iterations done. 3599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 370 iterations done. 3699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 380 iterations done. 3799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 390 iterations done. 3899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 400 iterations done. 3999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 410 iterations done. 4099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 420 iterations done. 4199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 430 iterations done. 4299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 440 iterations done. 4399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 450 iterations done. 4499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 460 iterations done. 4599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 470 iterations done. 4699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 480 iterations done. 4799999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 490 iterations done. 4899999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 500 iterations done. 4999999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 510 iterations done. 5099999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 520 iterations done. 5199999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 530 iterations done. 5299999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 540 iterations done. 5399999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 550 iterations done. 5499999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 560 iterations done. 5599999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 570 iterations done. 5699999994 characters processed.\n",
            "[BWTIncConstructFromPacked] 580 iterations done. 5799054074 characters processed.\n",
            "[BWTIncConstructFromPacked] 590 iterations done. 5888997130 characters processed.\n",
            "[BWTIncConstructFromPacked] 600 iterations done. 5968934746 characters processed.\n",
            "[BWTIncConstructFromPacked] 610 iterations done. 6039979482 characters processed.\n",
            "[BWTIncConstructFromPacked] 620 iterations done. 6103120202 characters processed.\n",
            "[BWTIncConstructFromPacked] 630 iterations done. 6159235834 characters processed.\n",
            "[BWTIncConstructFromPacked] 640 iterations done. 6209107546 characters processed.\n",
            "[BWTIncConstructFromPacked] 650 iterations done. 6253429626 characters processed.\n",
            "[BWTIncConstructFromPacked] 660 iterations done. 6292819178 characters processed.\n",
            "[BWTIncConstructFromPacked] 670 iterations done. 6327824746 characters processed.\n",
            "[BWTIncConstructFromPacked] 680 iterations done. 6358933706 characters processed.\n",
            "[BWTIncConstructFromPacked] 690 iterations done. 6386579386 characters processed.\n",
            "[BWTIncConstructFromPacked] 700 iterations done. 6411146922 characters processed.\n",
            "[BWTIncConstructFromPacked] 710 iterations done. 6432978554 characters processed.\n",
            "[bwt_gen] Finished constructing BWT in 711 iterations.\n",
            "[bwa_index] 3566.42 seconds elapse.\n",
            "[bwa_index] Update BWT... tcmalloc: large alloc 1608679424 bytes == 0x55698f02e000 @  0x7fb02c361001 0x5568fd38be9c 0x5568fd372ff7 0x5568fd38ff07 0x5568fd39051f 0x5568fd350043 0x7fb02b57abf7 0x5568fd35017a\n",
            "tcmalloc: large alloc 3217350656 bytes == 0x5569eee56000 @  0x7fb02c361001 0x5568fd38be9c 0x5568fd38facf 0x5568fd38ff12 0x5568fd39051f 0x5568fd350043 0x7fb02b57abf7 0x5568fd35017a\n",
            "23.29 sec\n",
            "[bwa_index] Pack forward-only FASTA... 20.05 sec\n",
            "[bwa_index] Construct SA from BWT and Occ... tcmalloc: large alloc 3217350656 bytes == 0x55698f028000 @  0x7fb02c361001 0x5568fd38be9c 0x5568fd372ff7 0x5568fd38ffc9 0x5568fd39051f 0x5568fd350043 0x7fb02b57abf7 0x5568fd35017a\n",
            "1806.85 sec\n",
            "[main] Version: 0.7.17-r1198-dirty\n",
            "[main] CMD: bwa/bwa index Homo_sapiens_assembly38.fasta\n",
            "[main] Real time: 5613.936 sec; CPU: 5445.372 sec\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Q1ZLsCLkAbrV",
        "outputId": "7a5f6691-6f1f-4465-87c8-341b664f15a4"
      },
      "source": [
        "!gsutil cp gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf . \n",
        "!gsutil cp gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf.idx .\n",
        "!gsutil cp gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta .\n",
        "!gsutil cp gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta.fai .\n",
        "!gsutil cp gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.dict ."
      ],
      "execution_count": 27,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Copying gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf...\n",
            "/ [1 files][ 10.2 GiB/ 10.2 GiB]   49.2 MiB/s                                   \n",
            "Operation completed over 1 objects/10.2 GiB.                                     \n",
            "Copying gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf.idx...\n",
            "/ [1 files][ 11.9 MiB/ 11.9 MiB]                                                \n",
            "Operation completed over 1 objects/11.9 MiB.                                     \n",
            "Copying gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta...\n",
            "- [1 files][  3.0 GiB/  3.0 GiB]   49.6 MiB/s                                   \n",
            "Operation completed over 1 objects/3.0 GiB.                                      \n",
            "Copying gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta.fai...\n",
            "/ [1 files][157.2 KiB/157.2 KiB]                                                \n",
            "Operation completed over 1 objects/157.2 KiB.                                    \n",
            "Copying gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.dict...\n",
            "/ [1 files][568.1 KiB/568.1 KiB]                                                \n",
            "Operation completed over 1 objects/568.1 KiB.                                    \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "22VXDCuHLA0e",
        "outputId": "5d8187f4-2526-4f8e-f0c7-fb07deb5c9e2"
      },
      "source": [
        "# !gdown --id 1-0U11U1Z3Dp8vLHoZOtagQl25dFdza-C"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Downloading...\n",
            "From: https://drive.google.com/uc?id=1-0U11U1Z3Dp8vLHoZOtagQl25dFdza-C\n",
            "To: /content/HG00096.chr20.exome.sorted.bam\n",
            "66.0MB [00:01, 52.0MB/s]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "d7rhNOpmLguX"
      },
      "source": [
        "!gsutil ls gs://genomics-public-data/resources/broad/hg38/v0"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "k1WGx121LuRJ"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1Dwq2-veQEeP",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b72be828-df77-4dcf-e915-90ebd3f13675"
      },
      "source": [
        "pe1 = '/content/drive/MyDrive/sample_25.chrom11.exome.pe1.fq'\n",
        "pe2 = '/content/drive/MyDrive/sample_25.chrom11.exome.pe2.fq'\n",
        "\n",
        "# !bwa/bwa mem -M Homo_sapiens_assembly38.fasta {pe1} {pe2} > aln-pe.sam\n",
        "# !bwa/bwa mem -M -R '@RG\\tID:1\\tPL:Illumina\\tSM:HG00096' -o HG00096.chr11.exome.sam Homo_sapiens_assembly38.fasta {pe1} {pe2} \n",
        "!./bwa/bwa mem -M -R '@RG\\tID:1\\tPL:Illumina\\tSM:HG00096' -o HG00096.chr11.exome.sam Homo_sapiens_assembly38.fasta {pe1} {pe2} "
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "tcmalloc: large alloc 3217350656 bytes == 0x55f378ae6000 @  0x7f1f7de01001 0x55f377f9be9c 0x55f377f82ff7 0x55f377f868a6 0x55f377f86ac8 0x55f377f79b4f 0x55f377f600d4 0x7f1f7d01abf7 0x55f377f6017a\n",
            "tcmalloc: large alloc 1608679424 bytes == 0x55f438734000 @  0x7f1f7de01001 0x55f377f9be9c 0x55f377f82eed 0x55f377f868cf 0x55f377f86ac8 0x55f377f79b4f 0x55f377f600d4 0x7f1f7d01abf7 0x55f377f6017a\n",
            "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 52123, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.79, 41.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 11.827 CPU sec, 13.820 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 56464, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.65, 41.83)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 11.030 CPU sec, 10.894 real sec\n",
            "[M::process] read 131294 sequences (10000091 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 54032, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.66, 41.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 10.751 CPU sec, 10.601 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (3, 54855, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.63, 41.71)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131294 reads in 11.044 CPU sec, 10.892 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 53354, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 166)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 284)\n",
            "[M::mem_pestat] mean and std.dev: (139.25, 42.28)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 343)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.503 CPU sec, 10.309 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 57033, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.50, 41.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.468 CPU sec, 10.356 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 50928, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.84, 41.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.679 CPU sec, 10.487 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 57094, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.54, 41.77)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.519 CPU sec, 10.355 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 51470, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.55, 41.99)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.820 CPU sec, 10.671 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 55506, 0, 4)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.67, 41.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.611 CPU sec, 10.473 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 54796, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.68, 41.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.950 CPU sec, 11.721 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 52932, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.83, 41.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.861 CPU sec, 10.729 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (3, 55109, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.75, 41.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.604 CPU sec, 10.420 real sec\n",
            "[M::process] read 130788 sequences (10000017 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 53564, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.67, 41.90)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.875 CPU sec, 10.678 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (5, 55149, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.69, 41.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 130788 reads in 10.825 CPU sec, 10.625 real sec\n",
            "[M::process] read 129702 sequences (10000116 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 55284, 0, 3)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.67, 41.58)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 10.917 CPU sec, 10.732 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 53766, 1, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.62, 41.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129702 reads in 10.832 CPU sec, 10.647 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 54362, 0, 4)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.45, 41.71)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.626 CPU sec, 10.457 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 54892, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.48, 41.73)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.473 CPU sec, 10.315 real sec\n",
            "[M::process] read 130526 sequences (10000046 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 52967, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.56, 41.81)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.747 CPU sec, 10.614 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 55852, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.63, 41.73)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 130526 reads in 10.582 CPU sec, 10.448 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 54477, 0, 3)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (106, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 283)\n",
            "[M::mem_pestat] mean and std.dev: (138.27, 41.87)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 342)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 11.069 CPU sec, 10.909 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 54940, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.77, 41.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 10.942 CPU sec, 10.855 real sec\n",
            "[M::process] read 129678 sequences (10000017 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 55518, 0, 3)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.22, 41.65)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 11.044 CPU sec, 10.864 real sec\n",
            "[M::process] read 129034 sequences (10000135 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 53818, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 166)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 284)\n",
            "[M::mem_pestat] mean and std.dev: (139.28, 42.21)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 343)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129678 reads in 10.969 CPU sec, 10.829 real sec\n",
            "[M::process] read 130816 sequences (10000117 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 54402, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.76, 41.79)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 129034 reads in 10.904 CPU sec, 10.730 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 54688, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.58, 41.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 130816 reads in 10.969 CPU sec, 10.853 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 55498, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 166)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 284)\n",
            "[M::mem_pestat] mean and std.dev: (139.16, 41.99)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 343)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 11.015 CPU sec, 10.867 real sec\n",
            "[M::process] read 131580 sequences (10000080 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 53762, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 166)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 284)\n",
            "[M::mem_pestat] mean and std.dev: (139.20, 42.05)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 343)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 11.154 CPU sec, 10.990 real sec\n",
            "[M::process] read 81550 sequences (10000150 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 56669, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 132, 165)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 281)\n",
            "[M::mem_pestat] mean and std.dev: (138.58, 41.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 339)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 131580 reads in 11.022 CPU sec, 10.818 real sec\n",
            "[M::process] read 66226 sequences (10000126 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 13957, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (107, 133, 166)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 284)\n",
            "[M::mem_pestat] mean and std.dev: (138.93, 42.28)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 343)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 81550 reads in 30.234 CPU sec, 30.059 real sec\n",
            "[M::process] read 66226 sequences (10000126 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation FR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 66226 reads in 26.642 CPU sec, 26.495 real sec\n",
            "[M::process] read 42484 sequences (6415084 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 2, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation FR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 66226 reads in 26.303 CPU sec, 26.219 real sec\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 1, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation FR as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 42484 reads in 17.007 CPU sec, 16.944 real sec\n",
            "[main] Version: 0.7.17-r1198-dirty\n",
            "[main] CMD: ./bwa/bwa mem -M -R @RG\\tID:1\\tPL:Illumina\\tSM:HG00096 -o HG00096.chr11.exome.sam Homo_sapiens_assembly38.fasta /content/drive/MyDrive/sample_25.chrom11.exome.pe1.fq /content/drive/MyDrive/sample_25.chrom11.exome.pe2.fq\n",
            "[main] Real time: 450.393 sec; CPU: 431.527 sec\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9W0juj6DA7J7"
      },
      "source": [
        "!samtools view -b -o HG00096.chr11.exome.bam HG00096.chr11.exome.sam"
      ],
      "execution_count": 10,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "H4MUgg7Xd61y",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "49418e3e-37e6-4851-8022-80abaa6454c2"
      },
      "source": [
        "!samtools sort -o HG00096.chr11.exome.sorted.bam HG00096.chr11.exome.bam\n",
        "!samtools index HG00096.chr11.exome.sorted.bam"
      ],
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "[bam_sort_core] merging from 1 files and 1 in-memory blocks...\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "DASmu9oR5nFM"
      },
      "source": [
        "import pysam\n",
        "\n",
        "samfile = pysam.AlignmentFile('HG00096.chr11.exome.sorted.bam', 'rb')\n",
        "\n",
        "cnt = 0\n",
        "successful = 0\n",
        "unsuccessful = 0\n",
        "both_mapped = 0\n",
        "template_lengths = []\n",
        "for read in samfile.fetch():\n",
        "    cnt+=1\n",
        "    if read.is_unmapped:\n",
        "        unsuccessful += 1\n",
        "    else:\n",
        "        if not read.mate_is_unmapped:\n",
        "            both_mapped += 1\n",
        "        successful += 1\n",
        "    template_lengths.append(read.template_length)\n",
        "\n",
        "    \n",
        "\n"
      ],
      "execution_count": 12,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "4pgyAaDVwPLq",
        "outputId": "bcc42fc6-5995-43f4-c50b-e8d4814f341f"
      },
      "source": [
        "print(f'Mapirano readova: {successful}\\nMapiranih parova readova: {both_mapped}\\nNemapirano redova:{unsuccessful}')"
      ],
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Mapirano readova: 3913079\n",
            "Mapiranih parova readova: 3887368\n",
            "Nemapirano redova:25695\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "dtLFGPOeRC0B"
      },
      "source": [
        "2c. Nacrtati histogram dužina sekvenciranih fragmenata (template_length)."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 311
        },
        "id": "UnJXG0nwPvHh",
        "outputId": "db042d36-06e6-4080-e738-8d51da8d20b9"
      },
      "source": [
        "import matplotlib.pyplot as plt\n",
        "\n",
        "plt.hist(template_lengths, bins=201, log=True) \n",
        "plt.title('Template lengths')\n",
        "\n"
      ],
      "execution_count": 158,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Text(0.5, 1.0, 'Template lengths')"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 158
        },
        {
          "output_type": "display_data",
          "data": {
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAXcAAAEVCAYAAAAb/KWvAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4yLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+WH4yJAAAWF0lEQVR4nO3df5BdZ33f8fcHm7UzdjE/7FAj2chg10HTTkK8tWnJTDwBigQRSgkEa0iLM8KKmXHa6Y8QpTCTmEwCdKZJSzFxFfCIkIyM4iGpVKu1IcQV05jUggZio7gIB6JVHEv+gagTB2P87R975d6u98e5e+/du/vo/Zq5s/c859znfM/e3e8++z3nPidVhSSpLc+ZdACSpNEzuUtSg0zuktQgk7skNcjkLkkNMrlLUoNM7mpSkruSvHOF9vX1JK9diX3N2e+GJJXkzJXet1Y/k7tGLsnjfY+nkzzRt/z2Scc316SS86DWSpxaHfyLr5GrqnNPPU/ydeCdVfWZyUUknX4cuWvFJHlOkp1JvpbkkSR7k7ywt+5UieGnkhxN8liS65P8/SRfTvLNJB/u6+vaJP8jyYeTnEzyp0les8B+X57ks719Ppzkt5M8v7fuE8DFwP7efxbv7rW/Kskf9vb7pSRXj/AY35Hkz3uxvKfvtd+T5OO9Yz+c5N1JZhaLs+ftC/R3ZZJDSb6V5KEkv9rlGNSIqvLhY2wP4OvAa3vP/znweWA9cBbwn4A9vXUbgAJuBs4G/hHwN8DvAd8LrAOOAz/c2/5a4CngXwDPBd4GnARe2Ft/F7P/MQBcCryut88LgIPAv58vxt7yOuAR4A3MDoBe11u+YETH+BvA9wDfD3wbeEVv/QeA/w68oPf6LwMzi8S5VH93A/+k9/xc4FWT/nnwsXIPR+5aSdcD76mqmar6NvCLwFvmnBD8par6m6q6E/grZhPj8ao6BnwOeGXftseZTdLfqapPAvcDb5y706o6UlWfrqpvV9UJ4FeBH14kzp8EDlTVgap6uqo+DRxiNtmP4hhvrKonqupLwJeYTcoAPwH8SlU9VlUzwIc67G+x/r4DXJrk/Kp6vKo+37E/NcDkrpX0UuB3e6WObwKHge8CL+7b5qG+50/Ms3xu3/Kxquqf+e4bwEvm7jTJi5PcmuRYkm8BvwWcv0Scbz0VZy/WHwIuXPoQOx3jX/Y9/+u+Y3oJcLRvXf/zxSzU33bg7wB/muSeJD/asT81wOSulXQU2FxVz+97nN0blS/HuiTpW74Y+It5tvsVZssXf6+qnsfsyLz/dXOnRj0KfGJOnOdU1Qc6xDTMMT7IbDnmlIvmrB9oCteq+mpVbWO2rPVB4LYk5wzSh9Yuk7tW0s3ALyd5KUCSC5JsHaK/7wX+WZLnJnkr8ArgwDzb/S3gceBkknXAz85Z/xDwsr7l3wK2JHl9kjOSnJ3k6iTrWdowx7gX+PkkL+jFecMScS4qyU8muaCqnga+2Wt+uuvrtbaZ3LWS/gOwD7gzyf9h9sTjVUP090fAZcDDwC8Db6mqR+bZ7kbgB5k94Xo78Kk5698PvLdXSvnXVXUU2Ar8G+AEs6Pxn6Xb78swx/g+YAb4M+AzwG3MniCdN84O/W0C7kvyeC+ua6rqiY6xaI3L/1+ylNaGJNcyezXMD006lnFJ8i5mE/JiJ3+leTlyl1aJJBcmeXXvWvnLgX8F/O6k49La5CdUpdVjitnr4i9htkZ+K/CRiUakNcuyjCQ1yLKMJDVoVZRlzj///NqwYcOkw5CkNeULX/jCw1V1wXzrJprck2wBtlx66aUcOnRokqFI0pqT5BsLrZtoWaaq9lfVjvPOO2+SYUhScyaa3JNsSbLr5MmTkwxDkprjyF2SGuTIXZIa5MhdkhrkyF2SGuTIXZIa5CdUJalBlmWkRWzYefukQ5CWxbKMJDXIsowkNcjkLkkNsuYuSQ2y5i5JDbIsI0kNMrlLUoNM7pLUIE+oSlKDPKEqSQ2yLCNJDTK5S1KDTO6S1CCTuyQ16MxRd5jkOcAvAc8DDlXVx0e9D0nS4jqN3JPckuR4knvntG9Kcn+SI0l29pq3AuuB7wAzow1XktRF17LMbmBTf0OSM4CbgM3ARmBbko3A5cAfVtW/BN41ulAlSV11Su5VdRB4dE7zlcCRqnqgqp4EbmV21D4DPNbb5rsL9ZlkR5JDSQ6dOHFi8MglSQsa5oTqOuBo3/JMr+1TwOuT/Efg4EIvrqpdwI3AF6empoYIQ5I018hPqFbVXwPbO267H9g/PT193ajjkKTT2TAj92PARX3L63ttnTm3jCSNxzDJ/R7gsiSXJJkCrgH2DdKBc8tI0nh0vRRyD3A3cHmSmSTbq+op4AbgDuAwsLeq7htk547cJWk8OtXcq2rbAu0HgAPL3bk1d0kaD+dzl6QGOZ+7JDXIkbskNciRuyQ1yCl/JalBlmUkqUGWZSSpQZZlJKlBJndJapA1d0lqkDV3SWqQZRlJapDJXZIaZHKXpAZ5QlWSGuQJVUlqkGUZSWqQyV2SGmRyl6QGmdwlqUEjT+5Jrk7yuSQ3J7l61P1LkpbWKbknuSXJ8ST3zmnflOT+JEeS7Ow1F/A4cDYwM9pwJUlddB257wY29TckOQO4CdgMbAS2JdkIfK6qNgM/B9w4ulAlSV11Su5VdRB4dE7zlcCRqnqgqp4EbgW2VtXTvfWPAWct1GeSHUkOJTl04sSJZYQuSVrImUO8dh1wtG95BrgqyZuB1wPPBz680IuraleSB4EtU1NTVwwRhyRpjpGfUK2qT1XVT1fV26rqriW29ROqkjQGwyT3Y8BFfcvre22dObeMJI3HMMn9HuCyJJckmQKuAfYN0oEjd0kaj66XQu4B7gYuTzKTZHtVPQXcANwBHAb2VtV9g+zckbskjUenE6pVtW2B9gPAgeXuvKr2A/unp6evW24fkqRncz53SWqQ87lLUoMcuUtSgxy5S1KDnPJXkhpkWUaSGmRZRpIaZFlGkhpkcpekBllzl6QGWXOXpAZZlpGkBpncJalBJndJapAnVCWpQZ5QlaQGWZaRpAaZ3CWpQSZ3SWqQyV2SGjSW5J7knCSHkvzoOPqXJC2uU3JPckuS40nundO+Kcn9SY4k2dm36ueAvaMMVJLUXdeR+25gU39DkjOAm4DNwEZgW5KNSV4HfAU4PsI4JUkDOLPLRlV1MMmGOc1XAkeq6gGAJLcCW4FzgXOYTfhPJDlQVU/P7TPJDmAHwMUXX7zc+CVJ8+iU3BewDjjatzwDXFVVNwAkuRZ4eL7EDlBVu5I8CGyZmpq6Yog4JElzjO1qmaraXVX/ZYlt/ISqJI3BMMn9GHBR3/L6Xltnzi0jSeMxTHK/B7gsySVJpoBrgH2DdODIXZLGo+ulkHuAu4HLk8wk2V5VTwE3AHcAh4G9VXXfIDt35C5J49H1apltC7QfAA4sd+dVtR/YPz09fd1y+5AkPZvzuUtSg5zPXZIa5MhdkhrkyF2SGuSUv5LUIMsyktQgyzKS1CDLMpLUIJO7JDXImrskNciauyQ1yLKMJDXI5C5JDTK5S1KDPKEqSQ3yhKokNciyjCQ1yOQuSQ0yuUtSg0zuktSgkSf3JK9IcnOS25K8a9T9S5KW1im5J7klyfEk985p35Tk/iRHkuwEqKrDVXU98BPAq0cfsiRpKV1H7ruBTf0NSc4AbgI2AxuBbUk29ta9CbgdODCySCVJnXVK7lV1EHh0TvOVwJGqeqCqngRuBbb2tt9XVZuBty/UZ5IdSQ4lOXTixInlRS9JmteZQ7x2HXC0b3kGuCrJ1cCbgbNYZOReVbuSPAhsmZqaumKIOCRJcwyT3OdVVXcBd3Xcdj+wf3p6+rpRxyFJp7NhrpY5BlzUt7y+19aZc8tI0ngMk9zvAS5LckmSKeAaYN8gHTi3jCSNR9dLIfcAdwOXJ5lJsr2qngJuAO4ADgN7q+q+QXbuyF2SxqNTzb2qti3QfoAhLne05i5J4+F87pLUIOdzl6QGOXKXpAY5cpekBjnlryQ1yLKMJDXIsowkNciyjCQ1yOQuSQ2y5i5JDbLmLkkNsiwjSQ0yuUtSg0zuktQgT6hKUoM8oSpJDbIsI0kNMrlLUoNM7pLUIJO7JDWo0w2yB5Xkx4A3As8DPlZVd45jP5Kk+XUeuSe5JcnxJPfOad+U5P4kR5LsBKiq36uq64DrgbeNNmRJ0lIGKcvsBjb1NyQ5A7gJ2AxsBLYl2di3yXt76yVJK6hzcq+qg8Cjc5qvBI5U1QNV9SRwK7A1sz4I/Neq+uJ8/SXZkeRQkkMnTpxYbvySpHkMW3NfBxztW54BrgJ+BngtcF6SS6vq5rkvrKpdSR4EtkxNTV0xZBySpD5juVqmqj5UVVdU1fXzJfa+7fyEqiSNwbDJ/RhwUd/y+l5bJ84tI0njMWxyvwe4LMklSaaAa4B9XV/syF2SxmOQSyH3AHcDlyeZSbK9qp4CbgDuAA4De6vqvgH6dOQuSWPQ+YRqVW1boP0AcGA5O6+q/cD+6enp65bzeknS/JzPXZIa5HzuktQgR+6S1CBH7pLUIKf8laQGWZaRpAZZlpGkBlmW0dht2Hn7pEOQTjsmd6lR/lE9vVlzl6QGWXOXpAZZlpGkBpncJalBJndpCRt23u7JSa05w95DdShJtgBbLr300kmGIT3LqJL5qX6+/oE3jqQ/qStPqKpZg464u2zvKF5rhWUZSWqQyV0T1dpIeCWPpbXvnUbL5K7TislQpwuTu6Rn+N9AO0ae3JO8LMnHktw26r41WcP84ps0pJXVKbknuSXJ8ST3zmnflOT+JEeS7ASoqgeqavs4gtVgTKjjM4nva9d9rqb3fbXEcTrqep37buDDwG+eakhyBnAT8DpgBrgnyb6q+sqog1Rblrr2e8PO2ztdFz7INeSjSDKDJNeV5vX0mqvTyL2qDgKPzmm+EjjSG6k/CdwKbO264yQ7khxKcujEiROdA9bKmvQosH/fw5SE5i7P1zbsfpZjsX1N+nuvtW2Ymvs64Gjf8gywLsmLktwMvDLJzy/04qraBdwIfHFqamqIMEarxV+mcSTIuesWS5aT0GX/p+JeTecRhu1zodd2+QM3zPZdt1nL1trxjfyEalU9UlXXV9XLq+r9S2zrJ1QlaQyGmVvmGHBR3/L6Xltnzi2z+nStdw/T/yi3Ww3GHeukvhfz1fH7Y1mofb5lrbxhRu73AJcluSTJFHANsG+QDhy5S9J4dBq5J9kDXA2cn2QG+IWq+liSG4A7gDOAW6rqvkF27sh95cxXd19o5DXoZFv9zydxtcak9jsJo7gqZtD3bNBRuKP21aFTcq+qbQu0HwAOLHfnVbUf2D89PX3dcvuQJD2b87mvcsu9lnulrv0epVFcJbJaR/D9I+SuV/KsBYP+l7da358WOZ+7JDXIkfsasZpGPYvFMora/SDrhtl2LVvOVUddr4EfdQzzvaZLnX+1/LyvVY7cJalBTvkrSQ2aaHJPsiXJrpMnT04yjFVhkI/wr4bSw6Q+ir3WPgLeb9CP/J9uuk55sNyfgdX8+zQOlmUkqUGWZSSpQSZ3SWqQl0IOYe7lWgvV7rpMvLRQn4vte5j1q8FaiHEtWO3fx4Xq5jD7OzCKS10HndDsdLjM0pq7JDXIsowkNcjkLkkNMrlLUoNO6xOqyz2ps1Afy31t13apNcPMkDnJO1T1z/C5Wk/OekJVkhpkWUaSGmRyl6QGmdwlqUEmd0lq0MivlklyDvAR4Engrqr67VHvQ5K0uE4j9yS3JDme5N457ZuS3J/kSJKdveY3A7dV1XXAm0YcrySpg65lmd3Apv6GJGcANwGbgY3AtiQbgfXA0d5m3x1NmJKkQXQqy1TVwSQb5jRfCRypqgcAktwKbAVmmE3wf8wifzyS7AB2AFx88cWDxv2MhT5ENF/7UjMyzrfd3O2X84GK02kmupXS0ge9WjqW1WLQWVPnyw2Dvi9LzUzZJUeN0jAnVNfx/0boMJvU1wGfAn48ya8D+xd6cVXtAm4Evjg1NTVEGJKkuUZ+QrWq/gr4qY7b7gf2T09PXzfqOCTpdDbMyP0YcFHf8vpeW2feIFuSxmOY5H4PcFmSS5JMAdcA+wbpwLllJGk8ul4KuQe4G7g8yUyS7VX1FHADcAdwGNhbVfcNsnNH7pI0Hl2vltm2QPsB4MByd27NXZLGY6LTDzhyl6TxcD53SWqQI3dJalCqatIxkOQE8I1JxzFi5wMPTzqIMfMY23E6HGeLx/jSqrpgvhWrIrm3KMmhqpqedBzj5DG243Q4ztPhGPs5n7skNcjkLkkNMrmPz65JB7ACPMZ2nA7HeToc4zOsuUtSgxy5S1KDTO6S1CCT+4gkeWuS+5I8nWTBy60WuO/smpDkhUk+neSrva8vWGC77yb5495joJlCJ2Wp9yXJWUk+2Vv/R/PcmWzV63CM1yY50ffevXMScQ5jofs9961Pkg/1vgdfTvKDKx3jSjG5j869zN4c/OBCGyxy39m1Yifw+1V1GfD7veX5PFFVP9B7rPqbpHd8X7YDj1XVpcCvAR9c2SiHM8DP3if73ruPrmiQo7GbOfd7nmMzcFnvsQP49RWIaSJM7iNSVYer6v4lNnvmvrNV9SRw6r6za8VW4OO95x8HfmyCsYxSl/el/9hvA16TJCsY47DW+s9eJ1V1EHh0kU22Ar9Zsz4PPD/JhSsT3coyua+she47u1a8uKoe7D3/S+DFC2x3dpJDST6fZC38AejyvjyzTe9eBieBF61IdKPR9Wfvx3vlituSXDTP+rVurf8Odjbye6i2LMlngL89z6r3VNV/Xul4xmGxY+xfqKpKstB1tC+tqmNJXgZ8NsmfVNXXRh2rRm4/sKeqvp3kp5n9T+VHJhyTlsnkPoCqeu2QXQx939lxW+wYkzyU5MKqerD3r+zxBfo41vv6QJK7gFcCqzm5d3lfTm0zk+RM4DzgkZUJbySWPMaq6j+ejwL/dgXiWmmr/ndwVCzLrKyh7zs7YfuAd/SevwN41n8rSV6Q5Kze8/OBVwNfWbEIl6fL+9J/7G8BPltr6xOASx7jnNrzm5i9fWZr9gH/tHfVzKuAk32lxrZUlY8RPIB/zGz97tvAQ8AdvfaXAAf6tnsD8L+ZHcm+Z9JxD3iML2L2KpmvAp8BXthrnwY+2nv+D4E/Ab7U+7p90nF3PLZnvS/A+4A39Z6fDfwOcAT4n8DLJh3zGI7x/cB9vffuD4Dvm3TMyzjGPcCDwHd6v4/bgeuB63vrw+xVQ1/r/XxOTzrmcT2cfkCSGmRZRpIaZHKXpAaZ3CWpQSZ3SWqQyV2SVthSE5zN2fbiJH+Q5H/1Pj38hi77MLlL0srbzeITnPV7L7C3ql7J7OcTPtLlRSZ3SVphNc8EZ0lenuS/JflCks8l+b5TmwPP6z0/D/iLLvtw+gFJWh12Mfthq68muYrZEfqPAL8I3JnkZ4BzgE7ToJjcJWnCkpzL7Ke7f6dvJumzel+3Abur6t8l+QfAJ5L83ap6erE+Te6SNHnPAb5ZVT8wz7rt9OrzVXV3krOB81lg4r7+DiVJE1RV3wL+LMlb4ZnbAX5/b/WfA6/ptb+C2XmOTizVp3PLSNIKS7IHuJrZEfhDwC8An2X2tn8XAs8Fbq2q9/Vuh/gbwLnMnlx9d1XdueQ+TO6S1B7LMpLUIJO7JDXI5C5JDTK5S1KDTO6S1CCTuyQ1yOQuSQ36v6urGFwkBK5yAAAAAElFTkSuQmCC\n",
            "text/plain": [
              "<Figure size 432x288 with 1 Axes>"
            ]
          },
          "metadata": {
            "tags": [],
            "needs_background": "light"
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "rNrBHFbSRcig"
      },
      "source": [
        "### 3. Izvršiti obradu dobijenog BAM fajla prema GATK protokolu (Markiranje Duplikata, rekalibracija kvaliteta baza) \n",
        "3a. Koliki su procenati PCR i optičkih duplikata?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rwXTShWQcsQg"
      },
      "source": [
        "# pe1 = '/content/drive/MyDrive/sample_25.chrom11.exome.pe1.fq'\n",
        "# pe2 = '/content/drive/MyDrive/sample_25.chrom11.exome.pe2.fq'\n",
        "# ref = 'Homo_sapiens_assembly38.fasta'\n",
        " \n",
        "# !./bwa/bwa mem -M -R '@RG\\tID:1\\tPL:Illumina\\tSM:HG00096' -o HG00096.chr11.exome.sam {ref} {pe1} {pe2} \n",
        "# !samtools view -b HG00096.chr11.exome.sam > HG00096.chr11.exome.bam\n",
        "# !samtools sort HG00096.chr11.exome.bam > HG00096.chr11.exome.sorted.bam"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "6_Z6WaWORBk5",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "d3644de4-999e-441d-e594-57e42258e6d3"
      },
      "source": [
        "ibam = 'HG00096.chr11.exome.sorted.bam'\n",
        "obam = 'HG00096.chr11.exome.deduped.bam'\n",
        "\n",
        "!java -jar picard.jar MarkDuplicates I={ibam} O={obam} M=dup_metrics.txt\n",
        "\n",
        "!grep -A2 '## METRICS CLASS' dup_metrics.txt | grep -v '## METRICS CLASS' > report.tsv"
      ],
      "execution_count": 36,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO\t2021-06-03 14:23:32\tMarkDuplicates\t\n",
            "\n",
            "********** NOTE: Picard's command line syntax is changing.\n",
            "**********\n",
            "********** For more information, please see:\n",
            "********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)\n",
            "**********\n",
            "********** The command line looks like this in the new syntax:\n",
            "**********\n",
            "**********    MarkDuplicates -I HG00096.chr11.exome.sorted.bam -O HG00096.chr11.exome.deduped.bam -M dup_metrics.txt\n",
            "**********\n",
            "\n",
            "\n",
            "14:23:33.268 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/picard.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "[Thu Jun 03 14:23:33 UTC 2021] MarkDuplicates INPUT=[HG00096.chr11.exome.sorted.bam] OUTPUT=HG00096.chr11.exome.deduped.bam METRICS_FILE=dup_metrics.txt    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false\n",
            "[Thu Jun 03 14:23:33 UTC 2021] Executing as root@43bfdb95a763 on Linux 5.4.109+ amd64; OpenJDK 64-Bit Server VM 11.0.11+9-Ubuntu-0ubuntu2.18.04; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.21.6-SNAPSHOT\n",
            "INFO\t2021-06-03 14:23:33\tMarkDuplicates\tStart of doWork freeMemory: 22263552; totalMemory: 28311552; maxMemory: 3407872000\n",
            "INFO\t2021-06-03 14:23:33\tMarkDuplicates\tReading input file and constructing read end information.\n",
            "INFO\t2021-06-03 14:23:33\tMarkDuplicates\tWill retain up to 12347362 data points before spilling to disk.\n",
            "WARNING\t2021-06-03 14:23:33\tAbstractOpticalDuplicateFinderCommandLineProgram\tA field field parsed out of a read name was expected to contain an integer and did not. Read name: SRR795846.1306863. Cause: String 'SRR795846.1306863' did not start with a parsable number.\n",
            "INFO\t2021-06-03 14:23:42\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    9s.  Last read position: chr11:34,123,755\n",
            "INFO\t2021-06-03 14:23:42\tMarkDuplicates\tTracking 2722 as yet unmatched pairs. 2125 records in RAM.\n",
            "INFO\t2021-06-03 14:23:48\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    5s.  Last read position: chr11:64,750,323\n",
            "INFO\t2021-06-03 14:23:48\tMarkDuplicates\tTracking 2734 as yet unmatched pairs. 1685 records in RAM.\n",
            "INFO\t2021-06-03 14:23:53\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    5s.  Last read position: chr11:104,954,942\n",
            "INFO\t2021-06-03 14:23:53\tMarkDuplicates\tTracking 2228 as yet unmatched pairs. 780 records in RAM.\n",
            "INFO\t2021-06-03 14:23:58\tMarkDuplicates\tRead 3938774 records. 0 pairs never matched.\n",
            "INFO\t2021-06-03 14:23:58\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 895641488; totalMemory: 1425014784; maxMemory: 3407872000\n",
            "INFO\t2021-06-03 14:23:58\tMarkDuplicates\tWill retain up to 106496000 duplicate indices before spilling to disk.\n",
            "INFO\t2021-06-03 14:23:59\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
            "INFO\t2021-06-03 14:24:01\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
            "INFO\t2021-06-03 14:24:02\tMarkDuplicates\tSorting list of duplicate records.\n",
            "INFO\t2021-06-03 14:24:02\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 1417705072; totalMemory: 2277507072; maxMemory: 3407872000\n",
            "INFO\t2021-06-03 14:24:02\tMarkDuplicates\tMarking 542269 records as duplicates.\n",
            "INFO\t2021-06-03 14:24:02\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
            "INFO\t2021-06-03 14:24:02\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
            "INFO\t2021-06-03 14:24:48\tMarkDuplicates\tWriting complete. Closing input iterator.\n",
            "INFO\t2021-06-03 14:24:48\tMarkDuplicates\tDuplicate Index cleanup.\n",
            "INFO\t2021-06-03 14:24:48\tMarkDuplicates\tGetting Memory Stats.\n",
            "INFO\t2021-06-03 14:24:48\tMarkDuplicates\tBefore output close freeMemory: 27108184; totalMemory: 35651584; maxMemory: 3407872000\n",
            "INFO\t2021-06-03 14:24:48\tMarkDuplicates\tClosed outputs. Getting more Memory Stats.\n",
            "INFO\t2021-06-03 14:24:48\tMarkDuplicates\tAfter output close freeMemory: 27108184; totalMemory: 35651584; maxMemory: 3407872000\n",
            "[Thu Jun 03 14:24:48 UTC 2021] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 1.25 minutes.\n",
            "Runtime.totalMemory()=35651584\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 117
        },
        "id": "J4BVlB6vUQQE",
        "outputId": "199b4254-4859-41d3-8c14-3e6082dbcf51"
      },
      "source": [
        "import pandas as pd\n",
        "pd.read_csv('report.tsv', sep='\\t')"
      ],
      "execution_count": 38,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>LIBRARY</th>\n",
              "      <th>UNPAIRED_READS_EXAMINED</th>\n",
              "      <th>READ_PAIRS_EXAMINED</th>\n",
              "      <th>SECONDARY_OR_SUPPLEMENTARY_RDS</th>\n",
              "      <th>UNMAPPED_READS</th>\n",
              "      <th>UNPAIRED_READ_DUPLICATES</th>\n",
              "      <th>READ_PAIR_DUPLICATES</th>\n",
              "      <th>READ_PAIR_OPTICAL_DUPLICATES</th>\n",
              "      <th>PERCENT_DUPLICATION</th>\n",
              "      <th>ESTIMATED_LIBRARY_SIZE</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Unknown Library</td>\n",
              "      <td>25695</td>\n",
              "      <td>1942489</td>\n",
              "      <td>2406</td>\n",
              "      <td>250893</td>\n",
              "      <td>15009</td>\n",
              "      <td>263630</td>\n",
              "      <td>0</td>\n",
              "      <td>0.138664</td>\n",
              "      <td>6493052</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "           LIBRARY  ...  ESTIMATED_LIBRARY_SIZE\n",
              "0  Unknown Library  ...                 6493052\n",
              "\n",
              "[1 rows x 10 columns]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 38
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "BlCo00QYXHay",
        "outputId": "39963b97-a290-4159-cfd6-9f09ce89c0b2"
      },
      "source": [
        "reader = pysam.AlignmentFile(obam)\n",
        "\n",
        "dups = [read for read in reader if read.is_duplicate]\n",
        "print(len(dups))"
      ],
      "execution_count": 39,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "542269\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "atFT600FFyay"
      },
      "source": [
        "ibam = 'HG00096.chr11.exome.deduped.bam'\n",
        "obam = 'HG00096.chr11.exome.recalibrated.bam'\n",
        "recal = 'HG00096.chr11.exome.recal.table'\n",
        "\n",
        "dbsnp = 'Homo_sapiens_assembly38.dbsnp138.vcf'\n",
        "ref = 'Homo_sapiens_assembly38.fasta'\n",
        "\n",
        "gatk = 'java -jar gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar'"
      ],
      "execution_count": 40,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "vM99Z1xdGKze",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4c1eeffb-9c67-4ba2-bacb-622091075f2c"
      },
      "source": [
        "!{gatk} BaseRecalibrator -I {ibam} --known-sites {dbsnp} -O {recal} --reference {ref}"
      ],
      "execution_count": 41,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:26:10.369 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:26:16.107 INFO  BaseRecalibrator - ------------------------------------------------------------\n",
            "14:26:16.107 INFO  BaseRecalibrator - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:26:16.107 INFO  BaseRecalibrator - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:26:16.108 INFO  BaseRecalibrator - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:26:16.108 INFO  BaseRecalibrator - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:26:16.108 INFO  BaseRecalibrator - Start Date/Time: June 3, 2021 at 2:26:10 PM UTC\n",
            "14:26:16.108 INFO  BaseRecalibrator - ------------------------------------------------------------\n",
            "14:26:16.108 INFO  BaseRecalibrator - ------------------------------------------------------------\n",
            "14:26:16.109 INFO  BaseRecalibrator - HTSJDK Version: 2.21.0\n",
            "14:26:16.109 INFO  BaseRecalibrator - Picard Version: 2.21.2\n",
            "14:26:16.110 INFO  BaseRecalibrator - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:26:16.110 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:26:16.110 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:26:16.110 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:26:16.110 INFO  BaseRecalibrator - Deflater: IntelDeflater\n",
            "14:26:16.110 INFO  BaseRecalibrator - Inflater: IntelInflater\n",
            "14:26:16.110 INFO  BaseRecalibrator - GCS max retries/reopens: 20\n",
            "14:26:16.110 INFO  BaseRecalibrator - Requester pays: disabled\n",
            "14:26:16.110 INFO  BaseRecalibrator - Initializing engine\n",
            "14:26:16.937 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/Homo_sapiens_assembly38.dbsnp138.vcf\n",
            "14:26:17.430 INFO  BaseRecalibrator - Done initializing engine\n",
            "14:26:17.450 INFO  BaseRecalibrationEngine - The covariates being used here: \n",
            "14:26:17.454 INFO  BaseRecalibrationEngine - \tReadGroupCovariate\n",
            "14:26:17.454 INFO  BaseRecalibrationEngine - \tQualityScoreCovariate\n",
            "14:26:17.454 INFO  BaseRecalibrationEngine - \tContextCovariate\n",
            "14:26:17.454 INFO  BaseRecalibrationEngine - \tCycleCovariate\n",
            "14:26:17.614 INFO  ProgressMeter - Starting traversal\n",
            "14:26:17.615 INFO  ProgressMeter -        Current Locus  Elapsed Minutes       Reads Processed     Reads/Minute\n",
            "14:26:28.636 INFO  ProgressMeter -          chr11:87490              0.2                  3000          16333.9\n",
            "14:26:38.660 INFO  ProgressMeter -       chr11:11271179              0.4                441000        1257305.8\n",
            "14:26:48.671 INFO  ProgressMeter -       chr11:45945793              0.5                926000        1789083.9\n",
            "14:26:58.678 INFO  ProgressMeter -       chr11:62608526              0.7               1476000        2156686.1\n",
            "14:27:08.680 INFO  ProgressMeter -       chr11:73365729              0.9               2014000        2366442.1\n",
            "14:27:18.684 INFO  ProgressMeter -      chr11:108244819              1.0               2493000        2449400.7\n",
            "14:27:28.700 INFO  ProgressMeter -      chr11:125086155              1.2               3010000        2540656.1\n",
            "14:27:37.679 INFO  BaseRecalibrator - 977106 read(s) filtered by: ((((((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter) AND WellformedReadFilter)\n",
            "  977106 read(s) filtered by: (((((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter)\n",
            "      977106 read(s) filtered by: ((((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter)\n",
            "          450220 read(s) filtered by: (((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter)\n",
            "              448472 read(s) filtered by: ((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter)\n",
            "                  448472 read(s) filtered by: (MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter)\n",
            "                      448472 read(s) filtered by: MappingQualityNotZeroReadFilter \n",
            "              1748 read(s) filtered by: NotSecondaryAlignmentReadFilter \n",
            "          526886 read(s) filtered by: NotDuplicateReadFilter \n",
            "\n",
            "14:27:37.680 INFO  ProgressMeter - chr11_KI270927v1_alt:150017              1.3               3186866        2388209.1\n",
            "14:27:37.680 INFO  ProgressMeter - Traversal complete. Processed 3186866 total reads in 1.3 minutes.\n",
            "14:27:37.864 INFO  BaseRecalibrator - Calculating quantized quality scores...\n",
            "14:27:37.896 INFO  BaseRecalibrator - Writing recalibration report...\n",
            "14:27:40.452 INFO  BaseRecalibrator - ...done!\n",
            "14:27:40.453 INFO  BaseRecalibrator - Shutting down engine\n",
            "[June 3, 2021 at 2:27:40 PM UTC] org.broadinstitute.hellbender.tools.walkers.bqsr.BaseRecalibrator done. Elapsed time: 1.50 minutes.\n",
            "Runtime.totalMemory()=377487360\n",
            "Tool returned:\n",
            "3186866\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "isDLDA5cGLXH",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "53844b13-8237-4e44-8b5b-799fd0bbfed9"
      },
      "source": [
        "!{gatk} ApplyBQSR -I {ibam} -bqsr {recal} -O {obam}"
      ],
      "execution_count": 42,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:27:43.744 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:27:49.483 INFO  ApplyBQSR - ------------------------------------------------------------\n",
            "14:27:49.483 INFO  ApplyBQSR - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:27:49.483 INFO  ApplyBQSR - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:27:49.484 INFO  ApplyBQSR - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:27:49.484 INFO  ApplyBQSR - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:27:49.484 INFO  ApplyBQSR - Start Date/Time: June 3, 2021 at 2:27:43 PM UTC\n",
            "14:27:49.484 INFO  ApplyBQSR - ------------------------------------------------------------\n",
            "14:27:49.484 INFO  ApplyBQSR - ------------------------------------------------------------\n",
            "14:27:49.485 INFO  ApplyBQSR - HTSJDK Version: 2.21.0\n",
            "14:27:49.486 INFO  ApplyBQSR - Picard Version: 2.21.2\n",
            "14:27:49.486 INFO  ApplyBQSR - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:27:49.486 INFO  ApplyBQSR - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:27:49.486 INFO  ApplyBQSR - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:27:49.486 INFO  ApplyBQSR - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:27:49.486 INFO  ApplyBQSR - Deflater: IntelDeflater\n",
            "14:27:49.486 INFO  ApplyBQSR - Inflater: IntelInflater\n",
            "14:27:49.486 INFO  ApplyBQSR - GCS max retries/reopens: 20\n",
            "14:27:49.486 INFO  ApplyBQSR - Requester pays: disabled\n",
            "14:27:49.486 INFO  ApplyBQSR - Initializing engine\n",
            "14:27:49.835 INFO  ApplyBQSR - Done initializing engine\n",
            "14:27:49.928 INFO  ProgressMeter - Starting traversal\n",
            "14:27:49.933 INFO  ProgressMeter -        Current Locus  Elapsed Minutes       Reads Processed     Reads/Minute\n",
            "14:27:59.938 INFO  ProgressMeter -        chr11:5301363              0.2                272000        1631347.5\n",
            "14:28:09.961 INFO  ProgressMeter -       chr11:17135147              0.3                682000        2043139.6\n",
            "14:28:19.983 INFO  ProgressMeter -       chr11:44051698              0.5               1095000        2186356.1\n",
            "14:28:29.983 INFO  ProgressMeter -       chr11:57646322              0.7               1503000        2251685.4\n",
            "14:28:39.986 INFO  ProgressMeter -       chr11:63463446              0.8               1910000        2289573.1\n",
            "14:28:50.006 INFO  ProgressMeter -       chr11:68900526              1.0               2321000        2318179.5\n",
            "14:29:00.026 INFO  ProgressMeter -       chr11:86671717              1.2               2732000        2338607.3\n",
            "14:29:10.032 INFO  ProgressMeter -      chr11:111870098              1.3               3143000        2354336.5\n",
            "14:29:20.039 INFO  ProgressMeter -      chr11:122782992              1.5               3552000        2365240.6\n",
            "14:29:30.045 INFO  ProgressMeter -             unmapped              1.7               3947000        2365574.2\n",
            "14:29:37.631 INFO  ApplyBQSR - No reads filtered by: WellformedReadFilter\n",
            "14:29:37.631 INFO  ProgressMeter -             unmapped              1.8               4163972        2319804.6\n",
            "14:29:37.631 INFO  ProgressMeter - Traversal complete. Processed 4163972 total reads in 1.8 minutes.\n",
            "14:29:37.716 INFO  ApplyBQSR - Shutting down engine\n",
            "[June 3, 2021 at 2:29:37 PM UTC] org.broadinstitute.hellbender.tools.walkers.bqsr.ApplyBQSR done. Elapsed time: 1.90 minutes.\n",
            "Runtime.totalMemory()=319815680\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "KDQcfFT2GLjO",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "c15dc7b8-9d09-4027-fff0-ea15b3db298f"
      },
      "source": [
        "post_recal = 'HG00096.chr11.exome.post_recal.table'\n",
        "!{gatk} BaseRecalibrator -I {obam} --known-sites {dbsnp} -O {post_recal} --reference {ref}"
      ],
      "execution_count": 43,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:29:41.154 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:29:46.837 INFO  BaseRecalibrator - ------------------------------------------------------------\n",
            "14:29:46.837 INFO  BaseRecalibrator - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:29:46.838 INFO  BaseRecalibrator - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:29:46.838 INFO  BaseRecalibrator - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:29:46.838 INFO  BaseRecalibrator - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:29:46.838 INFO  BaseRecalibrator - Start Date/Time: June 3, 2021 at 2:29:41 PM UTC\n",
            "14:29:46.839 INFO  BaseRecalibrator - ------------------------------------------------------------\n",
            "14:29:46.839 INFO  BaseRecalibrator - ------------------------------------------------------------\n",
            "14:29:46.839 INFO  BaseRecalibrator - HTSJDK Version: 2.21.0\n",
            "14:29:46.839 INFO  BaseRecalibrator - Picard Version: 2.21.2\n",
            "14:29:46.839 INFO  BaseRecalibrator - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:29:46.839 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:29:46.840 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:29:46.840 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:29:46.840 INFO  BaseRecalibrator - Deflater: IntelDeflater\n",
            "14:29:46.840 INFO  BaseRecalibrator - Inflater: IntelInflater\n",
            "14:29:46.840 INFO  BaseRecalibrator - GCS max retries/reopens: 20\n",
            "14:29:46.840 INFO  BaseRecalibrator - Requester pays: disabled\n",
            "14:29:46.840 INFO  BaseRecalibrator - Initializing engine\n",
            "14:29:47.540 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/Homo_sapiens_assembly38.dbsnp138.vcf\n",
            "14:29:48.111 INFO  BaseRecalibrator - Done initializing engine\n",
            "14:29:48.118 INFO  BaseRecalibrationEngine - The covariates being used here: \n",
            "14:29:48.119 INFO  BaseRecalibrationEngine - \tReadGroupCovariate\n",
            "14:29:48.119 INFO  BaseRecalibrationEngine - \tQualityScoreCovariate\n",
            "14:29:48.119 INFO  BaseRecalibrationEngine - \tContextCovariate\n",
            "14:29:48.119 INFO  BaseRecalibrationEngine - \tCycleCovariate\n",
            "14:29:48.347 INFO  ProgressMeter - Starting traversal\n",
            "14:29:48.348 INFO  ProgressMeter -        Current Locus  Elapsed Minutes       Reads Processed     Reads/Minute\n",
            "14:29:58.353 INFO  ProgressMeter -        chr11:1242728              0.2                 56000         335865.7\n",
            "14:30:08.355 INFO  ProgressMeter -       chr11:18235807              0.3                575000        1724396.5\n",
            "14:30:18.364 INFO  ProgressMeter -       chr11:55491779              0.5               1117000        2232809.2\n",
            "14:30:28.372 INFO  ProgressMeter -       chr11:65864597              0.7               1712000        2566524.2\n",
            "14:30:38.381 INFO  ProgressMeter -       chr11:89768257              0.8               2266000        2717406.5\n",
            "14:30:48.382 INFO  ProgressMeter -      chr11:119174536              1.0               2818000        2816404.0\n",
            "14:30:59.087 INFO  ProgressMeter - chr11_KI270832v1_alt:79533              1.2               3181000        2698087.3\n",
            "14:30:59.779 INFO  BaseRecalibrator - 977106 read(s) filtered by: ((((((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter) AND WellformedReadFilter)\n",
            "  977106 read(s) filtered by: (((((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter)\n",
            "      977106 read(s) filtered by: ((((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter)\n",
            "          450220 read(s) filtered by: (((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter)\n",
            "              448472 read(s) filtered by: ((MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter)\n",
            "                  448472 read(s) filtered by: (MappingQualityNotZeroReadFilter AND MappingQualityAvailableReadFilter)\n",
            "                      448472 read(s) filtered by: MappingQualityNotZeroReadFilter \n",
            "              1748 read(s) filtered by: NotSecondaryAlignmentReadFilter \n",
            "          526886 read(s) filtered by: NotDuplicateReadFilter \n",
            "\n",
            "14:30:59.779 INFO  ProgressMeter - chr11_KI270927v1_alt:150017              1.2               3186866        2676876.4\n",
            "14:30:59.780 INFO  ProgressMeter - Traversal complete. Processed 3186866 total reads in 1.2 minutes.\n",
            "14:30:59.996 INFO  BaseRecalibrator - Calculating quantized quality scores...\n",
            "14:31:00.024 INFO  BaseRecalibrator - Writing recalibration report...\n",
            "14:31:02.514 INFO  BaseRecalibrator - ...done!\n",
            "14:31:02.514 INFO  BaseRecalibrator - Shutting down engine\n",
            "[June 3, 2021 at 2:31:02 PM UTC] org.broadinstitute.hellbender.tools.walkers.bqsr.BaseRecalibrator done. Elapsed time: 1.36 minutes.\n",
            "Runtime.totalMemory()=488636416\n",
            "Tool returned:\n",
            "3186866\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "YKhp-gmHGLvG",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "1bdf3bfb-451d-4f19-8dea-a35c28602409"
      },
      "source": [
        "!{gatk} AnalyzeCovariates -before {recal} -after {post_recal} -plots 'compare.pdf'"
      ],
      "execution_count": 44,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:31:05.747 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:31:11.476 INFO  AnalyzeCovariates - ------------------------------------------------------------\n",
            "14:31:11.477 INFO  AnalyzeCovariates - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:31:11.477 INFO  AnalyzeCovariates - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:31:11.477 INFO  AnalyzeCovariates - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:31:11.477 INFO  AnalyzeCovariates - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:31:11.478 INFO  AnalyzeCovariates - Start Date/Time: June 3, 2021 at 2:31:05 PM UTC\n",
            "14:31:11.478 INFO  AnalyzeCovariates - ------------------------------------------------------------\n",
            "14:31:11.478 INFO  AnalyzeCovariates - ------------------------------------------------------------\n",
            "14:31:11.479 INFO  AnalyzeCovariates - HTSJDK Version: 2.21.0\n",
            "14:31:11.479 INFO  AnalyzeCovariates - Picard Version: 2.21.2\n",
            "14:31:11.479 INFO  AnalyzeCovariates - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:31:11.479 INFO  AnalyzeCovariates - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:31:11.479 INFO  AnalyzeCovariates - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:31:11.479 INFO  AnalyzeCovariates - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:31:11.479 INFO  AnalyzeCovariates - Deflater: IntelDeflater\n",
            "14:31:11.479 INFO  AnalyzeCovariates - Inflater: IntelInflater\n",
            "14:31:11.479 INFO  AnalyzeCovariates - GCS max retries/reopens: 20\n",
            "14:31:11.479 INFO  AnalyzeCovariates - Requester pays: disabled\n",
            "14:31:11.480 INFO  AnalyzeCovariates - Initializing engine\n",
            "14:31:11.480 INFO  AnalyzeCovariates - Done initializing engine\n",
            "14:31:12.598 INFO  AnalyzeCovariates - Generating csv file '/tmp/AnalyzeCovariates16294854876934872214.csv'\n",
            "14:31:12.778 INFO  AnalyzeCovariates - Generating plots file 'compare.pdf'\n",
            "14:31:14.836 INFO  AnalyzeCovariates - Shutting down engine\n",
            "[June 3, 2021 at 2:31:14 PM UTC] org.broadinstitute.hellbender.tools.walkers.bqsr.AnalyzeCovariates done. Elapsed time: 0.15 minutes.\n",
            "Runtime.totalMemory()=333447168\n",
            "org.broadinstitute.hellbender.utils.R.RScriptExecutorException: \n",
            "Rscript exited with 1\n",
            "Command Line: Rscript -e tempLibDir = '/tmp/Rlib.4819296818445452974';source('/tmp/BQSR.17682447959655344312.R'); /tmp/AnalyzeCovariates16294854876934872214.csv /content/HG00096.chr11.exome.recal.table /content/compare.pdf\n",
            "Stdout: \n",
            "Stderr: \n",
            "Attaching package: ‘gplots’\n",
            "\n",
            "The following object is masked from ‘package:stats’:\n",
            "\n",
            "    lowess\n",
            "\n",
            "Error in distributeGraphRows(list(a, b, c), c(1, 1, 1)) : \n",
            "  object 'a' not found\n",
            "Calls: source -> withVisible -> eval -> eval -> distributeGraphRows\n",
            "In addition: There were 25 warnings (use warnings() to see them)\n",
            "Execution halted\n",
            "\n",
            "\tat org.broadinstitute.hellbender.utils.R.RScriptExecutor.getScriptException(RScriptExecutor.java:80)\n",
            "\tat org.broadinstitute.hellbender.utils.R.RScriptExecutor.getScriptException(RScriptExecutor.java:19)\n",
            "\tat org.broadinstitute.hellbender.utils.runtime.ScriptExecutor.executeCuratedArgs(ScriptExecutor.java:126)\n",
            "\tat org.broadinstitute.hellbender.utils.R.RScriptExecutor.exec(RScriptExecutor.java:126)\n",
            "\tat org.broadinstitute.hellbender.utils.recalibration.RecalUtils.generatePlots(RecalUtils.java:360)\n",
            "\tat org.broadinstitute.hellbender.tools.walkers.bqsr.AnalyzeCovariates.generatePlots(AnalyzeCovariates.java:329)\n",
            "\tat org.broadinstitute.hellbender.tools.walkers.bqsr.AnalyzeCovariates.doWork(AnalyzeCovariates.java:341)\n",
            "\tat org.broadinstitute.hellbender.cmdline.CommandLineProgram.runTool(CommandLineProgram.java:139)\n",
            "\tat org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMainPostParseArgs(CommandLineProgram.java:191)\n",
            "\tat org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:210)\n",
            "\tat org.broadinstitute.hellbender.Main.runCommandLineProgram(Main.java:163)\n",
            "\tat org.broadinstitute.hellbender.Main.mainEntry(Main.java:206)\n",
            "\tat org.broadinstitute.hellbender.Main.main(Main.java:292)\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "oU_48vLVFRox"
      },
      "source": [
        "### 4. Identifikovati mutacije upotrebom alata Haplotype Caller i filtirtati mutacije predefinisanim filterima (hard filtering) prema Broad preporukama (kao što je rađeno na vežbama). 4a. Koliko je ukupno mutacija identifikovano, koliko od njih su SNP-ovi, a koliko INDEL-i? 4b. Koliko mutacija prolazi, a koliko ne prolazi kriterijume filtriranja. 4c. Izračunati Ti/Tv odnos pre i posle filtriranja.\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "lw4kJmDZFRKF",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f1699439-711d-4831-b04c-fd652bfc8166"
      },
      "source": [
        "gatk = 'java -jar gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar'\n",
        "dbsnp = 'Homo_sapiens_assembly38.dbsnp138.vcf'\n",
        "ref = 'Homo_sapiens_assembly38.fasta'\n",
        "\n",
        "ibam = 'HG00096.chr11.exome.recalibrated.bam'\n",
        "ovcf = 'HG00096.chr11.exome.vcf'\n",
        "\n",
        "!{gatk} HaplotypeCaller --input {ibam} --output {ovcf} --reference {ref} --dbsnp {dbsnp} -L chr11"
      ],
      "execution_count": 45,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:31:30.402 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:31:36.073 INFO  HaplotypeCaller - ------------------------------------------------------------\n",
            "14:31:36.073 INFO  HaplotypeCaller - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:31:36.073 INFO  HaplotypeCaller - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:31:36.073 INFO  HaplotypeCaller - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:31:36.074 INFO  HaplotypeCaller - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:31:36.074 INFO  HaplotypeCaller - Start Date/Time: June 3, 2021 at 2:31:30 PM UTC\n",
            "14:31:36.074 INFO  HaplotypeCaller - ------------------------------------------------------------\n",
            "14:31:36.074 INFO  HaplotypeCaller - ------------------------------------------------------------\n",
            "14:31:36.075 INFO  HaplotypeCaller - HTSJDK Version: 2.21.0\n",
            "14:31:36.075 INFO  HaplotypeCaller - Picard Version: 2.21.2\n",
            "14:31:36.075 INFO  HaplotypeCaller - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:31:36.075 INFO  HaplotypeCaller - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:31:36.075 INFO  HaplotypeCaller - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:31:36.075 INFO  HaplotypeCaller - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:31:36.075 INFO  HaplotypeCaller - Deflater: IntelDeflater\n",
            "14:31:36.075 INFO  HaplotypeCaller - Inflater: IntelInflater\n",
            "14:31:36.075 INFO  HaplotypeCaller - GCS max retries/reopens: 20\n",
            "14:31:36.075 INFO  HaplotypeCaller - Requester pays: disabled\n",
            "14:31:36.075 INFO  HaplotypeCaller - Initializing engine\n",
            "14:31:36.771 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/Homo_sapiens_assembly38.dbsnp138.vcf\n",
            "14:31:37.224 INFO  IntervalArgumentCollection - Processing 135086622 bp from intervals\n",
            "14:31:37.275 INFO  HaplotypeCaller - Done initializing engine\n",
            "14:31:37.401 INFO  HaplotypeCallerEngine - Disabling physical phasing, which is supported only for reference-model confidence output\n",
            "14:31:37.417 INFO  NativeLibraryLoader - Loading libgkl_utils.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_utils.so\n",
            "14:31:37.420 INFO  NativeLibraryLoader - Loading libgkl_pairhmm_omp.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_pairhmm_omp.so\n",
            "14:31:37.486 INFO  IntelPairHmm - Flush-to-zero (FTZ) is enabled when running PairHMM\n",
            "14:31:37.487 INFO  IntelPairHmm - Available threads: 2\n",
            "14:31:37.487 INFO  IntelPairHmm - Requested threads: 4\n",
            "14:31:37.487 WARN  IntelPairHmm - Using 2 available threads, but 4 were requested\n",
            "14:31:37.487 INFO  PairHMM - Using the OpenMP multi-threaded AVX-accelerated native PairHMM implementation\n",
            "14:31:37.570 INFO  ProgressMeter - Starting traversal\n",
            "14:31:37.572 INFO  ProgressMeter -        Current Locus  Elapsed Minutes     Regions Processed   Regions/Minute\n",
            "14:31:38.911 WARN  InbreedingCoeff - InbreedingCoeff will not be calculated; at least 10 samples must have called genotypes\n",
            "14:31:47.573 INFO  ProgressMeter -         chr11:715396              0.2                  2550          15298.5\n",
            "14:31:57.605 INFO  ProgressMeter -        chr11:1923010              0.3                  6920          20725.8\n",
            "14:32:07.606 INFO  ProgressMeter -        chr11:4744160              0.5                 16740          33443.2\n",
            "14:32:18.204 INFO  ProgressMeter -        chr11:5841846              0.7                 20620          30449.7\n",
            "14:32:28.203 INFO  ProgressMeter -        chr11:9629607              0.8                 33690          39924.2\n",
            "14:32:38.220 INFO  ProgressMeter -       chr11:17415352              1.0                 60280          59635.9\n",
            "14:32:48.256 INFO  ProgressMeter -       chr11:22275658              1.2                 77000          65361.3\n",
            "14:32:58.262 INFO  ProgressMeter -       chr11:33870112              1.3                116200          86404.8\n",
            "14:33:08.263 INFO  ProgressMeter -       chr11:46409062              1.5                158670         104974.0\n",
            "14:33:18.263 INFO  ProgressMeter -       chr11:53705232              1.7                183470         109326.6\n",
            "14:33:28.263 INFO  ProgressMeter -       chr11:58714549              1.8                200530         108697.2\n",
            "14:33:38.294 INFO  ProgressMeter -       chr11:62372935              2.0                213240         105983.2\n",
            "14:33:48.307 INFO  ProgressMeter -       chr11:65122822              2.2                222920         102307.7\n",
            "14:33:58.392 INFO  ProgressMeter -       chr11:67802977              2.3                232390          99015.8\n",
            "14:34:08.416 INFO  ProgressMeter -       chr11:72597126              2.5                249000          99042.7\n",
            "14:34:18.418 INFO  ProgressMeter -       chr11:78206370              2.7                268340         100098.2\n",
            "14:34:28.422 INFO  ProgressMeter -       chr11:90282183              2.8                309140         108565.4\n",
            "14:34:38.425 INFO  ProgressMeter -      chr11:101958155              3.0                348560         115638.7\n",
            "14:34:48.425 INFO  ProgressMeter -      chr11:110865459              3.2                378830         119095.8\n",
            "14:34:58.429 INFO  ProgressMeter -      chr11:117439546              3.3                401560         119954.0\n",
            "14:35:08.439 INFO  ProgressMeter -      chr11:120961138              3.5                413900         117770.9\n",
            "14:35:19.173 INFO  ProgressMeter -      chr11:124383515              3.7                425660         115250.4\n",
            "14:35:29.175 INFO  ProgressMeter -      chr11:131913886              3.9                451320         116920.8\n",
            "14:35:32.301 INFO  HaplotypeCaller - 693937 read(s) filtered by: ((((((((MappingQualityReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter) AND NonZeroReferenceLengthAlignmentReadFilter) AND GoodCigarReadFilter) AND WellformedReadFilter)\n",
            "  693937 read(s) filtered by: (((((((MappingQualityReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter) AND NonZeroReferenceLengthAlignmentReadFilter) AND GoodCigarReadFilter)\n",
            "      693937 read(s) filtered by: ((((((MappingQualityReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter) AND NonZeroReferenceLengthAlignmentReadFilter)\n",
            "          693937 read(s) filtered by: (((((MappingQualityReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter) AND PassesVendorQualityCheckReadFilter)\n",
            "              693937 read(s) filtered by: ((((MappingQualityReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter) AND NotDuplicateReadFilter)\n",
            "                  170091 read(s) filtered by: (((MappingQualityReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter) AND NotSecondaryAlignmentReadFilter)\n",
            "                      169107 read(s) filtered by: ((MappingQualityReadFilter AND MappingQualityAvailableReadFilter) AND MappedReadFilter)\n",
            "                          169107 read(s) filtered by: (MappingQualityReadFilter AND MappingQualityAvailableReadFilter)\n",
            "                              169107 read(s) filtered by: MappingQualityReadFilter \n",
            "                      984 read(s) filtered by: NotSecondaryAlignmentReadFilter \n",
            "                  523846 read(s) filtered by: NotDuplicateReadFilter \n",
            "\n",
            "14:35:32.301 INFO  ProgressMeter -      chr11:135084255              3.9                462117         118123.5\n",
            "14:35:32.301 INFO  ProgressMeter - Traversal complete. Processed 462117 total regions in 3.9 minutes.\n",
            "14:35:32.379 INFO  VectorLoglessPairHMM - Time spent in setup for JNI call : 0.270714929\n",
            "14:35:32.379 INFO  PairHMM - Total compute time in PairHMM computeLogLikelihoods() : 24.815262952\n",
            "14:35:32.379 INFO  SmithWatermanAligner - Total compute time in java Smith-Waterman : 12.46 sec\n",
            "14:35:32.379 INFO  HaplotypeCaller - Shutting down engine\n",
            "[June 3, 2021 at 2:35:32 PM UTC] org.broadinstitute.hellbender.tools.walkers.haplotypecaller.HaplotypeCaller done. Elapsed time: 4.03 minutes.\n",
            "Runtime.totalMemory()=1033895936\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "7A3iZV7pHj2c"
      },
      "source": [
        "def is_snp(variant):\n",
        "  return len(variant.ref) == 1 and len(variant.alts[0]) == 1\n",
        "\n",
        "def is_transitions(variant):\n",
        "  allels = set((variant.ref, variant.alts[0]))\n",
        "  return allels == {'A', 'G'} or allels == {'C', 'T'}"
      ],
      "execution_count": 46,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "5U80AR66HnVm",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "ddd1abcf-8453-48d2-8378-9fe0a9eb0e79"
      },
      "source": [
        "from collections import Counter\n",
        "\n",
        "Counter(v.id == None for v in pysam.VariantFile(ovcf))"
      ],
      "execution_count": 47,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Counter({False: 5314, True: 456})"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 47
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "v2MW1VmEHr5A",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e6ddf002-a6bf-41a0-d0e3-cc8327ff1ef8"
      },
      "source": [
        "ti = sum(1 for v in pysam.VariantFile(ovcf) \\\n",
        "         if is_snp(v)\\\n",
        "         and is_transitions(v))\n",
        "\n",
        "tv = sum(1 for v in pysam.VariantFile(ovcf) \\\n",
        "          if is_snp(v) \\\n",
        "          and not is_transitions(v))\n",
        "\n",
        "print('Before filter:')\n",
        "print('Ti:', ti)\n",
        "print('Tv:', tv)\n",
        "print('Ti/Tv', ti/tv)"
      ],
      "execution_count": 48,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Before filter:\n",
            "Ti: 3612\n",
            "Tv: 1625\n",
            "Ti/Tv 2.222769230769231\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "j3WwFi_FHzm5"
      },
      "source": [
        "gatk = 'java -jar gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar'\n",
        "\n",
        "snp_f = \"QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0\"\n",
        "indel_f = \"QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0\" \n",
        "\n",
        "ref = 'Homo_sapiens_assembly38.fasta'\n",
        "\n",
        "ivcf = 'HG00096.chr11.exome.vcf'\n",
        "isvcf = 'HG00096.chr11.exome.snp.vcf'\n",
        "iivcf = 'HG00096.chr11.exome.indel.vcf'\n",
        "\n",
        "\n",
        "osvcf = 'HG00096.chr11.exome.snp.filtered.vcf'\n",
        "oivcf = 'HG00096.chr11.exome.indel.filtered.vcf'\n",
        "ovcf = 'HG00096.chr11.exome.filtered.vcf'"
      ],
      "execution_count": 49,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "IIWBiUH1H2Hs",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "ed1dbbd8-2ce3-454f-adb6-708655a1683a"
      },
      "source": [
        "!{gatk} SelectVariants -R {ref} -V {ivcf} -O {isvcf} --select-type-to-include SNP\n",
        "!{gatk} VariantFiltration -R {ref} -V {isvcf} -O {osvcf} -filter \"{snp_f}\" --filter-name \"snp\"\n",
        "\n",
        "!{gatk} SelectVariants -R {ref} -V {ivcf} -O {iivcf} --select-type-to-include INDEL\n",
        "!{gatk} VariantFiltration -R {ref} -V {iivcf} -O {oivcf} -filter \"{indel_f}\" --filter-name \"indel\"\n",
        "\n",
        "!{gatk} MergeVcfs -I {osvcf} -I {oivcf} -O {ovcf}"
      ],
      "execution_count": 50,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:41:23.808 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:41:29.555 INFO  SelectVariants - ------------------------------------------------------------\n",
            "14:41:29.556 INFO  SelectVariants - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:41:29.556 INFO  SelectVariants - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:41:29.557 INFO  SelectVariants - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:41:29.557 INFO  SelectVariants - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:41:29.558 INFO  SelectVariants - Start Date/Time: June 3, 2021 at 2:41:23 PM UTC\n",
            "14:41:29.558 INFO  SelectVariants - ------------------------------------------------------------\n",
            "14:41:29.558 INFO  SelectVariants - ------------------------------------------------------------\n",
            "14:41:29.559 INFO  SelectVariants - HTSJDK Version: 2.21.0\n",
            "14:41:29.559 INFO  SelectVariants - Picard Version: 2.21.2\n",
            "14:41:29.559 INFO  SelectVariants - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:41:29.559 INFO  SelectVariants - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:41:29.559 INFO  SelectVariants - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:41:29.559 INFO  SelectVariants - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:41:29.559 INFO  SelectVariants - Deflater: IntelDeflater\n",
            "14:41:29.559 INFO  SelectVariants - Inflater: IntelInflater\n",
            "14:41:29.560 INFO  SelectVariants - GCS max retries/reopens: 20\n",
            "14:41:29.560 INFO  SelectVariants - Requester pays: disabled\n",
            "14:41:29.560 INFO  SelectVariants - Initializing engine\n",
            "14:41:30.014 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/HG00096.chr11.exome.vcf\n",
            "14:41:30.215 INFO  SelectVariants - Done initializing engine\n",
            "14:41:30.359 INFO  ProgressMeter - Starting traversal\n",
            "14:41:30.359 INFO  ProgressMeter -        Current Locus  Elapsed Minutes    Variants Processed  Variants/Minute\n",
            "14:41:31.324 INFO  ProgressMeter -      chr11:125619893              0.0                  5235         325829.9\n",
            "14:41:31.324 INFO  ProgressMeter - Traversal complete. Processed 5235 total variants in 0.0 minutes.\n",
            "14:41:31.425 INFO  SelectVariants - Shutting down engine\n",
            "[June 3, 2021 at 2:41:31 PM UTC] org.broadinstitute.hellbender.tools.walkers.variantutils.SelectVariants done. Elapsed time: 0.13 minutes.\n",
            "Runtime.totalMemory()=348127232\n",
            "14:41:34.806 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:41:40.505 INFO  VariantFiltration - ------------------------------------------------------------\n",
            "14:41:40.506 INFO  VariantFiltration - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:41:40.506 INFO  VariantFiltration - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:41:40.506 INFO  VariantFiltration - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:41:40.506 INFO  VariantFiltration - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:41:40.506 INFO  VariantFiltration - Start Date/Time: June 3, 2021 at 2:41:34 PM UTC\n",
            "14:41:40.507 INFO  VariantFiltration - ------------------------------------------------------------\n",
            "14:41:40.507 INFO  VariantFiltration - ------------------------------------------------------------\n",
            "14:41:40.508 INFO  VariantFiltration - HTSJDK Version: 2.21.0\n",
            "14:41:40.508 INFO  VariantFiltration - Picard Version: 2.21.2\n",
            "14:41:40.508 INFO  VariantFiltration - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:41:40.508 INFO  VariantFiltration - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:41:40.508 INFO  VariantFiltration - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:41:40.508 INFO  VariantFiltration - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:41:40.508 INFO  VariantFiltration - Deflater: IntelDeflater\n",
            "14:41:40.508 INFO  VariantFiltration - Inflater: IntelInflater\n",
            "14:41:40.508 INFO  VariantFiltration - GCS max retries/reopens: 20\n",
            "14:41:40.508 INFO  VariantFiltration - Requester pays: disabled\n",
            "14:41:40.508 INFO  VariantFiltration - Initializing engine\n",
            "14:41:41.004 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/HG00096.chr11.exome.snp.vcf\n",
            "14:41:41.264 INFO  VariantFiltration - Done initializing engine\n",
            "14:41:41.467 INFO  ProgressMeter - Starting traversal\n",
            "14:41:41.467 INFO  ProgressMeter -        Current Locus  Elapsed Minutes    Variants Processed  Variants/Minute\n",
            "14:41:41.513 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.515 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.521 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.522 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.522 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.523 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.526 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.529 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.529 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.530 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.535 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.536 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.537 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.538 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.543 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.543 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.545 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.545 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.546 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.547 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.548 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.548 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.551 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.552 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.552 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.553 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.556 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.558 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.564 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.568 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.569 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.570 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.570 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.573 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.573 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.574 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.574 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.575 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.576 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.576 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.577 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.580 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.581 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.581 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.583 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.588 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.590 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.595 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.595 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.598 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.599 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.599 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.600 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.600 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.601 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.602 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.603 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.603 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.604 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.604 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.605 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.605 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.606 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.606 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.607 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.609 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.609 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.610 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.610 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.611 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.612 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.617 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.620 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.622 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.622 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.624 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.624 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.626 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.627 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.638 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.639 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.641 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.681 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.682 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.682 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.683 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.683 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.684 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.685 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.685 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.687 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.688 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.689 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.689 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.690 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.692 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.693 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.694 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.694 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.695 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.695 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.696 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.696 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.697 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.697 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.699 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.700 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.700 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.701 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.703 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.703 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.704 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.704 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.705 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.705 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.707 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.707 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.707 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.708 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.709 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.710 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.712 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.715 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.715 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.716 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.716 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.716 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.717 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.721 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.721 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.722 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.722 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.723 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.724 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.724 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.725 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.725 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.725 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.726 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.727 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.727 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.729 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.731 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.732 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.735 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.735 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.736 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.736 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.739 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.739 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.741 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.742 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.743 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.745 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.745 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.746 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.746 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.748 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.750 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.750 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.753 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.753 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.756 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.756 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.757 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.767 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.767 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.768 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.771 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.771 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.773 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.776 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.776 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.777 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.778 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.778 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.779 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.781 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.782 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.784 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.785 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.787 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.787 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.846 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.847 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.848 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.848 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.848 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.849 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.849 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.850 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.851 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.860 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.861 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.862 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.862 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.865 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.875 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.878 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.880 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.885 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.885 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.886 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.886 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.887 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.888 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.891 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.891 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.892 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.892 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.892 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.893 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.896 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.897 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.902 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.903 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.903 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.904 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.904 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.905 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.906 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.913 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.918 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.919 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.920 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.922 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.922 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.923 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.928 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.931 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.931 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.931 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.932 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.936 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.936 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.937 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.937 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.939 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.942 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.942 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.943 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.943 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.944 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.946 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.946 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.947 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.951 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.956 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.956 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.968 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.976 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.976 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.977 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.977 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.978 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.978 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.978 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.979 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.980 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.987 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.988 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.988 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.988 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.988 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.989 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.989 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.989 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.990 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.990 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.990 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.991 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.992 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.992 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.992 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.993 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.993 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.998 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.999 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.999 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:41.999 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.000 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.000 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.001 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.001 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.001 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.001 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.002 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.002 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.004 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.005 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.006 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.006 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.007 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.009 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.011 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.012 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.013 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.013 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.014 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.015 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.015 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.016 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.016 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.016 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.017 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.018 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.018 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.018 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.019 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.020 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.020 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.021 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.025 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.028 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.035 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.036 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.036 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.036 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.037 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.037 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.037 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.038 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.038 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.039 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.039 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.040 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.045 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.045 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.046 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.046 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.046 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.049 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.049 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.053 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.054 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.055 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.060 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.060 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.062 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.063 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.064 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.065 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.068 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.069 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.069 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.070 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.070 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.072 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.072 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.073 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.073 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.073 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.078 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.078 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.078 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.079 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.079 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.079 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.080 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.081 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.081 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.081 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.081 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.082 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.083 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.083 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.083 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.084 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.084 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.084 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.085 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.086 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.086 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.087 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.087 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.087 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.087 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.089 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.089 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.089 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.089 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.090 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.090 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.093 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.099 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.100 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.101 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.101 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.105 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.106 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.106 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.107 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.107 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.108 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.110 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.110 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.111 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.111 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.111 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.112 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.113 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.113 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.113 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.114 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.114 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.115 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.115 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.116 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.116 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.117 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.117 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.117 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.119 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.120 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.120 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.122 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.123 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.123 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.125 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.126 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.126 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.133 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.133 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.133 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.134 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.134 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.134 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.134 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.135 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.135 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.135 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.136 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.136 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.136 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.137 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.137 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.137 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.138 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.138 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.138 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.139 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.141 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.141 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.141 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.142 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.142 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.144 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.147 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.147 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.147 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.147 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.148 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.150 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.150 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.151 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.151 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.151 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.151 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.152 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.152 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.157 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.159 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.160 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.160 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.161 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.161 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.162 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.163 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.164 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.165 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.167 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.170 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.173 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.173 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.173 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.174 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.174 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.175 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.175 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.176 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.176 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.178 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.178 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.179 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.179 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.179 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.187 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.188 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.188 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.196 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.196 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.196 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.198 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.198 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.198 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.198 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.199 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.199 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.199 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.199 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.199 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.200 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.200 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.200 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.200 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.201 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.201 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.201 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.201 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.206 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.206 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.206 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.207 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.207 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.207 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.207 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.207 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.208 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.208 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.208 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.208 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.209 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.209 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.209 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.209 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.209 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.209 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.216 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.217 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.217 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.217 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.217 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.218 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.218 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.218 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.218 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.227 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.227 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.228 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.228 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.228 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.228 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.228 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.229 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.229 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.229 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.229 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.230 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.230 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.230 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.230 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.231 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.231 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.231 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.231 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.231 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.231 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.232 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.232 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.232 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.232 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.233 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.234 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.238 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.239 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.239 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.239 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.239 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.239 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.240 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.240 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.240 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.240 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.240 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.240 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.241 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.241 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.241 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.241 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.241 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.241 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.242 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.253 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.253 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.255 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.255 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.255 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.257 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.259 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.267 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.267 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.268 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.268 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.269 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.269 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.269 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.270 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.270 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.270 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.271 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.271 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.272 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.272 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.275 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.275 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.275 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.276 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.276 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.276 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.276 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.277 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.277 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.277 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.278 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.278 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.278 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.278 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.279 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.330 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.331 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.331 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.331 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.331 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.332 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.332 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.332 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.333 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.333 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.333 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.333 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.333 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.334 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.334 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.334 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.334 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.334 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.334 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.334 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.335 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.335 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.335 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.336 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.336 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.336 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.336 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.337 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.337 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.337 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.337 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.338 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.338 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.338 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.338 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.339 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.339 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.339 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.340 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.340 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.340 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.340 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.340 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.340 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.341 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.341 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.341 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.341 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.341 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.341 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.342 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.342 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.342 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.342 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.342 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.342 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.342 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.343 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.352 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.356 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.356 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.357 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.357 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.359 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.367 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.367 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.367 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.367 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.367 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.367 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.368 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.368 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.368 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.368 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.368 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.368 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.369 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.369 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.369 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.369 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.369 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.369 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.370 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.370 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.370 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.370 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.370 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.371 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.371 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.371 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.371 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.372 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.372 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.372 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.433 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.433 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.434 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.434 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.434 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.434 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.434 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.434 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.435 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.435 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.436 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.436 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.436 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.436 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.436 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.437 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.438 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.438 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.438 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.438 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.438 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.438 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.439 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.439 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.439 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.439 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.439 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.440 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.440 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.440 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.441 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.442 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.442 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.443 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.443 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.443 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.444 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.444 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.444 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.444 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.445 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.445 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.445 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.445 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.445 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.451 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.451 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.451 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.457 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.457 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.457 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.459 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.459 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.459 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.460 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.460 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.460 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.460 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.460 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.533 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.533 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.533 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.533 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.533 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.534 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.534 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.534 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.534 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.534 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.535 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.535 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.535 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.535 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.536 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.536 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.536 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.539 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.544 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.544 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.544 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.545 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.545 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.545 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.545 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.546 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.546 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.546 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.547 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.547 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.547 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.547 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.548 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.548 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.549 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.549 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.549 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.551 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.551 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.551 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.551 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.552 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.552 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.552 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.553 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.553 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.553 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.553 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.553 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.554 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.554 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.554 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.555 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.555 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.567 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.567 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.570 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.570 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.574 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.574 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.574 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.574 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.574 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.575 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.575 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.575 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.575 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.579 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.580 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.581 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.581 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.581 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.582 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.595 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.595 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.595 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.598 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.598 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.598 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.636 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.636 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.636 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.637 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.638 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.638 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.638 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.639 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.639 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.639 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.639 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.639 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.640 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.640 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.640 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.641 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.641 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.641 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.641 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.641 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.641 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.642 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.642 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.643 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.643 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.644 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.645 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.645 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.645 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.645 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.645 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.646 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.646 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.646 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.646 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.646 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.647 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.647 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.647 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.647 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.647 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.647 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.648 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.648 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.648 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.649 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.649 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.649 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.649 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.650 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.650 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.650 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.650 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.650 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.651 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.651 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.651 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.651 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.651 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.652 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.652 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.652 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.653 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.653 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.653 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.653 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.654 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.654 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.654 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.654 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.654 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.655 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.655 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.655 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.655 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.655 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.656 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.656 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.656 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.656 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.657 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.657 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.657 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.657 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.657 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.657 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.658 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.658 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.658 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.659 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.659 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.659 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.659 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.660 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.660 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.661 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.663 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.663 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.663 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.664 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.665 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.737 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.737 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.737 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.737 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.738 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.738 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.738 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.738 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.738 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.738 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.738 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.739 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.739 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.740 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.740 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.740 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.740 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.740 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.740 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.740 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.741 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.741 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.741 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.741 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.741 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.741 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.742 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.742 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.742 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.742 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.742 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.742 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.743 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.743 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.743 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.743 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.744 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.744 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.745 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.745 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.745 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.746 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.746 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.746 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.747 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.747 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.747 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.747 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.747 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.748 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.748 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.748 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.748 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.748 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.748 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.749 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.749 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.749 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.749 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.750 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.750 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.750 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.750 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.751 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.751 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.752 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.752 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.752 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.753 WARN  JexlEngine - ![59,73]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable ReadPosRankSum\n",
            "14:41:42.754 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.754 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.754 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.754 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.755 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.755 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.755 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.756 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.756 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.756 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.756 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.757 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.757 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.757 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.757 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.757 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.757 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.758 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.759 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.759 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.759 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.759 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.759 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.759 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.760 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.760 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.760 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.761 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.761 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.761 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.761 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.761 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.761 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.762 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.762 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.762 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.762 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.762 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.762 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.764 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.764 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.764 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.765 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.765 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.856 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.856 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.857 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.857 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.857 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.857 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.858 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.858 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.858 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.859 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.859 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.859 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.859 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.859 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.859 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.859 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.860 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.866 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.866 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.866 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.866 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.867 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.867 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.867 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.867 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.867 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.869 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.870 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.870 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.870 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.870 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.871 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.877 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.877 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.877 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.878 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.878 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.878 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.879 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.879 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.879 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.879 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.879 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.879 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.880 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.880 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.888 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.888 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.888 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.889 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.889 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.889 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.889 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.889 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.890 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.890 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.890 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.890 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.891 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.891 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.891 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.891 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.891 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.892 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.892 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.892 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.892 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.893 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.893 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.893 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.893 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.893 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.893 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.894 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.894 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.894 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.894 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.894 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.894 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.894 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.895 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.895 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.895 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.895 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.895 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.895 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.896 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.896 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.896 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.896 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.896 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.896 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.897 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.898 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.898 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.898 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.898 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.898 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.898 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.899 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.899 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.899 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.899 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.899 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.899 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.900 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.900 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.900 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.900 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.901 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.901 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.901 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.901 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.901 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.901 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.901 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.902 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.903 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.903 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.903 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.940 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.940 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.940 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.940 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.941 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.941 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.942 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.942 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.942 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.942 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.942 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.943 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.943 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.943 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.943 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.943 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.944 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.944 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.944 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.944 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.944 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.944 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.945 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.945 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.945 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.945 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.946 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.946 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.947 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.947 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.947 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.947 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.948 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.949 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.950 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.951 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.951 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.951 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.952 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.952 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.952 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.952 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.952 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.952 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.952 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.953 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.954 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.955 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.956 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.956 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.957 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.958 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.959 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.960 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.960 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.960 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.960 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.960 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.960 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.961 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.961 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:42.961 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.041 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.042 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.043 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.043 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.043 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.044 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.045 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.045 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.045 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.045 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.045 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.046 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.046 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.049 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.049 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.049 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.050 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.050 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.050 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.051 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.051 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.052 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.052 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.052 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.052 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.052 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.053 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.053 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.053 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.053 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.053 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.053 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.054 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.054 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.054 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.054 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.055 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.055 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.055 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.055 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.055 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.055 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.056 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.056 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.056 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.056 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.056 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.057 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.057 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.057 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.057 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.058 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.058 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.058 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.058 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.058 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.058 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.058 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.059 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.060 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.060 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.060 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.060 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.060 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.061 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.061 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.061 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.061 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.062 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.062 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.063 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.063 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.064 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.064 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.065 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.065 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.066 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.066 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.066 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.067 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.067 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.067 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.067 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.067 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.067 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.067 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.068 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.068 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.068 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.068 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.068 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.069 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.069 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.155 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.156 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.158 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.159 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.159 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.159 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.161 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.162 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.163 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.163 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.167 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.173 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.174 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.174 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.174 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.175 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.175 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.175 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.176 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.176 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.176 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.177 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.177 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.177 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.177 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.177 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.177 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.177 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.178 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.178 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.178 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.179 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.179 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.179 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.179 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.180 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.181 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.181 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.181 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.181 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.181 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.181 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.181 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.182 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.183 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.184 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.184 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.184 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.187 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.187 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.188 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.188 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.189 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.189 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.189 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.189 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.189 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.190 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.191 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.191 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.191 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.192 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.192 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.192 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.192 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.192 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.193 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.193 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.193 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.193 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.193 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.193 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.193 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.194 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.194 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.194 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.194 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.194 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.194 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.195 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.195 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.195 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.195 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.195 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.195 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.196 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.196 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.197 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.197 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.197 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.197 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.197 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.197 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.198 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.198 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.243 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.245 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.245 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.245 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.246 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.246 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.246 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.246 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.246 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.246 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.247 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.247 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.247 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.248 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.248 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.248 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.248 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.249 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.250 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.251 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.252 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.253 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.253 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.253 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.253 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.253 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.254 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.255 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.255 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.255 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.255 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.256 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.257 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.257 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.257 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.257 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.257 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.258 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.258 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.258 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.258 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.259 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.259 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.259 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.259 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.259 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.259 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.260 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.261 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.262 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.263 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.264 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.265 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.266 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.266 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.266 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.349 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.350 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.351 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.352 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.352 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.352 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.352 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.352 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.352 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.353 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.354 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.355 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.356 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.356 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.356 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.356 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.357 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.357 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.357 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.357 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.357 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.358 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.358 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.358 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.358 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.358 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.359 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.359 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.359 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.359 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.359 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.360 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.361 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.362 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.363 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.364 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.365 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.366 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.367 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.448 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.450 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.451 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.452 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.453 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.454 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.455 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.456 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.457 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.457 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.457 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.457 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.458 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.459 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.460 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.460 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.461 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.462 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.463 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.464 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.465 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.466 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.467 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.467 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.467 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.467 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.467 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.468 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.468 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.468 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.468 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.469 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.469 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.469 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.469 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.469 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.469 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.469 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.470 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.470 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.470 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.470 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.470 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.470 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.471 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.471 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.471 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.471 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.471 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.472 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.472 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.472 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.473 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.473 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.473 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.473 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.474 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.474 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.474 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.474 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.474 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.474 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.474 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.475 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.475 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.475 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.560 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.560 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.561 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.561 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.562 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.562 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.562 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.563 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.565 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.566 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.567 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.567 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.567 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.570 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.570 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.570 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.571 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.572 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.576 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.577 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.577 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.577 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.577 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.577 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.578 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.581 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.581 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.582 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.582 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.582 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.582 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.582 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.582 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.587 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.587 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.587 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.587 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.587 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.587 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.587 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.588 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.588 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.588 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.589 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.590 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.590 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.590 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.590 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.590 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.591 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.592 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.593 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.594 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.595 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.596 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.597 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.598 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.599 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.599 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.599 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.599 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.600 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.600 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.600 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.659 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.659 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.660 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.660 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.660 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.660 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.660 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.661 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.661 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.661 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.661 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.661 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.661 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.662 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.662 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.662 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.662 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.662 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.662 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.662 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.663 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.663 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.663 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.663 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.664 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.664 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.664 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.664 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.664 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.665 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.665 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.665 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.666 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.667 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.668 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.669 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.669 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.669 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.669 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.669 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.669 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.670 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.671 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.671 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.671 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.671 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.671 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.671 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.672 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.672 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.672 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.672 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.673 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.673 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.673 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.673 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.673 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.674 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.675 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.676 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.677 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.677 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.677 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.677 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.677 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.677 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.677 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.678 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.679 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.679 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.679 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.762 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.764 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.766 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.767 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.767 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.767 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.770 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.770 WARN  JexlEngine - ![38,47]: 'QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0;' undefined variable MQRankSum\n",
            "14:41:43.771 INFO  ProgressMeter -      chr11:125619893              0.0                  5235         136387.3\n",
            "14:41:43.771 INFO  ProgressMeter - Traversal complete. Processed 5235 total variants in 0.0 minutes.\n",
            "14:41:43.862 INFO  VariantFiltration - Shutting down engine\n",
            "[June 3, 2021 at 2:41:43 PM UTC] org.broadinstitute.hellbender.tools.walkers.filters.VariantFiltration done. Elapsed time: 0.15 minutes.\n",
            "Runtime.totalMemory()=264241152\n",
            "14:41:47.126 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:41:52.865 INFO  SelectVariants - ------------------------------------------------------------\n",
            "14:41:52.866 INFO  SelectVariants - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:41:52.866 INFO  SelectVariants - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:41:52.867 INFO  SelectVariants - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:41:52.867 INFO  SelectVariants - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:41:52.867 INFO  SelectVariants - Start Date/Time: June 3, 2021 at 2:41:47 PM UTC\n",
            "14:41:52.867 INFO  SelectVariants - ------------------------------------------------------------\n",
            "14:41:52.867 INFO  SelectVariants - ------------------------------------------------------------\n",
            "14:41:52.868 INFO  SelectVariants - HTSJDK Version: 2.21.0\n",
            "14:41:52.869 INFO  SelectVariants - Picard Version: 2.21.2\n",
            "14:41:52.869 INFO  SelectVariants - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:41:52.869 INFO  SelectVariants - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:41:52.869 INFO  SelectVariants - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:41:52.869 INFO  SelectVariants - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:41:52.869 INFO  SelectVariants - Deflater: IntelDeflater\n",
            "14:41:52.869 INFO  SelectVariants - Inflater: IntelInflater\n",
            "14:41:52.869 INFO  SelectVariants - GCS max retries/reopens: 20\n",
            "14:41:52.870 INFO  SelectVariants - Requester pays: disabled\n",
            "14:41:52.870 INFO  SelectVariants - Initializing engine\n",
            "14:41:53.491 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/HG00096.chr11.exome.vcf\n",
            "14:41:53.738 INFO  SelectVariants - Done initializing engine\n",
            "14:41:53.931 INFO  ProgressMeter - Starting traversal\n",
            "14:41:53.931 INFO  ProgressMeter -        Current Locus  Elapsed Minutes    Variants Processed  Variants/Minute\n",
            "14:41:54.424 INFO  ProgressMeter -             unmapped              0.0                   532          64746.5\n",
            "14:41:54.424 INFO  ProgressMeter - Traversal complete. Processed 532 total variants in 0.0 minutes.\n",
            "14:41:54.526 INFO  SelectVariants - Shutting down engine\n",
            "[June 3, 2021 at 2:41:54 PM UTC] org.broadinstitute.hellbender.tools.walkers.variantutils.SelectVariants done. Elapsed time: 0.12 minutes.\n",
            "Runtime.totalMemory()=338690048\n",
            "14:41:57.829 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:42:03.590 INFO  VariantFiltration - ------------------------------------------------------------\n",
            "14:42:03.591 INFO  VariantFiltration - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:42:03.591 INFO  VariantFiltration - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:42:03.593 INFO  VariantFiltration - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:42:03.593 INFO  VariantFiltration - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:42:03.594 INFO  VariantFiltration - Start Date/Time: June 3, 2021 at 2:41:57 PM UTC\n",
            "14:42:03.594 INFO  VariantFiltration - ------------------------------------------------------------\n",
            "14:42:03.594 INFO  VariantFiltration - ------------------------------------------------------------\n",
            "14:42:03.596 INFO  VariantFiltration - HTSJDK Version: 2.21.0\n",
            "14:42:03.596 INFO  VariantFiltration - Picard Version: 2.21.2\n",
            "14:42:03.596 INFO  VariantFiltration - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:42:03.596 INFO  VariantFiltration - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:42:03.596 INFO  VariantFiltration - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:42:03.596 INFO  VariantFiltration - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:42:03.596 INFO  VariantFiltration - Deflater: IntelDeflater\n",
            "14:42:03.596 INFO  VariantFiltration - Inflater: IntelInflater\n",
            "14:42:03.597 INFO  VariantFiltration - GCS max retries/reopens: 20\n",
            "14:42:03.597 INFO  VariantFiltration - Requester pays: disabled\n",
            "14:42:03.597 INFO  VariantFiltration - Initializing engine\n",
            "14:42:04.096 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/HG00096.chr11.exome.indel.vcf\n",
            "14:42:04.337 INFO  VariantFiltration - Done initializing engine\n",
            "14:42:04.510 INFO  ProgressMeter - Starting traversal\n",
            "14:42:04.510 INFO  ProgressMeter -        Current Locus  Elapsed Minutes    Variants Processed  Variants/Minute\n",
            "14:42:04.541 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.546 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.547 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.548 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.548 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.550 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.551 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.552 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.553 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.554 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.555 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.557 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.558 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.559 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.560 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.562 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.564 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.565 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.566 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.567 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.569 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.570 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.571 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.573 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.576 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.577 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.579 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.590 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.594 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.599 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.600 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.604 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.605 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.607 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.607 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.609 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.617 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.618 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.620 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.621 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.622 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.622 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.623 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.633 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.633 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.634 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.634 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.641 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.651 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.653 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.655 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.657 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.658 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.659 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.662 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.663 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.665 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.667 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.668 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.676 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.677 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.682 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.682 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.683 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.683 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.684 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.684 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.685 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.685 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.688 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.689 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.694 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.695 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.695 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.697 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.704 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.705 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.706 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.707 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.707 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.708 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.708 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.709 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.709 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.711 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.711 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.713 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.714 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.715 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.716 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.718 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.720 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.721 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.721 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.723 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.725 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.727 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.727 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.727 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.728 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.730 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.730 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.730 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.732 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.734 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.735 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.736 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.737 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.738 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.739 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.739 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.740 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.740 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.741 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.741 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.742 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.742 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.743 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.744 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.746 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.747 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.747 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.748 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.748 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.749 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.749 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.750 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.751 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.752 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.753 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.754 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.756 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.757 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.757 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.759 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.759 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.760 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.763 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.764 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.764 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.765 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.766 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.767 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.768 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.768 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.786 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.786 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.787 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.787 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.788 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.788 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.790 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.790 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.791 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.791 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.792 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.792 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.793 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.794 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.794 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.795 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.796 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.797 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.798 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.799 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.800 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.802 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.802 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.804 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.805 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.805 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.806 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.812 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.816 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.816 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.817 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.817 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.819 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.820 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.821 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.828 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.829 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.829 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.830 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.830 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.842 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.845 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.845 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.846 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.846 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.846 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.847 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.847 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.848 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.849 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.853 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.853 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.854 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.854 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.855 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.855 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.855 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.856 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.856 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.856 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.857 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.857 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.858 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.859 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.861 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.862 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.863 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.863 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.864 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.865 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.865 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.865 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.866 WARN  JexlEngine - ![26,40]: 'QD < 2.0 || FS > 200.0 || ReadPosRankSum < -20.0;' undefined variable ReadPosRankSum\n",
            "14:42:04.868 INFO  ProgressMeter -             unmapped              0.0                   532          89411.8\n",
            "14:42:04.868 INFO  ProgressMeter - Traversal complete. Processed 532 total variants in 0.0 minutes.\n",
            "14:42:04.939 INFO  VariantFiltration - Shutting down engine\n",
            "[June 3, 2021 at 2:42:04 PM UTC] org.broadinstitute.hellbender.tools.walkers.filters.VariantFiltration done. Elapsed time: 0.12 minutes.\n",
            "Runtime.totalMemory()=271581184\n",
            "14:42:08.165 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "[Thu Jun 03 14:42:08 UTC 2021] MergeVcfs  --INPUT HG00096.chr11.exome.snp.filtered.vcf --INPUT HG00096.chr11.exome.indel.filtered.vcf --OUTPUT HG00096.chr11.exome.filtered.vcf  --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX true --CREATE_MD5_FILE false --GA4GH_CLIENT_SECRETS client_secrets.json --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n",
            "[Thu Jun 03 14:42:13 UTC 2021] Executing as root@43bfdb95a763 on Linux 5.4.109+ amd64; OpenJDK 64-Bit Server VM 11.0.11+9-Ubuntu-0ubuntu2.18.04; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.1.4.1\n",
            "[Thu Jun 03 14:42:15 UTC 2021] picard.vcf.MergeVcfs done. Elapsed time: 0.12 minutes.\n",
            "Runtime.totalMemory()=267386880\n",
            "Tool returned:\n",
            "0\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "7mO32O_iH8Dj"
      },
      "source": [
        "def is_filtered(variant):\n",
        "  return 'PASS' not in variant.filter\n",
        "\n",
        "def is_snp(variant):\n",
        "  return len(variant.ref) == 1 and len(variant.alts[0]) == 1\n",
        "\n",
        "def is_transitions(variant):\n",
        "  allels = set((variant.ref, variant.alts[0]))\n",
        "  return allels == {'A', 'G'} or allels == {'C', 'T'}"
      ],
      "execution_count": 51,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "L8A9vApzH9A8",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "54473db9-6da2-41f4-993b-3c51547345ce"
      },
      "source": [
        "ovcf = 'HG00096.chr11.exome.filtered.vcf'\n",
        "\n",
        "# print('SNPs:',\n",
        "#       sum(1 for v in pysam.VariantFile(ovcf) if is_filtered(v) and is_snp(v)))\n",
        "\n",
        "# print('INDELs:',\n",
        "#       sum(1 for v in pysam.VariantFile(ovcf) if is_filtered(v) and not is_snp(v)))\n",
        "\n",
        "snp = 0\n",
        "ind = 0\n",
        "\n",
        "for v in pysam.VariantFile(ovcf):\n",
        "    if is_snp(v):\n",
        "        snp += 1\n",
        "    else:\n",
        "        ind += 1\n",
        "\n",
        "print(f'SNPs: {snp}, INDELs: {ind}, Sum: {snp + ind}')"
      ],
      "execution_count": 52,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "SNPs: 5235, INDELs: 532, Sum: 5767\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "XeGm8Fe8Gbng",
        "outputId": "ee132f7b-c879-4fd6-a4d9-3463ea13a6ae"
      },
      "source": [
        "proslo = 0\n",
        "nije_proslo = 0\n",
        "\n",
        "for v in pysam.VariantFile(ovcf):\n",
        "    if is_filtered(v):\n",
        "        nije_proslo += 1\n",
        "    else:\n",
        "        proslo += 1\n",
        "\n",
        "print(f'Passed filter: {proslo}, Did not pass filter: {nije_proslo}')"
      ],
      "execution_count": 53,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Passed filter: 5577, Did not pass filter: 190\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0BF_UDqGICms",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2068b4ef-a53b-46c3-dcf0-f33534ac4a90"
      },
      "source": [
        "ti = sum(1 for v in pysam.VariantFile(ovcf) \\\n",
        "         if not is_filtered(v) \\\n",
        "         and is_snp(v)\\\n",
        "         and is_transitions(v))\n",
        "\n",
        "tv = sum(1 for v in pysam.VariantFile(ovcf) \\\n",
        "          if not is_filtered(v) \\\n",
        "          and is_snp(v) \\\n",
        "          and not is_transitions(v))\n",
        "\n",
        "print('After filter:')\n",
        "print('Ti:', ti)\n",
        "print('Tv:', tv)\n",
        "print('Ti/Tv', ti/tv)"
      ],
      "execution_count": 54,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "After filter:\n",
            "Ti: 3489\n",
            "Tv: 1567\n",
            "Ti/Tv 2.226547543075941\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "mDwNMSnHFmXu"
      },
      "source": [
        "###5. Anotirati mutacije alatom Funcotator 5a. Izbrojati različite vrednosti ClinVar značajnosti (anotacija ClinVar_VCF_CLNSIG, koliko mutacija je označeno kao Benign, Likely_benign, itd.)-."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pyHtWMefDAdh",
        "outputId": "022994e6-3f89-4587-f861-e413b2279b70"
      },
      "source": [
        "!{gatk} FuncotatorDataSourceDownloader --germline --extract-after-download"
      ],
      "execution_count": 55,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:42:18.824 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:42:24.531 INFO  FuncotatorDataSourceDownloader - ------------------------------------------------------------\n",
            "14:42:24.532 INFO  FuncotatorDataSourceDownloader - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:42:24.532 INFO  FuncotatorDataSourceDownloader - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:42:24.532 INFO  FuncotatorDataSourceDownloader - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:42:24.532 INFO  FuncotatorDataSourceDownloader - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:42:24.533 INFO  FuncotatorDataSourceDownloader - Start Date/Time: June 3, 2021 at 2:42:18 PM UTC\n",
            "14:42:24.533 INFO  FuncotatorDataSourceDownloader - ------------------------------------------------------------\n",
            "14:42:24.533 INFO  FuncotatorDataSourceDownloader - ------------------------------------------------------------\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - HTSJDK Version: 2.21.0\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - Picard Version: 2.21.2\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - Deflater: IntelDeflater\n",
            "14:42:24.534 INFO  FuncotatorDataSourceDownloader - Inflater: IntelInflater\n",
            "14:42:24.535 INFO  FuncotatorDataSourceDownloader - GCS max retries/reopens: 20\n",
            "14:42:24.535 INFO  FuncotatorDataSourceDownloader - Requester pays: disabled\n",
            "14:42:24.535 INFO  FuncotatorDataSourceDownloader - Initializing engine\n",
            "14:42:24.535 INFO  FuncotatorDataSourceDownloader - Done initializing engine\n",
            "14:42:24.535 INFO  FuncotatorDataSourceDownloader - Germline data sources selected.\n",
            "14:42:31.126 INFO  NioFileCopierWithProgressMeter - Initiating copy from gs://broad-public-datasets/funcotator/funcotator_dataSources.v1.6.20190124g.tar.gz to file:///content/funcotator_dataSources.v1.6.20190124g.tar.gz\n",
            "14:42:31.126 INFO  NioFileCopierWithProgressMeter - File size: 248385296 bytes (236 MB).\n",
            "14:42:31.127 INFO  NioFileCopierWithProgressMeter - Please wait.  This could take a while...\n",
            "14:42:32.675 INFO  NioFileCopierWithProgressMeter -     Transfer:  5.00% complete.  Est. time remaining: 02:32.149 (@1513.49 kbps)\n",
            "14:42:33.379 INFO  NioFileCopierWithProgressMeter -     Transfer:  10.00% complete.  Est. time remaining: 23.434 (@9301.95 kbps)\n",
            "14:42:33.989 INFO  NioFileCopierWithProgressMeter -     Transfer:  15.00% complete.  Est. time remaining: 15.912 (@12927.08 kbps)\n",
            "14:42:34.643 INFO  NioFileCopierWithProgressMeter -     Transfer:  20.00% complete.  Est. time remaining: 13.438 (@14392.56 kbps)\n",
            "14:42:35.221 INFO  NioFileCopierWithProgressMeter -     Transfer:  25.00% complete.  Est. time remaining: 11.482 (@15773.32 kbps)\n",
            "14:42:35.771 INFO  NioFileCopierWithProgressMeter -     Transfer:  30.00% complete.  Est. time remaining: 10.005 (@16874.85 kbps)\n",
            "14:42:36.329 INFO  NioFileCopierWithProgressMeter -     Transfer:  35.00% complete.  Est. time remaining: 09.080 (@17353.10 kbps)\n",
            "14:42:36.872 INFO  NioFileCopierWithProgressMeter -     Transfer:  40.00% complete.  Est. time remaining: 08.065 (@18012.69 kbps)\n",
            "14:42:37.344 INFO  NioFileCopierWithProgressMeter -     Transfer:  45.00% complete.  Est. time remaining: 07.033 (@18910.08 kbps)\n",
            "14:42:37.942 INFO  NioFileCopierWithProgressMeter -     Transfer:  50.00% complete.  Est. time remaining: 06.329 (@19070.49 kbps)\n",
            "14:42:38.435 INFO  NioFileCopierWithProgressMeter -     Transfer:  55.00% complete.  Est. time remaining: 05.062 (@21416.70 kbps)\n",
            "14:42:38.997 INFO  NioFileCopierWithProgressMeter -     Transfer:  60.00% complete.  Est. time remaining: 04.390 (@21894.14 kbps)\n",
            "14:42:39.413 INFO  NioFileCopierWithProgressMeter -     Transfer:  65.00% complete.  Est. time remaining: 03.756 (@22590.62 kbps)\n",
            "14:42:39.959 INFO  NioFileCopierWithProgressMeter -     Transfer:  70.00% complete.  Est. time remaining: 03.160 (@22962.27 kbps)\n",
            "14:42:40.534 INFO  NioFileCopierWithProgressMeter -     Transfer:  75.00% complete.  Est. time remaining: 02.624 (@22973.40 kbps)\n",
            "14:42:41.070 INFO  NioFileCopierWithProgressMeter -     Transfer:  80.00% complete.  Est. time remaining: 02.084 (@23031.97 kbps)\n",
            "14:42:41.536 INFO  NioFileCopierWithProgressMeter -     Transfer:  85.00% complete.  Est. time remaining: 01.510 (@23646.62 kbps)\n",
            "14:42:42.079 INFO  NioFileCopierWithProgressMeter -     Transfer:  90.00% complete.  Est. time remaining: 00.990 (@23646.62 kbps)\n",
            "14:42:42.595 INFO  NioFileCopierWithProgressMeter -     Transfer:  95.00% complete.  Est. time remaining: 00.475 (@23419.10 kbps)\n",
            "14:42:43.136 INFO  NioFileCopierWithProgressMeter -     Transfer:  100.00% complete.  Est. time remaining: 00.000 (@23438.27 kbps)\n",
            "14:42:43.136 INFO  NioFileCopierWithProgressMeter - Download Complete! - Total elapsed time: 18s\n",
            "14:42:43.137 INFO  IOUtils - Extracting data from archive: file:///content/funcotator_dataSources.v1.6.20190124g.tar.gz\n",
            "14:42:43.155 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/\n",
            "14:42:43.159 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/\n",
            "14:42:43.160 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg38/\n",
            "14:42:43.160 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.transcripts.dict\n",
            "14:42:43.576 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.chr_patch_hapl_scaff.annotation.REORDERED.gtf\n",
            "14:42:48.733 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.transcripts.fasta\n",
            "14:42:50.661 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.transcripts.fasta.fai\n",
            "14:42:50.944 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.chr_patch_hapl_scaff.annotation.REORDERED.gtf.idx\n",
            "14:42:50.979 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.config\n",
            "14:42:50.980 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg19/\n",
            "14:42:50.981 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg19/gencode.v19.chr_patch_hapl_scaff.annotation.REORDERED.gtf.idx\n",
            "14:42:50.982 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg19/gencode.v19.pc_transcripts.fasta\n",
            "14:42:53.842 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg19/gencode.v19.chr_patch_hapl_scaff.annotation.REORDERED.gtf\n",
            "14:43:15.731 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg19/gencode.v19.pc_transcripts.dict\n",
            "14:43:15.884 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg19/gencode.config\n",
            "14:43:15.884 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gencode/hg19/gencode.v19.pc_transcripts.fasta.fai\n",
            "14:43:15.992 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/\n",
            "14:43:15.997 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg38/\n",
            "14:43:15.998 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429_hg38.vcf.idx\n",
            "14:43:16.033 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429_hg38.vcf\n",
            "14:43:19.842 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg38/preprocess_clinvar_VCF_hg38.sh\n",
            "14:43:19.843 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429.vcf\n",
            "14:43:20.486 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_vcf.config\n",
            "14:43:20.487 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg19/\n",
            "14:43:20.487 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg19/clinvar_20180401.vcf.idx\n",
            "14:43:20.515 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg19/clinvar_vcf.config\n",
            "14:43:20.516 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/clinvar/hg19/clinvar_20180401.vcf\n",
            "14:43:25.423 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/template.config\n",
            "14:43:25.424 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/README.txt\n",
            "14:43:25.424 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/MANIFEST.txt\n",
            "14:43:25.425 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_rec/\n",
            "14:43:25.425 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_rec/hg38/\n",
            "14:43:25.425 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_rec/hg38/acmg_rec.config\n",
            "14:43:25.426 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_rec/hg38/acmg59_test_cleaned.txt\n",
            "14:43:25.426 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_rec/hg19/\n",
            "14:43:25.427 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_rec/hg19/acmg_rec.config\n",
            "14:43:25.427 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_rec/hg19/acmg59_test_cleaned.txt\n",
            "14:43:25.427 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/\n",
            "14:43:25.427 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/\n",
            "14:43:25.428 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/lmm_known.config\n",
            "14:43:25.428 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.vcf\n",
            "14:43:25.429 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/preprocess_lmm.sh\n",
            "14:43:25.430 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf\n",
            "14:43:25.432 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf.idx\n",
            "14:43:25.433 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg19/\n",
            "14:43:25.433 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg19/lmm_known.config\n",
            "14:43:25.434 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg19/LMM_Path_LP_VUS5-variants-6-12-18.sorted.vcf\n",
            "14:43:25.436 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/lmm_known/hg19/LMM_Path_LP_VUS5-variants-6-12-18.sorted.vcf.idx\n",
            "14:43:25.437 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gnomAD_exome.tar.gz\n",
            "14:43:25.437 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/gnomAD_genome.tar.gz\n",
            "14:43:25.438 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_lof/\n",
            "14:43:25.438 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_lof/hg38/\n",
            "14:43:25.439 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_lof/hg38/acmg_lof.tsv\n",
            "14:43:25.439 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_lof/hg38/acmg_lof.config\n",
            "14:43:25.440 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_lof/hg19/\n",
            "14:43:25.440 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_lof/hg19/acmg_lof.tsv\n",
            "14:43:25.441 INFO  IOUtils - Extracting file: funcotator_dataSources.v1.6.20190124g/acmg_lof/hg19/acmg_lof.config\n",
            "14:43:25.441 INFO  FuncotatorDataSourceDownloader - Shutting down engine\n",
            "[June 3, 2021 at 2:43:25 PM UTC] org.broadinstitute.hellbender.tools.funcotator.FuncotatorDataSourceDownloader done. Elapsed time: 1.11 minutes.\n",
            "Runtime.totalMemory()=312475648\n",
            "Tool returned:\n",
            "true\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "vC0ELniDHnga",
        "outputId": "1d457db2-efe4-47a7-c64b-db34ab6e2750"
      },
      "source": [
        "ivcf = 'HG00096.chr11.exome.filtered.vcf'\n",
        "ovcf = 'HG00096.chr11.exome.annotated.vcf'\n",
        "func = 'funcotator_dataSources.v1.6.20190124g/'\n",
        "\n",
        "!{gatk} Funcotator -O {ovcf} --ref-version hg38 -R {ref} -V {ivcf} --output-file-format VCF --data-sources-path {func}"
      ],
      "execution_count": 56,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "14:43:28.839 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.1.4.1/gatk-package-4.1.4.1-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "14:43:34.585 INFO  Funcotator - ------------------------------------------------------------\n",
            "14:43:34.586 INFO  Funcotator - The Genome Analysis Toolkit (GATK) v4.1.4.1\n",
            "14:43:34.586 INFO  Funcotator - For support and documentation go to https://software.broadinstitute.org/gatk/\n",
            "14:43:34.587 INFO  Funcotator - Executing as root@43bfdb95a763 on Linux v5.4.109+ amd64\n",
            "14:43:34.587 INFO  Funcotator - Java runtime: OpenJDK 64-Bit Server VM v11.0.11+9-Ubuntu-0ubuntu2.18.04\n",
            "14:43:34.587 INFO  Funcotator - Start Date/Time: June 3, 2021 at 2:43:28 PM UTC\n",
            "14:43:34.587 INFO  Funcotator - ------------------------------------------------------------\n",
            "14:43:34.587 INFO  Funcotator - ------------------------------------------------------------\n",
            "14:43:34.588 INFO  Funcotator - HTSJDK Version: 2.21.0\n",
            "14:43:34.588 INFO  Funcotator - Picard Version: 2.21.2\n",
            "14:43:34.588 INFO  Funcotator - HTSJDK Defaults.COMPRESSION_LEVEL : 2\n",
            "14:43:34.588 INFO  Funcotator - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false\n",
            "14:43:34.589 INFO  Funcotator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true\n",
            "14:43:34.589 INFO  Funcotator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false\n",
            "14:43:34.589 INFO  Funcotator - Deflater: IntelDeflater\n",
            "14:43:34.589 INFO  Funcotator - Inflater: IntelInflater\n",
            "14:43:34.589 INFO  Funcotator - GCS max retries/reopens: 20\n",
            "14:43:34.589 INFO  Funcotator - Requester pays: disabled\n",
            "14:43:34.589 INFO  Funcotator - Initializing engine\n",
            "14:43:35.164 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/HG00096.chr11.exome.filtered.vcf\n",
            "14:43:35.412 INFO  Funcotator - Done initializing engine\n",
            "14:43:35.412 INFO  Funcotator - Validating Sequence Dictionaries...\n",
            "14:43:35.452 INFO  Funcotator - Processing user transcripts/defaults/overrides...\n",
            "14:43:35.453 INFO  Funcotator - Initializing data sources...\n",
            "14:43:35.459 INFO  DataSourceUtils - Initializing data sources from directory: funcotator_dataSources.v1.6.20190124g/\n",
            "14:43:35.460 INFO  DataSourceUtils - Data sources version: 1.6.2019124g\n",
            "14:43:35.460 INFO  DataSourceUtils - Data sources source: ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/funcotator/funcotator_dataSources.v1.6.20190124g.tar.gz\n",
            "14:43:35.460 INFO  DataSourceUtils - Data sources alternate source: gs://broad-public-datasets/funcotator/funcotator_dataSources.v1.6.20190124g.tar.gz\n",
            "14:43:35.467 INFO  DataSourceUtils - Resolved data source file path: file:///content/clinvar_20180429_hg38.vcf -> file:///content/funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429_hg38.vcf\n",
            "14:43:35.469 INFO  DataSourceUtils - Resolved data source file path: file:///content/gencode.v27.chr_patch_hapl_scaff.annotation.REORDERED.gtf -> file:///content/funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.chr_patch_hapl_scaff.annotation.REORDERED.gtf\n",
            "14:43:35.469 INFO  DataSourceUtils - Resolved data source file path: file:///content/gencode.v27.transcripts.fasta -> file:///content/funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.transcripts.fasta\n",
            "14:43:35.471 INFO  DataSourceUtils - Resolved data source file path: file:///content/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf -> file:///content/funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf\n",
            "14:43:35.475 INFO  DataSourceUtils - Resolved data source file path: file:///content/acmg59_test_cleaned.txt -> file:///content/funcotator_dataSources.v1.6.20190124g/acmg_rec/hg38/acmg59_test_cleaned.txt\n",
            "14:43:35.477 INFO  DataSourceUtils - Resolved data source file path: file:///content/acmg_lof.tsv -> file:///content/funcotator_dataSources.v1.6.20190124g/acmg_lof/hg38/acmg_lof.tsv\n",
            "14:43:35.477 INFO  Funcotator - Finalizing data sources (this step can be long if data sources are cloud-based)...\n",
            "14:43:35.478 INFO  DataSourceUtils - Resolved data source file path: file:///content/clinvar_20180429_hg38.vcf -> file:///content/funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429_hg38.vcf\n",
            "14:43:35.478 INFO  DataSourceUtils - Setting lookahead cache for data source: ClinVar_VCF : 100000\n",
            "14:43:35.483 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429_hg38.vcf\n",
            "14:43:35.761 INFO  DataSourceUtils - Resolved data source file path: file:///content/clinvar_20180429_hg38.vcf -> file:///content/funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429_hg38.vcf\n",
            "14:43:35.863 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/funcotator_dataSources.v1.6.20190124g/clinvar/hg38/clinvar_20180429_hg38.vcf\n",
            "14:43:36.161 INFO  DataSourceUtils - Resolved data source file path: file:///content/gencode.v27.chr_patch_hapl_scaff.annotation.REORDERED.gtf -> file:///content/funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.chr_patch_hapl_scaff.annotation.REORDERED.gtf\n",
            "14:43:36.161 INFO  DataSourceUtils - Setting lookahead cache for data source: Gencode : 100000\n",
            "14:43:36.165 INFO  FeatureManager - Using codec GencodeGtfCodec to read file file:///content/funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.chr_patch_hapl_scaff.annotation.REORDERED.gtf\n",
            "14:43:36.190 INFO  DataSourceUtils - Resolved data source file path: file:///content/gencode.v27.transcripts.fasta -> file:///content/funcotator_dataSources.v1.6.20190124g/gencode/hg38/gencode.v27.transcripts.fasta\n",
            "14:43:45.182 INFO  DataSourceUtils - Resolved data source file path: file:///content/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf -> file:///content/funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf\n",
            "14:43:45.182 INFO  DataSourceUtils - Setting lookahead cache for data source: LMMKnown : 100000\n",
            "14:43:45.190 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf\n",
            "14:43:45.193 INFO  DataSourceUtils - Resolved data source file path: file:///content/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf -> file:///content/funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf\n",
            "14:43:45.200 INFO  FeatureManager - Using codec VCFCodec to read file file:///content/funcotator_dataSources.v1.6.20190124g/lmm_known/hg38/LMM_Path_LP_VUS5-variants-6-12-18.sorted_liftover_b38.corrected.vcf\n",
            "14:43:45.204 INFO  DataSourceUtils - Resolved data source file path: file:///content/acmg59_test_cleaned.txt -> file:///content/funcotator_dataSources.v1.6.20190124g/acmg_rec/hg38/acmg59_test_cleaned.txt\n",
            "14:43:45.209 INFO  DataSourceUtils - Resolved data source file path: file:///content/acmg_lof.tsv -> file:///content/funcotator_dataSources.v1.6.20190124g/acmg_lof/hg38/acmg_lof.tsv\n",
            "14:43:45.210 INFO  Funcotator - Initializing Funcotator Engine...\n",
            "14:43:45.238 INFO  Funcotator - Creating a VCF file for output: file:/content/HG00096.chr11.exome.annotated.vcf\n",
            "14:43:45.384 INFO  ProgressMeter - Starting traversal\n",
            "14:43:45.384 INFO  ProgressMeter -        Current Locus  Elapsed Minutes    Variants Processed  Variants/Minute\n",
            "14:43:45.718 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1307, ref allele: G)\n",
            "14:43:45.720 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1307, alt allele: A)\n",
            "14:43:45.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=764, ref allele: G)\n",
            "14:43:45.725 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=764, alt allele: A)\n",
            "14:43:45.726 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=386, ref allele: G)\n",
            "14:43:45.726 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=386, alt allele: A)\n",
            "14:43:45.803 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: G)\n",
            "14:43:45.803 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: A)\n",
            "14:43:45.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1475, ref allele: G)\n",
            "14:43:45.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1475, alt allele: A)\n",
            "14:43:45.809 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1235, ref allele: G)\n",
            "14:43:45.809 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1235, alt allele: A)\n",
            "14:43:45.810 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1225, ref allele: G)\n",
            "14:43:45.810 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1225, alt allele: A)\n",
            "14:43:45.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: G)\n",
            "14:43:45.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: T)\n",
            "14:43:45.830 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1475, ref allele: G)\n",
            "14:43:45.830 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1475, alt allele: T)\n",
            "14:43:45.834 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1235, ref allele: G)\n",
            "14:43:45.834 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1235, alt allele: T)\n",
            "14:43:45.835 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1225, ref allele: G)\n",
            "14:43:45.835 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1225, alt allele: T)\n",
            "14:43:45.851 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: C)\n",
            "14:43:45.852 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: T)\n",
            "14:43:45.854 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1235, ref allele: C)\n",
            "14:43:45.854 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1235, alt allele: T)\n",
            "14:43:45.861 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, ref allele: A)\n",
            "14:43:45.862 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, alt allele: G)\n",
            "14:43:45.863 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: A)\n",
            "14:43:45.863 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: G)\n",
            "14:43:45.865 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, ref allele: A)\n",
            "14:43:45.865 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, alt allele: G)\n",
            "14:43:45.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: A)\n",
            "14:43:45.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: G)\n",
            "14:43:45.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: A)\n",
            "14:43:45.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: G)\n",
            "14:43:45.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: G)\n",
            "14:43:45.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: A)\n",
            "14:43:45.946 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: C)\n",
            "14:43:45.946 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: T)\n",
            "14:43:45.958 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: G)\n",
            "14:43:45.959 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: A)\n",
            "14:43:45.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, ref allele: C)\n",
            "14:43:45.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, alt allele: T)\n",
            "14:43:45.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: C)\n",
            "14:43:45.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: T)\n",
            "14:43:45.972 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, ref allele: C)\n",
            "14:43:45.972 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, alt allele: T)\n",
            "14:43:45.980 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, ref allele: T)\n",
            "14:43:45.980 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, alt allele: C)\n",
            "14:43:45.981 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: T)\n",
            "14:43:45.981 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: C)\n",
            "14:43:45.984 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, ref allele: T)\n",
            "14:43:45.985 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, alt allele: C)\n",
            "14:43:45.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, ref allele: T)\n",
            "14:43:45.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, alt allele: C)\n",
            "14:43:45.993 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, ref allele: T)\n",
            "14:43:45.993 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1591, alt allele: C)\n",
            "14:43:45.996 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, ref allele: T)\n",
            "14:43:45.996 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1427, alt allele: C)\n",
            "14:43:46.047 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, ref allele: C)\n",
            "14:43:46.047 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, alt allele: T)\n",
            "14:43:46.058 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, ref allele: A)\n",
            "14:43:46.059 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, alt allele: T)\n",
            "14:43:46.069 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, ref allele: T)\n",
            "14:43:46.070 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, alt allele: C)\n",
            "14:43:46.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, ref allele: G)\n",
            "14:43:46.089 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, alt allele: A)\n",
            "14:43:46.095 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, ref allele: T)\n",
            "14:43:46.095 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, alt allele: C)\n",
            "14:43:46.109 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, ref allele: A)\n",
            "14:43:46.109 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2936, alt allele: G)\n",
            "14:43:46.166 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2963, ref allele: A)\n",
            "14:43:46.167 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2963, alt allele: G)\n",
            "14:43:46.168 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3062, ref allele: A)\n",
            "14:43:46.168 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3062, alt allele: G)\n",
            "14:43:46.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2963, ref allele: T)\n",
            "14:43:46.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2963, alt allele: C)\n",
            "14:43:46.184 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3062, ref allele: T)\n",
            "14:43:46.185 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3062, alt allele: C)\n",
            "14:43:46.192 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=736, ref allele: G)\n",
            "14:43:46.192 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=736, alt allele: C)\n",
            "14:43:46.213 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, ref allele: T)\n",
            "14:43:46.213 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, alt allele: C)\n",
            "14:43:46.214 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, ref allele: T)\n",
            "14:43:46.214 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, alt allele: C)\n",
            "14:43:46.215 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, ref allele: T)\n",
            "14:43:46.215 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, alt allele: C)\n",
            "14:43:46.219 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, ref allele: T)\n",
            "14:43:46.219 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, alt allele: C)\n",
            "14:43:46.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, ref allele: T)\n",
            "14:43:46.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, alt allele: C)\n",
            "14:43:46.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, ref allele: T)\n",
            "14:43:46.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, alt allele: C)\n",
            "14:43:46.225 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, ref allele: G)\n",
            "14:43:46.225 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, alt allele: C)\n",
            "14:43:46.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, ref allele: G)\n",
            "14:43:46.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, alt allele: C)\n",
            "14:43:46.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, ref allele: G)\n",
            "14:43:46.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, alt allele: C)\n",
            "14:43:46.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, ref allele: G)\n",
            "14:43:46.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, alt allele: C)\n",
            "14:43:46.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, ref allele: G)\n",
            "14:43:46.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, alt allele: C)\n",
            "14:43:46.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, ref allele: G)\n",
            "14:43:46.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, alt allele: C)\n",
            "14:43:46.236 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, ref allele: A)\n",
            "14:43:46.236 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, alt allele: G)\n",
            "14:43:46.237 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, ref allele: A)\n",
            "14:43:46.237 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=659, alt allele: G)\n",
            "14:43:46.237 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, ref allele: A)\n",
            "14:43:46.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=605, alt allele: G)\n",
            "14:43:46.242 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=730, ref allele: C)\n",
            "14:43:46.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=730, alt allele: G)\n",
            "14:43:46.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=844, ref allele: C)\n",
            "14:43:46.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=844, alt allele: G)\n",
            "14:43:46.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=808, ref allele: G)\n",
            "14:43:46.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=808, alt allele: A)\n",
            "14:43:46.249 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=713, ref allele: G)\n",
            "14:43:46.249 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=713, alt allele: A)\n",
            "14:43:46.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, ref allele: C)\n",
            "14:43:46.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, alt allele: T)\n",
            "14:43:46.522 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, ref allele: A)\n",
            "14:43:46.522 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, alt allele: G)\n",
            "14:43:46.526 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, ref allele: A)\n",
            "14:43:46.527 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, alt allele: G)\n",
            "14:43:46.538 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, ref allele: G)\n",
            "14:43:46.539 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, alt allele: A)\n",
            "14:43:46.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, ref allele: C)\n",
            "14:43:46.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2809, alt allele: T)\n",
            "14:43:46.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1720, ref allele: C)\n",
            "14:43:46.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1720, alt allele: T)\n",
            "14:43:46.557 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1598, ref allele: C)\n",
            "14:43:46.559 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1598, alt allele: T)\n",
            "14:43:46.560 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: C)\n",
            "14:43:46.560 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: T)\n",
            "14:43:46.562 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, ref allele: C)\n",
            "14:43:46.562 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, alt allele: T)\n",
            "14:43:46.566 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1720, ref allele: T)\n",
            "14:43:46.567 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1720, alt allele: G)\n",
            "14:43:46.567 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1598, ref allele: T)\n",
            "14:43:46.568 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1598, alt allele: G)\n",
            "14:43:46.572 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: T)\n",
            "14:43:46.572 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: G)\n",
            "14:43:46.574 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, ref allele: T)\n",
            "14:43:46.574 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, alt allele: G)\n",
            "14:43:46.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1720, ref allele: A)\n",
            "14:43:46.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1720, alt allele: G)\n",
            "14:43:46.582 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1598, ref allele: A)\n",
            "14:43:46.582 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1598, alt allele: G)\n",
            "14:43:46.584 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: A)\n",
            "14:43:46.585 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: G)\n",
            "14:43:46.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, ref allele: A)\n",
            "14:43:46.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, alt allele: G)\n",
            "14:43:46.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2542, ref allele: T)\n",
            "14:43:46.605 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2542, alt allele: C)\n",
            "14:43:46.612 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2542, ref allele: A)\n",
            "14:43:46.613 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2542, alt allele: G)\n",
            "14:43:46.623 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2542, ref allele: G)\n",
            "14:43:46.630 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2542, alt allele: T)\n",
            "14:43:46.782 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2500, ref allele: C)\n",
            "14:43:46.783 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2500, alt allele: T)\n",
            "14:43:46.894 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, ref allele: C)\n",
            "14:43:46.894 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, alt allele: T)\n",
            "14:43:46.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3266, ref allele: C)\n",
            "14:43:46.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3266, alt allele: T)\n",
            "14:43:46.896 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3188, ref allele: C)\n",
            "14:43:46.897 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3188, alt allele: T)\n",
            "14:43:46.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, ref allele: C)\n",
            "14:43:46.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, alt allele: T)\n",
            "14:43:46.905 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3266, ref allele: C)\n",
            "14:43:46.905 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3266, alt allele: T)\n",
            "14:43:46.908 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3188, ref allele: C)\n",
            "14:43:46.908 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3188, alt allele: T)\n",
            "14:43:47.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4357, ref allele: T)\n",
            "14:43:47.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4357, alt allele: C)\n",
            "14:43:47.184 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, ref allele: T)\n",
            "14:43:47.184 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, alt allele: C)\n",
            "14:43:47.188 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=790, ref allele: C)\n",
            "14:43:47.188 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=790, alt allele: T)\n",
            "14:43:47.189 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4357, ref allele: C)\n",
            "14:43:47.189 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4357, alt allele: T)\n",
            "14:43:47.190 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, ref allele: C)\n",
            "14:43:47.190 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, alt allele: T)\n",
            "14:43:47.195 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2692, ref allele: G)\n",
            "14:43:47.196 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2692, alt allele: A)\n",
            "14:43:47.196 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, ref allele: G)\n",
            "14:43:47.196 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, alt allele: A)\n",
            "14:43:47.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2998, ref allele: A)\n",
            "14:43:47.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2998, alt allele: G)\n",
            "14:43:47.229 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2998, ref allele: G)\n",
            "14:43:47.230 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2998, alt allele: T)\n",
            "14:43:47.237 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2428, ref allele: C)\n",
            "14:43:47.237 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2428, alt allele: T)\n",
            "14:43:47.246 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2428, ref allele: T)\n",
            "14:43:47.246 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2428, alt allele: C)\n",
            "14:43:47.255 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, ref allele: G)\n",
            "14:43:47.255 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, alt allele: A)\n",
            "14:43:47.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, ref allele: G)\n",
            "14:43:47.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, alt allele: A)\n",
            "14:43:47.262 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, ref allele: A)\n",
            "14:43:47.262 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, alt allele: G)\n",
            "14:43:47.262 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, ref allele: A)\n",
            "14:43:47.263 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, alt allele: G)\n",
            "14:43:47.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1648, ref allele: G)\n",
            "14:43:47.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1648, alt allele: A)\n",
            "14:43:47.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1486, ref allele: G)\n",
            "14:43:47.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1486, alt allele: A)\n",
            "14:43:47.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1369, ref allele: G)\n",
            "14:43:47.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1369, alt allele: A)\n",
            "14:43:47.297 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:837662-837663 due to alternate allele: *\n",
            "14:43:47.298 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:837662-837663 due to alternate allele: *\n",
            "14:43:47.298 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:837662-837663 due to alternate allele: *\n",
            "14:43:47.298 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:837662-837663 due to alternate allele: *\n",
            "14:43:47.298 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:837662-837663 due to alternate allele: *\n",
            "14:43:47.422 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, ref allele: T)\n",
            "14:43:47.422 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=892, alt allele: C)\n",
            "14:43:47.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1423, ref allele: T)\n",
            "14:43:47.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1423, alt allele: C)\n",
            "14:43:47.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1025, ref allele: T)\n",
            "14:43:47.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1025, alt allele: C)\n",
            "14:43:47.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1430, ref allele: T)\n",
            "14:43:47.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1430, alt allele: C)\n",
            "14:43:47.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1426, ref allele: T)\n",
            "14:43:47.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1426, alt allele: C)\n",
            "14:43:47.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1031, ref allele: T)\n",
            "14:43:47.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1031, alt allele: C)\n",
            "14:43:47.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1462, ref allele: T)\n",
            "14:43:47.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1462, alt allele: C)\n",
            "14:43:47.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1339, ref allele: T)\n",
            "14:43:47.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1339, alt allele: C)\n",
            "14:43:47.562 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3374, ref allele: C)\n",
            "14:43:47.562 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3374, alt allele: T)\n",
            "14:43:47.563 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1289, ref allele: C)\n",
            "14:43:47.563 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1289, alt allele: T)\n",
            "14:43:47.564 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1477, ref allele: C)\n",
            "14:43:47.564 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1477, alt allele: T)\n",
            "14:43:47.974 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:47.976 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: A)\n",
            "14:43:47.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:47.988 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:47.996 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:47.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.025 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: A)\n",
            "14:43:48.026 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: G)\n",
            "14:43:48.032 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: T)\n",
            "14:43:48.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: A)\n",
            "14:43:48.038 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: G)\n",
            "14:43:48.045 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.046 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.051 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: T)\n",
            "14:43:48.052 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.056 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.057 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: A)\n",
            "14:43:48.062 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.063 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: A)\n",
            "14:43:48.067 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:48.068 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:48.073 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:48.074 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:48.079 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: T)\n",
            "14:43:48.080 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.085 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:48.086 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:48.092 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: T)\n",
            "14:43:48.093 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.102 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: A)\n",
            "14:43:48.103 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: G)\n",
            "14:43:48.107 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:48.108 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:48.112 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.113 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.118 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: A)\n",
            "14:43:48.119 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: G)\n",
            "14:43:48.124 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: T)\n",
            "14:43:48.124 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.129 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:48.130 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:48.135 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: T)\n",
            "14:43:48.136 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.140 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.141 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: A)\n",
            "14:43:48.149 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.150 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: A)\n",
            "14:43:48.159 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.160 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:48.170 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:48.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: G)\n",
            "14:43:48.185 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: C)\n",
            "14:43:48.186 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: T)\n",
            "14:43:48.195 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: A)\n",
            "14:43:48.202 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: A)\n",
            "14:43:48.203 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: C)\n",
            "14:43:48.207 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, ref allele: G)\n",
            "14:43:48.208 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=17911, alt allele: A)\n",
            "14:43:48.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3617, ref allele: G)\n",
            "14:43:48.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3617, alt allele: A)\n",
            "14:43:48.465 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1207, ref allele: C)\n",
            "14:43:48.465 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1207, alt allele: G)\n",
            "14:43:48.467 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1207, ref allele: T)\n",
            "14:43:48.467 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1207, alt allele: C)\n",
            "14:43:48.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, ref allele: T)\n",
            "14:43:48.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, alt allele: C)\n",
            "14:43:48.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, ref allele: T)\n",
            "14:43:48.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, alt allele: C)\n",
            "14:43:48.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, ref allele: T)\n",
            "14:43:48.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, alt allele: C)\n",
            "14:43:48.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, ref allele: T)\n",
            "14:43:48.601 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, alt allele: C)\n",
            "14:43:48.603 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, ref allele: C)\n",
            "14:43:48.603 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, alt allele: G)\n",
            "14:43:48.604 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, ref allele: C)\n",
            "14:43:48.604 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, alt allele: G)\n",
            "14:43:48.608 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, ref allele: C)\n",
            "14:43:48.609 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, alt allele: T)\n",
            "14:43:48.609 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, ref allele: C)\n",
            "14:43:48.609 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1556, alt allele: T)\n",
            "14:43:48.614 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, ref allele: T)\n",
            "14:43:48.614 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, alt allele: C)\n",
            "14:43:48.614 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=745, ref allele: T)\n",
            "14:43:48.614 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=745, alt allele: C)\n",
            "14:43:48.615 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=706, ref allele: T)\n",
            "14:43:48.615 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=706, alt allele: C)\n",
            "14:43:48.996 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1910, ref allele: G)\n",
            "14:43:48.996 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1910, alt allele: A)\n",
            "14:43:49.000 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1898, ref allele: G)\n",
            "14:43:49.001 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1898, alt allele: A)\n",
            "14:43:49.005 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1889, ref allele: G)\n",
            "14:43:49.005 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1889, alt allele: A)\n",
            "14:43:49.010 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1910, ref allele: G)\n",
            "14:43:49.010 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1910, alt allele: A)\n",
            "14:43:49.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1898, ref allele: G)\n",
            "14:43:49.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1898, alt allele: A)\n",
            "14:43:49.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, ref allele: G)\n",
            "14:43:49.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, alt allele: A)\n",
            "14:43:49.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1889, ref allele: G)\n",
            "14:43:49.012 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1889, alt allele: A)\n",
            "14:43:49.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1544, ref allele: C)\n",
            "14:43:49.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1544, alt allele: G)\n",
            "14:43:49.465 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1423, ref allele: C)\n",
            "14:43:49.466 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1423, alt allele: G)\n",
            "14:43:49.471 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1544, ref allele: A)\n",
            "14:43:49.473 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1544, alt allele: C)\n",
            "14:43:49.474 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1423, ref allele: A)\n",
            "14:43:49.474 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1423, alt allele: C)\n",
            "14:43:49.478 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, ref allele: T)\n",
            "14:43:49.479 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, alt allele: G)\n",
            "14:43:49.479 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, ref allele: T)\n",
            "14:43:49.479 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, alt allele: G)\n",
            "14:43:49.480 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, ref allele: T)\n",
            "14:43:49.480 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, alt allele: G)\n",
            "14:43:49.488 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, ref allele: A)\n",
            "14:43:49.488 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, alt allele: G)\n",
            "14:43:49.493 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, ref allele: A)\n",
            "14:43:49.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, alt allele: G)\n",
            "14:43:49.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, ref allele: A)\n",
            "14:43:49.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, alt allele: G)\n",
            "14:43:49.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, ref allele: T)\n",
            "14:43:49.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, alt allele: C)\n",
            "14:43:49.505 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, ref allele: T)\n",
            "14:43:49.505 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, alt allele: C)\n",
            "14:43:49.506 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, ref allele: T)\n",
            "14:43:49.506 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, alt allele: C)\n",
            "14:43:49.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, ref allele: A)\n",
            "14:43:49.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3929, alt allele: G)\n",
            "14:43:49.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, ref allele: A)\n",
            "14:43:49.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3562, alt allele: G)\n",
            "14:43:49.511 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, ref allele: A)\n",
            "14:43:49.511 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3535, alt allele: G)\n",
            "14:43:50.040 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2653, ref allele: G)\n",
            "14:43:50.040 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2653, alt allele: A)\n",
            "14:43:50.040 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3889, ref allele: G)\n",
            "14:43:50.041 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3889, alt allele: A)\n",
            "14:43:50.041 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3619, ref allele: G)\n",
            "14:43:50.041 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3619, alt allele: A)\n",
            "14:43:50.042 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2798, ref allele: G)\n",
            "14:43:50.043 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2798, alt allele: A)\n",
            "14:43:50.071 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3889, ref allele: C)\n",
            "14:43:50.074 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3889, alt allele: T)\n",
            "14:43:50.075 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3619, ref allele: C)\n",
            "14:43:50.075 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3619, alt allele: T)\n",
            "14:43:50.076 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2798, ref allele: C)\n",
            "14:43:50.076 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2798, alt allele: T)\n",
            "14:43:50.150 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1409, ref allele: C)\n",
            "14:43:50.150 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1409, alt allele: T)\n",
            "14:43:50.451 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1504, ref allele: C)\n",
            "14:43:50.451 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1504, alt allele: A)\n",
            "14:43:50.453 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1241, ref allele: C)\n",
            "14:43:50.453 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1241, alt allele: A)\n",
            "14:43:50.455 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1504, ref allele: A)\n",
            "14:43:50.462 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1504, alt allele: G)\n",
            "14:43:50.463 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1154, ref allele: A)\n",
            "14:43:50.463 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1154, alt allele: G)\n",
            "14:43:50.463 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1241, ref allele: A)\n",
            "14:43:50.463 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1241, alt allele: G)\n",
            "14:43:50.465 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1510, ref allele: T)\n",
            "14:43:50.465 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1510, alt allele: CCAT)\n",
            "14:43:50.466 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1160, ref allele: T)\n",
            "14:43:50.466 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1160, alt allele: CCAT)\n",
            "14:43:50.470 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1247, ref allele: T)\n",
            "14:43:50.470 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1247, alt allele: CCAT)\n",
            "14:43:50.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1312, ref allele: T)\n",
            "14:43:50.487 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1312, alt allele: C)\n",
            "14:43:50.495 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6973, ref allele: G)\n",
            "14:43:50.496 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6973, alt allele: A)\n",
            "14:43:50.817 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2080, ref allele: A)\n",
            "14:43:50.818 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2080, alt allele: G)\n",
            "14:43:50.940 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3076, ref allele: C)\n",
            "14:43:50.940 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3076, alt allele: A)\n",
            "14:43:50.940 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, ref allele: C)\n",
            "14:43:50.941 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, alt allele: A)\n",
            "14:43:50.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3076, ref allele: A)\n",
            "14:43:50.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3076, alt allele: G)\n",
            "14:43:50.950 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, ref allele: A)\n",
            "14:43:50.950 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, alt allele: G)\n",
            "14:43:50.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3076, ref allele: G)\n",
            "14:43:50.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3076, alt allele: A)\n",
            "14:43:50.953 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, ref allele: G)\n",
            "14:43:50.953 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, alt allele: A)\n",
            "14:43:51.123 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5456, ref allele: A)\n",
            "14:43:51.123 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5456, alt allele: G)\n",
            "14:43:51.124 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, ref allele: A)\n",
            "14:43:51.124 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, alt allele: G)\n",
            "14:43:51.126 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5456, ref allele: T)\n",
            "14:43:51.126 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5456, alt allele: C)\n",
            "14:43:51.127 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, ref allele: T)\n",
            "14:43:51.127 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, alt allele: C)\n",
            "14:43:51.128 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5456, ref allele: C)\n",
            "14:43:51.129 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5456, alt allele: T)\n",
            "14:43:51.129 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, ref allele: C)\n",
            "14:43:51.129 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, alt allele: T)\n",
            "14:43:51.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, ref allele: A)\n",
            "14:43:51.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, alt allele: C)\n",
            "14:43:51.215 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, ref allele: A)\n",
            "14:43:51.215 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, alt allele: C)\n",
            "14:43:51.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, ref allele: C)\n",
            "14:43:51.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, alt allele: T)\n",
            "14:43:51.219 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, ref allele: C)\n",
            "14:43:51.219 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1115, alt allele: T)\n",
            "14:43:51.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2942, ref allele: G)\n",
            "14:43:51.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2942, alt allele: A)\n",
            "14:43:51.235 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2626, ref allele: G)\n",
            "14:43:51.236 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2626, alt allele: T)\n",
            "14:43:51.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2626, ref allele: G)\n",
            "14:43:51.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2626, alt allele: A)\n",
            "14:43:51.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1136, ref allele: C)\n",
            "14:43:51.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1136, alt allele: A)\n",
            "14:43:51.356 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1750, ref allele: T)\n",
            "14:43:51.357 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1750, alt allele: G)\n",
            "14:43:51.359 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1750, ref allele: T)\n",
            "14:43:51.359 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1750, alt allele: A)\n",
            "14:43:51.363 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2186, ref allele: C)\n",
            "14:43:51.363 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2186, alt allele: T)\n",
            "14:43:51.364 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1048, ref allele: C)\n",
            "14:43:51.364 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1048, alt allele: T)\n",
            "14:43:51.366 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3326, ref allele: C)\n",
            "14:43:51.366 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3326, alt allele: A)\n",
            "14:43:51.369 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, ref allele: C)\n",
            "14:43:51.370 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, alt allele: T)\n",
            "14:43:51.376 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, ref allele: C)\n",
            "14:43:51.377 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, alt allele: G)\n",
            "14:43:51.379 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, ref allele: T)\n",
            "14:43:51.380 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, alt allele: C)\n",
            "14:43:51.384 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, ref allele: G)\n",
            "14:43:51.385 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4393, alt allele: T)\n",
            "14:43:51.450 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7300, ref allele: A)\n",
            "14:43:51.450 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7300, alt allele: G)\n",
            "14:43:51.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: T)\n",
            "14:43:51.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: G)\n",
            "14:43:51.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: C)\n",
            "14:43:51.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: T)\n",
            "14:43:51.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: T)\n",
            "14:43:51.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: C)\n",
            "14:43:51.676 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: C)\n",
            "14:43:51.676 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: A)\n",
            "14:43:51.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: T)\n",
            "14:43:51.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: G)\n",
            "14:43:51.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: G)\n",
            "14:43:51.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: A)\n",
            "14:43:51.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: T)\n",
            "14:43:51.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: C)\n",
            "14:43:51.753 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3302, ref allele: G)\n",
            "14:43:51.753 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3302, alt allele: T)\n",
            "14:43:51.753 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: G)\n",
            "14:43:51.753 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: T)\n",
            "14:43:51.755 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3302, ref allele: G)\n",
            "14:43:51.755 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3302, alt allele: C)\n",
            "14:43:51.756 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: G)\n",
            "14:43:51.756 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: C)\n",
            "14:43:51.758 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3302, ref allele: G)\n",
            "14:43:51.758 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3302, alt allele: A)\n",
            "14:43:51.758 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: G)\n",
            "14:43:51.758 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: A)\n",
            "14:43:51.772 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2407, ref allele: C)\n",
            "14:43:51.773 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2407, alt allele: T)\n",
            "14:43:51.774 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2407, ref allele: C)\n",
            "14:43:51.774 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2407, alt allele: G)\n",
            "14:43:51.783 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, ref allele: T)\n",
            "14:43:51.784 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, alt allele: C)\n",
            "14:43:51.784 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, ref allele: T)\n",
            "14:43:51.784 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, alt allele: C)\n",
            "14:43:51.786 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, ref allele: G)\n",
            "14:43:51.786 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, alt allele: T)\n",
            "14:43:51.786 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, ref allele: G)\n",
            "14:43:51.786 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, alt allele: T)\n",
            "14:43:51.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, ref allele: G)\n",
            "14:43:51.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, alt allele: A)\n",
            "14:43:51.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, ref allele: G)\n",
            "14:43:51.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, alt allele: A)\n",
            "14:43:51.790 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, ref allele: A)\n",
            "14:43:51.790 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, alt allele: G)\n",
            "14:43:51.790 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, ref allele: A)\n",
            "14:43:51.790 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, alt allele: G)\n",
            "14:43:51.792 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, ref allele: G)\n",
            "14:43:51.792 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1061, alt allele: A)\n",
            "14:43:51.792 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, ref allele: G)\n",
            "14:43:51.792 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1057, alt allele: A)\n",
            "14:43:51.906 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3217, ref allele: A)\n",
            "14:43:51.906 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3217, alt allele: G)\n",
            "14:43:51.907 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2590, ref allele: A)\n",
            "14:43:51.907 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2590, alt allele: G)\n",
            "14:43:51.907 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3403, ref allele: A)\n",
            "14:43:51.907 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3403, alt allele: G)\n",
            "14:43:51.908 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3005, ref allele: A)\n",
            "14:43:51.908 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3005, alt allele: G)\n",
            "14:43:51.909 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, ref allele: A)\n",
            "14:43:51.909 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, alt allele: G)\n",
            "14:43:51.911 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3217, ref allele: G)\n",
            "14:43:51.911 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3217, alt allele: A)\n",
            "14:43:51.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2590, ref allele: G)\n",
            "14:43:51.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2590, alt allele: A)\n",
            "14:43:51.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3403, ref allele: G)\n",
            "14:43:51.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3403, alt allele: A)\n",
            "14:43:51.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2951, ref allele: G)\n",
            "14:43:51.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2951, alt allele: A)\n",
            "14:43:51.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1060, ref allele: G)\n",
            "14:43:51.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1060, alt allele: A)\n",
            "14:43:51.914 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3005, ref allele: G)\n",
            "14:43:51.914 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3005, alt allele: A)\n",
            "14:43:51.915 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, ref allele: G)\n",
            "14:43:51.915 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, alt allele: A)\n",
            "14:43:51.919 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3217, ref allele: G)\n",
            "14:43:51.919 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3217, alt allele: A)\n",
            "14:43:51.920 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2590, ref allele: G)\n",
            "14:43:51.920 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2590, alt allele: A)\n",
            "14:43:51.920 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3403, ref allele: G)\n",
            "14:43:51.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3403, alt allele: A)\n",
            "14:43:51.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2951, ref allele: G)\n",
            "14:43:51.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2951, alt allele: A)\n",
            "14:43:51.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1060, ref allele: G)\n",
            "14:43:51.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1060, alt allele: A)\n",
            "14:43:51.922 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3005, ref allele: G)\n",
            "14:43:51.922 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3005, alt allele: A)\n",
            "14:43:51.931 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, ref allele: C)\n",
            "14:43:51.932 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, alt allele: A)\n",
            "14:43:51.933 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, ref allele: C)\n",
            "14:43:51.933 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, alt allele: A)\n",
            "14:43:51.985 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2290, ref allele: A)\n",
            "14:43:51.986 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2290, alt allele: G)\n",
            "14:43:51.986 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3170, ref allele: A)\n",
            "14:43:51.986 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3170, alt allele: G)\n",
            "14:43:51.989 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2290, ref allele: C)\n",
            "14:43:51.989 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2290, alt allele: T)\n",
            "14:43:51.989 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3170, ref allele: C)\n",
            "14:43:51.989 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3170, alt allele: T)\n",
            "14:43:51.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, ref allele: T)\n",
            "14:43:51.993 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, alt allele: C)\n",
            "14:43:51.994 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, ref allele: G)\n",
            "14:43:51.994 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, alt allele: A)\n",
            "14:43:51.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, ref allele: T)\n",
            "14:43:51.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, alt allele: C)\n",
            "14:43:52.015 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, ref allele: G)\n",
            "14:43:52.016 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2989, alt allele: C)\n",
            "14:43:52.018 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, ref allele: T)\n",
            "14:43:52.018 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, alt allele: C)\n",
            "14:43:52.020 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, ref allele: A)\n",
            "14:43:52.020 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2075, alt allele: C)\n",
            "14:43:52.026 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, ref allele: C)\n",
            "14:43:52.026 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, alt allele: T)\n",
            "14:43:52.026 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, ref allele: C)\n",
            "14:43:52.026 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, alt allele: T)\n",
            "14:43:52.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, ref allele: T)\n",
            "14:43:52.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, alt allele: G)\n",
            "14:43:52.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, ref allele: T)\n",
            "14:43:52.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, alt allele: G)\n",
            "14:43:52.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, ref allele: T)\n",
            "14:43:52.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, alt allele: G)\n",
            "14:43:52.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, ref allele: T)\n",
            "14:43:52.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, alt allele: G)\n",
            "14:43:52.032 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, ref allele: A)\n",
            "14:43:52.032 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1276, alt allele: T)\n",
            "14:43:52.032 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, ref allele: A)\n",
            "14:43:52.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, alt allele: T)\n",
            "14:43:52.042 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1034, ref allele: G)\n",
            "14:43:52.042 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1034, alt allele: A)\n",
            "14:43:52.085 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, ref allele: C)\n",
            "14:43:52.086 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, alt allele: T)\n",
            "14:43:52.089 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, ref allele: T)\n",
            "14:43:52.089 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, alt allele: A)\n",
            "14:43:52.091 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, ref allele: C)\n",
            "14:43:52.091 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, alt allele: T)\n",
            "14:43:52.094 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, ref allele: C)\n",
            "14:43:52.094 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, alt allele: A)\n",
            "14:43:52.097 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, ref allele: T)\n",
            "14:43:52.097 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, alt allele: C)\n",
            "14:43:52.099 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, ref allele: C)\n",
            "14:43:52.100 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2545, alt allele: T)\n",
            "14:43:52.115 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: T)\n",
            "14:43:52.115 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: G)\n",
            "14:43:52.117 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: A)\n",
            "14:43:52.117 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: G)\n",
            "14:43:52.120 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: T)\n",
            "14:43:52.120 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: C)\n",
            "14:43:52.122 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: A)\n",
            "14:43:52.122 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: G)\n",
            "14:43:52.125 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: T)\n",
            "14:43:52.125 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: C)\n",
            "14:43:52.127 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: G)\n",
            "14:43:52.127 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: T)\n",
            "14:43:52.130 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: C)\n",
            "14:43:52.130 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: G)\n",
            "14:43:52.132 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: A)\n",
            "14:43:52.132 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: G)\n",
            "14:43:52.134 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, ref allele: C)\n",
            "14:43:52.135 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=970, alt allele: T)\n",
            "14:43:52.140 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, ref allele: G)\n",
            "14:43:52.140 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, alt allele: A)\n",
            "14:43:52.141 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, ref allele: G)\n",
            "14:43:52.141 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, alt allele: A)\n",
            "14:43:52.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, ref allele: G)\n",
            "14:43:52.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, alt allele: T)\n",
            "14:43:52.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, ref allele: G)\n",
            "14:43:52.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, alt allele: T)\n",
            "14:43:52.145 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, ref allele: A)\n",
            "14:43:52.145 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, alt allele: C)\n",
            "14:43:52.146 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, ref allele: A)\n",
            "14:43:52.146 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, alt allele: C)\n",
            "14:43:52.148 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, ref allele: G)\n",
            "14:43:52.148 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2947, alt allele: A)\n",
            "14:43:52.148 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, ref allele: G)\n",
            "14:43:52.148 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1027, alt allele: A)\n",
            "14:43:52.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4595, ref allele: T)\n",
            "14:43:52.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4595, alt allele: C)\n",
            "14:43:52.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4438, ref allele: T)\n",
            "14:43:52.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4438, alt allele: C)\n",
            "14:43:52.207 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, ref allele: G)\n",
            "14:43:52.207 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, alt allele: A)\n",
            "14:43:52.322 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3481, ref allele: C)\n",
            "14:43:52.322 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3481, alt allele: T)\n",
            "14:43:52.326 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3481, ref allele: T)\n",
            "14:43:52.326 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3481, alt allele: C)\n",
            "14:43:52.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, ref allele: C)\n",
            "14:43:52.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, alt allele: A)\n",
            "14:43:52.379 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1174, ref allele: T)\n",
            "14:43:52.379 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1174, alt allele: A)\n",
            "14:43:52.384 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1174, ref allele: T)\n",
            "14:43:52.384 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1174, alt allele: C)\n",
            "14:43:52.386 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1174, ref allele: G)\n",
            "14:43:52.386 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1174, alt allele: C)\n",
            "14:43:52.389 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2452, ref allele: T)\n",
            "14:43:52.389 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2452, alt allele: C)\n",
            "14:43:52.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2188, ref allele: T)\n",
            "14:43:52.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2188, alt allele: C)\n",
            "14:43:52.391 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2452, ref allele: G)\n",
            "14:43:52.391 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2452, alt allele: A)\n",
            "14:43:52.392 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2188, ref allele: G)\n",
            "14:43:52.392 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2188, alt allele: A)\n",
            "14:43:52.452 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, ref allele: G)\n",
            "14:43:52.452 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, alt allele: A)\n",
            "14:43:52.452 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2879, ref allele: G)\n",
            "14:43:52.453 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2879, alt allele: A)\n",
            "14:43:52.453 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2704, ref allele: G)\n",
            "14:43:52.453 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2704, alt allele: A)\n",
            "14:43:52.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2137, ref allele: G)\n",
            "14:43:52.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2137, alt allele: A)\n",
            "14:43:52.457 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, ref allele: C)\n",
            "14:43:52.457 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, alt allele: T)\n",
            "14:43:52.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2879, ref allele: C)\n",
            "14:43:52.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2879, alt allele: T)\n",
            "14:43:52.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2704, ref allele: C)\n",
            "14:43:52.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2704, alt allele: T)\n",
            "14:43:52.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2137, ref allele: C)\n",
            "14:43:52.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2137, alt allele: T)\n",
            "14:43:52.462 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3205, ref allele: A)\n",
            "14:43:52.462 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3205, alt allele: G)\n",
            "14:43:52.517 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6560, ref allele: C)\n",
            "14:43:52.517 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6560, alt allele: T)\n",
            "14:43:52.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, ref allele: G)\n",
            "14:43:52.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, alt allele: A)\n",
            "14:43:52.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, ref allele: G)\n",
            "14:43:52.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, alt allele: A)\n",
            "14:43:52.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, ref allele: G)\n",
            "14:43:52.537 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, alt allele: A)\n",
            "14:43:52.542 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, ref allele: C)\n",
            "14:43:52.543 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10765, alt allele: T)\n",
            "14:43:52.585 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1069, ref allele: C)\n",
            "14:43:52.585 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1069, alt allele: G)\n",
            "14:43:52.588 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1069, ref allele: G)\n",
            "14:43:52.588 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1069, alt allele: C)\n",
            "14:43:52.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1069, ref allele: A)\n",
            "14:43:52.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1069, alt allele: G)\n",
            "14:43:52.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4295, ref allele: T)\n",
            "14:43:52.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4295, alt allele: C)\n",
            "14:43:52.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1373, ref allele: T)\n",
            "14:43:52.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1373, alt allele: C)\n",
            "14:43:52.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4295, ref allele: C)\n",
            "14:43:52.597 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4295, alt allele: T)\n",
            "14:43:52.597 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1373, ref allele: C)\n",
            "14:43:52.597 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1373, alt allele: T)\n",
            "14:43:52.644 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, ref allele: A)\n",
            "14:43:52.649 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, alt allele: G)\n",
            "14:43:52.652 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, ref allele: C)\n",
            "14:43:52.653 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, alt allele: G)\n",
            "14:43:52.655 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, ref allele: A)\n",
            "14:43:52.655 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, alt allele: G)\n",
            "14:43:52.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, ref allele: T)\n",
            "14:43:52.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, alt allele: C)\n",
            "14:43:52.659 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, ref allele: A)\n",
            "14:43:52.660 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5390, alt allele: C)\n",
            "14:43:52.664 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, ref allele: G)\n",
            "14:43:52.664 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, alt allele: A)\n",
            "14:43:52.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: C)\n",
            "14:43:52.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: A)\n",
            "14:43:52.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: A)\n",
            "14:43:52.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: C)\n",
            "14:43:52.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: G)\n",
            "14:43:52.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: C)\n",
            "14:43:52.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: T)\n",
            "14:43:52.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: C)\n",
            "14:43:52.675 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: C)\n",
            "14:43:52.676 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: T)\n",
            "14:43:52.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3448, ref allele: A)\n",
            "14:43:52.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3448, alt allele: G)\n",
            "14:43:52.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, ref allele: A)\n",
            "14:43:52.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, alt allele: G)\n",
            "14:43:52.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1396, ref allele: A)\n",
            "14:43:52.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1396, alt allele: G)\n",
            "14:43:52.733 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3448, ref allele: A)\n",
            "14:43:52.733 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3448, alt allele: T)\n",
            "14:43:52.733 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, ref allele: A)\n",
            "14:43:52.733 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, alt allele: T)\n",
            "14:43:52.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1396, ref allele: A)\n",
            "14:43:52.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1396, alt allele: T)\n",
            "14:43:52.735 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, ref allele: T)\n",
            "14:43:52.735 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, alt allele: C)\n",
            "14:43:52.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, ref allele: T)\n",
            "14:43:52.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, alt allele: G)\n",
            "14:43:52.739 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, ref allele: T)\n",
            "14:43:52.739 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, alt allele: A)\n",
            "14:43:52.742 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, ref allele: A)\n",
            "14:43:52.742 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2681, alt allele: G)\n",
            "14:43:52.745 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: A)\n",
            "14:43:52.745 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: C)\n",
            "14:43:52.746 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: A)\n",
            "14:43:52.746 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: G)\n",
            "14:43:52.747 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: C)\n",
            "14:43:52.747 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: T)\n",
            "14:43:52.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: G)\n",
            "14:43:52.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: A)\n",
            "14:43:52.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: G)\n",
            "14:43:52.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: A)\n",
            "14:43:52.798 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1874, ref allele: A)\n",
            "14:43:52.798 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1874, alt allele: G)\n",
            "14:43:52.841 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3955, ref allele: T)\n",
            "14:43:52.842 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3955, alt allele: C)\n",
            "14:43:52.935 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, ref allele: C)\n",
            "14:43:52.936 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, alt allele: T)\n",
            "14:43:52.937 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, ref allele: C)\n",
            "14:43:52.937 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, alt allele: T)\n",
            "14:43:52.942 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, ref allele: A)\n",
            "14:43:52.942 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, alt allele: G)\n",
            "14:43:52.943 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, ref allele: A)\n",
            "14:43:52.943 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, alt allele: G)\n",
            "14:43:52.948 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, ref allele: G)\n",
            "14:43:52.948 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, alt allele: A)\n",
            "14:43:52.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, ref allele: G)\n",
            "14:43:52.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, alt allele: A)\n",
            "14:43:52.951 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, ref allele: T)\n",
            "14:43:52.951 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, alt allele: C)\n",
            "14:43:52.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, ref allele: T)\n",
            "14:43:52.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, alt allele: C)\n",
            "14:43:52.954 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, ref allele: C)\n",
            "14:43:52.954 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3557, alt allele: G)\n",
            "14:43:52.955 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, ref allele: C)\n",
            "14:43:52.955 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2965, alt allele: G)\n",
            "14:43:52.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1658, ref allele: A)\n",
            "14:43:52.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1658, alt allele: G)\n",
            "14:43:53.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1658, ref allele: A)\n",
            "14:43:53.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1658, alt allele: G)\n",
            "14:43:53.008 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1658, ref allele: G)\n",
            "14:43:53.008 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1658, alt allele: A)\n",
            "14:43:53.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, ref allele: A)\n",
            "14:43:53.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, alt allele: T)\n",
            "14:43:53.011 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1730, ref allele: A)\n",
            "14:43:53.012 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1730, alt allele: T)\n",
            "14:43:53.014 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, ref allele: C)\n",
            "14:43:53.015 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, alt allele: T)\n",
            "14:43:53.015 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1730, ref allele: C)\n",
            "14:43:53.015 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1730, alt allele: T)\n",
            "14:43:53.018 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, ref allele: G)\n",
            "14:43:53.018 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, alt allele: A)\n",
            "14:43:53.018 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1730, ref allele: G)\n",
            "14:43:53.018 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1730, alt allele: A)\n",
            "14:43:53.107 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7540, ref allele: C)\n",
            "14:43:53.107 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7540, alt allele: T)\n",
            "14:43:53.156 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1759, ref allele: T)\n",
            "14:43:53.156 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1759, alt allele: C)\n",
            "14:43:53.200 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1279, ref allele: C)\n",
            "14:43:53.200 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1279, alt allele: G)\n",
            "14:43:53.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=770, ref allele: C)\n",
            "14:43:53.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=770, alt allele: G)\n",
            "14:43:53.265 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3103, ref allele: G)\n",
            "14:43:53.265 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3103, alt allele: A)\n",
            "14:43:53.265 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, ref allele: G)\n",
            "14:43:53.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, alt allele: A)\n",
            "14:43:53.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, ref allele: G)\n",
            "14:43:53.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, alt allele: A)\n",
            "14:43:53.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, ref allele: G)\n",
            "14:43:53.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, alt allele: A)\n",
            "14:43:53.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, ref allele: G)\n",
            "14:43:53.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, alt allele: A)\n",
            "14:43:53.268 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3103, ref allele: T)\n",
            "14:43:53.268 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3103, alt allele: G)\n",
            "14:43:53.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, ref allele: T)\n",
            "14:43:53.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, alt allele: G)\n",
            "14:43:53.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, ref allele: T)\n",
            "14:43:53.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, alt allele: G)\n",
            "14:43:53.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, ref allele: T)\n",
            "14:43:53.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1991, alt allele: G)\n",
            "14:43:53.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, ref allele: T)\n",
            "14:43:53.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, alt allele: G)\n",
            "14:43:53.276 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3103, ref allele: G)\n",
            "14:43:53.276 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3103, alt allele: T)\n",
            "14:43:53.276 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, ref allele: G)\n",
            "14:43:53.276 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, alt allele: T)\n",
            "14:43:53.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, ref allele: G)\n",
            "14:43:53.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, alt allele: T)\n",
            "14:43:53.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, ref allele: G)\n",
            "14:43:53.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, alt allele: T)\n",
            "14:43:53.348 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4916, ref allele: T)\n",
            "14:43:53.348 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4916, alt allele: G)\n",
            "14:43:53.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=406, ref allele: T)\n",
            "14:43:53.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=406, alt allele: G)\n",
            "14:43:53.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=878, ref allele: T)\n",
            "14:43:53.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=878, alt allele: G)\n",
            "14:43:53.350 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=805, ref allele: T)\n",
            "14:43:53.350 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=805, alt allele: G)\n",
            "14:43:53.442 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1421, ref allele: C)\n",
            "14:43:53.442 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1421, alt allele: T)\n",
            "14:43:53.448 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, ref allele: T)\n",
            "14:43:53.448 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, alt allele: C)\n",
            "14:43:53.450 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, ref allele: C)\n",
            "14:43:53.450 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, alt allele: T)\n",
            "14:43:53.452 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, ref allele: C)\n",
            "14:43:53.452 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, alt allele: G)\n",
            "14:43:53.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, ref allele: T)\n",
            "14:43:53.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1685, alt allele: A)\n",
            "14:43:53.546 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4060, ref allele: G)\n",
            "14:43:53.547 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4060, alt allele: C)\n",
            "14:43:53.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6191, ref allele: T)\n",
            "14:43:53.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6191, alt allele: C)\n",
            "14:43:53.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3436, ref allele: A)\n",
            "14:43:53.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3436, alt allele: G)\n",
            "14:43:53.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3994, ref allele: G)\n",
            "14:43:53.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3994, alt allele: A)\n",
            "14:43:53.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3806, ref allele: G)\n",
            "14:43:53.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3806, alt allele: A)\n",
            "14:43:53.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3680, ref allele: G)\n",
            "14:43:53.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3680, alt allele: A)\n",
            "14:43:53.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2753, ref allele: G)\n",
            "14:43:53.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2753, alt allele: A)\n",
            "14:43:53.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3994, ref allele: T)\n",
            "14:43:53.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3994, alt allele: C)\n",
            "14:43:53.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3806, ref allele: T)\n",
            "14:43:53.821 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3806, alt allele: C)\n",
            "14:43:53.821 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3680, ref allele: T)\n",
            "14:43:53.821 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3680, alt allele: C)\n",
            "14:43:53.822 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2753, ref allele: T)\n",
            "14:43:53.822 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2753, alt allele: C)\n",
            "14:43:53.931 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5815, ref allele: C)\n",
            "14:43:53.931 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5815, alt allele: T)\n",
            "14:43:53.932 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6155, ref allele: C)\n",
            "14:43:53.932 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6155, alt allele: T)\n",
            "14:43:53.933 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: C)\n",
            "14:43:53.933 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: T)\n",
            "14:43:53.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2686, ref allele: C)\n",
            "14:43:53.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2686, alt allele: T)\n",
            "14:43:53.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2824, ref allele: C)\n",
            "14:43:53.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2824, alt allele: T)\n",
            "14:43:53.938 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2468, ref allele: G)\n",
            "14:43:53.938 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2468, alt allele: A)\n",
            "14:43:53.939 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: G)\n",
            "14:43:53.939 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: A)\n",
            "14:43:53.939 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2686, ref allele: G)\n",
            "14:43:53.939 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2686, alt allele: A)\n",
            "14:43:53.942 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2468, ref allele: G)\n",
            "14:43:53.942 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2468, alt allele: A)\n",
            "14:43:53.942 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: G)\n",
            "14:43:53.943 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: A)\n",
            "14:43:53.945 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: A)\n",
            "14:43:53.946 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: G)\n",
            "14:43:53.991 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4586, ref allele: C)\n",
            "14:43:53.991 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4586, alt allele: T)\n",
            "14:43:53.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4586, ref allele: C)\n",
            "14:43:53.993 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4586, alt allele: T)\n",
            "14:43:54.001 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4001, ref allele: G)\n",
            "14:43:54.001 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4001, alt allele: A)\n",
            "14:43:54.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3346, ref allele: G)\n",
            "14:43:54.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3346, alt allele: A)\n",
            "14:43:54.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7775, ref allele: G)\n",
            "14:43:54.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7775, alt allele: T)\n",
            "14:43:54.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4525, ref allele: G)\n",
            "14:43:54.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4525, alt allele: T)\n",
            "14:43:54.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7396, ref allele: G)\n",
            "14:43:54.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7396, alt allele: T)\n",
            "14:43:54.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7775, ref allele: G)\n",
            "14:43:54.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7775, alt allele: A)\n",
            "14:43:54.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4525, ref allele: G)\n",
            "14:43:54.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4525, alt allele: A)\n",
            "14:43:54.278 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7396, ref allele: G)\n",
            "14:43:54.278 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7396, alt allele: A)\n",
            "14:43:54.279 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7775, ref allele: T)\n",
            "14:43:54.279 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7775, alt allele: C)\n",
            "14:43:54.279 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4525, ref allele: T)\n",
            "14:43:54.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4525, alt allele: C)\n",
            "14:43:54.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7396, ref allele: T)\n",
            "14:43:54.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7396, alt allele: C)\n",
            "14:43:54.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2650, ref allele: A)\n",
            "14:43:54.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2650, alt allele: G)\n",
            "14:43:54.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2636, ref allele: A)\n",
            "14:43:54.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2636, alt allele: G)\n",
            "14:43:54.324 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, ref allele: C)\n",
            "14:43:54.324 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, alt allele: T)\n",
            "14:43:54.325 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, ref allele: C)\n",
            "14:43:54.326 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, alt allele: T)\n",
            "14:43:54.330 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, ref allele: T)\n",
            "14:43:54.330 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, alt allele: G)\n",
            "14:43:54.331 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, ref allele: T)\n",
            "14:43:54.332 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, alt allele: G)\n",
            "14:43:54.336 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, ref allele: T)\n",
            "14:43:54.336 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, alt allele: G)\n",
            "14:43:54.337 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, ref allele: T)\n",
            "14:43:54.337 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, alt allele: G)\n",
            "14:43:54.350 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, ref allele: G)\n",
            "14:43:54.350 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, alt allele: C)\n",
            "14:43:54.351 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, ref allele: G)\n",
            "14:43:54.352 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6625, alt allele: C)\n",
            "14:43:54.392 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, ref allele: G)\n",
            "14:43:54.392 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, alt allele: C)\n",
            "14:43:54.393 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, ref allele: C)\n",
            "14:43:54.393 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, alt allele: T)\n",
            "14:43:54.394 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, ref allele: G)\n",
            "14:43:54.394 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3044, alt allele: A)\n",
            "14:43:54.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8477, ref allele: C)\n",
            "14:43:54.484 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8477, alt allele: T)\n",
            "14:43:54.572 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=799, ref allele: C)\n",
            "14:43:54.572 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=799, alt allele: A)\n",
            "14:43:54.805 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6076, ref allele: A)\n",
            "14:43:54.805 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6076, alt allele: G)\n",
            "14:43:54.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, ref allele: A)\n",
            "14:43:54.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, alt allele: G)\n",
            "14:43:54.906 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2954, ref allele: T)\n",
            "14:43:54.907 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2954, alt allele: C)\n",
            "14:43:54.907 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2878, ref allele: T)\n",
            "14:43:54.907 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2878, alt allele: C)\n",
            "14:43:54.908 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, ref allele: T)\n",
            "14:43:54.908 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, alt allele: C)\n",
            "14:43:54.908 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1702, ref allele: T)\n",
            "14:43:54.909 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1702, alt allele: C)\n",
            "14:43:54.909 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1576, ref allele: T)\n",
            "14:43:54.909 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1576, alt allele: C)\n",
            "14:43:54.911 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2954, ref allele: C)\n",
            "14:43:54.911 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2954, alt allele: T)\n",
            "14:43:54.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2878, ref allele: C)\n",
            "14:43:54.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2878, alt allele: T)\n",
            "14:43:54.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, ref allele: C)\n",
            "14:43:54.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, alt allele: T)\n",
            "14:43:54.912 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1702, ref allele: C)\n",
            "14:43:54.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1702, alt allele: T)\n",
            "14:43:54.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1576, ref allele: C)\n",
            "14:43:54.913 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1576, alt allele: T)\n",
            "14:43:54.990 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, ref allele: T)\n",
            "14:43:54.990 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2707, alt allele: C)\n",
            "14:43:54.991 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2699, ref allele: T)\n",
            "14:43:54.991 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2699, alt allele: C)\n",
            "14:43:54.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, ref allele: T)\n",
            "14:43:54.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, alt allele: C)\n",
            "14:43:55.129 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4561, ref allele: T)\n",
            "14:43:55.129 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4561, alt allele: C)\n",
            "14:43:55.132 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=565, ref allele: T)\n",
            "14:43:55.132 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=565, alt allele: C)\n",
            "14:43:55.179 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8227, ref allele: A)\n",
            "14:43:55.180 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8227, alt allele: G)\n",
            "14:43:55.258 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1369, ref allele: TACA)\n",
            "14:43:55.258 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1369, alt allele: T)\n",
            "14:43:55.258 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1279, ref allele: TACA)\n",
            "14:43:55.259 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1279, alt allele: T)\n",
            "14:43:55.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2801, ref allele: G)\n",
            "14:43:55.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2801, alt allele: A)\n",
            "14:43:55.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2146, ref allele: G)\n",
            "14:43:55.268 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2146, alt allele: A)\n",
            "14:43:55.271 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2801, ref allele: C)\n",
            "14:43:55.272 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2801, alt allele: G)\n",
            "14:43:55.272 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2146, ref allele: C)\n",
            "14:43:55.273 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2146, alt allele: G)\n",
            "14:43:55.276 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2801, ref allele: A)\n",
            "14:43:55.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2801, alt allele: G)\n",
            "14:43:55.287 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, ref allele: G)\n",
            "14:43:55.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, alt allele: T)\n",
            "14:43:55.296 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, ref allele: G)\n",
            "14:43:55.296 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, alt allele: A)\n",
            "14:43:55.307 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, ref allele: G)\n",
            "14:43:55.307 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, alt allele: A)\n",
            "14:43:55.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, ref allele: C)\n",
            "14:43:55.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, alt allele: T)\n",
            "14:43:55.368 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, ref allele: T)\n",
            "14:43:55.368 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4921, alt allele: C)\n",
            "14:43:55.378 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, ref allele: G)\n",
            "14:43:55.378 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, alt allele: C)\n",
            "14:43:55.379 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2278, ref allele: G)\n",
            "14:43:55.379 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2278, alt allele: C)\n",
            "14:43:55.379 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2159, ref allele: G)\n",
            "14:43:55.379 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2159, alt allele: C)\n",
            "14:43:55.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, ref allele: A)\n",
            "14:43:55.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2237, alt allele: G)\n",
            "14:43:55.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2278, ref allele: A)\n",
            "14:43:55.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2278, alt allele: G)\n",
            "14:43:55.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2159, ref allele: A)\n",
            "14:43:55.391 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2159, alt allele: G)\n",
            "14:43:55.574 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1582, ref allele: A)\n",
            "14:43:55.575 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1582, alt allele: G)\n",
            "14:43:55.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: T)\n",
            "14:43:55.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: C)\n",
            "14:43:55.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, ref allele: T)\n",
            "14:43:55.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, alt allele: C)\n",
            "14:43:55.582 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: A)\n",
            "14:43:55.582 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: G)\n",
            "14:43:55.583 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, ref allele: A)\n",
            "14:43:55.583 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, alt allele: G)\n",
            "14:43:55.584 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: A)\n",
            "14:43:55.584 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: G)\n",
            "14:43:55.584 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, ref allele: A)\n",
            "14:43:55.584 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, alt allele: G)\n",
            "14:43:55.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: T)\n",
            "14:43:55.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: C)\n",
            "14:43:55.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, ref allele: T)\n",
            "14:43:55.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1429, alt allele: C)\n",
            "14:43:55.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1444, ref allele: T)\n",
            "14:43:55.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1444, alt allele: A)\n",
            "14:43:55.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1444, ref allele: T)\n",
            "14:43:55.592 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1444, alt allele: C)\n",
            "14:43:55.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=727, ref allele: G)\n",
            "14:43:55.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=727, alt allele: A)\n",
            "14:43:55.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, ref allele: G)\n",
            "14:43:55.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, alt allele: A)\n",
            "14:43:55.644 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, ref allele: G)\n",
            "14:43:55.644 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, alt allele: A)\n",
            "14:43:55.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, ref allele: G)\n",
            "14:43:55.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, alt allele: A)\n",
            "14:43:55.668 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, ref allele: T)\n",
            "14:43:55.668 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, alt allele: C)\n",
            "14:43:55.668 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1022, ref allele: T)\n",
            "14:43:55.668 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1022, alt allele: C)\n",
            "14:43:55.668 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=323, ref allele: T)\n",
            "14:43:55.668 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=323, alt allele: C)\n",
            "14:43:55.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, ref allele: T)\n",
            "14:43:55.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, alt allele: C)\n",
            "14:43:55.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, ref allele: T)\n",
            "14:43:55.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, alt allele: C)\n",
            "14:43:55.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, ref allele: T)\n",
            "14:43:55.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, alt allele: C)\n",
            "14:43:55.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1022, ref allele: T)\n",
            "14:43:55.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1022, alt allele: C)\n",
            "14:43:55.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=323, ref allele: T)\n",
            "14:43:55.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=323, alt allele: C)\n",
            "14:43:55.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, ref allele: T)\n",
            "14:43:55.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, alt allele: C)\n",
            "14:43:55.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, ref allele: T)\n",
            "14:43:55.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, alt allele: C)\n",
            "14:43:55.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, ref allele: C)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=743, alt allele: T)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1022, ref allele: C)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1022, alt allele: T)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=323, ref allele: C)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=323, alt allele: T)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, ref allele: C)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=563, alt allele: T)\n",
            "14:43:55.674 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, ref allele: C)\n",
            "14:43:55.675 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=610, alt allele: T)\n",
            "14:43:55.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=583, ref allele: C)\n",
            "14:43:55.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=583, alt allele: G)\n",
            "14:43:55.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=583, ref allele: T)\n",
            "14:43:55.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=583, alt allele: C)\n",
            "14:43:55.743 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4147, ref allele: C)\n",
            "14:43:55.743 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4147, alt allele: A)\n",
            "14:43:55.749 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4147, ref allele: C)\n",
            "14:43:55.750 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4147, alt allele: A)\n",
            "14:43:55.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2416, ref allele: C)\n",
            "14:43:55.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2416, alt allele: T)\n",
            "14:43:55.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1310, ref allele: C)\n",
            "14:43:55.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1310, alt allele: T)\n",
            "14:43:55.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1766, ref allele: C)\n",
            "14:43:55.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1766, alt allele: T)\n",
            "14:43:55.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2305, ref allele: C)\n",
            "14:43:55.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2305, alt allele: T)\n",
            "14:43:55.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1922, ref allele: C)\n",
            "14:43:55.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1922, alt allele: T)\n",
            "14:43:55.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1288, ref allele: C)\n",
            "14:43:55.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1288, alt allele: T)\n",
            "14:43:55.810 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2416, ref allele: A)\n",
            "14:43:55.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2416, alt allele: G)\n",
            "14:43:55.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1310, ref allele: A)\n",
            "14:43:55.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1310, alt allele: G)\n",
            "14:43:55.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1766, ref allele: A)\n",
            "14:43:55.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1766, alt allele: G)\n",
            "14:43:55.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2305, ref allele: A)\n",
            "14:43:55.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2305, alt allele: G)\n",
            "14:43:55.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1922, ref allele: A)\n",
            "14:43:55.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1922, alt allele: G)\n",
            "14:43:55.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1288, ref allele: A)\n",
            "14:43:55.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1288, alt allele: G)\n",
            "14:43:55.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2041, ref allele: T)\n",
            "14:43:55.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2041, alt allele: C)\n",
            "14:43:55.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1256, ref allele: T)\n",
            "14:43:55.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1256, alt allele: C)\n",
            "14:43:55.832 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2101, ref allele: C)\n",
            "14:43:55.832 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2101, alt allele: T)\n",
            "14:43:55.892 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4156, ref allele: G)\n",
            "14:43:55.892 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4156, alt allele: C)\n",
            "14:43:55.894 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4156, ref allele: A)\n",
            "14:43:55.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4156, alt allele: G)\n",
            "14:43:55.901 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4156, ref allele: A)\n",
            "14:43:55.901 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4156, alt allele: C)\n",
            "14:43:55.901 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2503, ref allele: A)\n",
            "14:43:55.901 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2503, alt allele: C)\n",
            "14:43:55.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1799, ref allele: A)\n",
            "14:43:55.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1799, alt allele: C)\n",
            "14:43:55.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=755, ref allele: A)\n",
            "14:43:55.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=755, alt allele: C)\n",
            "14:43:55.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, ref allele: A)\n",
            "14:43:55.903 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=995, alt allele: C)\n",
            "14:43:56.063 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3871, ref allele: C)\n",
            "14:43:56.063 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3871, alt allele: G)\n",
            "14:43:56.139 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1190, ref allele: G)\n",
            "14:43:56.139 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1190, alt allele: A)\n",
            "14:43:56.140 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1190, ref allele: G)\n",
            "14:43:56.140 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1190, alt allele: A)\n",
            "14:43:56.453 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: A)\n",
            "14:43:56.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: G)\n",
            "14:43:56.455 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: A)\n",
            "14:43:56.455 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: G)\n",
            "14:43:56.456 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: A)\n",
            "14:43:56.456 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: G)\n",
            "14:43:56.457 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: A)\n",
            "14:43:56.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: G)\n",
            "14:43:56.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: A)\n",
            "14:43:56.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: G)\n",
            "14:43:56.461 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: A)\n",
            "14:43:56.462 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: G)\n",
            "14:43:56.464 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: T)\n",
            "14:43:56.464 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: C)\n",
            "14:43:56.465 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: T)\n",
            "14:43:56.465 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: C)\n",
            "14:43:56.466 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: T)\n",
            "14:43:56.467 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: C)\n",
            "14:43:56.467 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: T)\n",
            "14:43:56.468 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: C)\n",
            "14:43:56.469 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: T)\n",
            "14:43:56.469 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: C)\n",
            "14:43:56.470 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: T)\n",
            "14:43:56.470 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: C)\n",
            "14:43:56.472 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: T)\n",
            "14:43:56.473 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: C)\n",
            "14:43:56.473 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: T)\n",
            "14:43:56.474 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: C)\n",
            "14:43:56.475 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: T)\n",
            "14:43:56.475 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: C)\n",
            "14:43:56.476 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: T)\n",
            "14:43:56.476 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: C)\n",
            "14:43:56.477 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: T)\n",
            "14:43:56.478 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: C)\n",
            "14:43:56.478 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: T)\n",
            "14:43:56.479 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: C)\n",
            "14:43:56.481 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: C)\n",
            "14:43:56.481 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: T)\n",
            "14:43:56.482 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: C)\n",
            "14:43:56.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: T)\n",
            "14:43:56.484 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: C)\n",
            "14:43:56.484 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: T)\n",
            "14:43:56.485 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: C)\n",
            "14:43:56.485 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: T)\n",
            "14:43:56.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: C)\n",
            "14:43:56.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: T)\n",
            "14:43:56.487 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: C)\n",
            "14:43:56.488 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: T)\n",
            "14:43:56.555 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: G)\n",
            "14:43:56.555 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: T)\n",
            "14:43:56.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: G)\n",
            "14:43:56.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: T)\n",
            "14:43:56.557 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: G)\n",
            "14:43:56.557 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: T)\n",
            "14:43:56.558 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: G)\n",
            "14:43:56.558 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: T)\n",
            "14:43:56.559 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: G)\n",
            "14:43:56.560 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: T)\n",
            "14:43:56.562 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: G)\n",
            "14:43:56.562 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: T)\n",
            "14:43:56.563 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: G)\n",
            "14:43:56.563 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: T)\n",
            "14:43:56.566 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: C)\n",
            "14:43:56.566 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: G)\n",
            "14:43:56.567 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: C)\n",
            "14:43:56.567 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: G)\n",
            "14:43:56.568 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: C)\n",
            "14:43:56.568 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: G)\n",
            "14:43:56.569 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: C)\n",
            "14:43:56.570 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: G)\n",
            "14:43:56.570 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: C)\n",
            "14:43:56.571 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: G)\n",
            "14:43:56.572 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: C)\n",
            "14:43:56.572 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: G)\n",
            "14:43:56.572 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: C)\n",
            "14:43:56.573 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: G)\n",
            "14:43:56.575 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: C)\n",
            "14:43:56.575 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: T)\n",
            "14:43:56.576 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: C)\n",
            "14:43:56.576 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: T)\n",
            "14:43:56.577 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: C)\n",
            "14:43:56.578 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: T)\n",
            "14:43:56.579 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: C)\n",
            "14:43:56.579 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: T)\n",
            "14:43:56.580 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: C)\n",
            "14:43:56.580 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: T)\n",
            "14:43:56.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: C)\n",
            "14:43:56.582 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: T)\n",
            "14:43:56.582 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: C)\n",
            "14:43:56.582 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: T)\n",
            "14:43:56.589 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: A)\n",
            "14:43:56.589 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: T)\n",
            "14:43:56.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: A)\n",
            "14:43:56.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: T)\n",
            "14:43:56.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: A)\n",
            "14:43:56.592 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: T)\n",
            "14:43:56.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: A)\n",
            "14:43:56.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: T)\n",
            "14:43:56.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: A)\n",
            "14:43:56.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: T)\n",
            "14:43:56.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: A)\n",
            "14:43:56.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: T)\n",
            "14:43:56.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: A)\n",
            "14:43:56.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: T)\n",
            "14:43:56.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: C)\n",
            "14:43:56.601 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: T)\n",
            "14:43:56.602 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: C)\n",
            "14:43:56.602 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: T)\n",
            "14:43:56.603 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: C)\n",
            "14:43:56.604 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: T)\n",
            "14:43:56.604 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: C)\n",
            "14:43:56.605 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: T)\n",
            "14:43:56.606 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: C)\n",
            "14:43:56.606 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: T)\n",
            "14:43:56.607 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: C)\n",
            "14:43:56.607 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: T)\n",
            "14:43:56.608 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: C)\n",
            "14:43:56.608 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: T)\n",
            "14:43:56.619 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: C)\n",
            "14:43:56.619 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: T)\n",
            "14:43:56.620 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: C)\n",
            "14:43:56.620 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: T)\n",
            "14:43:56.621 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: C)\n",
            "14:43:56.622 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: T)\n",
            "14:43:56.623 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: C)\n",
            "14:43:56.623 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: T)\n",
            "14:43:56.624 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: C)\n",
            "14:43:56.624 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: T)\n",
            "14:43:56.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: C)\n",
            "14:43:56.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: T)\n",
            "14:43:56.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: C)\n",
            "14:43:56.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: T)\n",
            "14:43:56.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: C)\n",
            "14:43:56.629 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: A)\n",
            "14:43:56.629 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: C)\n",
            "14:43:56.630 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: A)\n",
            "14:43:56.630 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: C)\n",
            "14:43:56.631 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: A)\n",
            "14:43:56.632 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: C)\n",
            "14:43:56.632 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: A)\n",
            "14:43:56.633 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: C)\n",
            "14:43:56.633 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: A)\n",
            "14:43:56.634 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: C)\n",
            "14:43:56.634 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: A)\n",
            "14:43:56.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: C)\n",
            "14:43:56.675 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: A)\n",
            "14:43:56.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: G)\n",
            "14:43:56.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: A)\n",
            "14:43:56.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: G)\n",
            "14:43:56.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: A)\n",
            "14:43:56.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: G)\n",
            "14:43:56.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: A)\n",
            "14:43:56.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: G)\n",
            "14:43:56.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: A)\n",
            "14:43:56.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: G)\n",
            "14:43:56.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: A)\n",
            "14:43:56.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: G)\n",
            "14:43:56.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: A)\n",
            "14:43:56.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: G)\n",
            "14:43:56.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: A)\n",
            "14:43:56.693 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: A)\n",
            "14:43:56.693 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: C)\n",
            "14:43:56.694 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: A)\n",
            "14:43:56.694 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: C)\n",
            "14:43:56.695 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: A)\n",
            "14:43:56.695 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: C)\n",
            "14:43:56.696 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: A)\n",
            "14:43:56.696 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: C)\n",
            "14:43:56.697 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: A)\n",
            "14:43:56.698 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: C)\n",
            "14:43:56.699 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: A)\n",
            "14:43:56.699 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: C)\n",
            "14:43:56.700 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: A)\n",
            "14:43:56.700 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: C)\n",
            "14:43:56.702 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: G)\n",
            "14:43:56.702 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: A)\n",
            "14:43:56.703 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: G)\n",
            "14:43:56.703 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: A)\n",
            "14:43:56.704 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: G)\n",
            "14:43:56.704 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: A)\n",
            "14:43:56.705 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: G)\n",
            "14:43:56.705 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: A)\n",
            "14:43:56.706 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: G)\n",
            "14:43:56.706 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: A)\n",
            "14:43:56.707 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: G)\n",
            "14:43:56.707 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: A)\n",
            "14:43:56.708 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: G)\n",
            "14:43:56.708 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: A)\n",
            "14:43:56.715 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, ref allele: A)\n",
            "14:43:56.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10667, alt allele: C)\n",
            "14:43:56.717 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, ref allele: A)\n",
            "14:43:56.717 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11000, alt allele: C)\n",
            "14:43:56.718 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, ref allele: A)\n",
            "14:43:56.718 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11003, alt allele: C)\n",
            "14:43:56.719 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, ref allele: A)\n",
            "14:43:56.719 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11171, alt allele: C)\n",
            "14:43:56.720 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, ref allele: A)\n",
            "14:43:56.720 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10960, alt allele: C)\n",
            "14:43:56.721 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, ref allele: A)\n",
            "14:43:56.721 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7882, alt allele: C)\n",
            "14:43:56.722 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, ref allele: A)\n",
            "14:43:56.722 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5084, alt allele: C)\n",
            "14:43:56.858 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=886, ref allele: T)\n",
            "14:43:56.858 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=886, alt allele: G)\n",
            "14:43:56.858 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=967, ref allele: T)\n",
            "14:43:56.858 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=967, alt allele: G)\n",
            "14:43:56.858 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1477, ref allele: T)\n",
            "14:43:56.858 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1477, alt allele: G)\n",
            "14:43:56.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1748, ref allele: T)\n",
            "14:43:56.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1748, alt allele: G)\n",
            "14:43:56.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1060, ref allele: T)\n",
            "14:43:56.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1060, alt allele: G)\n",
            "14:43:56.926 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, ref allele: C)\n",
            "14:43:56.926 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, alt allele: A)\n",
            "14:43:56.927 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, ref allele: C)\n",
            "14:43:56.927 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, alt allele: G)\n",
            "14:43:56.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, ref allele: A)\n",
            "14:43:56.929 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, alt allele: G)\n",
            "14:43:56.931 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, ref allele: G)\n",
            "14:43:56.931 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, alt allele: C)\n",
            "14:43:56.933 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, ref allele: G)\n",
            "14:43:56.933 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7084, alt allele: A)\n",
            "14:43:56.983 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, ref allele: G)\n",
            "14:43:56.983 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, alt allele: A)\n",
            "14:43:56.983 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, ref allele: G)\n",
            "14:43:56.983 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, alt allele: A)\n",
            "14:43:56.984 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, ref allele: G)\n",
            "14:43:56.984 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, alt allele: A)\n",
            "14:43:56.986 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, ref allele: C)\n",
            "14:43:56.986 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, alt allele: T)\n",
            "14:43:56.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, ref allele: C)\n",
            "14:43:56.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, alt allele: T)\n",
            "14:43:56.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, ref allele: C)\n",
            "14:43:56.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, alt allele: T)\n",
            "14:43:57.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, ref allele: C)\n",
            "14:43:57.040 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, alt allele: T)\n",
            "14:43:57.041 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, ref allele: C)\n",
            "14:43:57.041 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, alt allele: T)\n",
            "14:43:57.041 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, ref allele: C)\n",
            "14:43:57.041 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, alt allele: T)\n",
            "14:43:57.054 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, ref allele: C)\n",
            "14:43:57.054 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3094, alt allele: T)\n",
            "14:43:57.054 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, ref allele: C)\n",
            "14:43:57.055 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, alt allele: T)\n",
            "14:43:57.055 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, ref allele: C)\n",
            "14:43:57.055 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3022, alt allele: T)\n",
            "14:43:57.173 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3553, ref allele: G)\n",
            "14:43:57.173 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3553, alt allele: C)\n",
            "14:43:57.173 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4981, ref allele: G)\n",
            "14:43:57.173 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4981, alt allele: C)\n",
            "14:43:57.206 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, ref allele: A)\n",
            "14:43:57.206 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, alt allele: G)\n",
            "14:43:57.213 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:26553236-26553236 due to alternate allele: *\n",
            "14:43:57.213 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:26553236-26553236 due to alternate allele: *\n",
            "14:43:57.213 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:26553236-26553236 due to alternate allele: *\n",
            "14:43:57.223 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3118, ref allele: C)\n",
            "14:43:57.224 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3118, alt allele: T)\n",
            "14:43:57.224 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2968, ref allele: C)\n",
            "14:43:57.224 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2968, alt allele: T)\n",
            "14:43:57.224 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2084, ref allele: C)\n",
            "14:43:57.224 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2084, alt allele: T)\n",
            "14:43:57.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3118, ref allele: A)\n",
            "14:43:57.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3118, alt allele: G)\n",
            "14:43:57.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2968, ref allele: A)\n",
            "14:43:57.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2968, alt allele: G)\n",
            "14:43:57.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2084, ref allele: A)\n",
            "14:43:57.226 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2084, alt allele: G)\n",
            "14:43:57.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3118, ref allele: A)\n",
            "14:43:57.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3118, alt allele: G)\n",
            "14:43:57.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2968, ref allele: A)\n",
            "14:43:57.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2968, alt allele: G)\n",
            "14:43:57.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2084, ref allele: A)\n",
            "14:43:57.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2084, alt allele: G)\n",
            "14:43:57.236 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, ref allele: C)\n",
            "14:43:57.236 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, alt allele: T)\n",
            "14:43:57.242 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, ref allele: C)\n",
            "14:43:57.242 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3257, alt allele: T)\n",
            "14:43:57.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: A)\n",
            "14:43:57.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: G)\n",
            "14:43:57.253 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: C)\n",
            "14:43:57.254 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: T)\n",
            "14:43:57.257 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: T)\n",
            "14:43:57.257 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: C)\n",
            "14:43:57.258 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: G)\n",
            "14:43:57.258 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: A)\n",
            "14:43:57.259 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: T)\n",
            "14:43:57.259 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: C)\n",
            "14:43:57.263 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: G)\n",
            "14:43:57.263 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: T)\n",
            "14:43:57.264 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2269, ref allele: G)\n",
            "14:43:57.264 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2269, alt allele: T)\n",
            "14:43:57.268 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: C)\n",
            "14:43:57.268 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: T)\n",
            "14:43:57.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2269, ref allele: C)\n",
            "14:43:57.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2269, alt allele: T)\n",
            "14:43:57.274 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, ref allele: G)\n",
            "14:43:57.274 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6250, alt allele: A)\n",
            "14:43:57.275 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2269, ref allele: G)\n",
            "14:43:57.275 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2269, alt allele: A)\n",
            "14:43:57.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1886, ref allele: C)\n",
            "14:43:57.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1886, alt allele: T)\n",
            "14:43:57.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, ref allele: C)\n",
            "14:43:57.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, alt allele: T)\n",
            "14:43:57.289 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1886, ref allele: G)\n",
            "14:43:57.289 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1886, alt allele: A)\n",
            "14:43:57.289 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, ref allele: G)\n",
            "14:43:57.289 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, alt allele: A)\n",
            "14:43:57.298 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5206, ref allele: A)\n",
            "14:43:57.298 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5206, alt allele: G)\n",
            "14:43:57.305 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5206, ref allele: C)\n",
            "14:43:57.305 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5206, alt allele: T)\n",
            "14:43:57.366 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3568, ref allele: A)\n",
            "14:43:57.367 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3568, alt allele: G)\n",
            "14:43:57.368 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3568, ref allele: A)\n",
            "14:43:57.369 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3568, alt allele: G)\n",
            "14:43:57.370 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4304, ref allele: C)\n",
            "14:43:57.370 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4304, alt allele: A)\n",
            "14:43:57.371 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=718, ref allele: C)\n",
            "14:43:57.371 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=718, alt allele: A)\n",
            "14:43:57.444 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4172, ref allele: C)\n",
            "14:43:57.444 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4172, alt allele: T)\n",
            "14:43:57.547 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2572, ref allele: T)\n",
            "14:43:57.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2572, alt allele: C)\n",
            "14:43:57.553 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2572, ref allele: C)\n",
            "14:43:57.553 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2572, alt allele: T)\n",
            "14:43:57.589 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3122, ref allele: A)\n",
            "14:43:57.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3122, alt allele: G)\n",
            "14:43:57.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2768, ref allele: A)\n",
            "14:43:57.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2768, alt allele: G)\n",
            "14:43:57.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2114, ref allele: A)\n",
            "14:43:57.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2114, alt allele: G)\n",
            "14:43:57.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5530, ref allele: G)\n",
            "14:43:57.717 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5530, alt allele: A)\n",
            "14:43:57.718 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9335, ref allele: G)\n",
            "14:43:57.719 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9335, alt allele: A)\n",
            "14:43:57.720 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9335, ref allele: A)\n",
            "14:43:57.720 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9335, alt allele: G)\n",
            "14:43:57.721 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9335, ref allele: A)\n",
            "14:43:57.721 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9335, alt allele: G)\n",
            "14:43:57.858 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3668, ref allele: G)\n",
            "14:43:57.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3668, alt allele: T)\n",
            "14:43:57.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7408, ref allele: G)\n",
            "14:43:57.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7408, alt allele: T)\n",
            "14:43:57.859 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7345, ref allele: G)\n",
            "14:43:57.860 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7345, alt allele: T)\n",
            "14:43:57.860 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7399, ref allele: G)\n",
            "14:43:57.860 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7399, alt allele: T)\n",
            "14:43:57.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, ref allele: T)\n",
            "14:43:57.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, alt allele: C)\n",
            "14:43:57.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4492, ref allele: T)\n",
            "14:43:57.922 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4492, alt allele: C)\n",
            "14:43:57.923 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, ref allele: C)\n",
            "14:43:57.923 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, alt allele: T)\n",
            "14:43:57.924 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4492, ref allele: C)\n",
            "14:43:57.924 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4492, alt allele: T)\n",
            "14:43:57.926 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, ref allele: G)\n",
            "14:43:57.927 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, alt allele: A)\n",
            "14:43:57.927 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4492, ref allele: G)\n",
            "14:43:57.927 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4492, alt allele: A)\n",
            "14:43:57.930 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, ref allele: G)\n",
            "14:43:57.931 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, alt allele: A)\n",
            "14:43:57.932 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, ref allele: A)\n",
            "14:43:57.932 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, alt allele: C)\n",
            "14:43:57.962 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, ref allele: A)\n",
            "14:43:57.962 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, alt allele: G)\n",
            "14:43:57.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, ref allele: C)\n",
            "14:43:57.966 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=11678, alt allele: T)\n",
            "14:43:58.044 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1687, ref allele: T)\n",
            "14:43:58.044 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1687, alt allele: C)\n",
            "14:43:58.044 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2294, ref allele: T)\n",
            "14:43:58.044 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2294, alt allele: C)\n",
            "14:43:58.084 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3436, ref allele: T)\n",
            "14:43:58.084 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3436, alt allele: C)\n",
            "14:43:58.085 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, ref allele: T)\n",
            "14:43:58.085 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, alt allele: C)\n",
            "14:43:58.090 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3436, ref allele: C)\n",
            "14:43:58.090 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3436, alt allele: T)\n",
            "14:43:58.090 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, ref allele: C)\n",
            "14:43:58.091 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3329, alt allele: T)\n",
            "14:43:58.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2321, ref allele: A)\n",
            "14:43:58.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2321, alt allele: G)\n",
            "14:43:58.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2450, ref allele: A)\n",
            "14:43:58.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2450, alt allele: G)\n",
            "14:43:58.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=637, ref allele: A)\n",
            "14:43:58.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=637, alt allele: G)\n",
            "14:43:58.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2389, ref allele: C)\n",
            "14:43:58.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2389, alt allele: G)\n",
            "14:43:58.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=952, ref allele: C)\n",
            "14:43:58.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=952, alt allele: G)\n",
            "14:43:58.328 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4589, ref allele: C)\n",
            "14:43:58.328 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4589, alt allele: T)\n",
            "14:43:58.329 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2369, ref allele: C)\n",
            "14:43:58.329 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2369, alt allele: T)\n",
            "14:43:58.329 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3046, ref allele: C)\n",
            "14:43:58.329 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3046, alt allele: T)\n",
            "14:43:58.330 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1634, ref allele: C)\n",
            "14:43:58.330 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1634, alt allele: T)\n",
            "14:43:58.334 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2369, ref allele: A)\n",
            "14:43:58.334 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2369, alt allele: G)\n",
            "14:43:58.334 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3046, ref allele: A)\n",
            "14:43:58.334 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3046, alt allele: G)\n",
            "14:43:58.336 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2369, ref allele: T)\n",
            "14:43:58.336 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2369, alt allele: C)\n",
            "14:43:58.336 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3046, ref allele: T)\n",
            "14:43:58.336 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3046, alt allele: C)\n",
            "14:43:58.473 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1879, ref allele: A)\n",
            "14:43:58.473 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1879, alt allele: G)\n",
            "14:43:58.474 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1996, ref allele: A)\n",
            "14:43:58.474 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1996, alt allele: G)\n",
            "14:43:58.516 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2291, ref allele: C)\n",
            "14:43:58.516 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2291, alt allele: T)\n",
            "14:43:58.516 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=740, ref allele: C)\n",
            "14:43:58.516 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=740, alt allele: T)\n",
            "14:43:58.517 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2291, ref allele: G)\n",
            "14:43:58.517 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2291, alt allele: A)\n",
            "14:43:58.518 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=740, ref allele: G)\n",
            "14:43:58.518 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=740, alt allele: A)\n",
            "14:43:58.526 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2291, ref allele: T)\n",
            "14:43:58.526 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2291, alt allele: C)\n",
            "14:43:58.694 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:43391608-43391612 due to alternate allele: *\n",
            "14:43:58.694 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:43391608-43391612 due to alternate allele: *\n",
            "14:43:58.694 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:43391608-43391612 due to alternate allele: *\n",
            "14:43:58.739 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=866, ref allele: C)\n",
            "14:43:58.739 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=866, alt allele: T)\n",
            "14:43:58.743 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1474, ref allele: T)\n",
            "14:43:58.743 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1474, alt allele: C)\n",
            "14:43:58.754 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1474, ref allele: C)\n",
            "14:43:58.754 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1474, alt allele: G)\n",
            "14:43:58.756 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1474, ref allele: G)\n",
            "14:43:58.756 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1474, alt allele: T)\n",
            "14:43:58.772 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1820, ref allele: G)\n",
            "14:43:58.772 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1820, alt allele: A)\n",
            "14:43:58.774 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1820, ref allele: T)\n",
            "14:43:58.775 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1820, alt allele: C)\n",
            "14:43:58.791 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, ref allele: C)\n",
            "14:43:58.791 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, alt allele: T)\n",
            "14:43:58.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, ref allele: C)\n",
            "14:43:58.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, alt allele: T)\n",
            "14:43:58.796 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, ref allele: A)\n",
            "14:43:58.797 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, alt allele: G)\n",
            "14:43:58.806 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, ref allele: C)\n",
            "14:43:58.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, alt allele: T)\n",
            "14:43:58.809 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, ref allele: G)\n",
            "14:43:58.810 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5750, alt allele: C)\n",
            "14:43:58.844 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=967, ref allele: G)\n",
            "14:43:58.844 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=967, alt allele: A)\n",
            "14:43:58.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=967, ref allele: A)\n",
            "14:43:58.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=967, alt allele: G)\n",
            "14:43:58.869 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3316, ref allele: G)\n",
            "14:43:58.869 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3316, alt allele: A)\n",
            "14:43:58.870 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4229, ref allele: G)\n",
            "14:43:58.870 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4229, alt allele: A)\n",
            "14:43:58.985 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: T)\n",
            "14:43:58.985 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: C)\n",
            "14:43:58.988 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: G)\n",
            "14:43:58.988 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: A)\n",
            "14:43:58.993 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2459, ref allele: C)\n",
            "14:43:58.993 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2459, alt allele: T)\n",
            "14:43:58.994 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2528, ref allele: C)\n",
            "14:43:58.994 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2528, alt allele: T)\n",
            "14:43:58.994 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2516, ref allele: C)\n",
            "14:43:58.994 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2516, alt allele: T)\n",
            "14:43:58.995 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, ref allele: C)\n",
            "14:43:58.995 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, alt allele: T)\n",
            "14:43:59.087 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, ref allele: G)\n",
            "14:43:59.087 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, alt allele: A)\n",
            "14:43:59.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=985, ref allele: G)\n",
            "14:43:59.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=985, alt allele: A)\n",
            "14:43:59.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=635, ref allele: G)\n",
            "14:43:59.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=635, alt allele: A)\n",
            "14:43:59.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=686, ref allele: G)\n",
            "14:43:59.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=686, alt allele: A)\n",
            "14:43:59.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2566, ref allele: T)\n",
            "14:43:59.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2566, alt allele: C)\n",
            "14:43:59.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2030, ref allele: T)\n",
            "14:43:59.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2030, alt allele: C)\n",
            "14:43:59.177 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4199, ref allele: T)\n",
            "14:43:59.177 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4199, alt allele: C)\n",
            "14:43:59.178 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4348, ref allele: T)\n",
            "14:43:59.178 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4348, alt allele: C)\n",
            "14:43:59.178 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2351, ref allele: T)\n",
            "14:43:59.178 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2351, alt allele: C)\n",
            "14:43:59.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7121, ref allele: C)\n",
            "14:43:59.239 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7121, alt allele: T)\n",
            "14:43:59.239 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6532, ref allele: C)\n",
            "14:43:59.239 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6532, alt allele: T)\n",
            "14:43:59.240 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6428, ref allele: C)\n",
            "14:43:59.240 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6428, alt allele: T)\n",
            "14:43:59.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7121, ref allele: G)\n",
            "14:43:59.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7121, alt allele: A)\n",
            "14:43:59.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6532, ref allele: G)\n",
            "14:43:59.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6532, alt allele: A)\n",
            "14:43:59.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6428, ref allele: G)\n",
            "14:43:59.245 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6428, alt allele: A)\n",
            "14:43:59.337 INFO  ProgressMeter -       chr11:46890153              0.2                  2000           8600.3\n",
            "14:43:59.489 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, ref allele: G)\n",
            "14:43:59.490 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, alt allele: A)\n",
            "14:43:59.523 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2203, ref allele: G)\n",
            "14:43:59.523 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2203, alt allele: A)\n",
            "14:43:59.524 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1499, ref allele: G)\n",
            "14:43:59.524 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1499, alt allele: A)\n",
            "14:43:59.524 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=826, ref allele: G)\n",
            "14:43:59.524 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=826, alt allele: A)\n",
            "14:43:59.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, ref allele: G)\n",
            "14:43:59.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, alt allele: A)\n",
            "14:43:59.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5684, ref allele: G)\n",
            "14:43:59.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5684, alt allele: A)\n",
            "14:43:59.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5614, ref allele: G)\n",
            "14:43:59.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5614, alt allele: A)\n",
            "14:43:59.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, ref allele: G)\n",
            "14:43:59.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, alt allele: A)\n",
            "14:43:59.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, ref allele: G)\n",
            "14:43:59.644 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, alt allele: A)\n",
            "14:43:59.644 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5744, ref allele: G)\n",
            "14:43:59.644 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5744, alt allele: A)\n",
            "14:43:59.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4988, ref allele: G)\n",
            "14:43:59.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4988, alt allele: A)\n",
            "14:43:59.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, ref allele: G)\n",
            "14:43:59.646 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, alt allele: A)\n",
            "14:43:59.654 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, ref allele: T)\n",
            "14:43:59.655 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, alt allele: C)\n",
            "14:43:59.655 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5684, ref allele: T)\n",
            "14:43:59.655 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5684, alt allele: C)\n",
            "14:43:59.656 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5614, ref allele: T)\n",
            "14:43:59.656 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5614, alt allele: C)\n",
            "14:43:59.656 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, ref allele: T)\n",
            "14:43:59.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, alt allele: C)\n",
            "14:43:59.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, ref allele: T)\n",
            "14:43:59.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, alt allele: C)\n",
            "14:43:59.658 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5744, ref allele: T)\n",
            "14:43:59.658 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5744, alt allele: C)\n",
            "14:43:59.658 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4988, ref allele: T)\n",
            "14:43:59.659 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4988, alt allele: C)\n",
            "14:43:59.659 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, ref allele: T)\n",
            "14:43:59.659 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, alt allele: C)\n",
            "14:43:59.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, ref allele: G)\n",
            "14:43:59.669 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, alt allele: A)\n",
            "14:43:59.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, ref allele: G)\n",
            "14:43:59.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, alt allele: A)\n",
            "14:43:59.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, ref allele: G)\n",
            "14:43:59.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, alt allele: A)\n",
            "14:43:59.671 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, ref allele: G)\n",
            "14:43:59.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, alt allele: A)\n",
            "14:43:59.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, ref allele: T)\n",
            "14:43:59.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6005, alt allele: C)\n",
            "14:43:59.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5684, ref allele: T)\n",
            "14:43:59.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5684, alt allele: C)\n",
            "14:43:59.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5614, ref allele: T)\n",
            "14:43:59.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5614, alt allele: C)\n",
            "14:43:59.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, ref allele: T)\n",
            "14:43:59.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5873, alt allele: C)\n",
            "14:43:59.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, ref allele: T)\n",
            "14:43:59.682 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5990, alt allele: C)\n",
            "14:43:59.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5744, ref allele: T)\n",
            "14:43:59.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5744, alt allele: C)\n",
            "14:43:59.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4988, ref allele: T)\n",
            "14:43:59.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4988, alt allele: C)\n",
            "14:43:59.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, ref allele: T)\n",
            "14:43:59.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5942, alt allele: C)\n",
            "14:43:59.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1823, ref allele: T)\n",
            "14:43:59.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1823, alt allele: C)\n",
            "14:43:59.695 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4226, ref allele: G)\n",
            "14:43:59.695 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4226, alt allele: A)\n",
            "14:43:59.695 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4217, ref allele: G)\n",
            "14:43:59.696 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4217, alt allele: A)\n",
            "14:43:59.696 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4217, ref allele: G)\n",
            "14:43:59.696 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4217, alt allele: A)\n",
            "14:43:59.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1805, ref allele: A)\n",
            "14:43:59.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1805, alt allele: G)\n",
            "14:43:59.794 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=856, ref allele: A)\n",
            "14:43:59.794 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=856, alt allele: G)\n",
            "14:43:59.794 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2296, ref allele: A)\n",
            "14:43:59.794 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2296, alt allele: G)\n",
            "14:43:59.794 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3553, ref allele: A)\n",
            "14:43:59.795 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3553, alt allele: G)\n",
            "14:43:59.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1805, ref allele: G)\n",
            "14:43:59.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1805, alt allele: A)\n",
            "14:43:59.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2296, ref allele: G)\n",
            "14:43:59.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2296, alt allele: A)\n",
            "14:43:59.814 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3553, ref allele: G)\n",
            "14:43:59.816 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3553, alt allele: A)\n",
            "14:43:59.844 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1462, ref allele: T)\n",
            "14:43:59.844 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1462, alt allele: C)\n",
            "14:43:59.844 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1408, ref allele: T)\n",
            "14:43:59.844 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1408, alt allele: C)\n",
            "14:43:59.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1462, ref allele: T)\n",
            "14:43:59.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1462, alt allele: C)\n",
            "14:43:59.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1408, ref allele: T)\n",
            "14:43:59.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1408, alt allele: C)\n",
            "14:43:59.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2587, ref allele: C)\n",
            "14:43:59.992 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2587, alt allele: G)\n",
            "14:43:59.996 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3577, ref allele: G)\n",
            "14:43:59.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3577, alt allele: T)\n",
            "14:43:59.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3295, ref allele: G)\n",
            "14:43:59.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3295, alt allele: T)\n",
            "14:43:59.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2929, ref allele: G)\n",
            "14:43:59.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2929, alt allele: T)\n",
            "14:44:00.209 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5122, ref allele: T)\n",
            "14:44:00.209 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5122, alt allele: C)\n",
            "14:44:00.210 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3158, ref allele: T)\n",
            "14:44:00.210 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3158, alt allele: C)\n",
            "14:44:00.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5122, ref allele: G)\n",
            "14:44:00.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5122, alt allele: C)\n",
            "14:44:00.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1036, ref allele: T)\n",
            "14:44:00.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1036, alt allele: C)\n",
            "14:44:00.455 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1064, ref allele: A)\n",
            "14:44:00.455 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1064, alt allele: C)\n",
            "14:44:00.457 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1064, ref allele: G)\n",
            "14:44:00.457 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1064, alt allele: A)\n",
            "14:44:00.675 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, ref allele: T)\n",
            "14:44:00.675 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, alt allele: C)\n",
            "14:44:00.676 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1534, ref allele: T)\n",
            "14:44:00.676 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1534, alt allele: C)\n",
            "14:44:00.677 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, ref allele: C)\n",
            "14:44:00.677 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, alt allele: T)\n",
            "14:44:00.677 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1534, ref allele: C)\n",
            "14:44:00.677 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1534, alt allele: T)\n",
            "14:44:00.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, ref allele: C)\n",
            "14:44:00.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, alt allele: A)\n",
            "14:44:00.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1534, ref allele: C)\n",
            "14:44:00.680 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1534, alt allele: A)\n",
            "14:44:00.753 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2635, ref allele: C)\n",
            "14:44:00.753 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2635, alt allele: T)\n",
            "14:44:00.754 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, ref allele: C)\n",
            "14:44:00.754 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, alt allele: T)\n",
            "14:44:00.956 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1063, ref allele: G)\n",
            "14:44:00.956 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1063, alt allele: T)\n",
            "14:44:01.080 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1081, ref allele: C)\n",
            "14:44:01.080 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1081, alt allele: A)\n",
            "14:44:01.081 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1081, ref allele: A)\n",
            "14:44:01.081 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1081, alt allele: T)\n",
            "14:44:01.127 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2737, ref allele: T)\n",
            "14:44:01.127 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2737, alt allele: A)\n",
            "14:44:01.127 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1045, ref allele: T)\n",
            "14:44:01.128 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1045, alt allele: A)\n",
            "14:44:01.137 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=991, ref allele: T)\n",
            "14:44:01.137 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=991, alt allele: A)\n",
            "14:44:01.139 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=991, ref allele: A)\n",
            "14:44:01.139 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=991, alt allele: G)\n",
            "14:44:01.180 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3088, ref allele: C)\n",
            "14:44:01.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3088, alt allele: T)\n",
            "14:44:01.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3092, ref allele: C)\n",
            "14:44:01.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3092, alt allele: T)\n",
            "14:44:01.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3088, ref allele: A)\n",
            "14:44:01.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3088, alt allele: G)\n",
            "14:44:01.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3092, ref allele: A)\n",
            "14:44:01.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3092, alt allele: G)\n",
            "14:44:01.185 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3088, ref allele: G)\n",
            "14:44:01.185 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3088, alt allele: A)\n",
            "14:44:01.186 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3092, ref allele: G)\n",
            "14:44:01.186 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3092, alt allele: A)\n",
            "14:44:01.195 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3047, ref allele: A)\n",
            "14:44:01.195 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3047, alt allele: G)\n",
            "14:44:01.196 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3806, ref allele: A)\n",
            "14:44:01.196 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3806, alt allele: G)\n",
            "14:44:01.230 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1400, ref allele: T)\n",
            "14:44:01.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1400, alt allele: C)\n",
            "14:44:01.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, ref allele: T)\n",
            "14:44:01.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, alt allele: C)\n",
            "14:44:01.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1400, ref allele: A)\n",
            "14:44:01.233 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1400, alt allele: T)\n",
            "14:44:01.233 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, ref allele: A)\n",
            "14:44:01.233 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, alt allele: T)\n",
            "14:44:01.251 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1052, ref allele: T)\n",
            "14:44:01.251 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1052, alt allele: C)\n",
            "14:44:01.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1052, ref allele: G)\n",
            "14:44:01.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1052, alt allele: T)\n",
            "14:44:01.273 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, ref allele: T)\n",
            "14:44:01.273 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, alt allele: G)\n",
            "14:44:01.275 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, ref allele: A)\n",
            "14:44:01.275 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, alt allele: G)\n",
            "14:44:01.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, ref allele: C)\n",
            "14:44:01.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, alt allele: T)\n",
            "14:44:01.279 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, ref allele: G)\n",
            "14:44:01.279 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3611, alt allele: A)\n",
            "14:44:01.295 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1789, ref allele: C)\n",
            "14:44:01.296 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1789, alt allele: A)\n",
            "14:44:01.326 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5795, ref allele: C)\n",
            "14:44:01.326 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5795, alt allele: G)\n",
            "14:44:01.332 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1346, ref allele: G)\n",
            "14:44:01.333 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1346, alt allele: A)\n",
            "14:44:01.333 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1831, ref allele: G)\n",
            "14:44:01.333 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1831, alt allele: A)\n",
            "14:44:01.335 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1346, ref allele: C)\n",
            "14:44:01.335 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1346, alt allele: T)\n",
            "14:44:01.335 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1831, ref allele: C)\n",
            "14:44:01.335 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1831, alt allele: T)\n",
            "14:44:01.339 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=869, ref allele: T)\n",
            "14:44:01.339 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=869, alt allele: C)\n",
            "14:44:01.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=869, ref allele: T)\n",
            "14:44:01.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=869, alt allele: C)\n",
            "14:44:01.344 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, ref allele: C)\n",
            "14:44:01.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, alt allele: T)\n",
            "14:44:01.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1367, ref allele: C)\n",
            "14:44:01.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1367, alt allele: T)\n",
            "14:44:01.347 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, ref allele: A)\n",
            "14:44:01.347 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, alt allele: G)\n",
            "14:44:01.347 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1367, ref allele: A)\n",
            "14:44:01.347 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1367, alt allele: G)\n",
            "14:44:01.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, ref allele: T)\n",
            "14:44:01.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1424, alt allele: C)\n",
            "14:44:01.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1367, ref allele: T)\n",
            "14:44:01.349 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1367, alt allele: C)\n",
            "14:44:01.375 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2617, ref allele: T)\n",
            "14:44:01.375 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2617, alt allele: C)\n",
            "14:44:01.376 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2207, ref allele: T)\n",
            "14:44:01.376 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2207, alt allele: C)\n",
            "14:44:01.376 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1879, ref allele: T)\n",
            "14:44:01.376 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1879, alt allele: C)\n",
            "14:44:01.421 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, ref allele: C)\n",
            "14:44:01.421 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1649, alt allele: G)\n",
            "14:44:01.487 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4666, ref allele: T)\n",
            "14:44:01.487 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4666, alt allele: C)\n",
            "14:44:01.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6553, ref allele: C)\n",
            "14:44:01.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6553, alt allele: T)\n",
            "14:44:01.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4780, ref allele: C)\n",
            "14:44:01.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4780, alt allele: T)\n",
            "14:44:01.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6226, ref allele: C)\n",
            "14:44:01.532 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6226, alt allele: T)\n",
            "14:44:01.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5800, ref allele: C)\n",
            "14:44:01.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5800, alt allele: T)\n",
            "14:44:01.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6313, ref allele: C)\n",
            "14:44:01.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6313, alt allele: T)\n",
            "14:44:01.535 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6232, ref allele: C)\n",
            "14:44:01.535 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6232, alt allele: T)\n",
            "14:44:01.535 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6295, ref allele: C)\n",
            "14:44:01.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6295, alt allele: T)\n",
            "14:44:01.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5749, ref allele: C)\n",
            "14:44:01.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5749, alt allele: T)\n",
            "14:44:01.538 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4990, ref allele: C)\n",
            "14:44:01.539 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4990, alt allele: T)\n",
            "14:44:01.539 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6026, ref allele: C)\n",
            "14:44:01.539 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6026, alt allele: T)\n",
            "14:44:01.541 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6016, ref allele: C)\n",
            "14:44:01.541 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6016, alt allele: T)\n",
            "14:44:01.542 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5071, ref allele: C)\n",
            "14:44:01.542 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5071, alt allele: T)\n",
            "14:44:01.543 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5053, ref allele: C)\n",
            "14:44:01.543 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5053, alt allele: T)\n",
            "14:44:01.543 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4966, ref allele: C)\n",
            "14:44:01.546 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4966, alt allele: T)\n",
            "14:44:01.547 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4984, ref allele: C)\n",
            "14:44:01.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4984, alt allele: T)\n",
            "14:44:01.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2488, ref allele: C)\n",
            "14:44:01.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2488, alt allele: G)\n",
            "14:44:01.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2134, ref allele: C)\n",
            "14:44:01.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2134, alt allele: G)\n",
            "14:44:01.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1085, ref allele: C)\n",
            "14:44:01.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1085, alt allele: T)\n",
            "14:44:01.690 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, ref allele: G)\n",
            "14:44:01.691 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, alt allele: A)\n",
            "14:44:01.692 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, ref allele: C)\n",
            "14:44:01.692 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, alt allele: T)\n",
            "14:44:01.694 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, ref allele: C)\n",
            "14:44:01.694 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, alt allele: T)\n",
            "14:44:01.695 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, ref allele: C)\n",
            "14:44:01.695 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, alt allele: T)\n",
            "14:44:01.708 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, ref allele: G)\n",
            "14:44:01.708 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, alt allele: C)\n",
            "14:44:01.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1469, ref allele: G)\n",
            "14:44:01.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1469, alt allele: A)\n",
            "14:44:01.771 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=946, ref allele: A)\n",
            "14:44:01.771 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=946, alt allele: G)\n",
            "14:44:01.772 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=946, ref allele: T)\n",
            "14:44:01.772 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=946, alt allele: A)\n",
            "14:44:01.823 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1046, ref allele: A)\n",
            "14:44:01.823 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1046, alt allele: T)\n",
            "14:44:01.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1807, ref allele: G)\n",
            "14:44:01.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1807, alt allele: A)\n",
            "14:44:01.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1054, ref allele: G)\n",
            "14:44:01.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1054, alt allele: A)\n",
            "14:44:01.829 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1807, ref allele: T)\n",
            "14:44:01.829 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1807, alt allele: C)\n",
            "14:44:01.829 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1054, ref allele: T)\n",
            "14:44:01.829 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1054, alt allele: C)\n",
            "14:44:01.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1879, ref allele: G)\n",
            "14:44:01.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1879, alt allele: C)\n",
            "14:44:01.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1751, ref allele: G)\n",
            "14:44:01.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1751, alt allele: C)\n",
            "14:44:01.917 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1735, ref allele: A)\n",
            "14:44:01.918 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1735, alt allele: G)\n",
            "14:44:01.918 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, ref allele: A)\n",
            "14:44:01.918 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1070, alt allele: G)\n",
            "14:44:01.918 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1150, ref allele: A)\n",
            "14:44:01.918 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1150, alt allele: G)\n",
            "14:44:01.972 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1640, ref allele: G)\n",
            "14:44:01.973 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1640, alt allele: A)\n",
            "14:44:02.142 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4442, ref allele: C)\n",
            "14:44:02.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4442, alt allele: T)\n",
            "14:44:02.192 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8468, ref allele: G)\n",
            "14:44:02.192 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8468, alt allele: C)\n",
            "14:44:02.192 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, ref allele: G)\n",
            "14:44:02.192 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1037, alt allele: C)\n",
            "14:44:02.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8612, ref allele: G)\n",
            "14:44:02.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8612, alt allele: A)\n",
            "14:44:02.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, ref allele: G)\n",
            "14:44:02.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, alt allele: A)\n",
            "14:44:02.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8612, ref allele: T)\n",
            "14:44:02.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8612, alt allele: C)\n",
            "14:44:02.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, ref allele: T)\n",
            "14:44:02.199 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1051, alt allele: C)\n",
            "14:44:02.200 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1054, ref allele: A)\n",
            "14:44:02.200 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1054, alt allele: G)\n",
            "14:44:02.397 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1516, ref allele: C)\n",
            "14:44:02.397 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1516, alt allele: T)\n",
            "14:44:02.398 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1613, ref allele: C)\n",
            "14:44:02.398 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1613, alt allele: T)\n",
            "14:44:02.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=863, ref allele: A)\n",
            "14:44:02.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=863, alt allele: T)\n",
            "14:44:02.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1160, ref allele: A)\n",
            "14:44:02.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1160, alt allele: T)\n",
            "14:44:02.440 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1736, ref allele: A)\n",
            "14:44:02.440 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1736, alt allele: T)\n",
            "14:44:02.447 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=863, ref allele: A)\n",
            "14:44:02.447 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=863, alt allele: G)\n",
            "14:44:02.448 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1160, ref allele: A)\n",
            "14:44:02.448 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1160, alt allele: G)\n",
            "14:44:02.449 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1736, ref allele: A)\n",
            "14:44:02.449 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1736, alt allele: G)\n",
            "14:44:02.473 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=698, ref allele: C)\n",
            "14:44:02.473 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=698, alt allele: T)\n",
            "14:44:02.474 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1604, ref allele: C)\n",
            "14:44:02.474 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1604, alt allele: T)\n",
            "14:44:02.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=730, ref allele: T)\n",
            "14:44:02.484 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=730, alt allele: A)\n",
            "14:44:02.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=730, ref allele: G)\n",
            "14:44:02.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=730, alt allele: A)\n",
            "14:44:02.489 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1378, ref allele: C)\n",
            "14:44:02.489 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1378, alt allele: A)\n",
            "14:44:02.499 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, ref allele: ATT)\n",
            "14:44:02.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, ref allele: ATT)\n",
            "14:44:02.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, ref allele: A)\n",
            "14:44:02.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, alt allele: T)\n",
            "14:44:02.537 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, ref allele: A)\n",
            "14:44:02.537 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, alt allele: T)\n",
            "14:44:02.538 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, ref allele: A)\n",
            "14:44:02.538 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, alt allele: T)\n",
            "14:44:02.540 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, ref allele: C)\n",
            "14:44:02.540 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, alt allele: G)\n",
            "14:44:02.540 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, ref allele: C)\n",
            "14:44:02.540 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, alt allele: G)\n",
            "14:44:02.541 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, ref allele: C)\n",
            "14:44:02.542 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, alt allele: G)\n",
            "14:44:02.544 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, ref allele: T)\n",
            "14:44:02.544 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, alt allele: G)\n",
            "14:44:02.544 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, ref allele: T)\n",
            "14:44:02.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, alt allele: G)\n",
            "14:44:02.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, ref allele: T)\n",
            "14:44:02.546 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, alt allele: G)\n",
            "14:44:02.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, ref allele: G)\n",
            "14:44:02.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, alt allele: T)\n",
            "14:44:02.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, ref allele: G)\n",
            "14:44:02.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, alt allele: T)\n",
            "14:44:02.549 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, ref allele: G)\n",
            "14:44:02.549 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, alt allele: T)\n",
            "14:44:02.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, ref allele: A)\n",
            "14:44:02.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2537, alt allele: C)\n",
            "14:44:02.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, ref allele: A)\n",
            "14:44:02.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3401, alt allele: C)\n",
            "14:44:02.552 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, ref allele: A)\n",
            "14:44:02.552 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2813, alt allele: C)\n",
            "14:44:02.553 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=722, ref allele: C)\n",
            "14:44:02.553 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=722, alt allele: T)\n",
            "14:44:02.554 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=722, ref allele: T)\n",
            "14:44:02.554 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=722, alt allele: C)\n",
            "14:44:02.559 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1019, ref allele: C)\n",
            "14:44:02.559 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1019, alt allele: T)\n",
            "14:44:02.560 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1019, ref allele: C)\n",
            "14:44:02.560 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1019, alt allele: T)\n",
            "14:44:02.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=928, ref allele: A)\n",
            "14:44:02.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=928, alt allele: G)\n",
            "14:44:02.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=808, ref allele: A)\n",
            "14:44:02.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=808, alt allele: G)\n",
            "14:44:02.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=751, ref allele: A)\n",
            "14:44:02.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=751, alt allele: G)\n",
            "14:44:02.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=790, ref allele: A)\n",
            "14:44:02.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=790, alt allele: G)\n",
            "14:44:02.588 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=928, ref allele: C)\n",
            "14:44:02.588 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=928, alt allele: T)\n",
            "14:44:02.588 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=808, ref allele: C)\n",
            "14:44:02.588 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=808, alt allele: T)\n",
            "14:44:02.613 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=818, ref allele: T)\n",
            "14:44:02.614 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=818, alt allele: G)\n",
            "14:44:02.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1954, ref allele: T)\n",
            "14:44:02.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1954, alt allele: C)\n",
            "14:44:02.658 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1954, ref allele: C)\n",
            "14:44:02.658 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1954, alt allele: T)\n",
            "14:44:02.659 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1954, ref allele: C)\n",
            "14:44:02.659 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1954, alt allele: T)\n",
            "14:44:02.661 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2413, ref allele: T)\n",
            "14:44:02.661 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2413, alt allele: C)\n",
            "14:44:02.662 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3346, ref allele: C)\n",
            "14:44:02.662 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3346, alt allele: A)\n",
            "14:44:02.665 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3346, ref allele: G)\n",
            "14:44:02.666 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3346, alt allele: A)\n",
            "14:44:02.711 ERROR GencodeFuncotationFactory - Problem creating a GencodeFuncotation on transcript ENST00000452451.6 for variant: chr11:61019321-61019322(CG* -> C): Variant overlaps transcript but is not completely contained within it. Funcotator cannot currently handle this case. Transcript: ENST00000452451.6  Variant: [VC Unknown @ chr11:61019321-61019322 Q486.60 of type=INDEL alleles=[CG*, C] attr={AC=1, AF=0.500, AN=2, BaseQRankSum=0.776, DP=40, ExcessHet=3.0103, FS=5.391, MLEAC=1, MLEAF=0.500, MQ=60.00, MQRankSum=0.000, QD=13.90, ReadPosRankSum=-1.421, SOR=1.751} GT=GT:AD:DP:GQ:PL\t0/1:17,18:35:99:494,0,456 filters=\n",
            "14:44:02.712 WARN  GencodeFuncotationFactory - Creating default GencodeFuncotation on transcript ENST00000452451.6 for problem variant: chr11:61019321-61019322(CG* -> C)\n",
            "14:44:02.712 ERROR GencodeFuncotationFactory - Problem creating a GencodeFuncotation on transcript ENST00000352009.9 for variant: chr11:61019321-61019322(CG* -> C): Variant overlaps transcript but is not completely contained within it. Funcotator cannot currently handle this case. Transcript: ENST00000352009.9  Variant: [VC Unknown @ chr11:61019321-61019322 Q486.60 of type=INDEL alleles=[CG*, C] attr={AC=1, AF=0.500, AN=2, BaseQRankSum=0.776, DP=40, ExcessHet=3.0103, FS=5.391, MLEAC=1, MLEAF=0.500, MQ=60.00, MQRankSum=0.000, QD=13.90, ReadPosRankSum=-1.421, SOR=1.751} GT=GT:AD:DP:GQ:PL\t0/1:17,18:35:99:494,0,456 filters=\n",
            "14:44:02.712 WARN  GencodeFuncotationFactory - Creating default GencodeFuncotation on transcript ENST00000352009.9 for problem variant: chr11:61019321-61019322(CG* -> C)\n",
            "14:44:02.755 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3221, ref allele: C)\n",
            "14:44:02.755 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3221, alt allele: T)\n",
            "14:44:02.759 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3221, ref allele: A)\n",
            "14:44:02.759 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3221, alt allele: G)\n",
            "14:44:02.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3221, ref allele: C)\n",
            "14:44:02.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3221, alt allele: T)\n",
            "14:44:02.805 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1519, ref allele: G)\n",
            "14:44:02.805 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1519, alt allele: C)\n",
            "14:44:02.805 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=971, ref allele: G)\n",
            "14:44:02.805 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=971, alt allele: C)\n",
            "14:44:02.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1519, ref allele: C)\n",
            "14:44:02.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1519, alt allele: T)\n",
            "14:44:02.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=971, ref allele: C)\n",
            "14:44:02.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=971, alt allele: T)\n",
            "14:44:02.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, ref allele: C)\n",
            "14:44:02.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, alt allele: G)\n",
            "14:44:02.821 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3640, ref allele: C)\n",
            "14:44:02.825 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3640, alt allele: G)\n",
            "14:44:02.901 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2839, ref allele: G)\n",
            "14:44:02.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2839, alt allele: A)\n",
            "14:44:02.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1205, ref allele: G)\n",
            "14:44:02.902 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1205, alt allele: A)\n",
            "14:44:02.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1958, ref allele: G)\n",
            "14:44:02.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1958, alt allele: A)\n",
            "14:44:02.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1306, ref allele: G)\n",
            "14:44:02.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1306, alt allele: A)\n",
            "14:44:02.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=908, ref allele: G)\n",
            "14:44:02.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=908, alt allele: A)\n",
            "14:44:02.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1472, ref allele: C)\n",
            "14:44:02.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1472, alt allele: A)\n",
            "14:44:02.966 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, ref allele: C)\n",
            "14:44:02.966 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, alt allele: A)\n",
            "14:44:02.975 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1472, ref allele: G)\n",
            "14:44:02.975 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1472, alt allele: A)\n",
            "14:44:02.975 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, ref allele: G)\n",
            "14:44:02.975 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, alt allele: A)\n",
            "14:44:02.978 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1472, ref allele: G)\n",
            "14:44:02.978 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1472, alt allele: C)\n",
            "14:44:02.978 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, ref allele: G)\n",
            "14:44:02.978 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1535, alt allele: C)\n",
            "14:44:02.979 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=488, ref allele: G)\n",
            "14:44:02.979 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=488, alt allele: C)\n",
            "14:44:02.981 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1621, ref allele: C)\n",
            "14:44:02.982 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1621, alt allele: T)\n",
            "14:44:03.076 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5927, ref allele: T)\n",
            "14:44:03.077 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5927, alt allele: C)\n",
            "14:44:03.155 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2284, ref allele: C)\n",
            "14:44:03.155 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2284, alt allele: T)\n",
            "14:44:03.155 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2125, ref allele: C)\n",
            "14:44:03.155 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2125, alt allele: T)\n",
            "14:44:03.157 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2284, ref allele: A)\n",
            "14:44:03.157 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2284, alt allele: G)\n",
            "14:44:03.157 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2125, ref allele: A)\n",
            "14:44:03.157 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2125, alt allele: G)\n",
            "14:44:03.158 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2284, ref allele: A)\n",
            "14:44:03.158 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2284, alt allele: G)\n",
            "14:44:03.158 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2125, ref allele: A)\n",
            "14:44:03.158 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2125, alt allele: G)\n",
            "14:44:03.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, ref allele: T)\n",
            "14:44:03.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, alt allele: C)\n",
            "14:44:03.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, ref allele: C)\n",
            "14:44:03.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, alt allele: A)\n",
            "14:44:03.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4267, ref allele: C)\n",
            "14:44:03.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4267, alt allele: A)\n",
            "14:44:03.202 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, ref allele: C)\n",
            "14:44:03.203 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, alt allele: A)\n",
            "14:44:03.203 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4267, ref allele: C)\n",
            "14:44:03.203 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4267, alt allele: A)\n",
            "14:44:03.210 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, ref allele: T)\n",
            "14:44:03.210 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2740, alt allele: C)\n",
            "14:44:03.211 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4267, ref allele: T)\n",
            "14:44:03.211 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4267, alt allele: C)\n",
            "14:44:03.503 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: T)\n",
            "14:44:03.504 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: G)\n",
            "14:44:03.507 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: T)\n",
            "14:44:03.508 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: C)\n",
            "14:44:03.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: G)\n",
            "14:44:03.511 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: A)\n",
            "14:44:03.513 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: A)\n",
            "14:44:03.514 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: G)\n",
            "14:44:03.516 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: T)\n",
            "14:44:03.516 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: C)\n",
            "14:44:03.518 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: C)\n",
            "14:44:03.518 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: A)\n",
            "14:44:03.519 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: C)\n",
            "14:44:03.520 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: T)\n",
            "14:44:03.521 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: G)\n",
            "14:44:03.521 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: T)\n",
            "14:44:03.523 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: A)\n",
            "14:44:03.523 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: G)\n",
            "14:44:03.525 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: A)\n",
            "14:44:03.525 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: G)\n",
            "14:44:03.526 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: C)\n",
            "14:44:03.527 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: T)\n",
            "14:44:03.528 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: A)\n",
            "14:44:03.528 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: G)\n",
            "14:44:03.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, ref allele: T)\n",
            "14:44:03.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18787, alt allele: G)\n",
            "14:44:03.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=707, ref allele: A)\n",
            "14:44:03.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=707, alt allele: GAGA)\n",
            "14:44:03.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, ref allele: A)\n",
            "14:44:03.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1030, alt allele: GAGA)\n",
            "14:44:03.533 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18793, ref allele: A)\n",
            "14:44:03.534 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=18793, alt allele: GAGA)\n",
            "14:44:03.602 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2288, ref allele: G)\n",
            "14:44:03.602 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2288, alt allele: A)\n",
            "14:44:03.698 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2978, ref allele: A)\n",
            "14:44:03.699 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2978, alt allele: T)\n",
            "14:44:03.960 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2098, ref allele: T)\n",
            "14:44:03.960 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2098, alt allele: G)\n",
            "14:44:03.961 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2101, ref allele: T)\n",
            "14:44:03.961 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2101, alt allele: G)\n",
            "14:44:03.961 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, ref allele: T)\n",
            "14:44:03.961 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, alt allele: G)\n",
            "14:44:03.964 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2098, ref allele: C)\n",
            "14:44:03.964 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2098, alt allele: G)\n",
            "14:44:03.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2101, ref allele: C)\n",
            "14:44:03.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2101, alt allele: G)\n",
            "14:44:03.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, ref allele: C)\n",
            "14:44:03.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, alt allele: G)\n",
            "14:44:04.046 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2026, ref allele: G)\n",
            "14:44:04.046 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2026, alt allele: A)\n",
            "14:44:04.047 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2026, ref allele: T)\n",
            "14:44:04.047 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2026, alt allele: C)\n",
            "14:44:04.052 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2026, ref allele: C)\n",
            "14:44:04.052 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2026, alt allele: T)\n",
            "14:44:04.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, ref allele: T)\n",
            "14:44:04.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, alt allele: C)\n",
            "14:44:04.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, ref allele: T)\n",
            "14:44:04.177 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, alt allele: C)\n",
            "14:44:04.179 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, ref allele: G)\n",
            "14:44:04.179 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, alt allele: C)\n",
            "14:44:04.180 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, ref allele: G)\n",
            "14:44:04.180 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, alt allele: C)\n",
            "14:44:04.180 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, ref allele: G)\n",
            "14:44:04.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, alt allele: C)\n",
            "14:44:04.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, ref allele: G)\n",
            "14:44:04.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, alt allele: C)\n",
            "14:44:04.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, ref allele: A)\n",
            "14:44:04.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, alt allele: G)\n",
            "14:44:04.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, ref allele: A)\n",
            "14:44:04.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1166, alt allele: G)\n",
            "14:44:04.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7135, ref allele: A)\n",
            "14:44:04.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7135, alt allele: G)\n",
            "14:44:04.315 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4937, ref allele: T)\n",
            "14:44:04.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4937, alt allele: C)\n",
            "14:44:04.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4886, ref allele: T)\n",
            "14:44:04.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4886, alt allele: C)\n",
            "14:44:04.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3124, ref allele: T)\n",
            "14:44:04.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3124, alt allele: C)\n",
            "14:44:04.320 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2024, ref allele: C)\n",
            "14:44:04.320 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2024, alt allele: T)\n",
            "14:44:04.382 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3664, ref allele: G)\n",
            "14:44:04.382 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3664, alt allele: A)\n",
            "14:44:04.382 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=577, ref allele: G)\n",
            "14:44:04.382 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=577, alt allele: A)\n",
            "14:44:04.387 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1205, ref allele: G)\n",
            "14:44:04.387 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1205, alt allele: T)\n",
            "14:44:04.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1205, ref allele: T)\n",
            "14:44:04.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1205, alt allele: C)\n",
            "14:44:04.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, ref allele: T)\n",
            "14:44:04.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, alt allele: C)\n",
            "14:44:04.498 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, ref allele: C)\n",
            "14:44:04.499 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, alt allele: T)\n",
            "14:44:04.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, ref allele: C)\n",
            "14:44:04.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, alt allele: T)\n",
            "14:44:04.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, ref allele: C)\n",
            "14:44:04.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, alt allele: T)\n",
            "14:44:04.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, ref allele: A)\n",
            "14:44:04.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, alt allele: G)\n",
            "14:44:04.503 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, ref allele: C)\n",
            "14:44:04.504 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3949, alt allele: T)\n",
            "14:44:04.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2489, ref allele: C)\n",
            "14:44:04.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2489, alt allele: T)\n",
            "14:44:04.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=865, ref allele: A)\n",
            "14:44:04.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=865, alt allele: G)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, ref allele: A)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, alt allele: G)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, ref allele: A)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1058, alt allele: G)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=382, ref allele: A)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=382, alt allele: G)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=962, ref allele: A)\n",
            "14:44:04.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=962, alt allele: G)\n",
            "14:44:04.598 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=986, ref allele: G)\n",
            "14:44:04.599 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=986, alt allele: T)\n",
            "14:44:04.599 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, ref allele: G)\n",
            "14:44:04.599 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, alt allele: T)\n",
            "14:44:04.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=986, ref allele: A)\n",
            "14:44:04.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=986, alt allele: G)\n",
            "14:44:04.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, ref allele: A)\n",
            "14:44:04.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, alt allele: G)\n",
            "14:44:04.601 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=986, ref allele: T)\n",
            "14:44:04.601 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=986, alt allele: C)\n",
            "14:44:04.602 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, ref allele: T)\n",
            "14:44:04.602 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1094, alt allele: C)\n",
            "14:44:04.607 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=613, ref allele: T)\n",
            "14:44:04.607 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=613, alt allele: C)\n",
            "14:44:04.610 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=920, ref allele: C)\n",
            "14:44:04.610 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=920, alt allele: T)\n",
            "14:44:04.611 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4469, ref allele: A)\n",
            "14:44:04.611 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4469, alt allele: G)\n",
            "14:44:04.611 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3946, ref allele: A)\n",
            "14:44:04.612 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3946, alt allele: G)\n",
            "14:44:04.615 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4469, ref allele: C)\n",
            "14:44:04.615 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4469, alt allele: T)\n",
            "14:44:04.615 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3946, ref allele: C)\n",
            "14:44:04.616 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3946, alt allele: T)\n",
            "14:44:04.620 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4469, ref allele: G)\n",
            "14:44:04.620 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4469, alt allele: A)\n",
            "14:44:04.621 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3946, ref allele: G)\n",
            "14:44:04.621 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3946, alt allele: A)\n",
            "14:44:04.622 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1697, ref allele: G)\n",
            "14:44:04.622 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1697, alt allele: A)\n",
            "14:44:04.622 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1673, ref allele: G)\n",
            "14:44:04.622 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1673, alt allele: A)\n",
            "14:44:04.623 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1723, ref allele: G)\n",
            "14:44:04.623 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1723, alt allele: A)\n",
            "14:44:04.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: G)\n",
            "14:44:04.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: A)\n",
            "14:44:04.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: G)\n",
            "14:44:04.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: A)\n",
            "14:44:04.631 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: T)\n",
            "14:44:04.631 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: C)\n",
            "14:44:04.632 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: T)\n",
            "14:44:04.632 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: C)\n",
            "14:44:04.634 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: G)\n",
            "14:44:04.634 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: A)\n",
            "14:44:04.634 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: G)\n",
            "14:44:04.634 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: A)\n",
            "14:44:04.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: A)\n",
            "14:44:04.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: T)\n",
            "14:44:04.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: A)\n",
            "14:44:04.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: T)\n",
            "14:44:04.638 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: G)\n",
            "14:44:04.638 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: A)\n",
            "14:44:04.639 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: G)\n",
            "14:44:04.639 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: A)\n",
            "14:44:04.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: G)\n",
            "14:44:04.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: T)\n",
            "14:44:04.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: G)\n",
            "14:44:04.641 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: T)\n",
            "14:44:04.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: CGGG)\n",
            "14:44:04.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: C)\n",
            "14:44:04.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: CGGG)\n",
            "14:44:04.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: C)\n",
            "14:44:04.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: G)\n",
            "14:44:04.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: A)\n",
            "14:44:04.646 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: G)\n",
            "14:44:04.646 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: A)\n",
            "14:44:04.648 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: G)\n",
            "14:44:04.648 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: A)\n",
            "14:44:04.648 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: G)\n",
            "14:44:04.648 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: A)\n",
            "14:44:04.650 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: T)\n",
            "14:44:04.650 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: C)\n",
            "14:44:04.650 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: T)\n",
            "14:44:04.650 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: C)\n",
            "14:44:04.652 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: C)\n",
            "14:44:04.652 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: T)\n",
            "14:44:04.652 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: C)\n",
            "14:44:04.652 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: T)\n",
            "14:44:04.654 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, ref allele: C)\n",
            "14:44:04.654 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2218, alt allele: T)\n",
            "14:44:04.655 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, ref allele: C)\n",
            "14:44:04.655 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2215, alt allele: T)\n",
            "14:44:04.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=893, ref allele: A)\n",
            "14:44:04.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=893, alt allele: G)\n",
            "14:44:04.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=596, ref allele: A)\n",
            "14:44:04.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=596, alt allele: G)\n",
            "14:44:04.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=463, ref allele: A)\n",
            "14:44:04.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=463, alt allele: G)\n",
            "14:44:04.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, ref allele: T)\n",
            "14:44:04.725 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, alt allele: G)\n",
            "14:44:04.727 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, ref allele: T)\n",
            "14:44:04.727 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, alt allele: C)\n",
            "14:44:04.728 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, ref allele: A)\n",
            "14:44:04.729 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, alt allele: C)\n",
            "14:44:04.729 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2326, ref allele: A)\n",
            "14:44:04.729 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2326, alt allele: C)\n",
            "14:44:04.731 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, ref allele: T)\n",
            "14:44:04.732 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4915, alt allele: C)\n",
            "14:44:04.732 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2326, ref allele: T)\n",
            "14:44:04.732 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2326, alt allele: C)\n",
            "14:44:04.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3121, ref allele: T)\n",
            "14:44:04.737 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3121, alt allele: C)\n",
            "14:44:04.737 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3119, ref allele: T)\n",
            "14:44:04.737 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3119, alt allele: C)\n",
            "14:44:04.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3121, ref allele: T)\n",
            "14:44:04.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3121, alt allele: G)\n",
            "14:44:04.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3119, ref allele: T)\n",
            "14:44:04.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3119, alt allele: G)\n",
            "14:44:04.809 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, ref allele: C)\n",
            "14:44:04.810 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, alt allele: T)\n",
            "14:44:04.810 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, ref allele: C)\n",
            "14:44:04.810 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, alt allele: T)\n",
            "14:44:04.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: C)\n",
            "14:44:04.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: T)\n",
            "14:44:04.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, ref allele: T)\n",
            "14:44:04.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, alt allele: C)\n",
            "14:44:04.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, ref allele: T)\n",
            "14:44:04.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, alt allele: C)\n",
            "14:44:04.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: T)\n",
            "14:44:04.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: C)\n",
            "14:44:04.817 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, ref allele: A)\n",
            "14:44:04.817 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, alt allele: G)\n",
            "14:44:04.818 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, ref allele: A)\n",
            "14:44:04.818 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, alt allele: G)\n",
            "14:44:04.818 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: A)\n",
            "14:44:04.818 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: G)\n",
            "14:44:04.818 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, ref allele: A)\n",
            "14:44:04.818 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, alt allele: G)\n",
            "14:44:04.822 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, ref allele: T)\n",
            "14:44:04.822 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2599, alt allele: C)\n",
            "14:44:04.823 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, ref allele: T)\n",
            "14:44:04.823 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3206, alt allele: C)\n",
            "14:44:04.823 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, ref allele: T)\n",
            "14:44:04.823 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2887, alt allele: C)\n",
            "14:44:04.824 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, ref allele: T)\n",
            "14:44:04.824 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, alt allele: C)\n",
            "14:44:04.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6373, ref allele: C)\n",
            "14:44:04.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6373, alt allele: T)\n",
            "14:44:04.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6413, ref allele: C)\n",
            "14:44:04.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6413, alt allele: T)\n",
            "14:44:04.922 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6373, ref allele: C)\n",
            "14:44:04.923 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6373, alt allele: T)\n",
            "14:44:04.923 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6413, ref allele: C)\n",
            "14:44:04.923 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6413, alt allele: T)\n",
            "14:44:04.948 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2611, ref allele: C)\n",
            "14:44:04.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2611, alt allele: T)\n",
            "14:44:04.961 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, ref allele: A)\n",
            "14:44:04.961 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, alt allele: G)\n",
            "14:44:04.962 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2761, ref allele: A)\n",
            "14:44:04.962 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2761, alt allele: G)\n",
            "14:44:04.963 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, ref allele: A)\n",
            "14:44:04.963 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, alt allele: G)\n",
            "14:44:04.963 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3155, ref allele: A)\n",
            "14:44:04.963 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3155, alt allele: G)\n",
            "14:44:04.964 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3034, ref allele: A)\n",
            "14:44:04.964 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3034, alt allele: G)\n",
            "14:44:04.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1871, ref allele: A)\n",
            "14:44:04.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1871, alt allele: G)\n",
            "14:44:04.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, ref allele: T)\n",
            "14:44:04.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, alt allele: C)\n",
            "14:44:04.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2761, ref allele: T)\n",
            "14:44:04.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2761, alt allele: C)\n",
            "14:44:04.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, ref allele: T)\n",
            "14:44:04.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, alt allele: C)\n",
            "14:44:04.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3155, ref allele: T)\n",
            "14:44:04.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3155, alt allele: C)\n",
            "14:44:04.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3034, ref allele: T)\n",
            "14:44:04.969 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3034, alt allele: C)\n",
            "14:44:04.970 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1871, ref allele: T)\n",
            "14:44:04.970 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1871, alt allele: C)\n",
            "14:44:04.971 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, ref allele: C)\n",
            "14:44:04.971 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, alt allele: T)\n",
            "14:44:04.972 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2761, ref allele: C)\n",
            "14:44:04.972 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2761, alt allele: T)\n",
            "14:44:04.972 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, ref allele: C)\n",
            "14:44:04.972 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3148, alt allele: T)\n",
            "14:44:04.973 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3155, ref allele: C)\n",
            "14:44:04.973 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3155, alt allele: T)\n",
            "14:44:04.973 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3034, ref allele: C)\n",
            "14:44:04.974 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3034, alt allele: T)\n",
            "14:44:04.974 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1871, ref allele: C)\n",
            "14:44:04.974 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1871, alt allele: T)\n",
            "14:44:05.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1939, ref allele: C)\n",
            "14:44:05.249 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1939, alt allele: T)\n",
            "14:44:05.250 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1939, ref allele: C)\n",
            "14:44:05.250 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1939, alt allele: T)\n",
            "14:44:05.257 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2699, ref allele: C)\n",
            "14:44:05.257 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2699, alt allele: T)\n",
            "14:44:05.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=959, ref allele: C)\n",
            "14:44:05.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=959, alt allele: T)\n",
            "14:44:05.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1306, ref allele: C)\n",
            "14:44:05.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1306, alt allele: T)\n",
            "14:44:05.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=847, ref allele: G)\n",
            "14:44:05.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=847, alt allele: C)\n",
            "14:44:05.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=697, ref allele: G)\n",
            "14:44:05.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=697, alt allele: C)\n",
            "14:44:05.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=538, ref allele: G)\n",
            "14:44:05.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=538, alt allele: C)\n",
            "14:44:05.295 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=461, ref allele: G)\n",
            "14:44:05.295 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=461, alt allele: C)\n",
            "14:44:05.295 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=533, ref allele: G)\n",
            "14:44:05.295 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=533, alt allele: C)\n",
            "14:44:05.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, ref allele: A)\n",
            "14:44:05.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, alt allele: G)\n",
            "14:44:05.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2432, ref allele: A)\n",
            "14:44:05.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2432, alt allele: G)\n",
            "14:44:05.496 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, ref allele: T)\n",
            "14:44:05.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, alt allele: C)\n",
            "14:44:05.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2432, ref allele: T)\n",
            "14:44:05.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2432, alt allele: C)\n",
            "14:44:05.583 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3514, ref allele: C)\n",
            "14:44:05.583 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3514, alt allele: T)\n",
            "14:44:05.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3514, ref allele: C)\n",
            "14:44:05.586 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3514, alt allele: T)\n",
            "14:44:05.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2642, ref allele: G)\n",
            "14:44:05.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2642, alt allele: C)\n",
            "14:44:05.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2584, ref allele: G)\n",
            "14:44:05.716 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2584, alt allele: C)\n",
            "14:44:05.717 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2570, ref allele: G)\n",
            "14:44:05.717 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2570, alt allele: C)\n",
            "14:44:05.735 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, ref allele: G)\n",
            "14:44:05.735 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, alt allele: C)\n",
            "14:44:05.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, ref allele: G)\n",
            "14:44:05.737 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, alt allele: A)\n",
            "14:44:05.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, ref allele: T)\n",
            "14:44:05.738 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, alt allele: G)\n",
            "14:44:05.740 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, ref allele: A)\n",
            "14:44:05.740 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, alt allele: T)\n",
            "14:44:05.741 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, ref allele: G)\n",
            "14:44:05.742 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5185, alt allele: A)\n",
            "14:44:05.883 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3566, ref allele: C)\n",
            "14:44:05.884 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3566, alt allele: T)\n",
            "14:44:05.886 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7105, ref allele: A)\n",
            "14:44:05.887 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7105, alt allele: G)\n",
            "14:44:05.888 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7105, ref allele: T)\n",
            "14:44:05.888 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7105, alt allele: A)\n",
            "14:44:05.892 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3521, ref allele: G)\n",
            "14:44:05.892 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3521, alt allele: A)\n",
            "14:44:05.893 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3235, ref allele: G)\n",
            "14:44:05.893 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3235, alt allele: A)\n",
            "14:44:05.893 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: G)\n",
            "14:44:05.893 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: A)\n",
            "14:44:05.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3521, ref allele: T)\n",
            "14:44:05.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3521, alt allele: G)\n",
            "14:44:05.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3235, ref allele: T)\n",
            "14:44:05.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3235, alt allele: G)\n",
            "14:44:05.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: T)\n",
            "14:44:05.896 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: G)\n",
            "14:44:05.897 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3521, ref allele: G)\n",
            "14:44:05.897 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3521, alt allele: A)\n",
            "14:44:05.897 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3235, ref allele: G)\n",
            "14:44:05.898 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3235, alt allele: A)\n",
            "14:44:05.898 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, ref allele: G)\n",
            "14:44:05.898 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3628, alt allele: A)\n",
            "14:44:06.029 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2794, ref allele: T)\n",
            "14:44:06.029 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2794, alt allele: C)\n",
            "14:44:06.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=835, ref allele: T)\n",
            "14:44:06.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=835, alt allele: C)\n",
            "14:44:06.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1234, ref allele: C)\n",
            "14:44:06.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1234, alt allele: T)\n",
            "14:44:06.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=962, ref allele: C)\n",
            "14:44:06.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=962, alt allele: T)\n",
            "14:44:06.197 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=860, ref allele: C)\n",
            "14:44:06.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=860, alt allele: T)\n",
            "14:44:06.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=758, ref allele: C)\n",
            "14:44:06.198 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=758, alt allele: T)\n",
            "14:44:06.200 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, ref allele: G)\n",
            "14:44:06.200 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, alt allele: A)\n",
            "14:44:06.202 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, ref allele: T)\n",
            "14:44:06.202 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, alt allele: C)\n",
            "14:44:06.203 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, ref allele: T)\n",
            "14:44:06.203 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, alt allele: C)\n",
            "14:44:06.205 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, ref allele: G)\n",
            "14:44:06.205 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, alt allele: C)\n",
            "14:44:06.205 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, ref allele: G)\n",
            "14:44:06.205 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, alt allele: C)\n",
            "14:44:06.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, ref allele: C)\n",
            "14:44:06.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, alt allele: T)\n",
            "14:44:06.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, ref allele: C)\n",
            "14:44:06.213 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, alt allele: T)\n",
            "14:44:06.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, ref allele: G)\n",
            "14:44:06.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, alt allele: A)\n",
            "14:44:06.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, ref allele: G)\n",
            "14:44:06.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, alt allele: A)\n",
            "14:44:06.219 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, ref allele: G)\n",
            "14:44:06.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2180, alt allele: T)\n",
            "14:44:06.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, ref allele: G)\n",
            "14:44:06.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2392, alt allele: T)\n",
            "14:44:06.222 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1504, ref allele: A)\n",
            "14:44:06.222 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1504, alt allele: G)\n",
            "14:44:06.225 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, ref allele: A)\n",
            "14:44:06.225 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, alt allele: G)\n",
            "14:44:06.227 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, ref allele: T)\n",
            "14:44:06.227 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, alt allele: C)\n",
            "14:44:06.233 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1669, ref allele: A)\n",
            "14:44:06.233 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1669, alt allele: G)\n",
            "14:44:06.234 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1132, ref allele: A)\n",
            "14:44:06.234 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1132, alt allele: G)\n",
            "14:44:06.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1619, ref allele: A)\n",
            "14:44:06.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1619, alt allele: G)\n",
            "14:44:06.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1559, ref allele: A)\n",
            "14:44:06.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1559, alt allele: G)\n",
            "14:44:06.239 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1528, ref allele: A)\n",
            "14:44:06.239 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1528, alt allele: G)\n",
            "14:44:06.241 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1925, ref allele: C)\n",
            "14:44:06.242 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1925, alt allele: G)\n",
            "14:44:06.400 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3293, ref allele: G)\n",
            "14:44:06.400 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3293, alt allele: A)\n",
            "14:44:06.405 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3293, ref allele: C)\n",
            "14:44:06.405 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3293, alt allele: T)\n",
            "14:44:06.407 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3293, ref allele: G)\n",
            "14:44:06.407 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3293, alt allele: C)\n",
            "14:44:06.730 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, ref allele: C)\n",
            "14:44:06.730 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2558, alt allele: T)\n",
            "14:44:06.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2693, ref allele: G)\n",
            "14:44:06.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2693, alt allele: A)\n",
            "14:44:06.735 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2122, ref allele: G)\n",
            "14:44:06.735 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2122, alt allele: A)\n",
            "14:44:06.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2693, ref allele: A)\n",
            "14:44:06.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2693, alt allele: G)\n",
            "14:44:06.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2122, ref allele: A)\n",
            "14:44:06.737 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2122, alt allele: G)\n",
            "14:44:06.743 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1363, ref allele: C)\n",
            "14:44:06.743 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1363, alt allele: T)\n",
            "14:44:06.747 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1363, ref allele: T)\n",
            "14:44:06.747 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1363, alt allele: C)\n",
            "14:44:06.748 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1445, ref allele: T)\n",
            "14:44:06.748 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1445, alt allele: C)\n",
            "14:44:06.911 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=922, ref allele: G)\n",
            "14:44:06.911 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=922, alt allele: A)\n",
            "14:44:06.918 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2974, ref allele: A)\n",
            "14:44:06.918 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2974, alt allele: G)\n",
            "14:44:06.919 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2251, ref allele: A)\n",
            "14:44:06.919 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2251, alt allele: G)\n",
            "14:44:06.925 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, ref allele: G)\n",
            "14:44:06.925 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, alt allele: A)\n",
            "14:44:07.089 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, ref allele: G)\n",
            "14:44:07.089 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, alt allele: A)\n",
            "14:44:07.092 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, ref allele: C)\n",
            "14:44:07.092 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, alt allele: T)\n",
            "14:44:07.094 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2908, ref allele: G)\n",
            "14:44:07.094 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2908, alt allele: A)\n",
            "14:44:07.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8128, ref allele: T)\n",
            "14:44:07.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8128, alt allele: C)\n",
            "14:44:07.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8128, ref allele: A)\n",
            "14:44:07.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8128, alt allele: G)\n",
            "14:44:07.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1129, ref allele: C)\n",
            "14:44:07.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1129, alt allele: T)\n",
            "14:44:07.553 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6511, ref allele: T)\n",
            "14:44:07.553 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6511, alt allele: C)\n",
            "14:44:07.554 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6967, ref allele: T)\n",
            "14:44:07.554 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6967, alt allele: C)\n",
            "14:44:07.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3466, ref allele: C)\n",
            "14:44:07.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3466, alt allele: A)\n",
            "14:44:07.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2881, ref allele: T)\n",
            "14:44:07.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2881, alt allele: C)\n",
            "14:44:07.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2219, ref allele: T)\n",
            "14:44:07.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2219, alt allele: C)\n",
            "14:44:07.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, ref allele: T)\n",
            "14:44:07.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2138, alt allele: C)\n",
            "14:44:07.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1111, ref allele: T)\n",
            "14:44:07.643 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1111, alt allele: C)\n",
            "14:44:07.751 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4189, ref allele: T)\n",
            "14:44:07.751 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4189, alt allele: C)\n",
            "14:44:07.751 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, ref allele: T)\n",
            "14:44:07.751 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, alt allele: C)\n",
            "14:44:07.752 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4225, ref allele: T)\n",
            "14:44:07.752 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4225, alt allele: C)\n",
            "14:44:07.846 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=925, ref allele: G)\n",
            "14:44:07.846 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=925, alt allele: A)\n",
            "14:44:07.846 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, ref allele: G)\n",
            "14:44:07.846 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=941, alt allele: A)\n",
            "14:44:07.852 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=965, ref allele: A)\n",
            "14:44:07.853 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=965, alt allele: G)\n",
            "14:44:07.853 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=700, ref allele: A)\n",
            "14:44:07.853 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=700, alt allele: G)\n",
            "14:44:07.854 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=965, ref allele: T)\n",
            "14:44:07.854 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=965, alt allele: C)\n",
            "14:44:07.855 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=700, ref allele: T)\n",
            "14:44:07.855 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=700, alt allele: C)\n",
            "14:44:07.945 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=809, ref allele: C)\n",
            "14:44:07.945 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=809, alt allele: G)\n",
            "14:44:07.945 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=728, ref allele: C)\n",
            "14:44:07.945 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=728, alt allele: G)\n",
            "14:44:07.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, ref allele: C)\n",
            "14:44:07.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, alt allele: T)\n",
            "14:44:07.953 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, ref allele: G)\n",
            "14:44:07.953 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, alt allele: A)\n",
            "14:44:07.955 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, ref allele: A)\n",
            "14:44:07.956 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, alt allele: G)\n",
            "14:44:07.958 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, ref allele: A)\n",
            "14:44:07.958 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, alt allele: G)\n",
            "14:44:07.960 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, ref allele: A)\n",
            "14:44:07.960 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, alt allele: G)\n",
            "14:44:07.966 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, ref allele: G)\n",
            "14:44:07.966 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2480, alt allele: A)\n",
            "14:44:08.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, ref allele: G)\n",
            "14:44:08.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, alt allele: A)\n",
            "14:44:08.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, ref allele: G)\n",
            "14:44:08.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, alt allele: A)\n",
            "14:44:08.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, ref allele: G)\n",
            "14:44:08.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, alt allele: A)\n",
            "14:44:08.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, ref allele: G)\n",
            "14:44:08.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, alt allele: A)\n",
            "14:44:08.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, ref allele: C)\n",
            "14:44:08.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1300, alt allele: T)\n",
            "14:44:08.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, ref allele: C)\n",
            "14:44:08.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, alt allele: T)\n",
            "14:44:08.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, ref allele: C)\n",
            "14:44:08.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, alt allele: T)\n",
            "14:44:08.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, ref allele: C)\n",
            "14:44:08.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, alt allele: T)\n",
            "14:44:08.250 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, ref allele: C)\n",
            "14:44:08.250 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, alt allele: T)\n",
            "14:44:08.250 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, ref allele: C)\n",
            "14:44:08.250 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, alt allele: T)\n",
            "14:44:08.251 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, ref allele: T)\n",
            "14:44:08.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, alt allele: C)\n",
            "14:44:08.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, ref allele: T)\n",
            "14:44:08.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, alt allele: C)\n",
            "14:44:08.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, ref allele: T)\n",
            "14:44:08.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, alt allele: C)\n",
            "14:44:08.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, ref allele: C)\n",
            "14:44:08.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2786, alt allele: T)\n",
            "14:44:08.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, ref allele: C)\n",
            "14:44:08.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2078, alt allele: T)\n",
            "14:44:08.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, ref allele: C)\n",
            "14:44:08.256 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1231, alt allele: T)\n",
            "14:44:08.261 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2677, ref allele: G)\n",
            "14:44:08.262 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2677, alt allele: C)\n",
            "14:44:08.356 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2063, ref allele: C)\n",
            "14:44:08.356 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2063, alt allele: T)\n",
            "14:44:08.357 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2063, ref allele: G)\n",
            "14:44:08.357 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2063, alt allele: T)\n",
            "14:44:08.418 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, ref allele: G)\n",
            "14:44:08.418 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, alt allele: A)\n",
            "14:44:08.419 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, ref allele: T)\n",
            "14:44:08.419 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, alt allele: C)\n",
            "14:44:08.421 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, ref allele: C)\n",
            "14:44:08.422 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, alt allele: T)\n",
            "14:44:08.423 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, ref allele: G)\n",
            "14:44:08.423 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, alt allele: A)\n",
            "14:44:08.424 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, ref allele: G)\n",
            "14:44:08.424 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5159, alt allele: A)\n",
            "14:44:08.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2635, ref allele: T)\n",
            "14:44:08.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2635, alt allele: C)\n",
            "14:44:08.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3961, ref allele: T)\n",
            "14:44:08.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3961, alt allele: C)\n",
            "14:44:08.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=785, ref allele: T)\n",
            "14:44:08.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=785, alt allele: C)\n",
            "14:44:08.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3961, ref allele: A)\n",
            "14:44:08.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3961, alt allele: G)\n",
            "14:44:08.689 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3961, ref allele: C)\n",
            "14:44:08.689 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3961, alt allele: T)\n",
            "14:44:08.761 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2000, ref allele: T)\n",
            "14:44:08.761 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2000, alt allele: C)\n",
            "14:44:08.770 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2000, ref allele: A)\n",
            "14:44:08.770 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2000, alt allele: G)\n",
            "14:44:08.777 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2000, ref allele: A)\n",
            "14:44:08.777 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2000, alt allele: T)\n",
            "14:44:08.915 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4307, ref allele: G)\n",
            "14:44:08.915 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4307, alt allele: A)\n",
            "14:44:09.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5234, ref allele: G)\n",
            "14:44:09.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5234, alt allele: A)\n",
            "14:44:09.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4133, ref allele: G)\n",
            "14:44:09.088 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4133, alt allele: A)\n",
            "14:44:09.117 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3272, ref allele: C)\n",
            "14:44:09.117 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3272, alt allele: T)\n",
            "14:44:09.117 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3272, ref allele: C)\n",
            "14:44:09.117 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3272, alt allele: T)\n",
            "14:44:09.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, ref allele: T)\n",
            "14:44:09.252 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, alt allele: C)\n",
            "14:44:09.259 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, ref allele: T)\n",
            "14:44:09.259 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, alt allele: C)\n",
            "14:44:09.261 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, ref allele: G)\n",
            "14:44:09.261 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, alt allele: A)\n",
            "14:44:09.263 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, ref allele: T)\n",
            "14:44:09.263 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2666, alt allele: C)\n",
            "14:44:09.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2773, ref allele: A)\n",
            "14:44:09.270 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2773, alt allele: C)\n",
            "14:44:09.271 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2773, ref allele: T)\n",
            "14:44:09.271 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2773, alt allele: C)\n",
            "14:44:09.272 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2773, ref allele: T)\n",
            "14:44:09.272 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2773, alt allele: C)\n",
            "14:44:09.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1408, ref allele: G)\n",
            "14:44:09.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1408, alt allele: C)\n",
            "14:44:09.296 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1183, ref allele: T)\n",
            "14:44:09.296 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1183, alt allele: C)\n",
            "14:44:09.296 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1183, ref allele: T)\n",
            "14:44:09.296 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1183, alt allele: C)\n",
            "14:44:09.366 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2110, ref allele: G)\n",
            "14:44:09.366 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2110, alt allele: C)\n",
            "14:44:09.366 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=952, ref allele: G)\n",
            "14:44:09.366 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=952, alt allele: C)\n",
            "14:44:09.367 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2110, ref allele: T)\n",
            "14:44:09.367 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2110, alt allele: C)\n",
            "14:44:09.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=820, ref allele: CTA)\n",
            "14:44:09.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1138, ref allele: G)\n",
            "14:44:09.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1138, alt allele: A)\n",
            "14:44:09.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1126, ref allele: G)\n",
            "14:44:09.458 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1126, alt allele: A)\n",
            "14:44:09.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1138, ref allele: G)\n",
            "14:44:09.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1138, alt allele: A)\n",
            "14:44:09.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1126, ref allele: G)\n",
            "14:44:09.459 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1126, alt allele: A)\n",
            "14:44:09.566 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4315, ref allele: T)\n",
            "14:44:09.566 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4315, alt allele: C)\n",
            "14:44:09.567 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3355, ref allele: T)\n",
            "14:44:09.567 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3355, alt allele: C)\n",
            "14:44:09.568 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4193, ref allele: T)\n",
            "14:44:09.568 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4193, alt allele: C)\n",
            "14:44:09.568 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3119, ref allele: T)\n",
            "14:44:09.568 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3119, alt allele: C)\n",
            "14:44:09.569 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2855, ref allele: T)\n",
            "14:44:09.569 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2855, alt allele: C)\n",
            "14:44:09.664 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5767, ref allele: C)\n",
            "14:44:09.664 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5767, alt allele: G)\n",
            "14:44:09.665 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2683, ref allele: C)\n",
            "14:44:09.665 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2683, alt allele: G)\n",
            "14:44:09.665 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4778, ref allele: C)\n",
            "14:44:09.665 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4778, alt allele: G)\n",
            "14:44:09.666 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4942, ref allele: C)\n",
            "14:44:09.666 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4942, alt allele: G)\n",
            "14:44:09.667 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4066, ref allele: C)\n",
            "14:44:09.667 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4066, alt allele: G)\n",
            "14:44:09.667 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3883, ref allele: C)\n",
            "14:44:09.668 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3883, alt allele: G)\n",
            "14:44:09.678 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5773, ref allele: CA)\n",
            "14:44:09.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4784, ref allele: CA)\n",
            "14:44:09.679 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4948, ref allele: CA)\n",
            "14:44:09.681 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4072, ref allele: CA)\n",
            "14:44:09.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5767, ref allele: C)\n",
            "14:44:09.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5767, alt allele: T)\n",
            "14:44:09.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4778, ref allele: C)\n",
            "14:44:09.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4778, alt allele: T)\n",
            "14:44:09.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4942, ref allele: C)\n",
            "14:44:09.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4942, alt allele: T)\n",
            "14:44:09.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4066, ref allele: C)\n",
            "14:44:09.685 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4066, alt allele: T)\n",
            "14:44:09.710 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5767, ref allele: C)\n",
            "14:44:09.711 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5767, alt allele: T)\n",
            "14:44:09.711 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4778, ref allele: C)\n",
            "14:44:09.711 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4778, alt allele: T)\n",
            "14:44:09.712 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4942, ref allele: C)\n",
            "14:44:09.712 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4942, alt allele: T)\n",
            "14:44:09.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4066, ref allele: C)\n",
            "14:44:09.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4066, alt allele: T)\n",
            "14:44:09.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3883, ref allele: C)\n",
            "14:44:09.714 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3883, alt allele: T)\n",
            "14:44:09.948 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2107, ref allele: C)\n",
            "14:44:09.948 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2107, alt allele: T)\n",
            "14:44:09.948 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2660, ref allele: C)\n",
            "14:44:09.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2660, alt allele: T)\n",
            "14:44:09.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2174, ref allele: C)\n",
            "14:44:09.949 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2174, alt allele: T)\n",
            "14:44:09.951 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2107, ref allele: C)\n",
            "14:44:09.951 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2107, alt allele: T)\n",
            "14:44:09.951 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2660, ref allele: C)\n",
            "14:44:09.951 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2660, alt allele: T)\n",
            "14:44:09.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2174, ref allele: C)\n",
            "14:44:09.952 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2174, alt allele: T)\n",
            "14:44:09.953 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2107, ref allele: G)\n",
            "14:44:09.953 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2107, alt allele: A)\n",
            "14:44:09.954 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2660, ref allele: G)\n",
            "14:44:09.954 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2660, alt allele: A)\n",
            "14:44:09.954 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2174, ref allele: G)\n",
            "14:44:09.954 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2174, alt allele: A)\n",
            "14:44:10.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4994, ref allele: C)\n",
            "14:44:10.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4994, alt allele: G)\n",
            "14:44:10.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: C)\n",
            "14:44:10.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: G)\n",
            "14:44:10.172 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1612, ref allele: C)\n",
            "14:44:10.172 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1612, alt allele: G)\n",
            "14:44:10.172 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1802, ref allele: C)\n",
            "14:44:10.172 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1802, alt allele: G)\n",
            "14:44:10.186 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2113, ref allele: C)\n",
            "14:44:10.186 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2113, alt allele: T)\n",
            "14:44:10.234 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2623, ref allele: C)\n",
            "14:44:10.234 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2623, alt allele: T)\n",
            "14:44:10.234 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1406, ref allele: C)\n",
            "14:44:10.234 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1406, alt allele: T)\n",
            "14:44:10.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2623, ref allele: T)\n",
            "14:44:10.238 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2623, alt allele: C)\n",
            "14:44:10.239 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1406, ref allele: T)\n",
            "14:44:10.239 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1406, alt allele: C)\n",
            "14:44:10.240 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, ref allele: G)\n",
            "14:44:10.241 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, alt allele: A)\n",
            "14:44:10.241 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, ref allele: G)\n",
            "14:44:10.241 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, alt allele: A)\n",
            "14:44:10.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, ref allele: A)\n",
            "14:44:10.243 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, alt allele: G)\n",
            "14:44:10.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, ref allele: A)\n",
            "14:44:10.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, alt allele: G)\n",
            "14:44:10.246 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, ref allele: A)\n",
            "14:44:10.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, alt allele: G)\n",
            "14:44:10.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, ref allele: A)\n",
            "14:44:10.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, alt allele: G)\n",
            "14:44:10.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, ref allele: G)\n",
            "14:44:10.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, alt allele: A)\n",
            "14:44:10.249 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, ref allele: G)\n",
            "14:44:10.249 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, alt allele: A)\n",
            "14:44:10.254 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, ref allele: G)\n",
            "14:44:10.254 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, alt allele: A)\n",
            "14:44:10.255 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, ref allele: G)\n",
            "14:44:10.255 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, alt allele: A)\n",
            "14:44:10.303 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, ref allele: A)\n",
            "14:44:10.304 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, alt allele: G)\n",
            "14:44:10.305 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, ref allele: A)\n",
            "14:44:10.305 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, alt allele: G)\n",
            "14:44:10.308 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, ref allele: G)\n",
            "14:44:10.309 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7960, alt allele: A)\n",
            "14:44:10.309 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, ref allele: G)\n",
            "14:44:10.309 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6283, alt allele: A)\n",
            "14:44:10.310 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2722, ref allele: G)\n",
            "14:44:10.310 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2722, alt allele: A)\n",
            "14:44:10.361 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2248, ref allele: C)\n",
            "14:44:10.361 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2248, alt allele: T)\n",
            "14:44:10.363 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2248, ref allele: G)\n",
            "14:44:10.363 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2248, alt allele: A)\n",
            "14:44:10.367 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, ref allele: A)\n",
            "14:44:10.367 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, alt allele: T)\n",
            "14:44:10.368 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, ref allele: T)\n",
            "14:44:10.369 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, alt allele: C)\n",
            "14:44:10.373 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, ref allele: G)\n",
            "14:44:10.373 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, alt allele: A)\n",
            "14:44:10.464 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, ref allele: T)\n",
            "14:44:10.464 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=8515, alt allele: C)\n",
            "14:44:10.592 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3821, ref allele: T)\n",
            "14:44:10.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3821, alt allele: C)\n",
            "14:44:10.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2122, ref allele: T)\n",
            "14:44:10.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2122, alt allele: C)\n",
            "14:44:10.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1700, ref allele: T)\n",
            "14:44:10.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1700, alt allele: C)\n",
            "14:44:10.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, ref allele: T)\n",
            "14:44:10.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, alt allele: C)\n",
            "14:44:10.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1867, ref allele: T)\n",
            "14:44:10.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1867, alt allele: C)\n",
            "14:44:10.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, ref allele: T)\n",
            "14:44:10.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, alt allele: C)\n",
            "14:44:10.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, ref allele: G)\n",
            "14:44:10.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, alt allele: A)\n",
            "14:44:10.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1867, ref allele: G)\n",
            "14:44:10.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1867, alt allele: A)\n",
            "14:44:10.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, ref allele: G)\n",
            "14:44:10.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, alt allele: A)\n",
            "14:44:10.688 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, ref allele: G)\n",
            "14:44:10.688 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, alt allele: A)\n",
            "14:44:10.688 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1867, ref allele: G)\n",
            "14:44:10.688 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1867, alt allele: A)\n",
            "14:44:10.688 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, ref allele: G)\n",
            "14:44:10.688 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, alt allele: A)\n",
            "14:44:10.755 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7823, ref allele: G)\n",
            "14:44:10.755 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7823, alt allele: A)\n",
            "14:44:10.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6728, ref allele: T)\n",
            "14:44:10.807 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6728, alt allele: C)\n",
            "14:44:10.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2468, ref allele: T)\n",
            "14:44:10.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2468, alt allele: C)\n",
            "14:44:10.808 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=746, ref allele: T)\n",
            "14:44:10.809 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=746, alt allele: C)\n",
            "14:44:10.872 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2593, ref allele: G)\n",
            "14:44:10.873 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2593, alt allele: A)\n",
            "14:44:10.873 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2368, ref allele: G)\n",
            "14:44:10.874 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2368, alt allele: A)\n",
            "14:44:10.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, ref allele: C)\n",
            "14:44:10.934 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1895, alt allele: T)\n",
            "14:44:10.982 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1936, ref allele: C)\n",
            "14:44:10.982 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1936, alt allele: T)\n",
            "14:44:10.984 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4150, ref allele: G)\n",
            "14:44:10.984 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4150, alt allele: A)\n",
            "14:44:10.984 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2441, ref allele: G)\n",
            "14:44:10.985 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2441, alt allele: A)\n",
            "14:44:10.985 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3523, ref allele: G)\n",
            "14:44:10.985 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3523, alt allele: A)\n",
            "14:44:10.986 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4150, ref allele: C)\n",
            "14:44:10.986 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4150, alt allele: T)\n",
            "14:44:10.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2441, ref allele: C)\n",
            "14:44:10.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2441, alt allele: T)\n",
            "14:44:10.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3523, ref allele: C)\n",
            "14:44:10.987 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3523, alt allele: T)\n",
            "14:44:11.032 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, ref allele: A)\n",
            "14:44:11.032 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, alt allele: G)\n",
            "14:44:11.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, ref allele: A)\n",
            "14:44:11.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, alt allele: G)\n",
            "14:44:11.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1358, ref allele: A)\n",
            "14:44:11.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1358, alt allele: G)\n",
            "14:44:11.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, ref allele: A)\n",
            "14:44:11.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, alt allele: G)\n",
            "14:44:11.034 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, ref allele: C)\n",
            "14:44:11.034 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, alt allele: G)\n",
            "14:44:11.034 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, ref allele: C)\n",
            "14:44:11.035 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, alt allele: G)\n",
            "14:44:11.035 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1358, ref allele: C)\n",
            "14:44:11.035 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1358, alt allele: G)\n",
            "14:44:11.035 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, ref allele: C)\n",
            "14:44:11.035 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, alt allele: G)\n",
            "14:44:11.036 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, ref allele: C)\n",
            "14:44:11.036 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, alt allele: T)\n",
            "14:44:11.036 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, ref allele: C)\n",
            "14:44:11.036 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, alt allele: T)\n",
            "14:44:11.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1358, ref allele: C)\n",
            "14:44:11.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1358, alt allele: T)\n",
            "14:44:11.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, ref allele: C)\n",
            "14:44:11.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, alt allele: T)\n",
            "14:44:11.038 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, ref allele: G)\n",
            "14:44:11.038 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2299, alt allele: A)\n",
            "14:44:11.038 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, ref allele: G)\n",
            "14:44:11.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2225, alt allele: A)\n",
            "14:44:11.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, ref allele: G)\n",
            "14:44:11.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3391, alt allele: A)\n",
            "14:44:11.070 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3334, ref allele: T)\n",
            "14:44:11.070 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3334, alt allele: C)\n",
            "14:44:11.219 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1930, ref allele: C)\n",
            "14:44:11.219 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1930, alt allele: T)\n",
            "14:44:11.337 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4207, ref allele: T)\n",
            "14:44:11.338 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4207, alt allele: C)\n",
            "14:44:11.338 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, ref allele: T)\n",
            "14:44:11.338 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, alt allele: C)\n",
            "14:44:11.339 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4207, ref allele: C)\n",
            "14:44:11.340 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4207, alt allele: T)\n",
            "14:44:11.340 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, ref allele: C)\n",
            "14:44:11.340 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, alt allele: T)\n",
            "14:44:11.341 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4207, ref allele: A)\n",
            "14:44:11.341 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4207, alt allele: G)\n",
            "14:44:11.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, ref allele: A)\n",
            "14:44:11.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2656, alt allele: G)\n",
            "14:44:11.398 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7378, ref allele: G)\n",
            "14:44:11.399 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7378, alt allele: A)\n",
            "14:44:11.400 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7378, ref allele: G)\n",
            "14:44:11.400 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7378, alt allele: A)\n",
            "14:44:11.406 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7378, ref allele: G)\n",
            "14:44:11.407 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7378, alt allele: A)\n",
            "14:44:11.407 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=929, ref allele: G)\n",
            "14:44:11.407 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=929, alt allele: A)\n",
            "14:44:11.440 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2503, ref allele: A)\n",
            "14:44:11.440 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2503, alt allele: G)\n",
            "14:44:11.482 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, ref allele: T)\n",
            "14:44:11.482 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, alt allele: C)\n",
            "14:44:11.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4016, ref allele: T)\n",
            "14:44:11.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4016, alt allele: C)\n",
            "14:44:11.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, ref allele: T)\n",
            "14:44:11.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, alt allele: C)\n",
            "14:44:11.484 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, ref allele: T)\n",
            "14:44:11.484 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, alt allele: C)\n",
            "14:44:11.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, ref allele: C)\n",
            "14:44:11.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, alt allele: T)\n",
            "14:44:11.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, ref allele: C)\n",
            "14:44:11.498 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, alt allele: T)\n",
            "14:44:11.498 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, ref allele: C)\n",
            "14:44:11.498 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, alt allele: T)\n",
            "14:44:11.499 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, ref allele: G)\n",
            "14:44:11.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, alt allele: A)\n",
            "14:44:11.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, ref allele: G)\n",
            "14:44:11.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, alt allele: A)\n",
            "14:44:11.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, ref allele: G)\n",
            "14:44:11.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, alt allele: A)\n",
            "14:44:11.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, ref allele: A)\n",
            "14:44:11.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, alt allele: T)\n",
            "14:44:11.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, ref allele: A)\n",
            "14:44:11.503 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, alt allele: T)\n",
            "14:44:11.503 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, ref allele: A)\n",
            "14:44:11.503 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, alt allele: T)\n",
            "14:44:11.505 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, ref allele: A)\n",
            "14:44:11.505 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7462, alt allele: G)\n",
            "14:44:11.506 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, ref allele: A)\n",
            "14:44:11.506 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7336, alt allele: G)\n",
            "14:44:11.506 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, ref allele: A)\n",
            "14:44:11.507 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, alt allele: G)\n",
            "14:44:11.549 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2539, ref allele: A)\n",
            "14:44:11.550 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2533, ref allele: A)\n",
            "14:44:11.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2533, alt allele: G)\n",
            "14:44:11.552 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2533, ref allele: C)\n",
            "14:44:11.552 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2533, alt allele: T)\n",
            "14:44:11.554 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2533, ref allele: A)\n",
            "14:44:11.554 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2533, alt allele: G)\n",
            "14:44:11.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3737, ref allele: G)\n",
            "14:44:11.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3737, alt allele: A)\n",
            "14:44:11.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3001, ref allele: G)\n",
            "14:44:11.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3001, alt allele: A)\n",
            "14:44:11.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: G)\n",
            "14:44:11.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: A)\n",
            "14:44:11.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=619, ref allele: TG)\n",
            "14:44:11.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=542, ref allele: TG)\n",
            "14:44:11.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=421, ref allele: TG)\n",
            "14:44:11.670 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=511, ref allele: TG)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=619, ref allele: A)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=619, alt allele: T)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=542, ref allele: A)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=542, alt allele: T)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=421, ref allele: A)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=421, alt allele: T)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=511, ref allele: A)\n",
            "14:44:11.673 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=511, alt allele: T)\n",
            "14:44:11.696 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, ref allele: A)\n",
            "14:44:11.696 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1691, alt allele: G)\n",
            "14:44:11.722 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=566, ref allele: A)\n",
            "14:44:11.722 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=566, alt allele: G)\n",
            "14:44:11.722 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=656, ref allele: A)\n",
            "14:44:11.722 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=656, alt allele: G)\n",
            "14:44:11.723 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=386, ref allele: A)\n",
            "14:44:11.723 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=386, alt allele: G)\n",
            "14:44:11.723 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=530, ref allele: A)\n",
            "14:44:11.723 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=530, alt allele: G)\n",
            "14:44:11.723 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2518, ref allele: A)\n",
            "14:44:11.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2518, alt allele: G)\n",
            "14:44:11.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1888, ref allele: A)\n",
            "14:44:11.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1888, alt allele: G)\n",
            "14:44:11.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=422, ref allele: A)\n",
            "14:44:11.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=422, alt allele: G)\n",
            "14:44:11.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2419, ref allele: G)\n",
            "14:44:11.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2419, alt allele: T)\n",
            "14:44:11.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, ref allele: G)\n",
            "14:44:11.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, alt allele: T)\n",
            "14:44:11.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3770, ref allele: G)\n",
            "14:44:11.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3770, alt allele: T)\n",
            "14:44:11.761 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2419, ref allele: A)\n",
            "14:44:11.761 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2419, alt allele: G)\n",
            "14:44:11.761 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, ref allele: A)\n",
            "14:44:11.762 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, alt allele: G)\n",
            "14:44:11.762 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3770, ref allele: A)\n",
            "14:44:11.762 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3770, alt allele: G)\n",
            "14:44:11.763 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2419, ref allele: G)\n",
            "14:44:11.763 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2419, alt allele: C)\n",
            "14:44:11.763 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, ref allele: G)\n",
            "14:44:11.763 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4216, alt allele: C)\n",
            "14:44:11.763 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3770, ref allele: G)\n",
            "14:44:11.763 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3770, alt allele: C)\n",
            "14:44:11.766 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, ref allele: A)\n",
            "14:44:11.766 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, alt allele: G)\n",
            "14:44:11.767 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, ref allele: A)\n",
            "14:44:11.767 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, alt allele: G)\n",
            "14:44:11.770 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, ref allele: T)\n",
            "14:44:11.770 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, alt allele: C)\n",
            "14:44:11.770 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, ref allele: T)\n",
            "14:44:11.770 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, alt allele: C)\n",
            "14:44:11.771 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, ref allele: G)\n",
            "14:44:11.771 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, alt allele: A)\n",
            "14:44:11.771 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, ref allele: G)\n",
            "14:44:11.772 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, alt allele: A)\n",
            "14:44:11.772 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, ref allele: C)\n",
            "14:44:11.773 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6052, alt allele: T)\n",
            "14:44:11.773 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, ref allele: C)\n",
            "14:44:11.773 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6092, alt allele: T)\n",
            "14:44:11.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2519, ref allele: T)\n",
            "14:44:11.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2519, alt allele: C)\n",
            "14:44:11.850 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2519, ref allele: A)\n",
            "14:44:11.850 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2519, alt allele: C)\n",
            "14:44:11.892 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=14000, ref allele: C)\n",
            "14:44:11.893 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=14000, alt allele: T)\n",
            "14:44:11.899 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=14000, ref allele: C)\n",
            "14:44:11.900 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=14000, alt allele: T)\n",
            "14:44:12.145 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2120, ref allele: G)\n",
            "14:44:12.145 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2120, alt allele: A)\n",
            "14:44:12.178 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=719, ref allele: T)\n",
            "14:44:12.178 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=719, alt allele: G)\n",
            "14:44:12.451 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4037, ref allele: G)\n",
            "14:44:12.451 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4037, alt allele: A)\n",
            "14:44:12.452 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4037, ref allele: C)\n",
            "14:44:12.452 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4037, alt allele: T)\n",
            "14:44:12.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4037, ref allele: C)\n",
            "14:44:12.454 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4037, alt allele: T)\n",
            "14:44:12.472 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4123, ref allele: A)\n",
            "14:44:12.472 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4123, alt allele: G)\n",
            "14:44:12.475 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3536, ref allele: A)\n",
            "14:44:12.475 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3536, alt allele: G)\n",
            "14:44:12.475 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3049, ref allele: A)\n",
            "14:44:12.475 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3049, alt allele: G)\n",
            "14:44:12.478 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4123, ref allele: A)\n",
            "14:44:12.478 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4123, alt allele: G)\n",
            "14:44:12.479 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3536, ref allele: A)\n",
            "14:44:12.479 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3536, alt allele: G)\n",
            "14:44:12.480 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3049, ref allele: A)\n",
            "14:44:12.480 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3049, alt allele: G)\n",
            "14:44:12.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1969, ref allele: A)\n",
            "14:44:12.503 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1969, alt allele: G)\n",
            "14:44:12.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1969, ref allele: C)\n",
            "14:44:12.511 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1969, alt allele: T)\n",
            "14:44:12.529 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3613, ref allele: T)\n",
            "14:44:12.529 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3613, alt allele: G)\n",
            "14:44:12.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1958, ref allele: T)\n",
            "14:44:12.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1958, alt allele: G)\n",
            "14:44:12.535 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3613, ref allele: G)\n",
            "14:44:12.535 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3613, alt allele: A)\n",
            "14:44:12.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1958, ref allele: G)\n",
            "14:44:12.536 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1958, alt allele: A)\n",
            "14:44:12.565 INFO  ProgressMeter -       chr11:86268594              0.5                  4000           8830.0\n",
            "14:44:12.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2510, ref allele: G)\n",
            "14:44:12.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2510, alt allele: A)\n",
            "14:44:12.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2528, ref allele: G)\n",
            "14:44:12.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2528, alt allele: A)\n",
            "14:44:12.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1814, ref allele: G)\n",
            "14:44:12.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1814, alt allele: A)\n",
            "14:44:12.624 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, ref allele: G)\n",
            "14:44:12.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, alt allele: C)\n",
            "14:44:12.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, ref allele: G)\n",
            "14:44:12.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, alt allele: C)\n",
            "14:44:12.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, ref allele: G)\n",
            "14:44:12.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, alt allele: A)\n",
            "14:44:12.657 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, ref allele: G)\n",
            "14:44:12.658 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, alt allele: A)\n",
            "14:44:12.660 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, ref allele: G)\n",
            "14:44:12.660 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, alt allele: T)\n",
            "14:44:12.660 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, ref allele: G)\n",
            "14:44:12.660 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2311, alt allele: T)\n",
            "14:44:12.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3938, ref allele: G)\n",
            "14:44:12.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3938, alt allele: A)\n",
            "14:44:13.065 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1465, ref allele: T)\n",
            "14:44:13.065 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1465, alt allele: C)\n",
            "14:44:13.091 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, ref allele: T)\n",
            "14:44:13.097 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1921, alt allele: C)\n",
            "14:44:13.174 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7927, ref allele: A)\n",
            "14:44:13.174 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7927, alt allele: C)\n",
            "14:44:13.175 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4475, ref allele: A)\n",
            "14:44:13.175 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4475, alt allele: C)\n",
            "14:44:13.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4571, ref allele: A)\n",
            "14:44:13.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4571, alt allele: C)\n",
            "14:44:13.180 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7927, ref allele: C)\n",
            "14:44:13.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7927, alt allele: A)\n",
            "14:44:13.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4475, ref allele: C)\n",
            "14:44:13.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4475, alt allele: A)\n",
            "14:44:13.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4571, ref allele: C)\n",
            "14:44:13.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4571, alt allele: A)\n",
            "14:44:13.231 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4184, ref allele: G)\n",
            "14:44:13.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4184, alt allele: A)\n",
            "14:44:13.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3347, ref allele: G)\n",
            "14:44:13.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3347, alt allele: A)\n",
            "14:44:13.233 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2156, ref allele: G)\n",
            "14:44:13.233 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2156, alt allele: A)\n",
            "14:44:13.235 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2039, ref allele: G)\n",
            "14:44:13.235 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2039, alt allele: A)\n",
            "14:44:13.236 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1774, ref allele: G)\n",
            "14:44:13.236 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1774, alt allele: A)\n",
            "14:44:13.372 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2654, ref allele: C)\n",
            "14:44:13.372 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2654, alt allele: A)\n",
            "14:44:13.373 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2003, ref allele: C)\n",
            "14:44:13.373 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2003, alt allele: A)\n",
            "14:44:13.373 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4805, ref allele: C)\n",
            "14:44:13.373 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4805, alt allele: A)\n",
            "14:44:13.482 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, ref allele: A)\n",
            "14:44:13.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, alt allele: G)\n",
            "14:44:13.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, ref allele: T)\n",
            "14:44:13.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, alt allele: C)\n",
            "14:44:13.489 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, ref allele: A)\n",
            "14:44:13.489 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, alt allele: G)\n",
            "14:44:13.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, ref allele: G)\n",
            "14:44:13.497 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, alt allele: T)\n",
            "14:44:13.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, ref allele: G)\n",
            "14:44:13.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, alt allele: A)\n",
            "14:44:13.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7894, ref allele: G)\n",
            "14:44:13.503 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7894, alt allele: A)\n",
            "14:44:13.504 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, ref allele: A)\n",
            "14:44:13.504 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, alt allele: G)\n",
            "14:44:13.505 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7894, ref allele: A)\n",
            "14:44:13.506 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7894, alt allele: G)\n",
            "14:44:13.508 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, ref allele: G)\n",
            "14:44:13.509 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=19030, alt allele: A)\n",
            "14:44:13.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7894, ref allele: G)\n",
            "14:44:13.511 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7894, alt allele: A)\n",
            "14:44:13.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=889, ref allele: G)\n",
            "14:44:13.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=889, alt allele: A)\n",
            "14:44:13.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=472, ref allele: G)\n",
            "14:44:13.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=472, alt allele: A)\n",
            "14:44:13.546 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=889, ref allele: G)\n",
            "14:44:13.546 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=889, alt allele: A)\n",
            "14:44:13.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=472, ref allele: G)\n",
            "14:44:13.548 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=472, alt allele: A)\n",
            "14:44:13.589 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2417, ref allele: AAAG)\n",
            "14:44:13.589 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2417, alt allele: A)\n",
            "14:44:13.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, ref allele: AAAG)\n",
            "14:44:13.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2522, alt allele: A)\n",
            "14:44:13.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1862, ref allele: AAAG)\n",
            "14:44:13.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1862, alt allele: A)\n",
            "14:44:13.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1733, ref allele: AAAG)\n",
            "14:44:13.590 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1733, alt allele: A)\n",
            "14:44:13.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, ref allele: AAAG)\n",
            "14:44:13.591 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, alt allele: A)\n",
            "14:44:13.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, ref allele: C)\n",
            "14:44:13.595 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, alt allele: G)\n",
            "14:44:13.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2317, ref allele: C)\n",
            "14:44:13.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2317, alt allele: G)\n",
            "14:44:13.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2047, ref allele: C)\n",
            "14:44:13.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2047, alt allele: G)\n",
            "14:44:13.597 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, ref allele: G)\n",
            "14:44:13.597 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, alt allele: C)\n",
            "14:44:13.598 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2317, ref allele: G)\n",
            "14:44:13.598 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2317, alt allele: C)\n",
            "14:44:13.598 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2047, ref allele: G)\n",
            "14:44:13.598 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2047, alt allele: C)\n",
            "14:44:13.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5345, ref allele: A)\n",
            "14:44:13.642 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5345, alt allele: G)\n",
            "14:44:13.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5345, ref allele: A)\n",
            "14:44:13.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5345, alt allele: G)\n",
            "14:44:13.661 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2750, ref allele: A)\n",
            "14:44:13.662 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2750, alt allele: C)\n",
            "14:44:13.663 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2750, ref allele: A)\n",
            "14:44:13.663 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2750, alt allele: G)\n",
            "14:44:13.737 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1844, ref allele: C)\n",
            "14:44:13.737 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1844, alt allele: G)\n",
            "14:44:13.758 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, ref allele: T)\n",
            "14:44:13.758 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, alt allele: C)\n",
            "14:44:13.759 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, ref allele: A)\n",
            "14:44:13.759 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, alt allele: T)\n",
            "14:44:13.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, ref allele: G)\n",
            "14:44:13.760 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, alt allele: A)\n",
            "14:44:13.762 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, ref allele: G)\n",
            "14:44:13.762 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, alt allele: C)\n",
            "14:44:13.764 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, ref allele: C)\n",
            "14:44:13.764 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, alt allele: T)\n",
            "14:44:13.765 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, ref allele: C)\n",
            "14:44:13.765 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, alt allele: T)\n",
            "14:44:13.766 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, ref allele: C)\n",
            "14:44:13.766 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3199, alt allele: T)\n",
            "14:44:13.845 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2339, ref allele: A)\n",
            "14:44:13.845 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2339, alt allele: G)\n",
            "14:44:13.865 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4687, ref allele: G)\n",
            "14:44:13.865 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4687, alt allele: T)\n",
            "14:44:13.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3952, ref allele: T)\n",
            "14:44:13.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3952, alt allele: C)\n",
            "14:44:13.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2546, ref allele: T)\n",
            "14:44:13.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2546, alt allele: C)\n",
            "14:44:13.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7118, ref allele: GCAGCAGCAA)\n",
            "14:44:13.998 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7118, alt allele: A)\n",
            "14:44:13.998 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3166, ref allele: GCAGCAGCAA)\n",
            "14:44:13.998 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3166, alt allele: A)\n",
            "14:44:13.999 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, ref allele: T)\n",
            "14:44:14.000 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=7106, alt allele: C)\n",
            "14:44:14.000 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3154, ref allele: T)\n",
            "14:44:14.000 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3154, alt allele: C)\n",
            "14:44:14.043 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2941, ref allele: A)\n",
            "14:44:14.043 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2941, alt allele: G)\n",
            "14:44:14.132 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, ref allele: T)\n",
            "14:44:14.133 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, alt allele: G)\n",
            "14:44:14.133 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, ref allele: T)\n",
            "14:44:14.133 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, alt allele: G)\n",
            "14:44:14.136 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, ref allele: T)\n",
            "14:44:14.136 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, alt allele: G)\n",
            "14:44:14.136 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, ref allele: T)\n",
            "14:44:14.136 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, alt allele: G)\n",
            "14:44:14.139 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, ref allele: T)\n",
            "14:44:14.139 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, alt allele: C)\n",
            "14:44:14.140 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, ref allele: T)\n",
            "14:44:14.140 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, alt allele: C)\n",
            "14:44:14.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, ref allele: A)\n",
            "14:44:14.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, alt allele: G)\n",
            "14:44:14.144 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, ref allele: A)\n",
            "14:44:14.144 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, alt allele: G)\n",
            "14:44:14.145 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4303, ref allele: C)\n",
            "14:44:14.146 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, ref allele: C)\n",
            "14:44:14.168 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, ref allele: T)\n",
            "14:44:14.169 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3800, alt allele: A)\n",
            "14:44:14.169 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3679, ref allele: T)\n",
            "14:44:14.170 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3679, alt allele: A)\n",
            "14:44:14.170 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5734, ref allele: T)\n",
            "14:44:14.170 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5734, alt allele: A)\n",
            "14:44:14.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2368, ref allele: C)\n",
            "14:44:14.220 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2368, alt allele: T)\n",
            "14:44:14.222 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7039, ref allele: C)\n",
            "14:44:14.222 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7039, alt allele: T)\n",
            "14:44:14.223 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7039, ref allele: G)\n",
            "14:44:14.223 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7039, alt allele: A)\n",
            "14:44:14.278 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, ref allele: G)\n",
            "14:44:14.278 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, alt allele: A)\n",
            "14:44:14.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, ref allele: A)\n",
            "14:44:14.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, alt allele: T)\n",
            "14:44:14.293 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, ref allele: A)\n",
            "14:44:14.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, alt allele: G)\n",
            "14:44:14.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, ref allele: C)\n",
            "14:44:14.294 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, alt allele: T)\n",
            "14:44:14.295 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, ref allele: C)\n",
            "14:44:14.295 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1913, alt allele: T)\n",
            "14:44:14.299 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3056, ref allele: C)\n",
            "14:44:14.299 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3056, alt allele: T)\n",
            "14:44:14.301 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3056, ref allele: A)\n",
            "14:44:14.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3056, alt allele: G)\n",
            "14:44:14.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3056, ref allele: C)\n",
            "14:44:14.303 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3056, alt allele: T)\n",
            "14:44:14.315 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1970, ref allele: A)\n",
            "14:44:14.315 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1970, alt allele: G)\n",
            "14:44:14.386 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: C)\n",
            "14:44:14.387 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: T)\n",
            "14:44:14.388 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2984, ref allele: C)\n",
            "14:44:14.388 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2984, alt allele: T)\n",
            "14:44:14.393 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: A)\n",
            "14:44:14.393 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: T)\n",
            "14:44:14.397 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: G)\n",
            "14:44:14.398 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: A)\n",
            "14:44:14.405 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: A)\n",
            "14:44:14.405 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: G)\n",
            "14:44:14.410 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: A)\n",
            "14:44:14.411 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: G)\n",
            "14:44:14.416 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: C)\n",
            "14:44:14.417 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: A)\n",
            "14:44:14.425 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: T)\n",
            "14:44:14.425 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: C)\n",
            "14:44:14.441 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: G)\n",
            "14:44:14.441 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: A)\n",
            "14:44:14.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: T)\n",
            "14:44:14.483 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: G)\n",
            "14:44:14.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: G)\n",
            "14:44:14.495 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: A)\n",
            "14:44:14.498 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, ref allele: C)\n",
            "14:44:14.499 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=13678, alt allele: T)\n",
            "14:44:14.580 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4052, ref allele: T)\n",
            "14:44:14.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4052, alt allele: C)\n",
            "14:44:14.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2609, ref allele: G)\n",
            "14:44:14.600 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2609, alt allele: A)\n",
            "14:44:14.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2158, ref allele: A)\n",
            "14:44:14.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2158, alt allele: G)\n",
            "14:44:14.637 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1340, ref allele: A)\n",
            "14:44:14.637 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1340, alt allele: G)\n",
            "14:44:14.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1414, ref allele: A)\n",
            "14:44:14.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1414, alt allele: G)\n",
            "14:44:14.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1237, ref allele: A)\n",
            "14:44:14.645 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1237, alt allele: G)\n",
            "14:44:14.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=466, ref allele: C)\n",
            "14:44:14.672 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=466, alt allele: T)\n",
            "14:44:14.815 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3278, ref allele: A)\n",
            "14:44:14.815 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3278, alt allele: G)\n",
            "14:44:15.057 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8807, ref allele: A)\n",
            "14:44:15.057 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8807, alt allele: G)\n",
            "14:44:15.058 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9542, ref allele: A)\n",
            "14:44:15.058 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9542, alt allele: G)\n",
            "14:44:15.059 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8807, ref allele: A)\n",
            "14:44:15.060 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=8807, alt allele: G)\n",
            "14:44:15.060 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9542, ref allele: A)\n",
            "14:44:15.060 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=9542, alt allele: G)\n",
            "14:44:15.107 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2623, ref allele: T)\n",
            "14:44:15.107 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2623, alt allele: C)\n",
            "14:44:15.108 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, ref allele: T)\n",
            "14:44:15.109 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, alt allele: C)\n",
            "14:44:15.169 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1018, ref allele: C)\n",
            "14:44:15.169 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1018, alt allele: A)\n",
            "14:44:15.169 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1394, ref allele: C)\n",
            "14:44:15.169 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1394, alt allele: A)\n",
            "14:44:15.170 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2066, ref allele: A)\n",
            "14:44:15.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2066, alt allele: G)\n",
            "14:44:15.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1763, ref allele: A)\n",
            "14:44:15.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1763, alt allele: G)\n",
            "14:44:15.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1703, ref allele: A)\n",
            "14:44:15.171 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1703, alt allele: G)\n",
            "14:44:15.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, ref allele: T)\n",
            "14:44:15.244 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, alt allele: C)\n",
            "14:44:15.245 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, ref allele: T)\n",
            "14:44:15.245 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, alt allele: C)\n",
            "14:44:15.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, ref allele: T)\n",
            "14:44:15.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2939, alt allele: A)\n",
            "14:44:15.248 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, ref allele: T)\n",
            "14:44:15.249 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, alt allele: A)\n",
            "14:44:15.251 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2885, ref allele: A)\n",
            "14:44:15.251 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2885, alt allele: C)\n",
            "14:44:15.251 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=599, ref allele: A)\n",
            "14:44:15.251 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=599, alt allele: C)\n",
            "14:44:15.305 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2930, ref allele: T)\n",
            "14:44:15.305 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2930, alt allele: C)\n",
            "14:44:15.311 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2930, ref allele: A)\n",
            "14:44:15.311 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2930, alt allele: C)\n",
            "14:44:15.311 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1868, ref allele: A)\n",
            "14:44:15.311 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1868, alt allele: C)\n",
            "14:44:15.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4922, ref allele: T)\n",
            "14:44:15.486 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4922, alt allele: G)\n",
            "14:44:15.487 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, ref allele: T)\n",
            "14:44:15.487 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2993, alt allele: G)\n",
            "14:44:15.488 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2510, ref allele: T)\n",
            "14:44:15.488 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2510, alt allele: G)\n",
            "14:44:15.488 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5837, ref allele: T)\n",
            "14:44:15.488 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5837, alt allele: G)\n",
            "14:44:15.521 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2315, ref allele: A)\n",
            "14:44:15.521 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2315, alt allele: C)\n",
            "14:44:15.522 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2509, ref allele: A)\n",
            "14:44:15.522 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2509, alt allele: C)\n",
            "14:44:15.522 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, ref allele: A)\n",
            "14:44:15.522 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2240, alt allele: C)\n",
            "14:44:15.524 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: G)\n",
            "14:44:15.525 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: A)\n",
            "14:44:15.525 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: G)\n",
            "14:44:15.525 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: C)\n",
            "14:44:15.526 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: C)\n",
            "14:44:15.526 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: T)\n",
            "14:44:15.527 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, ref allele: G)\n",
            "14:44:15.527 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2543, alt allele: A)\n",
            "14:44:15.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, ref allele: T)\n",
            "14:44:15.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, alt allele: C)\n",
            "14:44:15.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2473, ref allele: T)\n",
            "14:44:15.530 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2473, alt allele: C)\n",
            "14:44:15.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2467, ref allele: T)\n",
            "14:44:15.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2467, alt allele: C)\n",
            "14:44:15.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1493, ref allele: T)\n",
            "14:44:15.531 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1493, alt allele: C)\n",
            "14:44:15.564 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, ref allele: C)\n",
            "14:44:15.564 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2789, alt allele: T)\n",
            "14:44:15.564 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2473, ref allele: C)\n",
            "14:44:15.564 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2473, alt allele: T)\n",
            "14:44:15.565 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2467, ref allele: C)\n",
            "14:44:15.565 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2467, alt allele: T)\n",
            "14:44:15.565 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1493, ref allele: C)\n",
            "14:44:15.565 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1493, alt allele: T)\n",
            "14:44:15.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1811, ref allele: T)\n",
            "14:44:15.636 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1811, alt allele: C)\n",
            "14:44:15.639 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1811, ref allele: T)\n",
            "14:44:15.639 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1811, alt allele: C)\n",
            "14:44:15.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4669, ref allele: G)\n",
            "14:44:15.686 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4669, alt allele: A)\n",
            "14:44:15.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3431, ref allele: G)\n",
            "14:44:15.687 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3431, alt allele: A)\n",
            "14:44:15.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1888, ref allele: T)\n",
            "14:44:15.921 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1888, alt allele: C)\n",
            "14:44:15.922 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2006, ref allele: T)\n",
            "14:44:15.922 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2006, alt allele: C)\n",
            "14:44:15.923 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1888, ref allele: G)\n",
            "14:44:15.923 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1888, alt allele: A)\n",
            "14:44:15.924 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2006, ref allele: G)\n",
            "14:44:15.924 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2006, alt allele: A)\n",
            "14:44:15.926 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2006, ref allele: T)\n",
            "14:44:15.926 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2006, alt allele: C)\n",
            "14:44:15.927 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1687, ref allele: T)\n",
            "14:44:15.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1687, alt allele: C)\n",
            "14:44:15.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1822, ref allele: T)\n",
            "14:44:15.928 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1822, alt allele: C)\n",
            "14:44:15.929 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1687, ref allele: T)\n",
            "14:44:15.929 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1687, alt allele: C)\n",
            "14:44:15.930 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1822, ref allele: T)\n",
            "14:44:15.930 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1822, alt allele: C)\n",
            "14:44:15.974 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1873, ref allele: A)\n",
            "14:44:15.975 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1873, alt allele: G)\n",
            "14:44:16.274 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1793, ref allele: G)\n",
            "14:44:16.274 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1793, alt allele: A)\n",
            "14:44:16.335 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=928, ref allele: C)\n",
            "14:44:16.335 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=928, alt allele: T)\n",
            "14:44:16.338 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4649, ref allele: T)\n",
            "14:44:16.338 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4649, alt allele: C)\n",
            "14:44:16.341 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4649, ref allele: C)\n",
            "14:44:16.341 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4649, alt allele: T)\n",
            "14:44:16.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3332, ref allele: C)\n",
            "14:44:16.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3332, alt allele: T)\n",
            "14:44:16.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4649, ref allele: G)\n",
            "14:44:16.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4649, alt allele: A)\n",
            "14:44:16.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3332, ref allele: G)\n",
            "14:44:16.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3332, alt allele: A)\n",
            "14:44:16.386 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, ref allele: T)\n",
            "14:44:16.386 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, alt allele: C)\n",
            "14:44:16.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, ref allele: A)\n",
            "14:44:16.390 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, alt allele: G)\n",
            "14:44:16.391 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, ref allele: C)\n",
            "14:44:16.392 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, alt allele: T)\n",
            "14:44:16.392 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, ref allele: G)\n",
            "14:44:16.393 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5630, alt allele: T)\n",
            "14:44:16.396 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, ref allele: A)\n",
            "14:44:16.396 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, alt allele: G)\n",
            "14:44:16.398 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, ref allele: C)\n",
            "14:44:16.399 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, alt allele: T)\n",
            "14:44:16.403 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, ref allele: C)\n",
            "14:44:16.403 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, alt allele: T)\n",
            "14:44:16.405 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, ref allele: A)\n",
            "14:44:16.405 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, alt allele: G)\n",
            "14:44:16.406 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, ref allele: A)\n",
            "14:44:16.406 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, alt allele: G)\n",
            "14:44:16.425 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, ref allele: C)\n",
            "14:44:16.425 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, alt allele: T)\n",
            "14:44:16.427 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, ref allele: C)\n",
            "14:44:16.427 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6899, alt allele: A)\n",
            "14:44:16.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, ref allele: T)\n",
            "14:44:16.494 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, alt allele: C)\n",
            "14:44:16.498 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, ref allele: G)\n",
            "14:44:16.498 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, alt allele: C)\n",
            "14:44:16.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, ref allele: A)\n",
            "14:44:16.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, alt allele: G)\n",
            "14:44:16.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, ref allele: C)\n",
            "14:44:16.502 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3269, alt allele: G)\n",
            "14:44:16.508 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, ref allele: A)\n",
            "14:44:16.508 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, alt allele: G)\n",
            "14:44:16.509 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, ref allele: G)\n",
            "14:44:16.509 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, alt allele: A)\n",
            "14:44:16.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, ref allele: A)\n",
            "14:44:16.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, alt allele: G)\n",
            "14:44:16.512 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, ref allele: A)\n",
            "14:44:16.512 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3695, alt allele: G)\n",
            "14:44:16.577 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1634, ref allele: T)\n",
            "14:44:16.577 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1634, alt allele: G)\n",
            "14:44:16.577 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2074, ref allele: T)\n",
            "14:44:16.577 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2074, alt allele: G)\n",
            "14:44:16.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1634, ref allele: T)\n",
            "14:44:16.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1634, alt allele: C)\n",
            "14:44:16.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2074, ref allele: T)\n",
            "14:44:16.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2074, alt allele: C)\n",
            "14:44:16.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1508, ref allele: T)\n",
            "14:44:16.593 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1508, alt allele: C)\n",
            "14:44:16.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1508, ref allele: T)\n",
            "14:44:16.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1508, alt allele: A)\n",
            "14:44:16.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3968, ref allele: T)\n",
            "14:44:16.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3968, alt allele: C)\n",
            "14:44:16.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1381, ref allele: T)\n",
            "14:44:16.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1381, alt allele: C)\n",
            "14:44:16.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3968, ref allele: A)\n",
            "14:44:16.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3968, alt allele: G)\n",
            "14:44:16.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1381, ref allele: A)\n",
            "14:44:16.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1381, alt allele: G)\n",
            "14:44:16.632 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1289, ref allele: C)\n",
            "14:44:16.632 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1289, alt allele: T)\n",
            "14:44:16.676 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2678, ref allele: T)\n",
            "14:44:16.676 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2678, alt allele: C)\n",
            "14:44:16.677 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2678, ref allele: G)\n",
            "14:44:16.677 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2678, alt allele: T)\n",
            "14:44:16.725 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=13655, ref allele: C)\n",
            "14:44:16.726 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=13655, alt allele: T)\n",
            "14:44:16.727 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2395, ref allele: A)\n",
            "14:44:16.727 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2395, alt allele: G)\n",
            "14:44:16.728 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2228, ref allele: A)\n",
            "14:44:16.728 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2228, alt allele: G)\n",
            "14:44:16.728 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1456, ref allele: A)\n",
            "14:44:16.728 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1456, alt allele: G)\n",
            "14:44:16.728 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2630, ref allele: A)\n",
            "14:44:16.729 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2630, alt allele: G)\n",
            "14:44:16.744 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1819, ref allele: G)\n",
            "14:44:16.744 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1819, alt allele: A)\n",
            "14:44:16.811 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1709, ref allele: T)\n",
            "14:44:16.812 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1709, alt allele: C)\n",
            "14:44:16.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3845, ref allele: T)\n",
            "14:44:16.813 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3845, alt allele: C)\n",
            "14:44:16.819 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5753, ref allele: A)\n",
            "14:44:16.819 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5753, alt allele: G)\n",
            "14:44:16.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5479, ref allele: A)\n",
            "14:44:16.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5479, alt allele: G)\n",
            "14:44:16.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5240, ref allele: A)\n",
            "14:44:16.820 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5240, alt allele: G)\n",
            "14:44:16.823 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5753, ref allele: C)\n",
            "14:44:16.824 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5753, alt allele: T)\n",
            "14:44:16.824 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5479, ref allele: C)\n",
            "14:44:16.824 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5479, alt allele: T)\n",
            "14:44:16.825 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5240, ref allele: C)\n",
            "14:44:16.826 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5240, alt allele: T)\n",
            "14:44:16.827 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2771, ref allele: C)\n",
            "14:44:16.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2771, alt allele: T)\n",
            "14:44:16.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1718, ref allele: C)\n",
            "14:44:16.828 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1718, alt allele: T)\n",
            "14:44:16.829 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2771, ref allele: G)\n",
            "14:44:16.829 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2771, alt allele: A)\n",
            "14:44:16.830 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1718, ref allele: G)\n",
            "14:44:16.830 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1718, alt allele: A)\n",
            "14:44:16.943 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=916, ref allele: A)\n",
            "14:44:16.943 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=916, alt allele: G)\n",
            "14:44:16.955 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1202, ref allele: C)\n",
            "14:44:16.955 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1202, alt allele: T)\n",
            "14:44:16.965 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1759, ref allele: C)\n",
            "14:44:16.966 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1759, alt allele: T)\n",
            "14:44:16.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2606, ref allele: G)\n",
            "14:44:16.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2606, alt allele: A)\n",
            "14:44:16.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2356, ref allele: G)\n",
            "14:44:16.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2356, alt allele: A)\n",
            "14:44:16.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2102, ref allele: G)\n",
            "14:44:16.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2102, alt allele: A)\n",
            "14:44:16.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2032, ref allele: G)\n",
            "14:44:16.968 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2032, alt allele: A)\n",
            "14:44:17.023 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4529, ref allele: A)\n",
            "14:44:17.023 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4529, alt allele: G)\n",
            "14:44:17.024 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4360, ref allele: A)\n",
            "14:44:17.024 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4360, alt allele: G)\n",
            "14:44:17.024 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, ref allele: A)\n",
            "14:44:17.024 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, alt allele: G)\n",
            "14:44:17.027 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4529, ref allele: C)\n",
            "14:44:17.027 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4529, alt allele: T)\n",
            "14:44:17.027 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4360, ref allele: C)\n",
            "14:44:17.027 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4360, alt allele: T)\n",
            "14:44:17.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, ref allele: C)\n",
            "14:44:17.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, alt allele: T)\n",
            "14:44:17.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, ref allele: C)\n",
            "14:44:17.028 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, alt allele: T)\n",
            "14:44:17.029 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4529, ref allele: C)\n",
            "14:44:17.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4529, alt allele: T)\n",
            "14:44:17.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4360, ref allele: C)\n",
            "14:44:17.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4360, alt allele: T)\n",
            "14:44:17.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, ref allele: C)\n",
            "14:44:17.030 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2162, alt allele: T)\n",
            "14:44:17.031 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, ref allele: C)\n",
            "14:44:17.031 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3358, alt allele: T)\n",
            "14:44:17.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, ref allele: C)\n",
            "14:44:17.033 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, alt allele: G)\n",
            "14:44:17.034 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, ref allele: C)\n",
            "14:44:17.034 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, alt allele: G)\n",
            "14:44:17.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, ref allele: C)\n",
            "14:44:17.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, alt allele: T)\n",
            "14:44:17.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, ref allele: C)\n",
            "14:44:17.037 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, alt allele: T)\n",
            "14:44:17.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, ref allele: C)\n",
            "14:44:17.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, alt allele: T)\n",
            "14:44:17.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, ref allele: C)\n",
            "14:44:17.039 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, alt allele: T)\n",
            "14:44:17.042 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, ref allele: A)\n",
            "14:44:17.042 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, alt allele: G)\n",
            "14:44:17.042 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, ref allele: A)\n",
            "14:44:17.042 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3388, alt allele: G)\n",
            "14:44:17.054 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3274, ref allele: A)\n",
            "14:44:17.055 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3274, alt allele: G)\n",
            "14:44:17.061 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4771, ref allele: C)\n",
            "14:44:17.061 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4771, alt allele: T)\n",
            "14:44:17.061 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2320, ref allele: C)\n",
            "14:44:17.062 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2320, alt allele: T)\n",
            "14:44:17.125 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3163, ref allele: C)\n",
            "14:44:17.125 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3163, alt allele: T)\n",
            "14:44:17.126 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3163, ref allele: C)\n",
            "14:44:17.126 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3163, alt allele: T)\n",
            "14:44:17.130 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, ref allele: C)\n",
            "14:44:17.130 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, alt allele: T)\n",
            "14:44:17.131 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3716, ref allele: C)\n",
            "14:44:17.131 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3716, alt allele: T)\n",
            "14:44:17.133 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, ref allele: C)\n",
            "14:44:17.133 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, alt allele: A)\n",
            "14:44:17.134 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3716, ref allele: C)\n",
            "14:44:17.134 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3716, alt allele: A)\n",
            "14:44:17.137 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, ref allele: C)\n",
            "14:44:17.137 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3665, alt allele: T)\n",
            "14:44:17.137 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3716, ref allele: C)\n",
            "14:44:17.137 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3716, alt allele: T)\n",
            "14:44:17.141 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1882, ref allele: T)\n",
            "14:44:17.141 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1882, alt allele: C)\n",
            "14:44:17.141 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2996, ref allele: T)\n",
            "14:44:17.142 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2996, alt allele: C)\n",
            "14:44:17.142 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2252, ref allele: T)\n",
            "14:44:17.142 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2252, alt allele: C)\n",
            "14:44:17.142 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2204, ref allele: T)\n",
            "14:44:17.142 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2204, alt allele: C)\n",
            "14:44:17.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1882, ref allele: G)\n",
            "14:44:17.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1882, alt allele: A)\n",
            "14:44:17.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2996, ref allele: G)\n",
            "14:44:17.143 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2996, alt allele: A)\n",
            "14:44:17.144 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2252, ref allele: G)\n",
            "14:44:17.144 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2252, alt allele: A)\n",
            "14:44:17.144 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2204, ref allele: G)\n",
            "14:44:17.144 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2204, alt allele: A)\n",
            "14:44:17.209 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, ref allele: A)\n",
            "14:44:17.209 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, alt allele: G)\n",
            "14:44:17.210 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, ref allele: A)\n",
            "14:44:17.210 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3313, alt allele: G)\n",
            "14:44:17.216 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3917, ref allele: T)\n",
            "14:44:17.216 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3917, alt allele: C)\n",
            "14:44:17.216 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, ref allele: T)\n",
            "14:44:17.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, alt allele: C)\n",
            "14:44:17.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3917, ref allele: C)\n",
            "14:44:17.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3917, alt allele: T)\n",
            "14:44:17.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, ref allele: C)\n",
            "14:44:17.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1663, alt allele: T)\n",
            "14:44:17.222 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2903, ref allele: C)\n",
            "14:44:17.222 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2903, alt allele: T)\n",
            "14:44:17.573 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3328, ref allele: T)\n",
            "14:44:17.574 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3328, alt allele: C)\n",
            "14:44:17.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2525, ref allele: A)\n",
            "14:44:17.625 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2525, alt allele: G)\n",
            "14:44:17.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=815, ref allele: A)\n",
            "14:44:17.626 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=815, alt allele: G)\n",
            "14:44:17.627 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2525, ref allele: G)\n",
            "14:44:17.627 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2525, alt allele: A)\n",
            "14:44:17.627 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=815, ref allele: G)\n",
            "14:44:17.627 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=815, alt allele: A)\n",
            "14:44:17.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2525, ref allele: C)\n",
            "14:44:17.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2525, alt allele: T)\n",
            "14:44:17.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=815, ref allele: C)\n",
            "14:44:17.628 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=815, alt allele: T)\n",
            "14:44:17.630 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2903, ref allele: G)\n",
            "14:44:17.630 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2903, alt allele: A)\n",
            "14:44:17.631 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2903, ref allele: A)\n",
            "14:44:17.631 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2903, alt allele: G)\n",
            "14:44:17.782 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, ref allele: C)\n",
            "14:44:17.782 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, alt allele: T)\n",
            "14:44:17.782 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, ref allele: C)\n",
            "14:44:17.783 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, alt allele: T)\n",
            "14:44:17.783 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, ref allele: C)\n",
            "14:44:17.783 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, alt allele: T)\n",
            "14:44:17.787 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, ref allele: G)\n",
            "14:44:17.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, alt allele: A)\n",
            "14:44:17.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, ref allele: G)\n",
            "14:44:17.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, alt allele: A)\n",
            "14:44:17.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, ref allele: G)\n",
            "14:44:17.788 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, alt allele: A)\n",
            "14:44:17.837 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, ref allele: G)\n",
            "14:44:17.838 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, alt allele: A)\n",
            "14:44:17.838 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, ref allele: G)\n",
            "14:44:17.838 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, alt allele: A)\n",
            "14:44:17.838 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, ref allele: G)\n",
            "14:44:17.838 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, alt allele: A)\n",
            "14:44:17.840 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, ref allele: A)\n",
            "14:44:17.840 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5861, alt allele: G)\n",
            "14:44:17.841 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, ref allele: A)\n",
            "14:44:17.841 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3593, alt allele: G)\n",
            "14:44:17.841 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, ref allele: A)\n",
            "14:44:17.841 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4214, alt allele: G)\n",
            "14:44:17.886 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, ref allele: C)\n",
            "14:44:17.886 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, alt allele: T)\n",
            "14:44:17.890 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, ref allele: A)\n",
            "14:44:17.890 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, alt allele: G)\n",
            "14:44:17.891 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, ref allele: C)\n",
            "14:44:17.892 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, alt allele: T)\n",
            "14:44:17.894 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, ref allele: T)\n",
            "14:44:17.894 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, alt allele: C)\n",
            "14:44:17.895 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, ref allele: G)\n",
            "14:44:17.896 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, alt allele: A)\n",
            "14:44:17.898 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, ref allele: C)\n",
            "14:44:17.898 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7426, alt allele: T)\n",
            "14:44:17.974 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, ref allele: T)\n",
            "14:44:17.974 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, alt allele: C)\n",
            "14:44:17.979 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, ref allele: C)\n",
            "14:44:17.979 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, alt allele: G)\n",
            "14:44:17.980 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, ref allele: C)\n",
            "14:44:17.980 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, alt allele: G)\n",
            "14:44:17.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, ref allele: T)\n",
            "14:44:17.997 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, alt allele: G)\n",
            "14:44:17.998 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, ref allele: T)\n",
            "14:44:17.998 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, alt allele: G)\n",
            "14:44:17.998 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3692, ref allele: T)\n",
            "14:44:17.998 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3692, alt allele: G)\n",
            "14:44:18.001 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, ref allele: T)\n",
            "14:44:18.001 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, alt allele: A)\n",
            "14:44:18.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, ref allele: T)\n",
            "14:44:18.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, alt allele: A)\n",
            "14:44:18.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3692, ref allele: T)\n",
            "14:44:18.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3692, alt allele: A)\n",
            "14:44:18.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, ref allele: T)\n",
            "14:44:18.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, alt allele: A)\n",
            "14:44:18.006 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, ref allele: G)\n",
            "14:44:18.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=10904, alt allele: A)\n",
            "14:44:18.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, ref allele: G)\n",
            "14:44:18.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3836, alt allele: A)\n",
            "14:44:18.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3692, ref allele: G)\n",
            "14:44:18.008 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3692, alt allele: A)\n",
            "14:44:18.008 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, ref allele: G)\n",
            "14:44:18.008 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3032, alt allele: A)\n",
            "14:44:18.175 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2665, ref allele: G)\n",
            "14:44:18.175 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2665, alt allele: A)\n",
            "14:44:18.176 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7723, ref allele: G)\n",
            "14:44:18.177 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7723, alt allele: A)\n",
            "14:44:18.177 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6266, ref allele: G)\n",
            "14:44:18.178 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6266, alt allele: A)\n",
            "14:44:18.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2665, ref allele: T)\n",
            "14:44:18.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2665, alt allele: C)\n",
            "14:44:18.181 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7723, ref allele: T)\n",
            "14:44:18.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=7723, alt allele: C)\n",
            "14:44:18.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6266, ref allele: T)\n",
            "14:44:18.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6266, alt allele: C)\n",
            "14:44:18.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2431, ref allele: T)\n",
            "14:44:18.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2431, alt allele: C)\n",
            "14:44:18.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4130, ref allele: T)\n",
            "14:44:18.201 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4130, alt allele: C)\n",
            "14:44:18.221 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, ref allele: T)\n",
            "14:44:18.221 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, alt allele: G)\n",
            "14:44:18.223 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, ref allele: A)\n",
            "14:44:18.223 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, alt allele: G)\n",
            "14:44:18.225 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, ref allele: G)\n",
            "14:44:18.225 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, alt allele: A)\n",
            "14:44:18.227 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, ref allele: G)\n",
            "14:44:18.227 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, alt allele: A)\n",
            "14:44:18.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, ref allele: T)\n",
            "14:44:18.228 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, alt allele: C)\n",
            "14:44:18.230 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, ref allele: T)\n",
            "14:44:18.230 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, alt allele: C)\n",
            "14:44:18.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, ref allele: G)\n",
            "14:44:18.232 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1331, alt allele: A)\n",
            "14:44:18.264 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: C)\n",
            "14:44:18.264 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: T)\n",
            "14:44:18.264 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: C)\n",
            "14:44:18.264 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: T)\n",
            "14:44:18.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: A)\n",
            "14:44:18.266 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: G)\n",
            "14:44:18.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: A)\n",
            "14:44:18.267 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: G)\n",
            "14:44:18.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: A)\n",
            "14:44:18.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: G)\n",
            "14:44:18.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: A)\n",
            "14:44:18.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: G)\n",
            "14:44:18.271 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: T)\n",
            "14:44:18.271 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: C)\n",
            "14:44:18.271 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: T)\n",
            "14:44:18.271 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: C)\n",
            "14:44:18.273 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: A)\n",
            "14:44:18.273 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: C)\n",
            "14:44:18.273 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: A)\n",
            "14:44:18.274 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: C)\n",
            "14:44:18.275 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: C)\n",
            "14:44:18.275 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: T)\n",
            "14:44:18.276 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: C)\n",
            "14:44:18.276 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: T)\n",
            "14:44:18.277 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: A)\n",
            "14:44:18.278 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: G)\n",
            "14:44:18.278 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: A)\n",
            "14:44:18.278 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: G)\n",
            "14:44:18.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: G)\n",
            "14:44:18.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: A)\n",
            "14:44:18.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: G)\n",
            "14:44:18.280 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: A)\n",
            "14:44:18.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: T)\n",
            "14:44:18.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: A)\n",
            "14:44:18.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: T)\n",
            "14:44:18.282 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: A)\n",
            "14:44:18.284 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: A)\n",
            "14:44:18.285 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: G)\n",
            "14:44:18.285 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: A)\n",
            "14:44:18.285 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: G)\n",
            "14:44:18.287 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: A)\n",
            "14:44:18.287 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: C)\n",
            "14:44:18.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: A)\n",
            "14:44:18.288 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: C)\n",
            "14:44:18.289 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, ref allele: T)\n",
            "14:44:18.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3301, alt allele: G)\n",
            "14:44:18.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, ref allele: T)\n",
            "14:44:18.290 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3623, alt allele: G)\n",
            "14:44:18.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: G)\n",
            "14:44:18.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: A)\n",
            "14:44:18.304 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: G)\n",
            "14:44:18.304 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: C)\n",
            "14:44:18.306 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: C)\n",
            "14:44:18.306 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: T)\n",
            "14:44:18.308 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: C)\n",
            "14:44:18.308 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: T)\n",
            "14:44:18.310 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: A)\n",
            "14:44:18.310 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: G)\n",
            "14:44:18.312 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: C)\n",
            "14:44:18.312 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: A)\n",
            "14:44:18.314 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: G)\n",
            "14:44:18.314 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: C)\n",
            "14:44:18.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: C)\n",
            "14:44:18.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: A)\n",
            "14:44:18.318 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: C)\n",
            "14:44:18.318 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: T)\n",
            "14:44:18.320 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, ref allele: C)\n",
            "14:44:18.320 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3449, alt allele: G)\n",
            "14:44:18.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4388, ref allele: G)\n",
            "14:44:18.342 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4388, alt allele: T)\n",
            "14:44:18.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3419, ref allele: G)\n",
            "14:44:18.343 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3419, alt allele: T)\n",
            "14:44:18.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4388, ref allele: G)\n",
            "14:44:18.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4388, alt allele: A)\n",
            "14:44:18.345 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3419, ref allele: G)\n",
            "14:44:18.346 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3419, alt allele: A)\n",
            "14:44:18.351 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4388, ref allele: C)\n",
            "14:44:18.351 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4388, alt allele: T)\n",
            "14:44:18.352 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3419, ref allele: C)\n",
            "14:44:18.352 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3419, alt allele: T)\n",
            "14:44:18.436 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2176, ref allele: T)\n",
            "14:44:18.438 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, ref allele: G)\n",
            "14:44:18.438 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, alt allele: A)\n",
            "14:44:18.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, ref allele: T)\n",
            "14:44:18.439 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, alt allele: C)\n",
            "14:44:18.441 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, ref allele: A)\n",
            "14:44:18.441 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, alt allele: G)\n",
            "14:44:18.443 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, ref allele: A)\n",
            "14:44:18.443 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2170, alt allele: G)\n",
            "14:44:18.499 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1145, ref allele: C)\n",
            "14:44:18.499 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1145, alt allele: G)\n",
            "14:44:18.500 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1145, ref allele: C)\n",
            "14:44:18.501 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1145, alt allele: T)\n",
            "14:44:18.555 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3602, ref allele: A)\n",
            "14:44:18.555 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3602, alt allele: G)\n",
            "14:44:18.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3602, ref allele: C)\n",
            "14:44:18.556 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3602, alt allele: T)\n",
            "14:44:18.580 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, ref allele: G)\n",
            "14:44:18.580 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, alt allele: A)\n",
            "14:44:18.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, ref allele: A)\n",
            "14:44:18.581 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, alt allele: G)\n",
            "14:44:18.583 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, ref allele: C)\n",
            "14:44:18.583 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, alt allele: A)\n",
            "14:44:18.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, ref allele: T)\n",
            "14:44:18.587 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3893, alt allele: C)\n",
            "14:44:18.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2696, ref allele: G)\n",
            "14:44:18.594 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2696, alt allele: A)\n",
            "14:44:18.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2696, ref allele: A)\n",
            "14:44:18.596 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2696, alt allele: G)\n",
            "14:44:18.601 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2696, ref allele: T)\n",
            "14:44:18.601 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2696, alt allele: C)\n",
            "14:44:18.712 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2455, ref allele: A)\n",
            "14:44:18.712 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2455, alt allele: G)\n",
            "14:44:18.712 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1544, ref allele: A)\n",
            "14:44:18.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1544, alt allele: G)\n",
            "14:44:18.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1747, ref allele: A)\n",
            "14:44:18.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1747, alt allele: G)\n",
            "14:44:18.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1718, ref allele: A)\n",
            "14:44:18.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1718, alt allele: G)\n",
            "14:44:18.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1625, ref allele: A)\n",
            "14:44:18.713 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1625, alt allele: G)\n",
            "14:44:18.717 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:125597249-125597249 due to alternate allele: *\n",
            "14:44:18.717 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:125597249-125597249 due to alternate allele: *\n",
            "14:44:18.717 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:125597249-125597249 due to alternate allele: *\n",
            "14:44:18.717 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:125597249-125597249 due to alternate allele: *\n",
            "14:44:18.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4558, ref allele: G)\n",
            "14:44:18.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4558, alt allele: A)\n",
            "14:44:18.724 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2402, ref allele: G)\n",
            "14:44:18.725 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2402, alt allele: A)\n",
            "14:44:18.733 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4051, ref allele: A)\n",
            "14:44:18.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4051, alt allele: G)\n",
            "14:44:18.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4153, ref allele: A)\n",
            "14:44:18.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4153, alt allele: G)\n",
            "14:44:18.734 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1670, ref allele: A)\n",
            "14:44:18.735 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1670, alt allele: G)\n",
            "14:44:18.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1799, ref allele: A)\n",
            "14:44:18.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1799, alt allele: G)\n",
            "14:44:18.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, ref allele: A)\n",
            "14:44:18.736 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1712, alt allele: G)\n",
            "14:44:18.783 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=457, ref allele: A)\n",
            "14:44:18.783 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=457, alt allele: G)\n",
            "14:44:18.791 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:125748836-125748836 due to alternate allele: *\n",
            "14:44:18.792 WARN  GencodeFuncotationFactory - Cannot create complete funcotation for variant at chr11:125748836-125748836 due to alternate allele: *\n",
            "14:44:18.846 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, ref allele: G)\n",
            "14:44:18.846 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, alt allele: C)\n",
            "14:44:18.847 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1843, ref allele: G)\n",
            "14:44:18.847 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1843, alt allele: C)\n",
            "14:44:18.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, ref allele: G)\n",
            "14:44:18.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, alt allele: T)\n",
            "14:44:18.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1843, ref allele: G)\n",
            "14:44:18.848 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1843, alt allele: T)\n",
            "14:44:18.849 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, ref allele: T)\n",
            "14:44:18.849 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1792, alt allele: G)\n",
            "14:44:18.849 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1843, ref allele: T)\n",
            "14:44:18.850 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1843, alt allele: G)\n",
            "14:44:18.852 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1810, ref allele: T)\n",
            "14:44:18.852 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1810, alt allele: C)\n",
            "14:44:18.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5803, ref allele: T)\n",
            "14:44:18.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5803, alt allele: A)\n",
            "14:44:18.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, ref allele: T)\n",
            "14:44:18.864 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, alt allele: A)\n",
            "14:44:18.867 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5803, ref allele: C)\n",
            "14:44:18.867 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5803, alt allele: T)\n",
            "14:44:18.868 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, ref allele: C)\n",
            "14:44:18.868 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, alt allele: T)\n",
            "14:44:18.871 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5803, ref allele: G)\n",
            "14:44:18.871 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5803, alt allele: A)\n",
            "14:44:18.871 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, ref allele: G)\n",
            "14:44:18.871 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, alt allele: A)\n",
            "14:44:18.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, ref allele: G)\n",
            "14:44:18.904 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4186, alt allele: A)\n",
            "14:44:18.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1970, ref allele: T)\n",
            "14:44:18.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1970, alt allele: G)\n",
            "14:44:18.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1825, ref allele: T)\n",
            "14:44:18.967 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1825, alt allele: G)\n",
            "14:44:19.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, ref allele: C)\n",
            "14:44:19.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, alt allele: T)\n",
            "14:44:19.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1459, ref allele: C)\n",
            "14:44:19.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1459, alt allele: T)\n",
            "14:44:19.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1919, ref allele: C)\n",
            "14:44:19.002 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1919, alt allele: T)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, ref allele: C)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, alt allele: T)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1745, ref allele: C)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1745, alt allele: T)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1967, ref allele: C)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1967, alt allele: T)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1669, ref allele: C)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1669, alt allele: T)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1825, ref allele: C)\n",
            "14:44:19.003 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1825, alt allele: T)\n",
            "14:44:19.005 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, ref allele: C)\n",
            "14:44:19.005 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1790, alt allele: T)\n",
            "14:44:19.006 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1459, ref allele: C)\n",
            "14:44:19.006 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1459, alt allele: T)\n",
            "14:44:19.006 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1919, ref allele: C)\n",
            "14:44:19.006 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1919, alt allele: T)\n",
            "14:44:19.006 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, ref allele: C)\n",
            "14:44:19.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1724, alt allele: T)\n",
            "14:44:19.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1745, ref allele: C)\n",
            "14:44:19.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1745, alt allele: T)\n",
            "14:44:19.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1967, ref allele: C)\n",
            "14:44:19.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1967, alt allele: T)\n",
            "14:44:19.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1669, ref allele: C)\n",
            "14:44:19.007 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1669, alt allele: T)\n",
            "14:44:19.008 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1825, ref allele: C)\n",
            "14:44:19.008 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1825, alt allele: T)\n",
            "14:44:19.013 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3808, ref allele: C)\n",
            "14:44:19.013 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3808, alt allele: T)\n",
            "14:44:19.014 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3397, ref allele: C)\n",
            "14:44:19.014 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3397, alt allele: T)\n",
            "14:44:19.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4594, ref allele: A)\n",
            "14:44:19.182 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4594, alt allele: G)\n",
            "14:44:19.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3389, ref allele: A)\n",
            "14:44:19.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3389, alt allele: G)\n",
            "14:44:19.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2231, ref allele: A)\n",
            "14:44:19.183 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2231, alt allele: G)\n",
            "14:44:19.205 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4151, ref allele: C)\n",
            "14:44:19.205 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4151, alt allele: T)\n",
            "14:44:19.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3427, ref allele: T)\n",
            "14:44:19.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3427, alt allele: A)\n",
            "14:44:19.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4127, ref allele: T)\n",
            "14:44:19.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4127, alt allele: A)\n",
            "14:44:19.212 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4151, ref allele: T)\n",
            "14:44:19.213 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4151, alt allele: A)\n",
            "14:44:19.213 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3859, ref allele: T)\n",
            "14:44:19.213 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3859, alt allele: A)\n",
            "14:44:19.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2621, ref allele: C)\n",
            "14:44:19.217 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2621, alt allele: T)\n",
            "14:44:19.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2539, ref allele: C)\n",
            "14:44:19.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2539, alt allele: T)\n",
            "14:44:19.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, ref allele: C)\n",
            "14:44:19.218 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=2614, alt allele: T)\n",
            "14:44:19.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10111, ref allele: G)\n",
            "14:44:19.247 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=10111, alt allele: A)\n",
            "14:44:19.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1813, ref allele: C)\n",
            "14:44:19.269 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1813, alt allele: T)\n",
            "14:44:19.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2234, ref allele: C)\n",
            "14:44:19.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=2234, alt allele: T)\n",
            "14:44:19.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, ref allele: C)\n",
            "14:44:19.302 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1639, alt allele: T)\n",
            "14:44:19.311 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5093, ref allele: T)\n",
            "14:44:19.311 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=5093, alt allele: C)\n",
            "14:44:19.312 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4351, ref allele: T)\n",
            "14:44:19.312 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4351, alt allele: C)\n",
            "14:44:19.313 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4114, ref allele: T)\n",
            "14:44:19.313 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=4114, alt allele: C)\n",
            "14:44:19.315 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, ref allele: C)\n",
            "14:44:19.315 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, alt allele: T)\n",
            "14:44:19.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, ref allele: C)\n",
            "14:44:19.316 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, alt allele: T)\n",
            "14:44:19.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, ref allele: C)\n",
            "14:44:19.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, alt allele: T)\n",
            "14:44:19.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, ref allele: C)\n",
            "14:44:19.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, alt allele: T)\n",
            "14:44:19.317 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, ref allele: C)\n",
            "14:44:19.318 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, alt allele: T)\n",
            "14:44:19.320 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, ref allele: A)\n",
            "14:44:19.321 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, alt allele: G)\n",
            "14:44:19.321 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, ref allele: A)\n",
            "14:44:19.321 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, alt allele: G)\n",
            "14:44:19.322 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, ref allele: A)\n",
            "14:44:19.322 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, alt allele: G)\n",
            "14:44:19.322 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, ref allele: A)\n",
            "14:44:19.322 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, alt allele: G)\n",
            "14:44:19.322 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, ref allele: A)\n",
            "14:44:19.323 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, alt allele: G)\n",
            "14:44:19.324 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, ref allele: G)\n",
            "14:44:19.324 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, alt allele: A)\n",
            "14:44:19.324 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, ref allele: G)\n",
            "14:44:19.324 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, alt allele: A)\n",
            "14:44:19.325 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, ref allele: G)\n",
            "14:44:19.325 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, alt allele: A)\n",
            "14:44:19.325 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, ref allele: G)\n",
            "14:44:19.325 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, alt allele: A)\n",
            "14:44:19.326 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, ref allele: G)\n",
            "14:44:19.326 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, alt allele: A)\n",
            "14:44:19.327 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, ref allele: A)\n",
            "14:44:19.327 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6322, alt allele: C)\n",
            "14:44:19.327 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, ref allele: A)\n",
            "14:44:19.328 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6301, alt allele: C)\n",
            "14:44:19.328 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, ref allele: A)\n",
            "14:44:19.328 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5998, alt allele: C)\n",
            "14:44:19.328 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, ref allele: A)\n",
            "14:44:19.328 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=6182, alt allele: C)\n",
            "14:44:19.329 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, ref allele: A)\n",
            "14:44:19.329 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3308, alt allele: C)\n",
            "14:44:19.381 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3539, ref allele: C)\n",
            "14:44:19.382 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3539, alt allele: T)\n",
            "14:44:19.436 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4010, ref allele: C)\n",
            "14:44:19.436 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4010, alt allele: T)\n",
            "14:44:19.440 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4010, ref allele: G)\n",
            "14:44:19.441 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4010, alt allele: C)\n",
            "14:44:19.442 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4010, ref allele: A)\n",
            "14:44:19.442 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=4010, alt allele: G)\n",
            "14:44:19.509 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, ref allele: T)\n",
            "14:44:19.510 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, alt allele: G)\n",
            "14:44:19.511 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, ref allele: T)\n",
            "14:44:19.511 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, alt allele: G)\n",
            "14:44:19.541 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, ref allele: A)\n",
            "14:44:19.541 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, alt allele: G)\n",
            "14:44:19.542 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, ref allele: A)\n",
            "14:44:19.542 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, alt allele: G)\n",
            "14:44:19.544 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, ref allele: T)\n",
            "14:44:19.544 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, alt allele: C)\n",
            "14:44:19.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, ref allele: T)\n",
            "14:44:19.545 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, alt allele: C)\n",
            "14:44:19.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, ref allele: G)\n",
            "14:44:19.551 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, alt allele: A)\n",
            "14:44:19.552 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, ref allele: G)\n",
            "14:44:19.552 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1562, alt allele: A)\n",
            "14:44:19.553 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, ref allele: G)\n",
            "14:44:19.554 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, alt allele: T)\n",
            "14:44:19.558 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, ref allele: G)\n",
            "14:44:19.559 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, alt allele: C)\n",
            "14:44:19.561 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, ref allele: G)\n",
            "14:44:19.561 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=6535, alt allele: C)\n",
            "14:44:19.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3040, ref allele: A)\n",
            "14:44:19.683 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3040, alt allele: C)\n",
            "14:44:19.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3190, ref allele: A)\n",
            "14:44:19.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=3190, alt allele: C)\n",
            "14:44:19.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1607, ref allele: A)\n",
            "14:44:19.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=1607, alt allele: C)\n",
            "14:44:19.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1315, ref allele: A)\n",
            "14:44:19.684 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=1315, alt allele: C)\n",
            "14:44:19.750 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5050, ref allele: C)\n",
            "14:44:19.750 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5050, alt allele: T)\n",
            "14:44:19.754 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5050, ref allele: G)\n",
            "14:44:19.754 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5050, alt allele: A)\n",
            "14:44:19.756 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5050, ref allele: T)\n",
            "14:44:19.756 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 1 (size=5050, alt allele: C)\n",
            "14:44:19.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3566, ref allele: G)\n",
            "14:44:19.793 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3566, alt allele: C)\n",
            "14:44:19.796 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3566, ref allele: A)\n",
            "14:44:19.796 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3566, alt allele: G)\n",
            "14:44:19.832 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3152, ref allele: C)\n",
            "14:44:19.832 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3152, alt allele: G)\n",
            "14:44:19.833 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3365, ref allele: C)\n",
            "14:44:19.833 WARN  FuncotatorUtils - createAminoAcidSequence given a coding sequence of length not divisible by 3.  Dropping bases from the end: 2 (size=3365, alt allele: G)\n",
            "14:44:19.859 INFO  ProgressMeter -      chr11:118390487              0.6                  5767          10036.8\n",
            "14:44:19.859 INFO  ProgressMeter - Traversal complete. Processed 5767 total variants in 0.6 minutes.\n",
            "14:44:19.859 INFO  VcfFuncotationFactory - ClinVar_VCF 20180429_hg38 cache hits/total: 0/331\n",
            "14:44:19.859 INFO  VcfFuncotationFactory - LMMKnown 20180618 cache hits/total: 0/1\n",
            "14:44:19.937 INFO  Funcotator - Shutting down engine\n",
            "[June 3, 2021 at 2:44:19 PM UTC] org.broadinstitute.hellbender.tools.funcotator.Funcotator done. Elapsed time: 0.85 minutes.\n",
            "Runtime.totalMemory()=2205155328\n",
            "Tool returned:\n",
            "true\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VPpBseSjHrWy",
        "outputId": "18c3a3b0-ee5b-462b-e9b5-99db0687472f"
      },
      "source": [
        "reader = pysam.VariantFile(ovcf)\n",
        "description = reader.header.info['FUNCOTATION'].description\n",
        "print(description)\n",
        "funcotations = description[73:].split('|')"
      ],
      "execution_count": 57,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Functional annotation from the Funcotator tool.  Funcotation fields are: Gencode_27_hugoSymbol|Gencode_27_ncbiBuild|Gencode_27_chromosome|Gencode_27_start|Gencode_27_end|Gencode_27_variantClassification|Gencode_27_secondaryVariantClassification|Gencode_27_variantType|Gencode_27_refAllele|Gencode_27_tumorSeqAllele1|Gencode_27_tumorSeqAllele2|Gencode_27_genomeChange|Gencode_27_annotationTranscript|Gencode_27_transcriptStrand|Gencode_27_transcriptExon|Gencode_27_transcriptPos|Gencode_27_cDnaChange|Gencode_27_codonChange|Gencode_27_proteinChange|Gencode_27_gcContent|Gencode_27_referenceContext|Gencode_27_otherTranscripts|ACMGLMMLof_LOF_Mechanism|ACMGLMMLof_Mode_of_Inheritance|ACMGLMMLof_Notes|ACMG_recommendation_Disease_Name|ClinVar_VCF_AF_ESP|ClinVar_VCF_AF_EXAC|ClinVar_VCF_AF_TGP|ClinVar_VCF_ALLELEID|ClinVar_VCF_CLNDISDB|ClinVar_VCF_CLNDISDBINCL|ClinVar_VCF_CLNDN|ClinVar_VCF_CLNDNINCL|ClinVar_VCF_CLNHGVS|ClinVar_VCF_CLNREVSTAT|ClinVar_VCF_CLNSIG|ClinVar_VCF_CLNSIGCONF|ClinVar_VCF_CLNSIGINCL|ClinVar_VCF_CLNVC|ClinVar_VCF_CLNVCSO|ClinVar_VCF_CLNVI|ClinVar_VCF_DBVARID|ClinVar_VCF_GENEINFO|ClinVar_VCF_MC|ClinVar_VCF_ORIGIN|ClinVar_VCF_RS|ClinVar_VCF_SSR|ClinVar_VCF_ID|ClinVar_VCF_FILTER|LMMKnown_LMM_FLAGGED|LMMKnown_ID|LMMKnown_FILTER\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Zogh5fUQHusz",
        "outputId": "8f427520-67c2-4709-87d1-139668c234c6"
      },
      "source": [
        "print('\\n'.join(funcotations))"
      ],
      "execution_count": 58,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Gencode_27_hugoSymbol\n",
            "Gencode_27_ncbiBuild\n",
            "Gencode_27_chromosome\n",
            "Gencode_27_start\n",
            "Gencode_27_end\n",
            "Gencode_27_variantClassification\n",
            "Gencode_27_secondaryVariantClassification\n",
            "Gencode_27_variantType\n",
            "Gencode_27_refAllele\n",
            "Gencode_27_tumorSeqAllele1\n",
            "Gencode_27_tumorSeqAllele2\n",
            "Gencode_27_genomeChange\n",
            "Gencode_27_annotationTranscript\n",
            "Gencode_27_transcriptStrand\n",
            "Gencode_27_transcriptExon\n",
            "Gencode_27_transcriptPos\n",
            "Gencode_27_cDnaChange\n",
            "Gencode_27_codonChange\n",
            "Gencode_27_proteinChange\n",
            "Gencode_27_gcContent\n",
            "Gencode_27_referenceContext\n",
            "Gencode_27_otherTranscripts\n",
            "ACMGLMMLof_LOF_Mechanism\n",
            "ACMGLMMLof_Mode_of_Inheritance\n",
            "ACMGLMMLof_Notes\n",
            "ACMG_recommendation_Disease_Name\n",
            "ClinVar_VCF_AF_ESP\n",
            "ClinVar_VCF_AF_EXAC\n",
            "ClinVar_VCF_AF_TGP\n",
            "ClinVar_VCF_ALLELEID\n",
            "ClinVar_VCF_CLNDISDB\n",
            "ClinVar_VCF_CLNDISDBINCL\n",
            "ClinVar_VCF_CLNDN\n",
            "ClinVar_VCF_CLNDNINCL\n",
            "ClinVar_VCF_CLNHGVS\n",
            "ClinVar_VCF_CLNREVSTAT\n",
            "ClinVar_VCF_CLNSIG\n",
            "ClinVar_VCF_CLNSIGCONF\n",
            "ClinVar_VCF_CLNSIGINCL\n",
            "ClinVar_VCF_CLNVC\n",
            "ClinVar_VCF_CLNVCSO\n",
            "ClinVar_VCF_CLNVI\n",
            "ClinVar_VCF_DBVARID\n",
            "ClinVar_VCF_GENEINFO\n",
            "ClinVar_VCF_MC\n",
            "ClinVar_VCF_ORIGIN\n",
            "ClinVar_VCF_RS\n",
            "ClinVar_VCF_SSR\n",
            "ClinVar_VCF_ID\n",
            "ClinVar_VCF_FILTER\n",
            "LMMKnown_LMM_FLAGGED\n",
            "LMMKnown_ID\n",
            "LMMKnown_FILTER\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ihcf95biFmB_",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "fd21e1f4-5d01-4e33-d487-e5cce98abd91"
      },
      "source": [
        "cnt = 0\n",
        "results = dict()\n",
        "\n",
        "for variant in pysam.VariantFile(ovcf):\n",
        "    try:\n",
        "        ann = {x:y for x, y in zip(funcotations, variant.info['FUNCOTATION'][0][1:-1].split('|'))}\n",
        "    except:\n",
        "        cnt += 1\n",
        "        pass\n",
        "\n",
        "    if ann['ClinVar_VCF_CLNSIG']:\n",
        "        # print(ann['ClinVar_VCF_CLNSIG'])\n",
        "        if ann['ClinVar_VCF_CLNSIG'] not in results.keys():\n",
        "            results[ann['ClinVar_VCF_CLNSIG']] = 0\n",
        "        else:\n",
        "            results[str(ann['ClinVar_VCF_CLNSIG'])] += 1\n",
        "\n",
        "\n",
        "for tp, num in results.items():\n",
        "    if num == 0:\n",
        "        continue\n",
        "    print('{} {}'.format(tp, num))"
      ],
      "execution_count": 159,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Benign 235\n",
            "Benign/Likely_benign 47\n",
            "not_provided 4\n",
            "Likely_benign 23\n",
            "Uncertain_significance 1\n",
            "drug_response 1\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "k0klSqtLFwdq"
      },
      "source": [
        "### 6. Svi uzorici sadrže određenu količinu kontaminacije DNK materijalom bakterijskog ili virusnog porekla. Većina ovakvih readova se neće mapirati na ljudski genom. Izvući readove koji nisu mapirani u procesu mapiranja, asemblovati ih alatom abyss, i identifikovati organizam od kojeg potiče najduži skafold upotrebom alata Blast."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "mUbLdr3OUnPW"
      },
      "source": [
        "# !pip install abyss\n",
        "\n",
        "!sudo apt-get install autotools-dev automake\n",
        "!git clone https://github.com/bcgsc/abyss.git\n",
        "!cd abyss && ./autogen.sh\n",
        "!cd abyss && ./configure\n",
        "!cd abyss && make\n",
        "!cd abyss && sudo make install\n",
        " \n",
        "!sudo apt-get install abyss"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "XUoTFXBXWfaK"
      },
      "source": [
        "#%%time\n",
        "!git clone https://github.com/sparsehash/sparsehash.git\n",
        "!cd sparsehash && ./configure\n",
        "!cd sparsehash && make\n",
        "!cd sparsehash && sudo make install"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "X980-qZIFwEo"
      },
      "source": [
        "# HG00096.chr11.exome.sorted.bam\n",
        "\n",
        "file = 'HG00096.chr11.filtered.exome.bam'\n",
        "\n",
        "sf = pysam.AlignmentFile('HG00096.chr11.exome.sorted.bam', 'rb')\n",
        "f = pysam.AlignmentFile(file, 'wb', template=sf)\n",
        "\n",
        "for read in sf.fetch():\n",
        "    if read.is_unmapped:\n",
        "        f.write(read)\n",
        "\n"
      ],
      "execution_count": 146,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pBnCU36X4Cog",
        "outputId": "96268899-2c55-4ed8-ca63-e5de146ccae6"
      },
      "source": [
        "!samtools split HG00096.chr11.filtered.exome.bam"
      ],
      "execution_count": 144,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "[W::bam_hdr_read] EOF marker is absent. The input is probably truncated\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "C3lBJqm9i55D",
        "outputId": "4ea9d2ff-8f9b-4bb5-ed6b-f694642d0c7e"
      },
      "source": [
        "!samtools fastq -1 reads_1.fq -2 reads_2.fq HG00096.chr11.filtered.exome.bam"
      ],
      "execution_count": 147,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "[M::bam2fq_mainloop] discarded 0 singletons\n",
            "[M::bam2fq_mainloop] processed 25695 reads\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "DrwnLZC76BBW"
      },
      "source": [
        "# !sed 's|.1 |/1 |g' <reads1.fq >reads2.fq"
      ],
      "execution_count": 140,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "asIGqZlu3iHV",
        "outputId": "e87ea5a4-c3d0-408a-a1c5-2f786f171023"
      },
      "source": [
        "!abyss-pe name=human1 k=64 in='reads_1.fq reads_2.fq'"
      ],
      "execution_count": 148,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "ABYSS -k64 -q3    --coverage-hist=coverage.hist -s human1-bubbles.fa  -o human1-1.fa reads_1.fq reads_2.fq \n",
            "ABySS 2.0.2\n",
            "ABYSS -k64 -q3 --coverage-hist=coverage.hist -s human1-bubbles.fa -o human1-1.fa reads_1.fq reads_2.fq\n",
            "Reading `reads_1.fq'...\n",
            "`reads_1.fq': discarded 19854 reads shorter than 64 bases\n",
            "Reading `reads_2.fq'...\n",
            "`reads_2.fq': discarded 5720 reads shorter than 64 bases\n",
            "Loaded 6737 k-mer\n",
            "Unable to determine minimum k-mer coverage\n",
            "Using a coverage threshold of 1...\n",
            "The median k-mer coverage is 1\n",
            "The reconstruction is 6737\n",
            "The k-mer coverage threshold is 1\n",
            "Setting parameter e (erode) to 2\n",
            "Setting parameter E (erodeStrand) to 0\n",
            "Setting parameter c (coverage) to 2\n",
            "Generating adjacency\n",
            "Added 13326 edges.\n",
            "Eroding tips\n",
            "Eroded 5610 tips.\n",
            "Eroded 0 tips.\n",
            "Pruning tips shorter than 1 bp...\n",
            "Pruning tips shorter than 2 bp...\n",
            "Pruning tips shorter than 4 bp...\n",
            "Pruning tips shorter than 8 bp...\n",
            "Pruning tips shorter than 16 bp...\n",
            "Pruned 26 k-mer in 4 tips.\n",
            "Pruning tips shorter than 32 bp...\n",
            "Pruned 59 k-mer in 4 tips.\n",
            "Pruning tips shorter than 64 bp...\n",
            "Pruned 34 k-mer in 2 tips.\n",
            "Pruning tips shorter than 64 bp...\n",
            "Pruned 10 tips in 7 rounds.\n",
            "Marked 4 edges of 2 ambiguous vertices.\n",
            "Removing low-coverage contigs (mean k-mer coverage < 2)\n",
            "Found 1008 k-mer in 8 contigs before removing low-coverage contigs.\n",
            "Removed 0 k-mer in 0 low-coverage contigs.\n",
            "Split 0 ambigiuous branches.\n",
            "Eroding tips\n",
            "Eroded 0 tips.\n",
            "Eroded 0 tips.\n",
            "Pruning tips shorter than 1 bp...\n",
            "Pruning tips shorter than 2 bp...\n",
            "Pruning tips shorter than 4 bp...\n",
            "Pruning tips shorter than 8 bp...\n",
            "Pruning tips shorter than 16 bp...\n",
            "Pruning tips shorter than 32 bp...\n",
            "Pruning tips shorter than 64 bp...\n",
            "Pruned 0 tips in 6 rounds.\n",
            "Popping bubbles\n",
            "Removed 0 bubbles\n",
            "Marked 4 edges of 2 ambiguous vertices.\n",
            "Assembled 1008 k-mer in 8 contigs.\n",
            "Removed 5729 k-mer.\n",
            "The signal-to-noise ratio (SNR) is -7.54618 dB.\n",
            "AdjList    -k64 -m50 --dot human1-1.fa >human1-1.dot\n",
            "abyss-filtergraph  --dot   -k64 -g human1-2.dot1 human1-1.dot human1-1.fa >human1-1.path\n",
            "MergeContigs   -k64 -g human1-2.dot -o human1-2.fa human1-1.fa human1-2.dot1 human1-1.path\n",
            "PopBubbles  --dot -j2 -k64  -p0.9  -g human1-3.dot human1-2.fa human1-2.dot >human1-2.path\n",
            "MergeContigs   -k64 -o human1-3.fa human1-2.fa human1-2.dot human1-2.path\n",
            "The minimum coverage of single-end contigs is 2.\n",
            "The minimum coverage of merged contigs is 3.69444.\n",
            "Consider increasing the coverage threshold parameter, c, to 3.69444.\n",
            "awk '!/^>/ {x[\">\" $1]=1; next} {getline s} $1 in x {print $0 \"\\n\" s}' \\\n",
            "\thuman1-2.path human1-1.fa >human1-indel.fa\n",
            "ln -sf human1-3.fa human1-unitigs.fa\n",
            "abyss-map   -j2 -l40    reads_1.fq reads_2.fq human1-3.fa \\\n",
            "\t|abyss-fixmate   -l40  -h human1-3.hist \\\n",
            "\t|sort -snk3 -k4 \\\n",
            "\t|DistanceEst   -j2 -k64 -l40 -s1000 -n10   -o human1-3.dist human1-3.hist\n",
            "Building the suffix array...\n",
            "Building the Burrows-Wheeler transform...\n",
            "Building the character occurrence table...\n",
            "Mateless   25695  100%\n",
            "Unaligned      0\n",
            "Singleton      0\n",
            "FR             0\n",
            "RF             0\n",
            "FF             0\n",
            "Different      0\n",
            "Total      25695\n",
            "abyss-fixmate: error: All reads are mateless. This can happen when first and second read IDs do not match.\n",
            "error: `human1-3.hist': No such file or directory\n",
            "/usr/bin/abyss-pe:561: recipe for target 'human1-3.dist' failed\n",
            "make: *** [human1-3.dist] Error 1\n",
            "make: *** Deleting file 'human1-3.dist'\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-J32mCeW5GBH",
        "outputId": "39ab0b5f-5917-467f-8487-5019cc2006be"
      },
      "source": [
        "from glob import glob\n",
        "\n",
        "for f in glob('*unitigs.fa'):\n",
        "  try:\n",
        "      scaffolds = pysam.FastaFile(f)\n",
        "  except:\n",
        "      pass\n",
        "  longest = sorted(zip(scaffolds.lengths, scaffolds.references), \n",
        "                   reverse=True)[0][1]\n",
        "  un = scaffolds.fetch(longest)\n",
        "  print(f, len(un), un)"
      ],
      "execution_count": 154,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "human1-unitigs.fa 466 TAAAAAGTCCTTTAAATTCTGGATTAACTGAAATTATAACAAGTTGTTTATCTCGTATAATGCTATATTTTGTGAAAGCAACAAATTTAGTTAAAGTATTTCCATCATCATTCCAAGCATTAATACTTAATAATTTGTCATACGTGCTACTAAGATCTTTTATAAATTGTTCCCCACGTTCTGTATAACCGCTTAGTTTTTTCAAATCGTGAAATGAAAAAATAATCTCATCTACACCTTTTTCACGTACACGTGAAGCAATTGAAAAAAATAAGTTCATTTCACGACTATTAAATTTTCTCAGAGGAATATGATTTAATCTATTTTCATATTTTACAATTTCATTTGCCATAAGCTTCTCCTACTTTTATAAAGTATGAAATAAAAGTAACATACTTTATAAAAATACTCAATAATGTCGGAAAACCTACTTGAAAAGCGTAGAAAACCTACTTGAAAAGCGTAG\n",
            "human-unitigs.fa 466 TAAAAAGTCCTTTAAATTCTGGATTAACTGAAATTATAACAAGTTGTTTATCTCGTATAATGCTATATTTTGTGAAAGCAACAAATTTAGTTAAAGTATTTCCATCATCATTCCAAGCATTAATACTTAATAATTTGTCATACGTGCTACTAAGATCTTTTATAAATTGTTCCCCACGTTCTGTATAACCGCTTAGTTTTTTCAAATCGTGAAATGAAAAAATAATCTCATCTACACCTTTTTCACGTACACGTGAAGCAATTGAAAAAAATAAGTTCATTTCACGACTATTAAATTTTCTCAGAGGAATATGATTTAATCTATTTTCATATTTTACAATTTCATTTGCCATAAGCTTCTCCTACTTTTATAAAGTATGAAATAAAAGTAACATACTTTATAAAAATACTCAATAATGTCGGAAAACCTACTTGAAAAGCGTAGAAAACCTACTTGAAAAGCGTAG\n"
          ],
          "name": "stdout"
        }
      ]
    }
  ]
}